import {
  BaseResponse,
  ExportRequest,
  ExportResultRequest,
  ExportType,
  FindVariantExonsRequest,
  MarkReadRequest,
  MarkReadResponse,
  PagingRequest,
  UpdateNoteRequest,
  Variant,
  VariantData,
} from 'src/common';
import { Response as ExpressResponse } from 'express';
import { VariantController, VariantService } from 'src/variantsInfo';

describe('VariantController', () => {
  let controller: VariantController;
  let variantService: VariantService;
  const mockFindVariants = jest.fn();
  const mockFindVariantExons = jest.fn();
  const mockSaveRead = jest.fn();

  beforeEach(() => {
    variantService = (<any>{
      findVariants: mockFindVariants,
      findVariantExons: mockFindVariantExons,
      saveRead: mockSaveRead,
      saveNote: jest.fn(),
      exportVcf: jest.fn(),
      exportTsv: jest.fn(),
      getExportResult: jest.fn(),
    }) as VariantService;
    controller = new VariantController(variantService);
  });

  describe('findVariants', () => {
    it('should return a BaseResponse with the correct variants and track number', async () => {
      const expectedTrackNumber = 'bab-test02:366208570385';
      const pagingRequest: PagingRequest = {
        selected_database: 'family_duopat_20230105-152056',
        track_number: expectedTrackNumber,
        page: 0,
        per_page: 300,
        sort: [
          {
            column: 'chrom',
            sort: 'asc',
          },
        ],
        filter: {
          impact: ['frameshift_variant', 'missense_variant'],
          chrom: 'chr1',
          start: {
            $gte: 1234,
          },
          end: {
            $lte: 5678,
          },
          scenario: 'any',
          pass_filter: ['PASS'],
          quality: {
            $gte: 1000,
          },
          snv_type: 'snp',
          is_coding: 1,
          is_exonic: 1,
          hkgi_high_impact: 1,
          sv_type: 'INS',
          p_lof: ['LOF', 'DUP_LOF'],
          polyphen_pred: ['probably_damaging'],
          sift_pred: ['deleterious'],
          clnsig: [
            'Pathogenic',
            'Pathogenic,_drug_response,_protective,_risk_factor',
          ],
          is_pathogenic: true,
          clingen_hi: ['emerging'],
          clingen_ts: ['emerging'],
          is_read: true,
          variant_type: 'small',
          highest_af: {
            $lte: 0.05,
          },
          gnomad_eas_af: {
            $lte: 0.05,
          },
          spliceai_pred_ds_ag: {
            $gte: 0.5,
          },
          spliceai_pred_ds_al: {
            $gte: 0.5,
          },
          spliceai_pred_ds_dg: {
            $gte: 0.5,
          },
          spliceai_pred_ds_dl: {
            $gte: 0.5,
          },
          revel: {
            $lte: 0.736,
          },
          cadd_phred: {
            $gte: 20,
          },
          polyphen_score: {
            $lte: 0.05,
          },
          sift_score: {
            $lte: 0.05,
          },
          constraint_mis_z: {
            $gte: 2,
          },
          constraint_syn_z: {
            $gte: 2,
          },
          constraint_oe_lof_upper: {
            $lte: 0.5,
          },
          constraint_oe_mis_upper: {
            $lte: 0.5,
          },
          highest_exomiser_scombi: {
            $gte: 0.5,
          },
          exomiser_ad_exgenescombi: {
            $gte: 0.5,
          },
          exomiser_ar_exgenescombi: {
            $gte: 0.5,
          },
          exomiser_xd_exgenescombi: {
            $gte: 0.5,
          },
          exomiser_xr_exgenescombi: {
            $gte: 0.5,
          },
          exomiser_mt_exgenescombi: {
            $gte: 0.5,
          },
        },
        samples: [
          {
            i: 0,
            name: 'sample1',
            sample_id: 1,
            family_id: 'family_duopat_20230105-152056',
            mother_id: '-9',
            father_id: 'family_duopat_20230105-152056',
            sex: 'F',
            phenotype: '2',
            group: 'affected',
            active: true,
          },
          {
            i: 1,
            name: 'sample2',
            sample_id: 2,
            family_id: 'family_duopat_20230105-152056',
            mother_id: '-9',
            father_id: 'family_duopat_20230105-152056',
            sex: 'M',
            phenotype: '1',
            group: 'not_affected',
            active: true,
          },
        ],
        panels: ['Colon Cancer'],
      } as PagingRequest;
      const userInfo = { username: 'user' };

      const expectedVariants: VariantData = {
        result: [
          {
            allelic_depths: [],
            genotype_qualities: [],
            is_read: false,
            _id: '64e5a2209b592486e3891b9e',
            chrom: 'chr1',
            start: 51937,
            end: 65417,
            len: 13480,
            ref: 'A',
            alt: '<DUP>',
            p_lof: 'False',
            constraint_pli: '',
            constraint_oe_lof_upper: [],
            variant_type: 'structural',
            type: 'DUP',
            impact: '',
            gene_objs: [],
            highest_af: 0.00149,
            highest_af_info: {
              source: 'HKBC - CNVkit',
            },
            gnomadv3_af: -1,
            gnomadv3_af_afr: -1,
            gnomadv3_af_amr: -1,
            gnomadv3_af_eas: -1,
            gnomadv3_af_nfe: -1,
            gnomadv3_af_sas: -1,
            gnomadv3_af_xx: -1,
            gnomadv3_af_xy: -1,
            gnomadv3_af_ami: -1,
            gnomadv3_af_asj: -1,
            gnomadv3_af_fin: -1,
            gnomadv3_af_mid: -1,
            gnomadv3_af_oth: -1,
            gnomad_af: -1,
            gnomad_eas_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_nfe_af: -1,
            gnomad_sas_af: -1,
            gnomad_asj_af: -1,
            gnomad_fin_af: -1,
            gnomad_oth_af: -1,
            is_pathogenic: false,
            variant_id: 'S:cnvkit:1',
            sv_id: 'S:cnvkit:1',
            afs: [
              {
                source_start: 51439,
                source_end: 65417,
                source_svType: 'DUP',
                AF: 0.00149,
                SAMPLE_COUNT: 1680,
                reciOverlap: 0.964,
                annoOverlap: 0.964,
                source_filter: 'hkbc',
                display_source: 'Hong Kong Birth Cohort',
                source: 'HKBC - CNVkit',
                svLength: 13979,
              },
            ],
            exts: [],
            intpns: [],
            caller: 'cnvkit',
            genotypes_index: [
              [0, 0],
              [0, 1],
              [0, 0],
            ],
            hg19_chrom: 'chr1',
            hg19_start: 51937,
            hg19_strand: '+',
            hg19_end: 65417,
            gene_symbol: [],
            ensembl_gene_id: [],
            entrez_gene_id: [],
            ensembl_transcript_id: [],
          },
          {
            is_read: false,
            _id: '64e59ef69c2aec9c8bd9f984',
            variant_id: '1',
            end: 69511,
            ref: 'A',
            alt: 'G',
            start: 69511,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'OR4F5',
                ensembl_gene_id: 'ENSG00000186092',
                transcript: 'ENST00000641515',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '3/3',
                codon_change: 'Aca/Gca',
                aa_change: 'T/A',
                aa_length: 162,
                polyphen_pred: 'benign',
                polyphen_score: 0,
                sift_pred: 'tolerated',
                sift_score: 0.85,
                allele: 'G',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000641515.2:c.484A>G',
                hgvsp: 'ENSP00000493376.2:p.Thr162Ala',
                cds_position: 484,
                existing_variation: 'rs2691305&COSV58736924',
                strand: 1,
                hgnc_id: 'HGNC:14825',
                mane_select: 'NM_001005484.2',
                source: 'Ensembl',
                domains:
                  'Transmembrane_helices:TMhelix&PROSITE_profiles:PS50262&CDD:cd15226&PANTHER:PTHR48002:SF51&PANTHER:PTHR48002&Pfam:PF13853&Gene3D:1.20.1070.10&Superfamily:SSF81321',
                pheno: '0&1',
                revel: 0.053,
                cadd_phred: 2.209,
                cadd_raw: 0.112916,
                spliceai_pred_dp_ag: 26,
                spliceai_pred_dp_al: 32,
                spliceai_pred_dp_dg: 26,
                spliceai_pred_dp_dl: 34,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.02,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'OR4F5',
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'OR4F5',
                entrez_gene_id: '79501',
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000641515',
            ensembl_gene_id: ['ENSG00000186092'],
            is_exonic: 1,
            is_coding: 1,
            aa_change: 'T/A',
            polyphen_pred: ['benign'],
            polyphen_score: [0],
            sift_pred: ['tolerated'],
            sift_score: [0.85],
            impact: 'missense_variant',
            hgvsc: 'ENST00000641515.2:c.484A>G',
            hgvsp: 'ENSP00000493376.2:p.Thr162Ala',
            existing_variation: 'rs2691305&COSV58736924',
            source: 'Ensembl',
            domains:
              'Transmembrane_helices:TMhelix&PROSITE_profiles:PS50262&CDD:cd15226&PANTHER:PTHR48002:SF51&PANTHER:PTHR48002&Pfam:PF13853&Gene3D:1.20.1070.10&Superfamily:SSF81321',
            gnomad_af: 0.9497,
            gnomad_afr_af: 0.6075,
            gnomad_amr_af: 0.9514,
            gnomad_asj_af: 0.9767,
            gnomad_eas_af: 0.9994,
            gnomad_fin_af: 0.9916,
            gnomad_nfe_af: 0.9726,
            gnomad_oth_af: 0.9506,
            gnomad_sas_af: 0.9854,
            pheno: '0&1',
            revel: [0.053],
            cadd_phred: 2.209,
            cadd_raw: 0.112916,
            spliceai_pred_dp_ag: 26,
            spliceai_pred_dp_al: 32,
            spliceai_pred_dp_dg: 26,
            spliceai_pred_dp_dl: 34,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0.02],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'OR4F5',
            exomiser_ad_exgenespheno: 0.5,
            exomiser_xr_exgenescombi: 0.00030623,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.5948,
            gnomadv3_af_nfe: 0.9674,
            gnomadv3_af_xy: 0.8496,
            exomiser_xd_exgenescombi: 0.00030623,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9,
            gnomadv3_af: 0.846,
            exomiser_mt_exgenespheno: 0.5,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00030623,
            gnomadv3_af_eas: 0.9998,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9784,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00030623,
            exomiser_ad_exgenescombi: 0.00030623,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.998,
            gnomadv3_af_fin: 0.9907,
            exomiser_ar_exgenespheno: 0.5,
            gnomadv3_af_sas: 0.9772,
            gnomadv3_af_amr: 0.8951,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.8427,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.8624,
            exomiser_xr_exgenespheno: 0.5,
            exomiser_xd_exgenespheno: 0.5,
            impact_severity: 'MED',
            quality: 2010.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 20],
              [0, 11],
              [0, 37],
            ],
            genotype_qualities: [[93, 33, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['OR4F5'],
            ensembl_transcript_id: ['ENST00000641515'],
            variant_type: 'small',
            highest_af: 0.9998,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 69511,
            hg19_strand: '+',
            hg19_end: 69511,
            entrez_gene_id: ['79501'],
          },
          {
            is_read: false,
            _id: '64e59ef69c2aec9c8bd9f985',
            variant_id: '2',
            end: 71175,
            ref: 'G',
            alt: 'GA',
            start: 71175,
            chrom: 'chr1',
            type: 'indel',
            gene_objs: [
              {
                gene: 'OR4F5',
                ensembl_gene_id: 'ENSG00000186092',
                transcript: 'ENST00000641515',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '3/3',
                allele: 'A',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000641515.2:c.*1178dup',
                existing_variation: 'rs753554581',
                strand: 1,
                hgnc_id: 'HGNC:14825',
                mane_select: 'NM_001005484.2',
                source: 'Ensembl',
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'OR4F5',
                entrez_gene_id: '79501',
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000641515',
            ensembl_gene_id: ['ENSG00000186092'],
            is_exonic: 1,
            is_coding: 0,
            impact: '3_prime_UTR_variant',
            hgvsc: 'ENST00000641515.2:c.*1178dup',
            existing_variation: 'rs753554581',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.0019062,
            gnomadv3_af_nfe: 0.00064683,
            gnomadv3_af_xy: 0.00081806,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0,
            gnomadv3_af: 0.0012491,
            clndisdb: 'None',
            gnomadv3_af_eas: 0.00067295,
            gnomadv3_af_asj: 0,
            clnsigincl: 'None',
            clnvi: 'None',
            clnsig: 'None',
            gnomadv3_af_ami: 0,
            gnomadv3_af_fin: 0,
            gnomadv3_af_sas: 0.0018315,
            gnomadv3_af_amr: 0,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            gnomadv3_af_xx: 0.0016004,
            gnomadv3_af_oth: 0,
            impact_severity: 'LOW',
            quality: 30.3,
            genotypes_index: [
              [0, 0],
              [0, 1],
              [0, 0],
            ],
            allelic_depths: [
              [21, 0],
              [8, 2],
              [32, 0],
            ],
            genotype_qualities: [[42, 39, 68]],
            pass_filter: 'PASS',
            gene_symbol: ['OR4F5'],
            ensembl_transcript_id: ['ENST00000641515'],
            variant_type: 'small',
            highest_af: 0.0012491,
            highest_af_info: {
              source: 'gnomad v3 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 71175,
            hg19_strand: '+',
            hg19_end: 71175,
            entrez_gene_id: ['79501'],
          },
          {
            allelic_depths: [],
            genotype_qualities: [],
            is_read: false,
            _id: '64e5a2209b592486e3891b9f',
            chrom: 'chr1',
            start: 182131,
            end: 189641,
            len: 7510,
            ref: 'G',
            alt: '<DUP>',
            p_lof: 'False',
            constraint_pli: '',
            constraint_oe_lof_upper: [],
            variant_type: 'structural',
            type: 'DUP',
            impact: '',
            gene_objs: [],
            highest_af: 0.000893,
            highest_af_info: {
              source: 'HKBC - CNVkit',
            },
            gnomadv3_af: -1,
            gnomadv3_af_afr: -1,
            gnomadv3_af_amr: -1,
            gnomadv3_af_eas: -1,
            gnomadv3_af_nfe: -1,
            gnomadv3_af_sas: -1,
            gnomadv3_af_xx: -1,
            gnomadv3_af_xy: -1,
            gnomadv3_af_ami: -1,
            gnomadv3_af_asj: -1,
            gnomadv3_af_fin: -1,
            gnomadv3_af_mid: -1,
            gnomadv3_af_oth: -1,
            gnomad_af: -1,
            gnomad_eas_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_nfe_af: -1,
            gnomad_sas_af: -1,
            gnomad_asj_af: -1,
            gnomad_fin_af: -1,
            gnomad_oth_af: -1,
            is_pathogenic: false,
            variant_id: 'S:cnvkit:2',
            sv_id: 'S:cnvkit:2',
            afs: [
              {
                source_start: 182132,
                source_end: 189141,
                source_svType: 'DUP',
                AF: 0.000893,
                SAMPLE_COUNT: 1680,
                reciOverlap: 0.933,
                annoOverlap: 1,
                source_filter: 'hkbc',
                display_source: 'Hong Kong Birth Cohort',
                source: 'HKBC - CNVkit',
                svLength: 7010,
              },
            ],
            exts: [],
            intpns: [],
            caller: 'cnvkit',
            genotypes_index: [
              [0, 0],
              [0, 0],
              [0, 1],
            ],
            hg19_chrom: 'chr2',
            hg19_start: 114359401,
            hg19_strand: '-',
            hg19_end: 114351884,
            gene_symbol: [],
            ensembl_gene_id: [],
            entrez_gene_id: [],
            ensembl_transcript_id: [],
          },
          {
            allelic_depths: [],
            genotype_qualities: [],
            is_read: false,
            _id: '64e5a2209b592486e3891ba0',
            chrom: 'chr1',
            start: 350969,
            end: 623011,
            len: 272042,
            ref: 'G',
            alt: '<DEL>',
            p_lof: 'LOF',
            constraint_pli: '',
            constraint_oe_lof_upper: [],
            mane_select: 'ENST00000426406',
            variant_type: 'structural',
            type: 'DEL',
            impact: '',
            gene_objs: [
              {
                is_related: false,
                transcript: 'ENST00000426406',
                gene: 'OR4F29',
                gene_filter: 'OR4F29',
                entrez_gene_id: '729759',
                ensembl_gene_id: 'ENSG00000284733',
                is_canonical: true,
                is_mane_select: true,
                annotation_overlap: 1,
                reciprocal_overlap: 0.00345,
                bkpt_start: 450740,
                bkpt_end: 451678,
                default_show: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            highest_af: -1,
            gnomadv3_af: -1,
            gnomadv3_af_afr: -1,
            gnomadv3_af_amr: -1,
            gnomadv3_af_eas: -1,
            gnomadv3_af_nfe: -1,
            gnomadv3_af_sas: -1,
            gnomadv3_af_xx: -1,
            gnomadv3_af_xy: -1,
            gnomadv3_af_ami: -1,
            gnomadv3_af_asj: -1,
            gnomadv3_af_fin: -1,
            gnomadv3_af_mid: -1,
            gnomadv3_af_oth: -1,
            gnomad_af: -1,
            gnomad_eas_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_nfe_af: -1,
            gnomad_sas_af: -1,
            gnomad_asj_af: -1,
            gnomad_fin_af: -1,
            gnomad_oth_af: -1,
            is_pathogenic: false,
            variant_id: 'S:cnvkit:3',
            sv_id: 'S:cnvkit:3',
            afs: [],
            exts: [],
            intpns: [],
            caller: 'cnvkit',
            genotypes_index: [
              [0, 1],
              [0, 0],
              [0, 0],
            ],
            hg19_chrom: 'chr5',
            hg19_start: 180896423,
            hg19_strand: '-',
            hg19_end: 558391,
            gene_symbol: [],
            ensembl_gene_id: [],
            entrez_gene_id: [],
            ensembl_transcript_id: [],
          },
          {
            allelic_depths: [],
            genotype_qualities: [],
            is_read: false,
            _id: '64e5a2209b592486e3891ba1',
            chrom: 'chr1',
            start: 434977,
            end: 451479,
            len: 16502,
            ref: 'T',
            alt: '<DEL>',
            p_lof: 'LOF',
            constraint_pli: '',
            constraint_oe_lof_upper: [],
            mane_select: 'ENST00000426406',
            variant_type: 'structural',
            type: 'DEL',
            impact: '',
            gene_objs: [
              {
                is_related: true,
                transcript: 'ENST00000426406',
                gene: 'OR4F29',
                gene_filter: 'OR4F29',
                entrez_gene_id: '729759',
                ensembl_gene_id: 'ENSG00000284733',
                is_canonical: true,
                is_mane_select: true,
                annotation_overlap: 0.78807,
                reciprocal_overlap: 0.04484,
                bkpt_start: 450740,
                bkpt_end: 451678,
                default_show: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            highest_af: 0.00298,
            highest_af_info: {
              source: 'HKBC - CNVkit',
            },
            gnomadv3_af: -1,
            gnomadv3_af_afr: -1,
            gnomadv3_af_amr: -1,
            gnomadv3_af_eas: -1,
            gnomadv3_af_nfe: -1,
            gnomadv3_af_sas: -1,
            gnomadv3_af_xx: -1,
            gnomadv3_af_xy: -1,
            gnomadv3_af_ami: -1,
            gnomadv3_af_asj: -1,
            gnomadv3_af_fin: -1,
            gnomadv3_af_mid: -1,
            gnomadv3_af_oth: -1,
            gnomad_af: -1,
            gnomad_eas_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_nfe_af: -1,
            gnomad_sas_af: -1,
            gnomad_asj_af: -1,
            gnomad_fin_af: -1,
            gnomad_oth_af: -1,
            is_pathogenic: false,
            variant_id: 'S:cnvkit:4',
            sv_id: 'S:cnvkit:4',
            afs: [
              {
                source_start: 433978,
                source_end: 451979,
                source_svType: 'DEL',
                AF: 0.00298,
                SAMPLE_COUNT: 1680,
                reciOverlap: 0.917,
                annoOverlap: 0.917,
                source_filter: 'hkbc',
                display_source: 'Hong Kong Birth Cohort',
                source: 'HKBC - CNVkit',
                svLength: -18002,
              },
            ],
            exts: [],
            intpns: [],
            caller: 'cnvkit',
            genotypes_index: [
              [0, 0],
              [0, 0],
              [0, 1],
            ],
            hg19_chrom: 'chr5',
            hg19_start: 180810987,
            hg19_strand: '-',
            hg19_end: 180794485,
            gene_symbol: ['OR4F29'],
            ensembl_gene_id: ['ENSG00000284733'],
            entrez_gene_id: ['729759'],
            ensembl_transcript_id: ['ENST00000426406'],
          },
          {
            allelic_depths: [],
            genotype_qualities: [],
            is_read: false,
            _id: '64e5a2209b592486e3891ba2',
            chrom: 'chr1',
            start: 439978,
            end: 446979,
            len: 7001,
            ref: 'A',
            alt: '<DUP>',
            p_lof: 'False',
            constraint_pli: '',
            constraint_oe_lof_upper: [],
            variant_type: 'structural',
            type: 'DUP',
            impact: '',
            gene_objs: [],
            highest_af: 0.000893,
            highest_af_info: {
              source: 'HKBC - CNVkit',
            },
            gnomadv3_af: -1,
            gnomadv3_af_afr: -1,
            gnomadv3_af_amr: -1,
            gnomadv3_af_eas: -1,
            gnomadv3_af_nfe: -1,
            gnomadv3_af_sas: -1,
            gnomadv3_af_xx: -1,
            gnomadv3_af_xy: -1,
            gnomadv3_af_ami: -1,
            gnomadv3_af_asj: -1,
            gnomadv3_af_fin: -1,
            gnomadv3_af_mid: -1,
            gnomadv3_af_oth: -1,
            gnomad_af: -1,
            gnomad_eas_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_nfe_af: -1,
            gnomad_sas_af: -1,
            gnomad_asj_af: -1,
            gnomad_fin_af: -1,
            gnomad_oth_af: -1,
            is_pathogenic: false,
            variant_id: 'S:cnvkit:5',
            sv_id: 'S:cnvkit:5',
            afs: [
              {
                source_start: 440479,
                source_end: 446979,
                source_svType: 'DUP',
                AF: 0.000893,
                SAMPLE_COUNT: 1680,
                reciOverlap: 0.929,
                annoOverlap: 1,
                source_filter: 'hkbc',
                display_source: 'Hong Kong Birth Cohort',
                source: 'HKBC - CNVkit',
                svLength: 6501,
              },
            ],
            exts: [],
            intpns: [],
            caller: 'cnvkit',
            genotypes_index: [
              [0, 0],
              [0, 1],
              [0, 0],
            ],
            hg19_chrom: 'chr5',
            hg19_start: 180805986,
            hg19_strand: '-',
            hg19_end: 180798985,
            gene_symbol: [],
            ensembl_gene_id: [],
            entrez_gene_id: [],
            ensembl_transcript_id: [],
          },
          {
            allelic_depths: [],
            genotype_qualities: [],
            is_read: false,
            _id: '64e5a2209b592486e3891ba3',
            chrom: 'chr1',
            start: 458480,
            end: 464480,
            len: 6000,
            ref: 'G',
            alt: '<DUP>',
            p_lof: 'False',
            constraint_pli: '',
            constraint_oe_lof_upper: [],
            variant_type: 'structural',
            type: 'DUP',
            impact: '',
            gene_objs: [],
            highest_af: 0.00119,
            highest_af_info: {
              source: 'HKBC - CNVkit',
            },
            gnomadv3_af: -1,
            gnomadv3_af_afr: -1,
            gnomadv3_af_amr: -1,
            gnomadv3_af_eas: -1,
            gnomadv3_af_nfe: -1,
            gnomadv3_af_sas: -1,
            gnomadv3_af_xx: -1,
            gnomadv3_af_xy: -1,
            gnomadv3_af_ami: -1,
            gnomadv3_af_asj: -1,
            gnomadv3_af_fin: -1,
            gnomadv3_af_mid: -1,
            gnomadv3_af_oth: -1,
            gnomad_af: -1,
            gnomad_eas_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_nfe_af: -1,
            gnomad_sas_af: -1,
            gnomad_asj_af: -1,
            gnomad_fin_af: -1,
            gnomad_oth_af: -1,
            is_pathogenic: false,
            variant_id: 'S:cnvkit:6',
            sv_id: 'S:cnvkit:6',
            afs: [
              {
                source_start: 458981,
                source_end: 464480,
                source_svType: 'DUP',
                AF: 0.00119,
                SAMPLE_COUNT: 1680,
                reciOverlap: 0.917,
                annoOverlap: 1,
                source_filter: 'hkbc',
                display_source: 'Hong Kong Birth Cohort',
                source: 'HKBC - CNVkit',
                svLength: 5500,
              },
            ],
            exts: [],
            intpns: [],
            caller: 'cnvkit',
            genotypes_index: [
              [0, 0],
              [0, 0],
              [0, 1],
            ],
            hg19_chrom: 'chr5',
            hg19_start: 180787486,
            hg19_strand: '-',
            hg19_end: 180781469,
            gene_symbol: [],
            ensembl_gene_id: [],
            entrez_gene_id: [],
            ensembl_transcript_id: [],
          },
          {
            allelic_depths: [],
            genotype_qualities: [],
            is_read: false,
            _id: '64e5a2209b592486e3891ba4',
            chrom: 'chr1',
            start: 586989,
            end: 600997,
            len: 14008,
            ref: 'A',
            alt: '<DEL>',
            p_lof: 'False',
            constraint_pli: '',
            constraint_oe_lof_upper: [],
            variant_type: 'structural',
            type: 'DEL',
            impact: '',
            gene_objs: [],
            highest_af: 0.000893,
            highest_af_info: {
              source: 'HKBC - CNVkit',
            },
            gnomadv3_af: -1,
            gnomadv3_af_afr: -1,
            gnomadv3_af_amr: -1,
            gnomadv3_af_eas: -1,
            gnomadv3_af_nfe: -1,
            gnomadv3_af_sas: -1,
            gnomadv3_af_xx: -1,
            gnomadv3_af_xy: -1,
            gnomadv3_af_ami: -1,
            gnomadv3_af_asj: -1,
            gnomadv3_af_fin: -1,
            gnomadv3_af_mid: -1,
            gnomadv3_af_oth: -1,
            gnomad_af: -1,
            gnomad_eas_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_nfe_af: -1,
            gnomad_sas_af: -1,
            gnomad_asj_af: -1,
            gnomad_fin_af: -1,
            gnomad_oth_af: -1,
            is_pathogenic: false,
            variant_id: 'S:cnvkit:7',
            sv_id: 'S:cnvkit:7',
            afs: [
              {
                source_start: 587490,
                source_end: 600497,
                source_svType: 'DEL',
                AF: 0.000893,
                SAMPLE_COUNT: 1680,
                reciOverlap: 0.929,
                annoOverlap: 1,
                source_filter: 'hkbc',
                display_source: 'Hong Kong Birth Cohort',
                source: 'HKBC - CNVkit',
                svLength: -13008,
              },
            ],
            exts: [],
            intpns: [],
            caller: 'cnvkit',
            genotypes_index: [
              [0, 0],
              [0, 1],
              [0, 0],
            ],
            hg19_chrom: 'chr1',
            hg19_start: 522369,
            hg19_strand: '+',
            hg19_end: 536377,
            gene_symbol: [],
            ensembl_gene_id: [],
            entrez_gene_id: [],
            ensembl_transcript_id: [],
          },
          {
            allelic_depths: [],
            genotype_qualities: [],
            is_read: false,
            _id: '64e5a2209b592486e3891ba5',
            chrom: 'chr1',
            start: 586989,
            end: 601498,
            len: 14509,
            ref: 'A',
            alt: '<DEL>',
            p_lof: 'False',
            constraint_pli: '',
            constraint_oe_lof_upper: [],
            variant_type: 'structural',
            type: 'DEL',
            impact: '',
            gene_objs: [],
            highest_af: 0.000298,
            highest_af_info: {
              source: 'HKBC - CNVkit',
            },
            gnomadv3_af: -1,
            gnomadv3_af_afr: -1,
            gnomadv3_af_amr: -1,
            gnomadv3_af_eas: -1,
            gnomadv3_af_nfe: -1,
            gnomadv3_af_sas: -1,
            gnomadv3_af_xx: -1,
            gnomadv3_af_xy: -1,
            gnomadv3_af_ami: -1,
            gnomadv3_af_asj: -1,
            gnomadv3_af_fin: -1,
            gnomadv3_af_mid: -1,
            gnomadv3_af_oth: -1,
            gnomad_af: -1,
            gnomad_eas_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_nfe_af: -1,
            gnomad_sas_af: -1,
            gnomad_asj_af: -1,
            gnomad_fin_af: -1,
            gnomad_oth_af: -1,
            is_pathogenic: false,
            variant_id: 'S:cnvkit:8',
            sv_id: 'S:cnvkit:8',
            afs: [
              {
                source_start: 587490,
                source_end: 600997,
                source_svType: 'DEL',
                AF: 0.000298,
                SAMPLE_COUNT: 1680,
                reciOverlap: 0.931,
                annoOverlap: 1,
                source_filter: 'hkbc',
                display_source: 'Hong Kong Birth Cohort',
                source: 'HKBC - CNVkit',
                svLength: -13508,
              },
            ],
            exts: [],
            intpns: [],
            caller: 'cnvkit',
            genotypes_index: [
              [0, 0],
              [0, 0],
              [0, 1],
            ],
            hg19_chrom: 'chr1',
            hg19_start: 522369,
            hg19_strand: '+',
            hg19_end: 536878,
            gene_symbol: [],
            ensembl_gene_id: [],
            entrez_gene_id: [],
            ensembl_transcript_id: [],
          },
          {
            allelic_depths: [],
            genotype_qualities: [],
            is_read: false,
            _id: '64e5a2209b592486e3891ba6',
            chrom: 'chr1',
            start: 659033,
            end: 668539,
            len: 9506,
            ref: 'A',
            alt: '<DEL>',
            p_lof: 'False',
            constraint_pli: '',
            constraint_oe_lof_upper: [],
            variant_type: 'structural',
            type: 'DEL',
            impact: '',
            gene_objs: [],
            highest_af: -1,
            gnomadv3_af: -1,
            gnomadv3_af_afr: -1,
            gnomadv3_af_amr: -1,
            gnomadv3_af_eas: -1,
            gnomadv3_af_nfe: -1,
            gnomadv3_af_sas: -1,
            gnomadv3_af_xx: -1,
            gnomadv3_af_xy: -1,
            gnomadv3_af_ami: -1,
            gnomadv3_af_asj: -1,
            gnomadv3_af_fin: -1,
            gnomadv3_af_mid: -1,
            gnomadv3_af_oth: -1,
            gnomad_af: -1,
            gnomad_eas_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_nfe_af: -1,
            gnomad_sas_af: -1,
            gnomad_asj_af: -1,
            gnomad_fin_af: -1,
            gnomad_oth_af: -1,
            is_pathogenic: false,
            variant_id: 'S:cnvkit:9',
            sv_id: 'S:cnvkit:9',
            afs: [],
            exts: [],
            intpns: [],
            caller: 'cnvkit',
            genotypes_index: [
              [0, 1],
              [0, 0],
              [0, 0],
            ],
            hg19_chrom: 'chr1',
            hg19_start: 594413,
            hg19_strand: '+',
            hg19_end: 603919,
            gene_symbol: [],
            ensembl_gene_id: [],
            entrez_gene_id: [],
            ensembl_transcript_id: [],
          },
          {
            allelic_depths: [],
            genotype_qualities: [],
            is_read: false,
            _id: '64e5a2209b592486e3891ba7',
            chrom: 'chr1',
            start: 670040,
            end: 682047,
            len: 12007,
            ref: 'T',
            alt: '<DEL>',
            p_lof: 'False',
            constraint_pli: '',
            constraint_oe_lof_upper: [],
            variant_type: 'structural',
            type: 'DEL',
            impact: '',
            gene_objs: [],
            highest_af: 0.00149,
            highest_af_info: {
              source: 'HKBC - CNVkit',
            },
            gnomadv3_af: -1,
            gnomadv3_af_afr: -1,
            gnomadv3_af_amr: -1,
            gnomadv3_af_eas: -1,
            gnomadv3_af_nfe: -1,
            gnomadv3_af_sas: -1,
            gnomadv3_af_xx: -1,
            gnomadv3_af_xy: -1,
            gnomadv3_af_ami: -1,
            gnomadv3_af_asj: -1,
            gnomadv3_af_fin: -1,
            gnomadv3_af_mid: -1,
            gnomadv3_af_oth: -1,
            gnomad_af: -1,
            gnomad_eas_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_nfe_af: -1,
            gnomad_sas_af: -1,
            gnomad_asj_af: -1,
            gnomad_fin_af: -1,
            gnomad_oth_af: -1,
            is_pathogenic: false,
            variant_id: 'S:cnvkit:10',
            sv_id: 'S:cnvkit:10',
            afs: [
              {
                source_start: 669541,
                source_end: 682548,
                source_svType: 'DEL',
                AF: 0.00149,
                SAMPLE_COUNT: 1680,
                reciOverlap: 0.923,
                annoOverlap: 0.923,
                source_filter: 'hkbc',
                display_source: 'Hong Kong Birth Cohort',
                source: 'HKBC - CNVkit',
                svLength: -13008,
              },
            ],
            exts: [],
            intpns: [],
            caller: 'cnvkit',
            genotypes_index: [
              [0, 0],
              [0, 0],
              [0, 1],
            ],
            hg19_chrom: 'chr1',
            hg19_start: 605420,
            hg19_strand: '+',
            hg19_end: 617427,
            gene_symbol: [],
            ensembl_gene_id: [],
            entrez_gene_id: [],
            ensembl_transcript_id: [],
          },
          {
            is_read: false,
            _id: '64e59ef69c2aec9c8bd9f986',
            variant_id: '3',
            end: 924024,
            ref: 'C',
            alt: 'G',
            start: 924024,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616016',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '1/14',
                allele: 'G',
                impact: ['5_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000616016.5:c.-408C>G',
                existing_variation: 'rs71509444',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                mane_select: 'NM_001385641.1',
                source: 'Ensembl',
                existing_inframe_oorfs: 0,
                existing_outofframe_oorfs: 0,
                existing_uorfs: 1,
                cadd_phred: 10.6,
                cadd_raw: 0.915468,
                impact_so: '5_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000342066',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509444',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                ccds: 'CCDS2.2',
                source: 'Ensembl',
                cadd_phred: 10.6,
                cadd_raw: 0.915468,
                constraint_mis_z: -3.4361,
                constraint_oe_lof: 0.89656,
                constraint_oe_lof_lower: 0.64,
                constraint_oe_lof_upper: 1.28,
                constraint_oe_mis: 1.5082,
                constraint_oe_mis_lower: 1.405,
                constraint_oe_mis_upper: 1.619,
                constraint_oe_syn: 1.6983,
                constraint_oe_syn_lower: 1.537,
                constraint_oe_syn_upper: 1.875,
                constraint_pli: 4.7681e-14,
                constraint_syn_z: -6.9212,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 4.7681e-14,
                loeuf: 1.28,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000437963',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509444',
                strand: 1,
                flags: 'cds_end_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 10.6,
                cadd_raw: 0.915468,
                constraint_mis_z: -0.44824,
                constraint_oe_lof: 0.54514,
                constraint_oe_lof_lower: 0.247,
                constraint_oe_lof_upper: 1.385,
                constraint_oe_mis: 1.1465,
                constraint_oe_mis_lower: 0.961,
                constraint_oe_mis_upper: 1.373,
                constraint_oe_syn: 1.1572,
                constraint_oe_syn_lower: 0.883,
                constraint_oe_syn_upper: 1.534,
                constraint_pli: 0.024691,
                constraint_syn_z: -0.67958,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 0.024691,
                loeuf: 1.385,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616125',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509444',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 10.6,
                cadd_raw: 0.915468,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000617307',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509444',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 10.6,
                cadd_raw: 0.915468,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618181',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509444',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 10.6,
                cadd_raw: 0.915468,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618323',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '1/14',
                allele: 'G',
                impact: ['5_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000618323.5:c.-408C>G',
                existing_variation: 'rs71509444',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                existing_inframe_oorfs: 0,
                existing_outofframe_oorfs: 0,
                existing_uorfs: 1,
                cadd_phred: 10.6,
                cadd_raw: 0.915468,
                impact_so: '5_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618779',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509444',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 10.6,
                cadd_raw: 0.915468,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000622503',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509444',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 10.6,
                cadd_raw: 0.915468,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000616016',
            ensembl_gene_id: ['ENSG00000187634'],
            is_exonic: 1,
            is_coding: 0,
            impact: '5_prime_UTR_variant',
            hgvsc: 'ENST00000616016.5:c.-408C>G',
            existing_variation: 'rs71509444',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            existing_inframe_oorfs: '0',
            existing_outofframe_oorfs: '0',
            existing_uorfs: '1',
            cadd_phred: 10.6,
            cadd_raw: 0.915468,
            exomiser_ad_exgenespheno: 0.2042,
            exomiser_xr_exgenescombi: 0.000014154,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.3942,
            gnomadv3_af_nfe: 0.9736,
            gnomadv3_af_xy: 0.7781,
            exomiser_xd_exgenescombi: 0.000014154,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.914,
            gnomadv3_af: 0.7788,
            exomiser_mt_exgenespheno: 0.2042,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.000014154,
            gnomadv3_af_eas: 0.5666,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.957,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.000014154,
            exomiser_ad_exgenescombi: 0.000014154,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9693,
            gnomadv3_af_fin: 0.9571,
            exomiser_ar_exgenespheno: 0.2042,
            gnomadv3_af_sas: 0.877,
            gnomadv3_af_amr: 0.8205,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.7795,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.8135,
            exomiser_xr_exgenespheno: 0.2042,
            exomiser_xd_exgenespheno: 0.2042,
            impact_severity: 'LOW',
            quality: 1467.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [17, 18],
              [17, 22],
              [30, 20],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['SAMD11'],
            ensembl_transcript_id: [
              'ENST00000616016',
              'ENST00000342066',
              'ENST00000437963',
              'ENST00000616125',
              'ENST00000617307',
              'ENST00000618181',
              'ENST00000618323',
              'ENST00000618779',
              'ENST00000622503',
            ],
            variant_type: 'small',
            highest_af: 0.7788,
            highest_af_info: {
              source: 'gnomad v3 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 859404,
            hg19_strand: '+',
            hg19_end: 859404,
            entrez_gene_id: ['148398'],
          },
          {
            is_read: false,
            _id: '64e59ef69c2aec9c8bd9f987',
            variant_id: '4',
            end: 924310,
            ref: 'C',
            alt: 'G',
            start: 924310,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616016',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '1/14',
                allele: 'G',
                impact: ['5_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000616016.5:c.-122C>G',
                existing_variation: 'rs71509445',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                mane_select: 'NM_001385641.1',
                source: 'Ensembl',
                existing_inframe_oorfs: 0,
                existing_outofframe_oorfs: 0,
                existing_uorfs: 1,
                cadd_phred: 15.41,
                cadd_raw: 1.450099,
                impact_so: '5_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000342066',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509445',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                ccds: 'CCDS2.2',
                source: 'Ensembl',
                cadd_phred: 15.41,
                cadd_raw: 1.450099,
                constraint_mis_z: -3.4361,
                constraint_oe_lof: 0.89656,
                constraint_oe_lof_lower: 0.64,
                constraint_oe_lof_upper: 1.28,
                constraint_oe_mis: 1.5082,
                constraint_oe_mis_lower: 1.405,
                constraint_oe_mis_upper: 1.619,
                constraint_oe_syn: 1.6983,
                constraint_oe_syn_lower: 1.537,
                constraint_oe_syn_upper: 1.875,
                constraint_pli: 4.7681e-14,
                constraint_syn_z: -6.9212,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 4.7681e-14,
                loeuf: 1.28,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000437963',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509445',
                strand: 1,
                flags: 'cds_end_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 15.41,
                cadd_raw: 1.450099,
                constraint_mis_z: -0.44824,
                constraint_oe_lof: 0.54514,
                constraint_oe_lof_lower: 0.247,
                constraint_oe_lof_upper: 1.385,
                constraint_oe_mis: 1.1465,
                constraint_oe_mis_lower: 0.961,
                constraint_oe_mis_upper: 1.373,
                constraint_oe_syn: 1.1572,
                constraint_oe_syn_lower: 0.883,
                constraint_oe_syn_upper: 1.534,
                constraint_pli: 0.024691,
                constraint_syn_z: -0.67958,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 0.024691,
                loeuf: 1.385,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616125',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509445',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 15.41,
                cadd_raw: 1.450099,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000617307',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509445',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 15.41,
                cadd_raw: 1.450099,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618181',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509445',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 15.41,
                cadd_raw: 1.450099,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618323',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '1/14',
                allele: 'G',
                impact: ['5_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000618323.5:c.-122C>G',
                existing_variation: 'rs71509445',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                existing_inframe_oorfs: 0,
                existing_outofframe_oorfs: 0,
                existing_uorfs: 1,
                cadd_phred: 15.41,
                cadd_raw: 1.450099,
                impact_so: '5_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618779',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509445',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 15.41,
                cadd_raw: 1.450099,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000622503',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509445',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 15.41,
                cadd_raw: 1.450099,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000616016',
            ensembl_gene_id: ['ENSG00000187634'],
            is_exonic: 1,
            is_coding: 0,
            impact: '5_prime_UTR_variant',
            hgvsc: 'ENST00000616016.5:c.-122C>G',
            existing_variation: 'rs71509445',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            existing_inframe_oorfs: '0',
            existing_outofframe_oorfs: '0',
            existing_uorfs: '1',
            cadd_phred: 15.41,
            cadd_raw: 1.450099,
            exomiser_ad_exgenespheno: 0.2042,
            exomiser_xr_exgenescombi: 0.000014154,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.4464,
            gnomadv3_af_nfe: 0.9791,
            gnomadv3_af_xy: 0.7954,
            exomiser_xd_exgenescombi: 0.000014154,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9391,
            gnomadv3_af: 0.7968,
            exomiser_mt_exgenespheno: 0.2042,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.000014154,
            gnomadv3_af_eas: 0.564,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9635,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.000014154,
            exomiser_ad_exgenescombi: 0.000014154,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9693,
            gnomadv3_af_fin: 0.9606,
            exomiser_ar_exgenespheno: 0.2042,
            gnomadv3_af_sas: 0.8798,
            gnomadv3_af_amr: 0.8318,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.7981,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.8283,
            exomiser_xr_exgenespheno: 0.2042,
            exomiser_xd_exgenespheno: 0.2042,
            impact_severity: 'LOW',
            quality: 1768.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [15, 19],
              [24, 17],
              [18, 13],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['SAMD11'],
            ensembl_transcript_id: [
              'ENST00000616016',
              'ENST00000342066',
              'ENST00000437963',
              'ENST00000616125',
              'ENST00000617307',
              'ENST00000618181',
              'ENST00000618323',
              'ENST00000618779',
              'ENST00000622503',
            ],
            variant_type: 'small',
            highest_af: 0.7968,
            highest_af_info: {
              source: 'gnomad v3 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 859690,
            hg19_strand: '+',
            hg19_end: 859690,
            entrez_gene_id: ['148398'],
          },
          {
            is_read: false,
            _id: '64e59ef69c2aec9c8bd9f988',
            variant_id: '5',
            end: 924321,
            ref: 'C',
            alt: 'G',
            start: 924321,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616016',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '1/14',
                allele: 'G',
                impact: ['5_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000616016.5:c.-111C>G',
                existing_variation: 'rs71509446',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                mane_select: 'NM_001385641.1',
                source: 'Ensembl',
                existing_inframe_oorfs: 0,
                existing_outofframe_oorfs: 0,
                existing_uorfs: 1,
                cadd_phred: 11.91,
                cadd_raw: 1.022879,
                impact_so: '5_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000342066',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509446',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                ccds: 'CCDS2.2',
                source: 'Ensembl',
                cadd_phred: 11.91,
                cadd_raw: 1.022879,
                constraint_mis_z: -3.4361,
                constraint_oe_lof: 0.89656,
                constraint_oe_lof_lower: 0.64,
                constraint_oe_lof_upper: 1.28,
                constraint_oe_mis: 1.5082,
                constraint_oe_mis_lower: 1.405,
                constraint_oe_mis_upper: 1.619,
                constraint_oe_syn: 1.6983,
                constraint_oe_syn_lower: 1.537,
                constraint_oe_syn_upper: 1.875,
                constraint_pli: 4.7681e-14,
                constraint_syn_z: -6.9212,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 4.7681e-14,
                loeuf: 1.28,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000437963',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509446',
                strand: 1,
                flags: 'cds_end_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 11.91,
                cadd_raw: 1.022879,
                constraint_mis_z: -0.44824,
                constraint_oe_lof: 0.54514,
                constraint_oe_lof_lower: 0.247,
                constraint_oe_lof_upper: 1.385,
                constraint_oe_mis: 1.1465,
                constraint_oe_mis_lower: 0.961,
                constraint_oe_mis_upper: 1.373,
                constraint_oe_syn: 1.1572,
                constraint_oe_syn_lower: 0.883,
                constraint_oe_syn_upper: 1.534,
                constraint_pli: 0.024691,
                constraint_syn_z: -0.67958,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 0.024691,
                loeuf: 1.385,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616125',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509446',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 11.91,
                cadd_raw: 1.022879,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000617307',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509446',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 11.91,
                cadd_raw: 1.022879,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618181',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509446',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 11.91,
                cadd_raw: 1.022879,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618323',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '1/14',
                allele: 'G',
                impact: ['5_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000618323.5:c.-111C>G',
                existing_variation: 'rs71509446',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                existing_inframe_oorfs: 0,
                existing_outofframe_oorfs: 0,
                existing_uorfs: 1,
                cadd_phred: 11.91,
                cadd_raw: 1.022879,
                impact_so: '5_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618779',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509446',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 11.91,
                cadd_raw: 1.022879,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000622503',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509446',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 11.91,
                cadd_raw: 1.022879,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000616016',
            ensembl_gene_id: ['ENSG00000187634'],
            is_exonic: 1,
            is_coding: 0,
            impact: '5_prime_UTR_variant',
            hgvsc: 'ENST00000616016.5:c.-111C>G',
            existing_variation: 'rs71509446',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            existing_inframe_oorfs: '0',
            existing_outofframe_oorfs: '0',
            existing_uorfs: '1',
            cadd_phred: 11.91,
            cadd_raw: 1.022879,
            exomiser_ad_exgenespheno: 0.2042,
            exomiser_xr_exgenescombi: 0.000014154,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.4469,
            gnomadv3_af_nfe: 0.9792,
            gnomadv3_af_xy: 0.7955,
            exomiser_xd_exgenescombi: 0.000014154,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9391,
            gnomadv3_af: 0.7969,
            exomiser_mt_exgenespheno: 0.2042,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.000014154,
            gnomadv3_af_eas: 0.5645,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9635,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.000014154,
            exomiser_ad_exgenescombi: 0.000014154,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9693,
            gnomadv3_af_fin: 0.9608,
            exomiser_ar_exgenespheno: 0.2042,
            gnomadv3_af_sas: 0.8798,
            gnomadv3_af_amr: 0.8318,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.7983,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.829,
            exomiser_xr_exgenespheno: 0.2042,
            exomiser_xd_exgenespheno: 0.2042,
            impact_severity: 'LOW',
            quality: 1744.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [16, 16],
              [23, 17],
              [18, 15],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['SAMD11'],
            ensembl_transcript_id: [
              'ENST00000616016',
              'ENST00000342066',
              'ENST00000437963',
              'ENST00000616125',
              'ENST00000617307',
              'ENST00000618181',
              'ENST00000618323',
              'ENST00000618779',
              'ENST00000622503',
            ],
            variant_type: 'small',
            highest_af: 0.7969,
            highest_af_info: {
              source: 'gnomad v3 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 859701,
            hg19_strand: '+',
            hg19_end: 859701,
            entrez_gene_id: ['148398'],
          },
          {
            is_read: false,
            _id: '64e59ef69c2aec9c8bd9f989',
            variant_id: '6',
            end: 924533,
            ref: 'A',
            alt: 'G',
            start: 924533,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616016',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '1/14',
                codon_change: 'ccA/ccG',
                aa_change: 'P',
                aa_length: 34,
                allele: 'G',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000616016.5:c.102A>G',
                hgvsp: 'ENSP00000478421.2:p.Pro34%3D',
                cds_position: 102,
                existing_variation: 'rs112703963',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                mane_select: 'NM_001385641.1',
                source: 'Ensembl',
                domains: 'Low_complexity_(Seg):seg',
                cadd_phred: 7.649,
                cadd_raw: 0.614071,
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000342066',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs112703963',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                ccds: 'CCDS2.2',
                source: 'Ensembl',
                cadd_phred: 7.649,
                cadd_raw: 0.614071,
                constraint_mis_z: -3.4361,
                constraint_oe_lof: 0.89656,
                constraint_oe_lof_lower: 0.64,
                constraint_oe_lof_upper: 1.28,
                constraint_oe_mis: 1.5082,
                constraint_oe_mis_lower: 1.405,
                constraint_oe_mis_upper: 1.619,
                constraint_oe_syn: 1.6983,
                constraint_oe_syn_lower: 1.537,
                constraint_oe_syn_upper: 1.875,
                constraint_pli: 4.7681e-14,
                constraint_syn_z: -6.9212,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 4.7681e-14,
                loeuf: 1.28,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000437963',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs112703963',
                strand: 1,
                flags: 'cds_end_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 7.649,
                cadd_raw: 0.614071,
                constraint_mis_z: -0.44824,
                constraint_oe_lof: 0.54514,
                constraint_oe_lof_lower: 0.247,
                constraint_oe_lof_upper: 1.385,
                constraint_oe_mis: 1.1465,
                constraint_oe_mis_lower: 0.961,
                constraint_oe_mis_upper: 1.373,
                constraint_oe_syn: 1.1572,
                constraint_oe_syn_lower: 0.883,
                constraint_oe_syn_upper: 1.534,
                constraint_pli: 0.024691,
                constraint_syn_z: -0.67958,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 0.024691,
                loeuf: 1.385,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616125',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs112703963',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 7.649,
                cadd_raw: 0.614071,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000617307',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs112703963',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 7.649,
                cadd_raw: 0.614071,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618181',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs112703963',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 7.649,
                cadd_raw: 0.614071,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618323',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '1/14',
                codon_change: 'ccA/ccG',
                aa_change: 'P',
                aa_length: 34,
                allele: 'G',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000618323.5:c.102A>G',
                hgvsp: 'ENSP00000480678.2:p.Pro34%3D',
                cds_position: 102,
                existing_variation: 'rs112703963',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                domains: 'Low_complexity_(Seg):seg',
                cadd_phred: 7.649,
                cadd_raw: 0.614071,
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618779',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs112703963',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 7.649,
                cadd_raw: 0.614071,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000622503',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs112703963',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 7.649,
                cadd_raw: 0.614071,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000616016',
            ensembl_gene_id: ['ENSG00000187634'],
            is_exonic: 1,
            is_coding: 1,
            aa_change: 'P',
            impact: 'synonymous_variant',
            hgvsc: 'ENST00000616016.5:c.102A>G',
            hgvsp: 'ENSP00000478421.2:p.Pro34%3D',
            existing_variation: 'rs112703963',
            source: 'Ensembl',
            domains: 'Low_complexity_(Seg):seg',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            cadd_phred: 7.649,
            cadd_raw: 0.614071,
            exomiser_ad_exgenespheno: 0.2042,
            exomiser_xr_exgenescombi: 0.000014154,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.4795,
            gnomadv3_af_nfe: 0.9802,
            gnomadv3_af_xy: 0.8227,
            exomiser_xd_exgenescombi: 0.000014154,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9427,
            gnomadv3_af: 0.8215,
            exomiser_mt_exgenespheno: 0.2042,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.000014154,
            gnomadv3_af_eas: 0.6886,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9732,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.000014154,
            exomiser_ad_exgenescombi: 0.000014154,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9693,
            gnomadv3_af_fin: 0.9618,
            exomiser_ar_exgenespheno: 0.2042,
            gnomadv3_af_sas: 0.9098,
            gnomadv3_af_amr: 0.9213,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.8204,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.8549,
            exomiser_xr_exgenespheno: 0.2042,
            exomiser_xd_exgenespheno: 0.2042,
            impact_severity: 'LOW',
            quality: 1214.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [18, 24],
              [27, 15],
              [23, 15],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['SAMD11'],
            ensembl_transcript_id: [
              'ENST00000616016',
              'ENST00000342066',
              'ENST00000437963',
              'ENST00000616125',
              'ENST00000617307',
              'ENST00000618181',
              'ENST00000618323',
              'ENST00000618779',
              'ENST00000622503',
            ],
            variant_type: 'small',
            highest_af: 0.8215,
            highest_af_info: {
              source: 'gnomad v3 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 859913,
            hg19_strand: '+',
            hg19_end: 859913,
            entrez_gene_id: ['148398'],
          },
          {
            is_read: false,
            _id: '64e59ef69c2aec9c8bd9f98a',
            variant_id: '7',
            end: 925036,
            ref: 'G',
            alt: 'A',
            start: 925036,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616016',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '1/13',
                hgvsc: 'ENST00000616016.5:c.517+88G>A',
                existing_variation: 'rs61464428',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                mane_select: 'NM_001385641.1',
                source: 'Ensembl',
                cadd_phred: 8.136,
                cadd_raw: 0.665529,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000342066',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs61464428',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                ccds: 'CCDS2.2',
                source: 'Ensembl',
                cadd_phred: 8.136,
                cadd_raw: 0.665529,
                constraint_mis_z: -3.4361,
                constraint_oe_lof: 0.89656,
                constraint_oe_lof_lower: 0.64,
                constraint_oe_lof_upper: 1.28,
                constraint_oe_mis: 1.5082,
                constraint_oe_mis_lower: 1.405,
                constraint_oe_mis_upper: 1.619,
                constraint_oe_syn: 1.6983,
                constraint_oe_syn_lower: 1.537,
                constraint_oe_syn_upper: 1.875,
                constraint_pli: 4.7681e-14,
                constraint_syn_z: -6.9212,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 4.7681e-14,
                loeuf: 1.28,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000437963',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs61464428',
                strand: 1,
                flags: 'cds_end_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 8.136,
                cadd_raw: 0.665529,
                constraint_mis_z: -0.44824,
                constraint_oe_lof: 0.54514,
                constraint_oe_lof_lower: 0.247,
                constraint_oe_lof_upper: 1.385,
                constraint_oe_mis: 1.1465,
                constraint_oe_mis_lower: 0.961,
                constraint_oe_mis_upper: 1.373,
                constraint_oe_syn: 1.1572,
                constraint_oe_syn_lower: 0.883,
                constraint_oe_syn_upper: 1.534,
                constraint_pli: 0.024691,
                constraint_syn_z: -0.67958,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 0.024691,
                loeuf: 1.385,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616125',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs61464428',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 8.136,
                cadd_raw: 0.665529,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000617307',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs61464428',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 8.136,
                cadd_raw: 0.665529,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618181',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs61464428',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 8.136,
                cadd_raw: 0.665529,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618323',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '1/13',
                hgvsc: 'ENST00000618323.5:c.517+88G>A',
                existing_variation: 'rs61464428',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 8.136,
                cadd_raw: 0.665529,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618779',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs61464428',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 8.136,
                cadd_raw: 0.665529,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000622503',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs61464428',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 8.136,
                cadd_raw: 0.665529,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000616016',
            ensembl_gene_id: ['ENSG00000187634'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant',
            hgvsc: 'ENST00000616016.5:c.517+88G>A',
            existing_variation: 'rs61464428',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            cadd_phred: 8.136,
            cadd_raw: 0.665529,
            exomiser_ad_exgenespheno: 0.2042,
            exomiser_xr_exgenescombi: 0.000014154,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.3145,
            gnomadv3_af_nfe: 0.7635,
            gnomadv3_af_xy: 0.607,
            exomiser_xd_exgenescombi: 0.000014154,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.7308,
            gnomadv3_af: 0.6079,
            exomiser_mt_exgenespheno: 0.2042,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.000014154,
            gnomadv3_af_eas: 0.4694,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.7361,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.000014154,
            exomiser_ad_exgenescombi: 0.000014154,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.5899,
            gnomadv3_af_fin: 0.7727,
            exomiser_ar_exgenespheno: 0.2042,
            gnomadv3_af_sas: 0.6107,
            gnomadv3_af_amr: 0.6062,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.6088,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.6334,
            exomiser_xr_exgenespheno: 0.2042,
            exomiser_xd_exgenespheno: 0.2042,
            impact_severity: 'LOW',
            quality: 1302.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [19, 24],
              [15, 13],
              [20, 14],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['SAMD11'],
            ensembl_transcript_id: [
              'ENST00000616016',
              'ENST00000342066',
              'ENST00000437963',
              'ENST00000616125',
              'ENST00000617307',
              'ENST00000618181',
              'ENST00000618323',
              'ENST00000618779',
              'ENST00000622503',
            ],
            variant_type: 'small',
            highest_af: 0.6079,
            highest_af_info: {
              source: 'gnomad v3 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 860416,
            hg19_strand: '+',
            hg19_end: 860416,
            entrez_gene_id: ['148398'],
          },
          {
            is_read: false,
            _id: '64e59ef69c2aec9c8bd9f98b',
            variant_id: '8',
            end: 930939,
            ref: 'G',
            alt: 'A',
            start: 930939,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616016',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '3/13',
                hgvsc: 'ENST00000616016.5:c.792-100G>A',
                existing_variation: 'rs9988021',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                mane_select: 'NM_001385641.1',
                source: 'Ensembl',
                cadd_phred: 0.192,
                cadd_raw: -0.400011,
                spliceai_pred_dp_ag: 2,
                spliceai_pred_dp_al: -14,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: 19,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000341065',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '1/11',
                hgvsc: 'ENST00000341065.8:c.26-100G>A',
                existing_variation: 'rs9988021',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 0.192,
                cadd_raw: -0.400011,
                spliceai_pred_dp_ag: 2,
                spliceai_pred_dp_al: -14,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: 19,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.7709,
                constraint_oe_lof: 1.0444,
                constraint_oe_lof_lower: 0.745,
                constraint_oe_lof_upper: 1.49,
                constraint_oe_mis: 1.6125,
                constraint_oe_mis_lower: 1.496,
                constraint_oe_mis_upper: 1.738,
                constraint_oe_syn: 1.8448,
                constraint_oe_syn_lower: 1.655,
                constraint_oe_syn_upper: 1.973,
                constraint_pli: 2.1045e-15,
                constraint_syn_z: -7.6532,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 2.1045e-15,
                loeuf: 1.49,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000342066',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '3/13',
                hgvsc: 'ENST00000342066.8:c.255-100G>A',
                existing_variation: 'rs9988021',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                ccds: 'CCDS2.2',
                source: 'Ensembl',
                cadd_phred: 0.192,
                cadd_raw: -0.400011,
                spliceai_pred_dp_ag: 2,
                spliceai_pred_dp_al: -14,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: 19,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.4361,
                constraint_oe_lof: 0.89656,
                constraint_oe_lof_lower: 0.64,
                constraint_oe_lof_upper: 1.28,
                constraint_oe_mis: 1.5082,
                constraint_oe_mis_lower: 1.405,
                constraint_oe_mis_upper: 1.619,
                constraint_oe_syn: 1.6983,
                constraint_oe_syn_lower: 1.537,
                constraint_oe_syn_upper: 1.875,
                constraint_pli: 4.7681e-14,
                constraint_syn_z: -6.9212,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 4.7681e-14,
                loeuf: 1.28,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000437963',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '3/4',
                hgvsc: 'ENST00000437963.5:c.255-100G>A',
                existing_variation: 'rs9988021',
                strand: 1,
                flags: 'cds_end_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 0.192,
                cadd_raw: -0.400011,
                spliceai_pred_dp_ag: 2,
                spliceai_pred_dp_al: -14,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: 19,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -0.44824,
                constraint_oe_lof: 0.54514,
                constraint_oe_lof_lower: 0.247,
                constraint_oe_lof_upper: 1.385,
                constraint_oe_mis: 1.1465,
                constraint_oe_mis_lower: 0.961,
                constraint_oe_mis_upper: 1.373,
                constraint_oe_syn: 1.1572,
                constraint_oe_syn_lower: 0.883,
                constraint_oe_syn_upper: 1.534,
                constraint_pli: 0.024691,
                constraint_syn_z: -0.67958,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 0.024691,
                loeuf: 1.385,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616125',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '2/10',
                hgvsc: 'ENST00000616125.5:c.255-100G>A',
                existing_variation: 'rs9988021',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 0.192,
                cadd_raw: -0.400011,
                spliceai_pred_dp_ag: 2,
                spliceai_pred_dp_al: -14,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: 19,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000617307',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '2/12',
                hgvsc: 'ENST00000617307.5:c.255-100G>A',
                existing_variation: 'rs9988021',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 0.192,
                cadd_raw: -0.400011,
                spliceai_pred_dp_ag: 2,
                spliceai_pred_dp_al: -14,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: 19,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618181',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '2/9',
                hgvsc: 'ENST00000618181.5:c.254+603G>A',
                existing_variation: 'rs9988021',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 0.192,
                cadd_raw: -0.400011,
                spliceai_pred_dp_ag: 2,
                spliceai_pred_dp_al: -14,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: 19,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618323',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '3/13',
                hgvsc: 'ENST00000618323.5:c.792-100G>A',
                existing_variation: 'rs9988021',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 0.192,
                cadd_raw: -0.400011,
                spliceai_pred_dp_ag: 2,
                spliceai_pred_dp_al: -14,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: 19,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618779',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '2/11',
                hgvsc: 'ENST00000618779.5:c.255-100G>A',
                existing_variation: 'rs9988021',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 0.192,
                cadd_raw: -0.400011,
                spliceai_pred_dp_ag: 2,
                spliceai_pred_dp_al: -14,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: 19,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000622503',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '2/12',
                hgvsc: 'ENST00000622503.5:c.255-100G>A',
                existing_variation: 'rs9988021',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 0.192,
                cadd_raw: -0.400011,
                spliceai_pred_dp_ag: 2,
                spliceai_pred_dp_al: -14,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: 19,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000616016',
            ensembl_gene_id: ['ENSG00000187634'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant',
            hgvsc: 'ENST00000616016.5:c.792-100G>A',
            existing_variation: 'rs9988021',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            cadd_phred: 0.192,
            cadd_raw: -0.400011,
            spliceai_pred_dp_ag: 2,
            spliceai_pred_dp_al: -14,
            spliceai_pred_dp_dg: 2,
            spliceai_pred_dp_dl: 19,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'SAMD11',
            exomiser_ad_exgenespheno: 0.2042,
            exomiser_xr_exgenescombi: 0.000014154,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.8656,
            gnomadv3_af_nfe: 0.9988,
            gnomadv3_af_xy: 0.9619,
            exomiser_xd_exgenescombi: 0.000014154,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9747,
            gnomadv3_af: 0.9603,
            exomiser_mt_exgenespheno: 0.2042,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.000014154,
            gnomadv3_af_eas: 0.9994,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 1,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.000014154,
            exomiser_ad_exgenescombi: 0.000014154,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 1,
            gnomadv3_af_fin: 0.999,
            exomiser_ar_exgenespheno: 0.2042,
            gnomadv3_af_sas: 0.9969,
            gnomadv3_af_amr: 0.9819,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.9589,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.9651,
            exomiser_xr_exgenespheno: 0.2042,
            exomiser_xd_exgenespheno: 0.2042,
            impact_severity: 'LOW',
            quality: 5511.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 57],
              [0, 47],
              [0, 58],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['SAMD11'],
            ensembl_transcript_id: [
              'ENST00000616016',
              'ENST00000341065',
              'ENST00000342066',
              'ENST00000437963',
              'ENST00000616125',
              'ENST00000617307',
              'ENST00000618181',
              'ENST00000618323',
              'ENST00000618779',
              'ENST00000622503',
            ],
            variant_type: 'small',
            highest_af: 0.9994,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 866319,
            hg19_strand: '+',
            hg19_end: 866319,
            entrez_gene_id: ['148398'],
          },
          {
            is_read: false,
            _id: '64e59ef69c2aec9c8bd9f98c',
            variant_id: '9',
            end: 931131,
            ref: 'C',
            alt: 'CCCCT',
            start: 931131,
            chrom: 'chr1',
            type: 'indel',
            gene_objs: [
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616016',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'CCCT',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '4/13',
                hgvsc: 'ENST00000616016.5:c.842+54_842+57dup',
                existing_variation: 'rs60722469',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                mane_select: 'NM_001385641.1',
                source: 'Ensembl',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000341065',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'CCCT',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '2/11',
                hgvsc: 'ENST00000341065.8:c.76+54_76+57dup',
                existing_variation: 'rs60722469',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                constraint_mis_z: -3.7709,
                constraint_oe_lof: 1.0444,
                constraint_oe_lof_lower: 0.745,
                constraint_oe_lof_upper: 1.49,
                constraint_oe_mis: 1.6125,
                constraint_oe_mis_lower: 1.496,
                constraint_oe_mis_upper: 1.738,
                constraint_oe_syn: 1.8448,
                constraint_oe_syn_lower: 1.655,
                constraint_oe_syn_upper: 1.973,
                constraint_pli: 2.1045e-15,
                constraint_syn_z: -7.6532,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 2.1045e-15,
                loeuf: 1.49,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000342066',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'CCCT',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '4/13',
                hgvsc: 'ENST00000342066.8:c.305+54_305+57dup',
                existing_variation: 'rs60722469',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                ccds: 'CCDS2.2',
                source: 'Ensembl',
                constraint_mis_z: -3.4361,
                constraint_oe_lof: 0.89656,
                constraint_oe_lof_lower: 0.64,
                constraint_oe_lof_upper: 1.28,
                constraint_oe_mis: 1.5082,
                constraint_oe_mis_lower: 1.405,
                constraint_oe_mis_upper: 1.619,
                constraint_oe_syn: 1.6983,
                constraint_oe_syn_lower: 1.537,
                constraint_oe_syn_upper: 1.875,
                constraint_pli: 4.7681e-14,
                constraint_syn_z: -6.9212,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 4.7681e-14,
                loeuf: 1.28,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000437963',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'CCCT',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '4/4',
                hgvsc: 'ENST00000437963.5:c.305+54_305+57dup',
                existing_variation: 'rs60722469',
                strand: 1,
                flags: 'cds_end_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                constraint_mis_z: -0.44824,
                constraint_oe_lof: 0.54514,
                constraint_oe_lof_lower: 0.247,
                constraint_oe_lof_upper: 1.385,
                constraint_oe_mis: 1.1465,
                constraint_oe_mis_lower: 0.961,
                constraint_oe_mis_upper: 1.373,
                constraint_oe_syn: 1.1572,
                constraint_oe_syn_lower: 0.883,
                constraint_oe_syn_upper: 1.534,
                constraint_pli: 0.024691,
                constraint_syn_z: -0.67958,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 0.024691,
                loeuf: 1.385,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616125',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'CCCT',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '3/10',
                hgvsc: 'ENST00000616125.5:c.305+54_305+57dup',
                existing_variation: 'rs60722469',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000617307',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'CCCT',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '3/12',
                hgvsc: 'ENST00000617307.5:c.305+54_305+57dup',
                existing_variation: 'rs60722469',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618181',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'CCCT',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '2/9',
                hgvsc: 'ENST00000618181.5:c.254+807_254+810dup',
                existing_variation: 'rs60722469',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618323',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'CCCT',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '4/13',
                hgvsc: 'ENST00000618323.5:c.842+54_842+57dup',
                existing_variation: 'rs60722469',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618779',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'CCCT',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '3/11',
                hgvsc: 'ENST00000618779.5:c.305+54_305+57dup',
                existing_variation: 'rs60722469',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000622503',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'CCCT',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '3/12',
                hgvsc: 'ENST00000622503.5:c.305+54_305+57dup',
                existing_variation: 'rs60722469',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000616016',
            ensembl_gene_id: ['ENSG00000187634'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant',
            hgvsc: 'ENST00000616016.5:c.842+54_842+57dup',
            existing_variation: 'rs60722469',
            source: 'Ensembl',
            gnomad_af: 0.6075,
            gnomad_afr_af: 0.2536,
            gnomad_amr_af: 0.5401,
            gnomad_asj_af: 0.6369,
            gnomad_eas_af: 0.4539,
            gnomad_fin_af: 0.7017,
            gnomad_nfe_af: 0.7002,
            gnomad_oth_af: 0.6342,
            gnomad_sas_af: 0.5774,
            exomiser_ad_exgenespheno: 0.2042,
            exomiser_xr_exgenescombi: 0.000014154,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.2611,
            gnomadv3_af_nfe: 0.6223,
            gnomadv3_af_xy: 0.5011,
            exomiser_xd_exgenescombi: 0.000014154,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.6821,
            gnomadv3_af: 0.5009,
            exomiser_mt_exgenespheno: 0.2042,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.000014154,
            gnomadv3_af_eas: 0.4619,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.6004,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.000014154,
            exomiser_ad_exgenescombi: 0.000014154,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.5,
            gnomadv3_af_fin: 0.6271,
            exomiser_ar_exgenespheno: 0.2042,
            gnomadv3_af_sas: 0.5616,
            gnomadv3_af_amr: 0.5169,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.5008,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.5197,
            exomiser_xr_exgenespheno: 0.2042,
            exomiser_xd_exgenespheno: 0.2042,
            impact_severity: 'LOW',
            quality: 1402,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [10, 14],
              [14, 7],
              [11, 14],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['SAMD11'],
            ensembl_transcript_id: [
              'ENST00000616016',
              'ENST00000341065',
              'ENST00000342066',
              'ENST00000437963',
              'ENST00000616125',
              'ENST00000617307',
              'ENST00000618181',
              'ENST00000618323',
              'ENST00000618779',
              'ENST00000622503',
            ],
            variant_type: 'small',
            highest_af: 0.6075,
            highest_af_info: {
              source: 'gnomad V2 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 866511,
            hg19_strand: '+',
            hg19_end: 866511,
            entrez_gene_id: ['148398'],
          },
          {
            allelic_depths: [],
            genotype_qualities: [],
            is_read: false,
            _id: '64e5a3c599107f1eb02eac4b',
            chrom: 'chr1',
            start: 934064,
            end: 934904,
            len: 840,
            ref: 'A',
            alt: '<DEL>',
            p_lof: 'False',
            constraint_pli: '',
            constraint_oe_lof_upper: [],
            mane_select: 'ENST00000616016',
            variant_type: 'structural',
            type: 'DEL',
            impact: '',
            gene_objs: [
              {
                is_related: true,
                transcript: 'ENST00000616016',
                gene: 'SAMD11',
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                ensembl_gene_id: 'ENSG00000187634',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_canonical: true,
                is_mane_select: true,
                annotation_overlap: 0.04067,
                reciprocal_overlap: 0.04067,
                bkpt_start: 923923,
                bkpt_end: 944574,
                default_show: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                is_related: true,
                transcript: 'ENST00000618323',
                gene: 'SAMD11',
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                ensembl_gene_id: 'ENSG00000187634',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                annotation_overlap: 0.04067,
                reciprocal_overlap: 0.04067,
                bkpt_start: 923923,
                bkpt_end: 944574,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                is_related: true,
                transcript: 'ENST00000437963',
                gene: 'SAMD11',
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                ensembl_gene_id: 'ENSG00000187634',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 0.024691,
                loeuf: 1.385,
                annotation_overlap: 0.07892,
                reciprocal_overlap: 0.07892,
                bkpt_start: 925150,
                bkpt_end: 935793,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                is_related: true,
                transcript: 'ENST00000342066',
                gene: 'SAMD11',
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                ensembl_gene_id: 'ENSG00000187634',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 4.7681e-14,
                loeuf: 1.28,
                annotation_overlap: 0.04458,
                reciprocal_overlap: 0.04458,
                bkpt_start: 925731,
                bkpt_end: 944574,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                is_related: true,
                transcript: 'ENST00000616125',
                gene: 'SAMD11',
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                ensembl_gene_id: 'ENSG00000187634',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                annotation_overlap: 0.04612,
                reciprocal_overlap: 0.04612,
                bkpt_start: 925942,
                bkpt_end: 944153,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                is_related: true,
                transcript: 'ENST00000617307',
                gene: 'SAMD11',
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                ensembl_gene_id: 'ENSG00000187634',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                annotation_overlap: 0.04612,
                reciprocal_overlap: 0.04612,
                bkpt_start: 925942,
                bkpt_end: 944153,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                is_related: true,
                transcript: 'ENST00000618181',
                gene: 'SAMD11',
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                ensembl_gene_id: 'ENSG00000187634',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                annotation_overlap: 0.04612,
                reciprocal_overlap: 0.04612,
                bkpt_start: 925942,
                bkpt_end: 944153,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                is_related: true,
                transcript: 'ENST00000618779',
                gene: 'SAMD11',
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                ensembl_gene_id: 'ENSG00000187634',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                annotation_overlap: 0.04612,
                reciprocal_overlap: 0.04612,
                bkpt_start: 925942,
                bkpt_end: 944153,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                is_related: true,
                transcript: 'ENST00000622503',
                gene: 'SAMD11',
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                ensembl_gene_id: 'ENSG00000187634',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                annotation_overlap: 0.04612,
                reciprocal_overlap: 0.04612,
                bkpt_start: 925942,
                bkpt_end: 944153,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                is_related: true,
                transcript: 'ENST00000341065',
                gene: 'SAMD11',
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                ensembl_gene_id: 'ENSG00000187634',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 2.1045e-15,
                loeuf: 1.49,
                annotation_overlap: 0.05889,
                reciprocal_overlap: 0.05889,
                bkpt_start: 930312,
                bkpt_end: 944575,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            highest_af: 0.863,
            highest_af_info: {
              source: '1KGP - Survtyper - East Asian',
            },
            gnomadv3_af: -1,
            gnomadv3_af_afr: -1,
            gnomadv3_af_amr: -1,
            gnomadv3_af_eas: -1,
            gnomadv3_af_nfe: -1,
            gnomadv3_af_sas: -1,
            gnomadv3_af_xx: -1,
            gnomadv3_af_xy: -1,
            gnomadv3_af_ami: -1,
            gnomadv3_af_asj: -1,
            gnomadv3_af_fin: -1,
            gnomadv3_af_mid: -1,
            gnomadv3_af_oth: -1,
            gnomad_af: -1,
            gnomad_eas_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_nfe_af: -1,
            gnomad_sas_af: -1,
            gnomad_asj_af: -1,
            gnomad_fin_af: -1,
            gnomad_oth_af: -1,
            is_pathogenic: false,
            variant_id: 'S:manta:1',
            sv_id: 'S:manta:1',
            afs: [
              {
                source_start: 934061,
                source_end: 934904,
                source_svType: 'DEL',
                svLength: -840,
                reciOverlap: 0.995,
                annoOverlap: 0.995,
                source: '1KGP - Survtyper - East Asian',
                source_filter: 'one_kg_sur_eas',
                AF: 0.863,
                SAMPLE_COUNT: 382,
              },
              {
                source_start: 934061,
                source_end: 934904,
                source_svType: 'DEL',
                svLength: -840,
                reciOverlap: 0.995,
                annoOverlap: 0.995,
                source: '1KGP - Survtyper - Global',
                source_filter: 'one_kg_sur',
                AF: 0.615,
                SAMPLE_COUNT: 2080,
              },
            ],
            exts: [],
            intpns: [],
            caller: 'manta',
            genotypes_index: [
              [0, 0],
              [0, 1],
              [1, 1],
            ],
            hg19_chrom: 'chr1',
            hg19_start: 869444,
            hg19_strand: '+',
            hg19_end: 870284,
            gene_symbol: ['SAMD11'],
            ensembl_gene_id: ['ENSG00000187634'],
            entrez_gene_id: ['148398'],
            ensembl_transcript_id: [
              'ENST00000616016',
              'ENST00000618323',
              'ENST00000437963',
              'ENST00000342066',
              'ENST00000616125',
              'ENST00000617307',
              'ENST00000618181',
              'ENST00000618779',
              'ENST00000622503',
              'ENST00000341065',
            ],
          },
          {
            allelic_depths: [],
            genotype_qualities: [],
            is_read: false,
            _id: '64e5a4cff99efe31718b84a3',
            chrom: 'chr1',
            start: 934321,
            end: 934937,
            len: 616,
            ref: 'A',
            alt: '<DEL>',
            p_lof: 'False',
            constraint_pli: '',
            constraint_oe_lof_upper: [],
            mane_select: 'ENST00000616016',
            variant_type: 'structural',
            type: 'DEL',
            impact: '',
            gene_objs: [
              {
                is_related: true,
                transcript: 'ENST00000616016',
                gene: 'SAMD11',
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                ensembl_gene_id: 'ENSG00000187634',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_canonical: true,
                is_mane_select: true,
                annotation_overlap: 0.02983,
                reciprocal_overlap: 0.02983,
                bkpt_start: 923923,
                bkpt_end: 944574,
                default_show: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                is_related: true,
                transcript: 'ENST00000618323',
                gene: 'SAMD11',
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                ensembl_gene_id: 'ENSG00000187634',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                annotation_overlap: 0.02983,
                reciprocal_overlap: 0.02983,
                bkpt_start: 923923,
                bkpt_end: 944574,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                is_related: true,
                transcript: 'ENST00000437963',
                gene: 'SAMD11',
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                ensembl_gene_id: 'ENSG00000187634',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 0.024691,
                loeuf: 1.385,
                annotation_overlap: 0.05787,
                reciprocal_overlap: 0.05787,
                bkpt_start: 925150,
                bkpt_end: 935793,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                is_related: true,
                transcript: 'ENST00000342066',
                gene: 'SAMD11',
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                ensembl_gene_id: 'ENSG00000187634',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 4.7681e-14,
                loeuf: 1.28,
                annotation_overlap: 0.03269,
                reciprocal_overlap: 0.03269,
                bkpt_start: 925731,
                bkpt_end: 944574,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                is_related: true,
                transcript: 'ENST00000616125',
                gene: 'SAMD11',
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                ensembl_gene_id: 'ENSG00000187634',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                annotation_overlap: 0.03382,
                reciprocal_overlap: 0.03382,
                bkpt_start: 925942,
                bkpt_end: 944153,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                is_related: true,
                transcript: 'ENST00000617307',
                gene: 'SAMD11',
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                ensembl_gene_id: 'ENSG00000187634',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                annotation_overlap: 0.03382,
                reciprocal_overlap: 0.03382,
                bkpt_start: 925942,
                bkpt_end: 944153,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                is_related: true,
                transcript: 'ENST00000618181',
                gene: 'SAMD11',
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                ensembl_gene_id: 'ENSG00000187634',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                annotation_overlap: 0.03382,
                reciprocal_overlap: 0.03382,
                bkpt_start: 925942,
                bkpt_end: 944153,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                is_related: true,
                transcript: 'ENST00000618779',
                gene: 'SAMD11',
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                ensembl_gene_id: 'ENSG00000187634',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                annotation_overlap: 0.03382,
                reciprocal_overlap: 0.03382,
                bkpt_start: 925942,
                bkpt_end: 944153,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                is_related: true,
                transcript: 'ENST00000622503',
                gene: 'SAMD11',
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                ensembl_gene_id: 'ENSG00000187634',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                annotation_overlap: 0.03382,
                reciprocal_overlap: 0.03382,
                bkpt_start: 925942,
                bkpt_end: 944153,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                is_related: true,
                transcript: 'ENST00000341065',
                gene: 'SAMD11',
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                ensembl_gene_id: 'ENSG00000187634',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 2.1045e-15,
                loeuf: 1.49,
                annotation_overlap: 0.04319,
                reciprocal_overlap: 0.04319,
                bkpt_start: 930312,
                bkpt_end: 944575,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            highest_af: 0.84,
            highest_af_info: {
              source: '1KGP - Survtyper - East Asian',
            },
            gnomadv3_af: -1,
            gnomadv3_af_afr: -1,
            gnomadv3_af_amr: -1,
            gnomadv3_af_eas: -1,
            gnomadv3_af_nfe: -1,
            gnomadv3_af_sas: -1,
            gnomadv3_af_xx: -1,
            gnomadv3_af_xy: -1,
            gnomadv3_af_ami: -1,
            gnomadv3_af_asj: -1,
            gnomadv3_af_fin: -1,
            gnomadv3_af_mid: -1,
            gnomadv3_af_oth: -1,
            gnomad_af: -1,
            gnomad_eas_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_nfe_af: -1,
            gnomad_sas_af: -1,
            gnomad_asj_af: -1,
            gnomad_fin_af: -1,
            gnomad_oth_af: -1,
            is_pathogenic: false,
            variant_id: 'S:survindel:1',
            sv_id: 'S:survindel:1',
            afs: [
              {
                source_start: 934322,
                source_end: 934937,
                source_svType: 'DEL',
                svLength: -616,
                reciOverlap: 1,
                annoOverlap: 1,
                source: '1KGP - Survtyper - East Asian',
                source_filter: 'one_kg_sur_eas',
                AF: 0.84,
                SAMPLE_COUNT: 134,
              },
              {
                source_start: 934322,
                source_end: 934937,
                source_svType: 'DEL',
                svLength: -616,
                reciOverlap: 1,
                annoOverlap: 1,
                source: '1KGP - Survtyper - Global',
                source_filter: 'one_kg_sur',
                AF: 0.635,
                SAMPLE_COUNT: 943,
              },
              {
                source_start: 934350,
                source_end: 934937,
                source_svType: 'DEL',
                svLength: -588,
                reciOverlap: 0.955,
                annoOverlap: 1,
                source: '1KGP - Survtyper - East Asian',
                source_filter: 'one_kg_sur_eas',
                AF: 0.769,
                SAMPLE_COUNT: 52,
              },
              {
                source_start: 934350,
                source_end: 934937,
                source_svType: 'DEL',
                svLength: -588,
                reciOverlap: 0.955,
                annoOverlap: 1,
                source: '1KGP - Survtyper - Global',
                source_filter: 'one_kg_sur',
                AF: 0.629,
                SAMPLE_COUNT: 337,
              },
            ],
            exts: [],
            intpns: [],
            caller: 'survindel',
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            hg19_chrom: 'chr1',
            hg19_start: 869701,
            hg19_strand: '+',
            hg19_end: 870317,
            gene_symbol: ['SAMD11'],
            ensembl_gene_id: ['ENSG00000187634'],
            entrez_gene_id: ['148398'],
            ensembl_transcript_id: [
              'ENST00000616016',
              'ENST00000618323',
              'ENST00000437963',
              'ENST00000342066',
              'ENST00000616125',
              'ENST00000617307',
              'ENST00000618181',
              'ENST00000618779',
              'ENST00000622503',
              'ENST00000341065',
            ],
          },
          {
            is_read: false,
            _id: '64e59ef69c2aec9c8bd9f98d',
            variant_id: '10',
            end: 935954,
            ref: 'G',
            alt: 'T',
            start: 935954,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616016',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'T',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '5/13',
                hgvsc: 'ENST00000616016.5:c.967+58G>T',
                existing_variation: 'rs4072383&COSV59704945',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                mane_select: 'NM_001385641.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 8.634,
                cadd_raw: 0.719531,
                spliceai_pred_dp_ag: 14,
                spliceai_pred_dp_al: 23,
                spliceai_pred_dp_dg: 43,
                spliceai_pred_dp_dl: 47,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000341065',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '3/11',
                hgvsc: 'ENST00000341065.8:c.201+58G>T',
                existing_variation: 'rs4072383&COSV59704945',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 8.634,
                cadd_raw: 0.719531,
                spliceai_pred_dp_ag: 14,
                spliceai_pred_dp_al: 23,
                spliceai_pred_dp_dg: 43,
                spliceai_pred_dp_dl: 47,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.7709,
                constraint_oe_lof: 1.0444,
                constraint_oe_lof_lower: 0.745,
                constraint_oe_lof_upper: 1.49,
                constraint_oe_mis: 1.6125,
                constraint_oe_mis_lower: 1.496,
                constraint_oe_mis_upper: 1.738,
                constraint_oe_syn: 1.8448,
                constraint_oe_syn_lower: 1.655,
                constraint_oe_syn_upper: 1.973,
                constraint_pli: 2.1045e-15,
                constraint_syn_z: -7.6532,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 2.1045e-15,
                loeuf: 1.49,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000342066',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '5/13',
                hgvsc: 'ENST00000342066.8:c.430+58G>T',
                existing_variation: 'rs4072383&COSV59704945',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                ccds: 'CCDS2.2',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 8.634,
                cadd_raw: 0.719531,
                spliceai_pred_dp_ag: 14,
                spliceai_pred_dp_al: 23,
                spliceai_pred_dp_dg: 43,
                spliceai_pred_dp_dl: 47,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.4361,
                constraint_oe_lof: 0.89656,
                constraint_oe_lof_lower: 0.64,
                constraint_oe_lof_upper: 1.28,
                constraint_oe_mis: 1.5082,
                constraint_oe_mis_lower: 1.405,
                constraint_oe_mis_upper: 1.619,
                constraint_oe_syn: 1.6983,
                constraint_oe_syn_lower: 1.537,
                constraint_oe_syn_upper: 1.875,
                constraint_pli: 4.7681e-14,
                constraint_syn_z: -6.9212,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 4.7681e-14,
                loeuf: 1.28,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000437963',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4072383&COSV59704945',
                strand: 1,
                flags: 'cds_end_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 8.634,
                cadd_raw: 0.719531,
                spliceai_pred_dp_ag: 14,
                spliceai_pred_dp_al: 23,
                spliceai_pred_dp_dg: 43,
                spliceai_pred_dp_dl: 47,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -0.44824,
                constraint_oe_lof: 0.54514,
                constraint_oe_lof_lower: 0.247,
                constraint_oe_lof_upper: 1.385,
                constraint_oe_mis: 1.1465,
                constraint_oe_mis_lower: 0.961,
                constraint_oe_mis_upper: 1.373,
                constraint_oe_syn: 1.1572,
                constraint_oe_syn_lower: 0.883,
                constraint_oe_syn_upper: 1.534,
                constraint_pli: 0.024691,
                constraint_syn_z: -0.67958,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 0.024691,
                loeuf: 1.385,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000455979',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4072383&COSV59704945',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 8.634,
                cadd_raw: 0.719531,
                spliceai_pred_dp_ag: 14,
                spliceai_pred_dp_al: 23,
                spliceai_pred_dp_dg: 43,
                spliceai_pred_dp_dl: 47,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.7139,
                constraint_oe_lof: 1.0978,
                constraint_oe_lof_lower: 0.733,
                constraint_oe_lof_upper: 1.666,
                constraint_oe_mis: 1.6449,
                constraint_oe_mis_lower: 1.52,
                constraint_oe_mis_upper: 1.781,
                constraint_oe_syn: 1.8162,
                constraint_oe_syn_lower: 1.621,
                constraint_oe_syn_upper: 1.966,
                constraint_pli: 5.471e-11,
                constraint_syn_z: -7.0453,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 5.471e-11,
                loeuf: 1.666,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616125',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '4/10',
                hgvsc: 'ENST00000616125.5:c.430+58G>T',
                existing_variation: 'rs4072383&COSV59704945',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 8.634,
                cadd_raw: 0.719531,
                spliceai_pred_dp_ag: 14,
                spliceai_pred_dp_al: 23,
                spliceai_pred_dp_dg: 43,
                spliceai_pred_dp_dl: 47,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000617307',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '4/12',
                hgvsc: 'ENST00000617307.5:c.430+58G>T',
                existing_variation: 'rs4072383&COSV59704945',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 8.634,
                cadd_raw: 0.719531,
                spliceai_pred_dp_ag: 14,
                spliceai_pred_dp_al: 23,
                spliceai_pred_dp_dg: 43,
                spliceai_pred_dp_dl: 47,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618181',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '3/9',
                hgvsc: 'ENST00000618181.5:c.379+58G>T',
                existing_variation: 'rs4072383&COSV59704945',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 8.634,
                cadd_raw: 0.719531,
                spliceai_pred_dp_ag: 14,
                spliceai_pred_dp_al: 23,
                spliceai_pred_dp_dg: 43,
                spliceai_pred_dp_dl: 47,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618323',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '5/13',
                hgvsc: 'ENST00000618323.5:c.967+58G>T',
                existing_variation: 'rs4072383&COSV59704945',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 8.634,
                cadd_raw: 0.719531,
                spliceai_pred_dp_ag: 14,
                spliceai_pred_dp_al: 23,
                spliceai_pred_dp_dg: 43,
                spliceai_pred_dp_dl: 47,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618779',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '4/11',
                hgvsc: 'ENST00000618779.5:c.430+58G>T',
                existing_variation: 'rs4072383&COSV59704945',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 8.634,
                cadd_raw: 0.719531,
                spliceai_pred_dp_ag: 14,
                spliceai_pred_dp_al: 23,
                spliceai_pred_dp_dg: 43,
                spliceai_pred_dp_dl: 47,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000622503',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '4/12',
                hgvsc: 'ENST00000622503.5:c.430+58G>T',
                existing_variation: 'rs4072383&COSV59704945',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 8.634,
                cadd_raw: 0.719531,
                spliceai_pred_dp_ag: 14,
                spliceai_pred_dp_al: 23,
                spliceai_pred_dp_dg: 43,
                spliceai_pred_dp_dl: 47,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000616016',
            ensembl_gene_id: ['ENSG00000187634'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant',
            hgvsc: 'ENST00000616016.5:c.967+58G>T',
            existing_variation: 'rs4072383&COSV59704945',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            pheno: '0&1',
            cadd_phred: 8.634,
            cadd_raw: 0.719531,
            spliceai_pred_dp_ag: 14,
            spliceai_pred_dp_al: 23,
            spliceai_pred_dp_dg: 43,
            spliceai_pred_dp_dl: 47,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'SAMD11',
            exomiser_ad_exgenespheno: 0.2042,
            exomiser_xr_exgenescombi: 0.000014154,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.277,
            gnomadv3_af_nfe: 0.7091,
            gnomadv3_af_xy: 0.5587,
            exomiser_xd_exgenescombi: 0.000014154,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.7057,
            gnomadv3_af: 0.562,
            exomiser_mt_exgenespheno: 0.2042,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.000014154,
            gnomadv3_af_eas: 0.5106,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.6756,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.000014154,
            exomiser_ad_exgenescombi: 0.000014154,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.5526,
            gnomadv3_af_fin: 0.6446,
            exomiser_ar_exgenespheno: 0.2042,
            gnomadv3_af_sas: 0.6398,
            gnomadv3_af_amr: 0.5823,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.5651,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.5943,
            exomiser_xr_exgenespheno: 0.2042,
            exomiser_xd_exgenespheno: 0.2042,
            impact_severity: 'LOW',
            quality: 4146.9,
            genotypes_index: [
              [1, 1],
              [0, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 43],
              [27, 31],
              [0, 53],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['SAMD11'],
            ensembl_transcript_id: [
              'ENST00000616016',
              'ENST00000341065',
              'ENST00000342066',
              'ENST00000437963',
              'ENST00000455979',
              'ENST00000616125',
              'ENST00000617307',
              'ENST00000618181',
              'ENST00000618323',
              'ENST00000618779',
              'ENST00000622503',
            ],
            variant_type: 'small',
            highest_af: 0.562,
            highest_af_info: {
              source: 'gnomad v3 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 871334,
            hg19_strand: '+',
            hg19_end: 871334,
            entrez_gene_id: ['148398'],
          },
          {
            is_read: false,
            _id: '64e59ef69c2aec9c8bd9f98e',
            variant_id: '11',
            end: 941119,
            ref: 'A',
            alt: 'G',
            start: 941119,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4372192',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                cadd_phred: 8.425,
                cadd_raw: 0.696706,
                spliceai_pred_dp_ag: 45,
                spliceai_pred_dp_al: 25,
                spliceai_pred_dp_dg: -1,
                spliceai_pred_dp_dl: 35,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.05,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616016',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '7/13',
                hgvsc: 'ENST00000616016.5:c.1196-25A>G',
                existing_variation: 'rs4372192',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                mane_select: 'NM_001385641.1',
                source: 'Ensembl',
                cadd_phred: 8.425,
                cadd_raw: 0.696706,
                spliceai_pred_dp_ag: 45,
                spliceai_pred_dp_al: 25,
                spliceai_pred_dp_dg: -1,
                spliceai_pred_dp_dl: 35,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.05,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000341065',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '5/11',
                hgvsc: 'ENST00000341065.8:c.430-25A>G',
                existing_variation: 'rs4372192',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 8.425,
                cadd_raw: 0.696706,
                spliceai_pred_dp_ag: 45,
                spliceai_pred_dp_al: 25,
                spliceai_pred_dp_dg: -1,
                spliceai_pred_dp_dl: 35,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.05,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.7709,
                constraint_oe_lof: 1.0444,
                constraint_oe_lof_lower: 0.745,
                constraint_oe_lof_upper: 1.49,
                constraint_oe_mis: 1.6125,
                constraint_oe_mis_lower: 1.496,
                constraint_oe_mis_upper: 1.738,
                constraint_oe_syn: 1.8448,
                constraint_oe_syn_lower: 1.655,
                constraint_oe_syn_upper: 1.973,
                constraint_pli: 2.1045e-15,
                constraint_syn_z: -7.6532,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 2.1045e-15,
                loeuf: 1.49,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000342066',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '7/13',
                hgvsc: 'ENST00000342066.8:c.707-25A>G',
                existing_variation: 'rs4372192',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                ccds: 'CCDS2.2',
                source: 'Ensembl',
                cadd_phred: 8.425,
                cadd_raw: 0.696706,
                spliceai_pred_dp_ag: 45,
                spliceai_pred_dp_al: 25,
                spliceai_pred_dp_dg: -1,
                spliceai_pred_dp_dl: 35,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.05,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.4361,
                constraint_oe_lof: 0.89656,
                constraint_oe_lof_lower: 0.64,
                constraint_oe_lof_upper: 1.28,
                constraint_oe_mis: 1.5082,
                constraint_oe_mis_lower: 1.405,
                constraint_oe_mis_upper: 1.619,
                constraint_oe_syn: 1.6983,
                constraint_oe_syn_lower: 1.537,
                constraint_oe_syn_upper: 1.875,
                constraint_pli: 4.7681e-14,
                constraint_syn_z: -6.9212,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 4.7681e-14,
                loeuf: 1.28,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000455979',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '1/6',
                hgvsc: 'ENST00000455979.1:c.187-25A>G',
                existing_variation: 'rs4372192',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 8.425,
                cadd_raw: 0.696706,
                spliceai_pred_dp_ag: 45,
                spliceai_pred_dp_al: 25,
                spliceai_pred_dp_dg: -1,
                spliceai_pred_dp_dl: 35,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.05,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.7139,
                constraint_oe_lof: 1.0978,
                constraint_oe_lof_lower: 0.733,
                constraint_oe_lof_upper: 1.666,
                constraint_oe_mis: 1.6449,
                constraint_oe_mis_lower: 1.52,
                constraint_oe_mis_upper: 1.781,
                constraint_oe_syn: 1.8162,
                constraint_oe_syn_lower: 1.621,
                constraint_oe_syn_upper: 1.966,
                constraint_pli: 5.471e-11,
                constraint_syn_z: -7.0453,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 5.471e-11,
                loeuf: 1.666,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616125',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '6/10',
                hgvsc: 'ENST00000616125.5:c.662-1291A>G',
                existing_variation: 'rs4372192',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 8.425,
                cadd_raw: 0.696706,
                spliceai_pred_dp_ag: 45,
                spliceai_pred_dp_al: 25,
                spliceai_pred_dp_dg: -1,
                spliceai_pred_dp_dl: 35,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.05,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000617307',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '6/12',
                hgvsc: 'ENST00000617307.5:c.710-25A>G',
                existing_variation: 'rs4372192',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 8.425,
                cadd_raw: 0.696706,
                spliceai_pred_dp_ag: 45,
                spliceai_pred_dp_al: 25,
                spliceai_pred_dp_dg: -1,
                spliceai_pred_dp_dl: 35,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.05,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618181',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '5/9',
                hgvsc: 'ENST00000618181.5:c.611-1291A>G',
                existing_variation: 'rs4372192',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 8.425,
                cadd_raw: 0.696706,
                spliceai_pred_dp_ag: 45,
                spliceai_pred_dp_al: 25,
                spliceai_pred_dp_dg: -1,
                spliceai_pred_dp_dl: 35,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.05,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618323',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '7/13',
                hgvsc: 'ENST00000618323.5:c.1199-25A>G',
                existing_variation: 'rs4372192',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 8.425,
                cadd_raw: 0.696706,
                spliceai_pred_dp_ag: 45,
                spliceai_pred_dp_al: 25,
                spliceai_pred_dp_dg: -1,
                spliceai_pred_dp_dl: 35,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.05,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618779',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '5/11',
                hgvsc: 'ENST00000618779.5:c.521-25A>G',
                existing_variation: 'rs4372192',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 8.425,
                cadd_raw: 0.696706,
                spliceai_pred_dp_ag: 45,
                spliceai_pred_dp_al: 25,
                spliceai_pred_dp_dg: -1,
                spliceai_pred_dp_dl: 35,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.05,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000622503',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '6/12',
                hgvsc: 'ENST00000622503.5:c.710-25A>G',
                existing_variation: 'rs4372192',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 8.425,
                cadd_raw: 0.696706,
                spliceai_pred_dp_ag: 45,
                spliceai_pred_dp_al: 25,
                spliceai_pred_dp_dg: -1,
                spliceai_pred_dp_dl: 35,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.05,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: ['ENSG00000188976', 'ENSG00000187634'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'downstream_gene_variant&intron_variant',
            existing_variation: 'rs4372192',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            gnomad_af: 0.9335,
            gnomad_afr_af: 0.8725,
            gnomad_amr_af: 0.9513,
            gnomad_asj_af: 0.9329,
            gnomad_eas_af: 0.9095,
            gnomad_fin_af: 0.9605,
            gnomad_nfe_af: 0.939,
            gnomad_oth_af: 0.9342,
            gnomad_sas_af: 0.9173,
            cadd_phred: 8.425,
            cadd_raw: 0.696706,
            spliceai_pred_dp_ag: 45,
            spliceai_pred_dp_al: 25,
            spliceai_pred_dp_dg: -1,
            spliceai_pred_dp_dl: 35,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0.05],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'SAMD11',
            constraint_mis_z: [-1.901],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147],
            exomiser_ad_exgenespheno: 0.2042,
            exomiser_xr_exgenescombi: 0.000014154,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.8847,
            gnomadv3_af_nfe: 0.9461,
            gnomadv3_af_xy: 0.9271,
            exomiser_xd_exgenescombi: 0.000014154,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9304,
            gnomadv3_af: 0.927,
            exomiser_mt_exgenespheno: 0.2042,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.000014154,
            gnomadv3_af_eas: 0.9248,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9277,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.000014154,
            exomiser_ad_exgenescombi: 0.000014154,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9638,
            gnomadv3_af_fin: 0.9666,
            exomiser_ar_exgenespheno: 0.2042,
            gnomadv3_af_sas: 0.9204,
            gnomadv3_af_amr: 0.9299,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.9269,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.925,
            exomiser_xr_exgenespheno: 0.2042,
            exomiser_xd_exgenespheno: 0.2042,
            impact_severity: 'LOW',
            quality: 1584.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [22, 15],
              [22, 19],
              [14, 26],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L', 'SAMD11'],
            ensembl_transcript_id: [
              'ENST00000327044',
              'ENST00000616016',
              'ENST00000341065',
              'ENST00000342066',
              'ENST00000455979',
              'ENST00000616125',
              'ENST00000617307',
              'ENST00000618181',
              'ENST00000618323',
              'ENST00000618779',
              'ENST00000622503',
            ],
            variant_type: 'small',
            highest_af: 0.9335,
            highest_af_info: {
              source: 'gnomad V2 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 876499,
            hg19_strand: '+',
            hg19_end: 876499,
            entrez_gene_id: ['26155', '148398'],
          },
          {
            is_read: false,
            _id: '64e59ef69c2aec9c8bd9f98f',
            variant_id: '12',
            end: 942335,
            ref: 'C',
            alt: 'G',
            start: 942335,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6605066&COSV58986459',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.482,
                cadd_raw: -0.21184,
                spliceai_pred_dp_ag: 7,
                spliceai_pred_dp_al: 2,
                spliceai_pred_dp_dg: -19,
                spliceai_pred_dp_dl: 2,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.07,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616016',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '9/13',
                hgvsc: 'ENST00000616016.5:c.1475-75C>G',
                existing_variation: 'rs6605066&COSV58986459',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                mane_select: 'NM_001385641.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.482,
                cadd_raw: -0.21184,
                spliceai_pred_dp_ag: 7,
                spliceai_pred_dp_al: 2,
                spliceai_pred_dp_dg: -19,
                spliceai_pred_dp_dl: 2,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.07,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000341065',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '7/11',
                hgvsc: 'ENST00000341065.8:c.709-75C>G',
                existing_variation: 'rs6605066&COSV58986459',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.482,
                cadd_raw: -0.21184,
                spliceai_pred_dp_ag: 7,
                spliceai_pred_dp_al: 2,
                spliceai_pred_dp_dg: -19,
                spliceai_pred_dp_dl: 2,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.07,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.7709,
                constraint_oe_lof: 1.0444,
                constraint_oe_lof_lower: 0.745,
                constraint_oe_lof_upper: 1.49,
                constraint_oe_mis: 1.6125,
                constraint_oe_mis_lower: 1.496,
                constraint_oe_mis_upper: 1.738,
                constraint_oe_syn: 1.8448,
                constraint_oe_syn_lower: 1.655,
                constraint_oe_syn_upper: 1.973,
                constraint_pli: 2.1045e-15,
                constraint_syn_z: -7.6532,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 2.1045e-15,
                loeuf: 1.49,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000342066',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '9/13',
                hgvsc: 'ENST00000342066.8:c.986-75C>G',
                existing_variation: 'rs6605066&COSV58986459',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                ccds: 'CCDS2.2',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.482,
                cadd_raw: -0.21184,
                spliceai_pred_dp_ag: 7,
                spliceai_pred_dp_al: 2,
                spliceai_pred_dp_dg: -19,
                spliceai_pred_dp_dl: 2,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.07,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.4361,
                constraint_oe_lof: 0.89656,
                constraint_oe_lof_lower: 0.64,
                constraint_oe_lof_upper: 1.28,
                constraint_oe_mis: 1.5082,
                constraint_oe_mis_lower: 1.405,
                constraint_oe_mis_upper: 1.619,
                constraint_oe_syn: 1.6983,
                constraint_oe_syn_lower: 1.537,
                constraint_oe_syn_upper: 1.875,
                constraint_pli: 4.7681e-14,
                constraint_syn_z: -6.9212,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 4.7681e-14,
                loeuf: 1.28,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000455979',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '3/6',
                hgvsc: 'ENST00000455979.1:c.466-75C>G',
                existing_variation: 'rs6605066&COSV58986459',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.482,
                cadd_raw: -0.21184,
                spliceai_pred_dp_ag: 7,
                spliceai_pred_dp_al: 2,
                spliceai_pred_dp_dg: -19,
                spliceai_pred_dp_dl: 2,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.07,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.7139,
                constraint_oe_lof: 1.0978,
                constraint_oe_lof_lower: 0.733,
                constraint_oe_lof_upper: 1.666,
                constraint_oe_mis: 1.6449,
                constraint_oe_mis_lower: 1.52,
                constraint_oe_mis_upper: 1.781,
                constraint_oe_syn: 1.8162,
                constraint_oe_syn_lower: 1.621,
                constraint_oe_syn_upper: 1.966,
                constraint_pli: 5.471e-11,
                constraint_syn_z: -7.0453,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 5.471e-11,
                loeuf: 1.666,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616125',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '6/10',
                hgvsc: 'ENST00000616125.5:c.662-75C>G',
                existing_variation: 'rs6605066&COSV58986459',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.482,
                cadd_raw: -0.21184,
                spliceai_pred_dp_ag: 7,
                spliceai_pred_dp_al: 2,
                spliceai_pred_dp_dg: -19,
                spliceai_pred_dp_dl: 2,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.07,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000617307',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '8/12',
                hgvsc: 'ENST00000617307.5:c.989-75C>G',
                existing_variation: 'rs6605066&COSV58986459',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.482,
                cadd_raw: -0.21184,
                spliceai_pred_dp_ag: 7,
                spliceai_pred_dp_al: 2,
                spliceai_pred_dp_dg: -19,
                spliceai_pred_dp_dl: 2,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.07,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618181',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '5/9',
                hgvsc: 'ENST00000618181.5:c.611-75C>G',
                existing_variation: 'rs6605066&COSV58986459',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.482,
                cadd_raw: -0.21184,
                spliceai_pred_dp_ag: 7,
                spliceai_pred_dp_al: 2,
                spliceai_pred_dp_dg: -19,
                spliceai_pred_dp_dl: 2,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.07,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618323',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '9/13',
                hgvsc: 'ENST00000618323.5:c.1478-75C>G',
                existing_variation: 'rs6605066&COSV58986459',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.482,
                cadd_raw: -0.21184,
                spliceai_pred_dp_ag: 7,
                spliceai_pred_dp_al: 2,
                spliceai_pred_dp_dg: -19,
                spliceai_pred_dp_dl: 2,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.07,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618779',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '7/11',
                hgvsc: 'ENST00000618779.5:c.800-75C>G',
                existing_variation: 'rs6605066&COSV58986459',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.482,
                cadd_raw: -0.21184,
                spliceai_pred_dp_ag: 7,
                spliceai_pred_dp_al: 2,
                spliceai_pred_dp_dg: -19,
                spliceai_pred_dp_dl: 2,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.07,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000622503',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '8/12',
                hgvsc: 'ENST00000622503.5:c.989-75C>G',
                existing_variation: 'rs6605066&COSV58986459',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.482,
                cadd_raw: -0.21184,
                spliceai_pred_dp_ag: 7,
                spliceai_pred_dp_al: 2,
                spliceai_pred_dp_dg: -19,
                spliceai_pred_dp_dl: 2,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.07,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: ['ENSG00000188976', 'ENSG00000187634'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'downstream_gene_variant&intron_variant',
            existing_variation: 'rs6605066&COSV58986459',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            pheno: '0&1',
            cadd_phred: 0.482,
            cadd_raw: -0.21184,
            spliceai_pred_dp_ag: 7,
            spliceai_pred_dp_al: 2,
            spliceai_pred_dp_dg: -19,
            spliceai_pred_dp_dl: 2,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0.07],
            spliceai_pred_symbol: 'SAMD11',
            constraint_mis_z: [-1.901],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147],
            exomiser_ad_exgenespheno: 0.2042,
            exomiser_xr_exgenescombi: 0.000014154,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.7967,
            gnomadv3_af_nfe: 0.933,
            gnomadv3_af_xy: 0.8925,
            exomiser_xd_exgenescombi: 0.000014154,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9114,
            gnomadv3_af: 0.8921,
            exomiser_mt_exgenespheno: 0.2042,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.000014154,
            gnomadv3_af_eas: 0.9261,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9116,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.000014154,
            exomiser_ad_exgenescombi: 0.000014154,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.8813,
            gnomadv3_af_fin: 0.9422,
            exomiser_ar_exgenespheno: 0.2042,
            gnomadv3_af_sas: 0.9047,
            gnomadv3_af_amr: 0.913,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.8918,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.902,
            exomiser_xr_exgenespheno: 0.2042,
            exomiser_xd_exgenespheno: 0.2042,
            impact_severity: 'LOW',
            quality: 1248.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [29, 23],
              [15, 15],
              [28, 17],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L', 'SAMD11'],
            ensembl_transcript_id: [
              'ENST00000327044',
              'ENST00000616016',
              'ENST00000341065',
              'ENST00000342066',
              'ENST00000455979',
              'ENST00000616125',
              'ENST00000617307',
              'ENST00000618181',
              'ENST00000618323',
              'ENST00000618779',
              'ENST00000622503',
            ],
            variant_type: 'small',
            highest_af: 0.9261,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 877715,
            hg19_strand: '+',
            hg19_end: 877715,
            entrez_gene_id: ['26155', '148398'],
          },
          {
            is_read: false,
            _id: '64e59ef69c2aec9c8bd9f990',
            variant_id: '13',
            end: 942451,
            ref: 'T',
            alt: 'C',
            start: 942451,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616016',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '10/14',
                codon_change: 'Tgg/Cgg',
                aa_change: 'W/R',
                aa_length: 506,
                polyphen_pred: 'benign',
                polyphen_score: 0,
                sift_pred: 'tolerated',
                sift_score: 1,
                allele: 'C',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000616016.5:c.1516T>C',
                hgvsp: 'ENSP00000478421.2:p.Trp506Arg',
                cds_position: 1516,
                existing_variation: 'rs6672356&COSV58985794',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                mane_select: 'NM_001385641.1',
                source: 'Ensembl',
                domains: 'PANTHER:PTHR12247&PANTHER:PTHR12247:SF67',
                pheno: '0&1',
                revel: 0.145,
                cadd_phred: 19.24,
                cadd_raw: 1.997533,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: 9,
                spliceai_pred_dp_dg: 41,
                spliceai_pred_dp_dl: 9,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6672356&COSV58985794',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 19.24,
                cadd_raw: 1.997533,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: 9,
                spliceai_pred_dp_dg: 41,
                spliceai_pred_dp_dl: 9,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000341065',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '8/12',
                codon_change: 'Tgg/Cgg',
                aa_change: 'W/R',
                aa_length: 251,
                polyphen_pred: 'benign',
                polyphen_score: 0,
                sift_pred: 'tolerated',
                sift_score: 1,
                allele: 'C',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000341065.8:c.751T>C',
                hgvsp: 'ENSP00000349216.4:p.Trp251Arg',
                cds_position: 751,
                existing_variation: 'rs6672356&COSV58985794',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                domains: 'PANTHER:PTHR12247&PANTHER:PTHR12247:SF67',
                pheno: '0&1',
                revel: 0.145,
                cadd_phred: 19.24,
                cadd_raw: 1.997533,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: 9,
                spliceai_pred_dp_dg: 41,
                spliceai_pred_dp_dl: 9,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.7709,
                constraint_oe_lof: 1.0444,
                constraint_oe_lof_lower: 0.745,
                constraint_oe_lof_upper: 1.49,
                constraint_oe_mis: 1.6125,
                constraint_oe_mis_lower: 1.496,
                constraint_oe_mis_upper: 1.738,
                constraint_oe_syn: 1.8448,
                constraint_oe_syn_lower: 1.655,
                constraint_oe_syn_upper: 1.973,
                constraint_pli: 2.1045e-15,
                constraint_syn_z: -7.6532,
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 2.1045e-15,
                loeuf: 1.49,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000342066',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '10/14',
                codon_change: 'Tgg/Cgg',
                aa_change: 'W/R',
                aa_length: 343,
                polyphen_pred: 'benign',
                polyphen_score: 0,
                sift_pred: 'tolerated',
                sift_score: 1,
                allele: 'C',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000342066.8:c.1027T>C',
                hgvsp: 'ENSP00000342313.3:p.Trp343Arg',
                cds_position: 1027,
                existing_variation: 'rs6672356&COSV58985794',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                ccds: 'CCDS2.2',
                source: 'Ensembl',
                domains: 'PANTHER:PTHR12247&PANTHER:PTHR12247:SF67',
                pheno: '0&1',
                revel: 0.145,
                cadd_phred: 19.24,
                cadd_raw: 1.997533,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: 9,
                spliceai_pred_dp_dg: 41,
                spliceai_pred_dp_dl: 9,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.4361,
                constraint_oe_lof: 0.89656,
                constraint_oe_lof_lower: 0.64,
                constraint_oe_lof_upper: 1.28,
                constraint_oe_mis: 1.5082,
                constraint_oe_mis_lower: 1.405,
                constraint_oe_mis_upper: 1.619,
                constraint_oe_syn: 1.6983,
                constraint_oe_syn_lower: 1.537,
                constraint_oe_syn_upper: 1.875,
                constraint_pli: 4.7681e-14,
                constraint_syn_z: -6.9212,
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 4.7681e-14,
                loeuf: 1.28,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000455979',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '4/7',
                codon_change: 'Tgg/Cgg',
                aa_change: 'W/R',
                aa_length: 170,
                polyphen_pred: 'benign',
                polyphen_score: 0,
                sift_pred: 'tolerated',
                sift_score: 1,
                allele: 'C',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000455979.1:c.508T>C',
                hgvsp: 'ENSP00000412228.1:p.Trp170Arg',
                cds_position: 508,
                existing_variation: 'rs6672356&COSV58985794',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                domains: 'PANTHER:PTHR12247&PANTHER:PTHR12247:SF67',
                pheno: '0&1',
                revel: 0.145,
                cadd_phred: 19.24,
                cadd_raw: 1.997533,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: 9,
                spliceai_pred_dp_dg: 41,
                spliceai_pred_dp_dl: 9,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.7139,
                constraint_oe_lof: 1.0978,
                constraint_oe_lof_lower: 0.733,
                constraint_oe_lof_upper: 1.666,
                constraint_oe_mis: 1.6449,
                constraint_oe_mis_lower: 1.52,
                constraint_oe_mis_upper: 1.781,
                constraint_oe_syn: 1.8162,
                constraint_oe_syn_lower: 1.621,
                constraint_oe_syn_upper: 1.966,
                constraint_pli: 5.471e-11,
                constraint_syn_z: -7.0453,
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 5.471e-11,
                loeuf: 1.666,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616125',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '7/11',
                codon_change: 'Tgg/Cgg',
                aa_change: 'W/R',
                aa_length: 235,
                polyphen_pred: 'benign',
                polyphen_score: 0,
                sift_pred: 'tolerated',
                sift_score: 1,
                allele: 'C',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000616125.5:c.703T>C',
                hgvsp: 'ENSP00000484643.1:p.Trp235Arg',
                cds_position: 703,
                existing_variation: 'rs6672356&COSV58985794',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                domains: 'PANTHER:PTHR12247&PANTHER:PTHR12247:SF67',
                pheno: '0&1',
                revel: 0.145,
                cadd_phred: 19.24,
                cadd_raw: 1.997533,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: 9,
                spliceai_pred_dp_dg: 41,
                spliceai_pred_dp_dl: 9,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000617307',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '9/13',
                codon_change: 'Tgg/Cgg',
                aa_change: 'W/R',
                aa_length: 344,
                polyphen_pred: 'benign',
                polyphen_score: 0,
                sift_pred: 'tolerated',
                sift_score: 1,
                allele: 'C',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000617307.5:c.1030T>C',
                hgvsp: 'ENSP00000482090.2:p.Trp344Arg',
                cds_position: 1030,
                existing_variation: 'rs6672356&COSV58985794',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                domains: 'PANTHER:PTHR12247&PANTHER:PTHR12247:SF67',
                pheno: '0&1',
                revel: 0.145,
                cadd_phred: 19.24,
                cadd_raw: 1.997533,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: 9,
                spliceai_pred_dp_dg: 41,
                spliceai_pred_dp_dl: 9,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618181',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '6/10',
                codon_change: 'Tgg/Cgg',
                aa_change: 'W/R',
                aa_length: 218,
                polyphen_pred: 'benign',
                polyphen_score: 0,
                sift_pred: 'tolerated',
                sift_score: 1,
                allele: 'C',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000618181.5:c.652T>C',
                hgvsp: 'ENSP00000480870.1:p.Trp218Arg',
                cds_position: 652,
                existing_variation: 'rs6672356&COSV58985794',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                domains: 'PANTHER:PTHR12247&PANTHER:PTHR12247:SF67',
                pheno: '0&1',
                revel: 0.145,
                cadd_phred: 19.24,
                cadd_raw: 1.997533,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: 9,
                spliceai_pred_dp_dg: 41,
                spliceai_pred_dp_dl: 9,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618323',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '10/14',
                codon_change: 'Tgg/Cgg',
                aa_change: 'W/R',
                aa_length: 507,
                polyphen_pred: 'benign',
                polyphen_score: 0,
                sift_pred: 'tolerated',
                sift_score: 1,
                allele: 'C',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000618323.5:c.1519T>C',
                hgvsp: 'ENSP00000480678.2:p.Trp507Arg',
                cds_position: 1519,
                existing_variation: 'rs6672356&COSV58985794',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                domains: 'PANTHER:PTHR12247&PANTHER:PTHR12247:SF67',
                pheno: '0&1',
                revel: 0.145,
                cadd_phred: 19.24,
                cadd_raw: 1.997533,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: 9,
                spliceai_pred_dp_dg: 41,
                spliceai_pred_dp_dl: 9,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618779',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '8/12',
                codon_change: 'Tgg/Cgg',
                aa_change: 'W/R',
                aa_length: 281,
                polyphen_pred: 'benign',
                polyphen_score: 0,
                sift_pred: 'tolerated',
                sift_score: 1,
                allele: 'C',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000618779.5:c.841T>C',
                hgvsp: 'ENSP00000484256.1:p.Trp281Arg',
                cds_position: 841,
                existing_variation: 'rs6672356&COSV58985794',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                domains: 'PANTHER:PTHR12247&PANTHER:PTHR12247:SF67',
                pheno: '0&1',
                revel: 0.145,
                cadd_phred: 19.24,
                cadd_raw: 1.997533,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: 9,
                spliceai_pred_dp_dg: 41,
                spliceai_pred_dp_dl: 9,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000622503',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '9/13',
                codon_change: 'Tgg/Cgg',
                aa_change: 'W/R',
                aa_length: 344,
                polyphen_pred: 'benign',
                polyphen_score: 0,
                sift_pred: 'tolerated',
                sift_score: 1,
                allele: 'C',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000622503.5:c.1030T>C',
                hgvsp: 'ENSP00000482138.1:p.Trp344Arg',
                cds_position: 1030,
                existing_variation: 'rs6672356&COSV58985794',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                domains: 'PANTHER:PTHR12247&PANTHER:PTHR12247:SF67',
                pheno: '0&1',
                revel: 0.145,
                cadd_phred: 19.24,
                cadd_raw: 1.997533,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: 9,
                spliceai_pred_dp_dg: 41,
                spliceai_pred_dp_dl: 9,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000616016',
            ensembl_gene_id: ['ENSG00000187634', 'ENSG00000188976'],
            is_exonic: 1,
            is_coding: 1,
            aa_change: 'W/R',
            polyphen_pred: ['benign'],
            polyphen_score: [0],
            sift_pred: ['tolerated'],
            sift_score: [1],
            impact: 'missense_variant&downstream_gene_variant',
            hgvsc: 'ENST00000616016.5:c.1516T>C',
            hgvsp: 'ENSP00000478421.2:p.Trp506Arg',
            existing_variation: 'rs6672356&COSV58985794',
            source: 'Ensembl',
            domains: 'PANTHER:PTHR12247&PANTHER:PTHR12247:SF67',
            gnomad_af: 0.9999,
            gnomad_afr_af: 0.9994,
            gnomad_amr_af: 0.9999,
            gnomad_asj_af: 1,
            gnomad_eas_af: 1,
            gnomad_fin_af: 1,
            gnomad_nfe_af: 0.9999,
            gnomad_oth_af: 1,
            gnomad_sas_af: 0.9997,
            pheno: '0&1',
            revel: [0.145],
            cadd_phred: 19.24,
            cadd_raw: 1.997533,
            spliceai_pred_dp_ag: -15,
            spliceai_pred_dp_al: 9,
            spliceai_pred_dp_dg: 41,
            spliceai_pred_dp_dl: 9,
            spliceai_pred_ds_ag: [0.02],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'SAMD11',
            constraint_mis_z: [-1.901],
            constraint_oe_lof_upper: [1.327],
            constraint_oe_mis_upper: [1.338],
            constraint_syn_z: [-6.7147],
            exomiser_ad_exgenespheno: 0.2042,
            exomiser_xr_exgenescombi: 0.000014154,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'criteria_provided,_single_submitter',
            clndn: 'not_provided',
            gnomadv3_af_afr: 0.9996,
            gnomadv3_af_nfe: 0.9999,
            gnomadv3_af_xy: 0.9999,
            exomiser_xd_exgenescombi: 0.000014154,
            clnsigconf: 'None',
            gnomadv3_af_mid: 1,
            clnvcso: 'SO:0001483',
            gnomadv3_af: 0.9998,
            exomiser_mt_exgenespheno: 0.2042,
            clndisdb: 'MedGen:CN517202',
            exomiser_ar_exgenescombi: 0.000014154,
            gnomadv3_af_eas: 1,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 1,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.000014154,
            exomiser_ad_exgenescombi: 0.000014154,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'Benign',
            clnvc: 'single_nucleotide_variant',
            gnomadv3_af_ami: 1,
            gnomadv3_af_fin: 1,
            exomiser_ar_exgenespheno: 0.2042,
            gnomadv3_af_sas: 0.9996,
            gnomadv3_af_amr: 1,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'NC_000001.11:g.942451T>C',
            exomiser_ad_exgenesvar: 0,
            clnid: '1166513',
            gnomadv3_af_xx: 0.9998,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 1,
            exomiser_xr_exgenespheno: 0.2042,
            exomiser_xd_exgenespheno: 0.2042,
            impact_severity: 'MED',
            quality: 3373.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 38],
              [0, 42],
              [0, 35],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['SAMD11', 'NOC2L'],
            ensembl_transcript_id: [
              'ENST00000616016',
              'ENST00000327044',
              'ENST00000341065',
              'ENST00000342066',
              'ENST00000455979',
              'ENST00000616125',
              'ENST00000617307',
              'ENST00000618181',
              'ENST00000618323',
              'ENST00000618779',
              'ENST00000622503',
            ],
            variant_type: 'small',
            highest_af: 1,
            highest_af_info: {
              source: 'gnomad V2 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 877831,
            hg19_strand: '+',
            hg19_end: 877831,
            entrez_gene_id: ['148398', '26155'],
          },
          {
            is_read: false,
            _id: '64e59ef69c2aec9c8bd9f991',
            variant_id: '14',
            end: 944296,
            ref: 'G',
            alt: 'A',
            start: 944296,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '19/19',
                allele: 'A',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000327044.7:c.*398C>T',
                existing_variation: 'rs6605067&COSV58988623',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.59,
                cadd_raw: 0.151347,
                spliceai_pred_dp_ag: -13,
                spliceai_pred_dp_al: -46,
                spliceai_pred_dp_dg: 19,
                spliceai_pred_dp_dl: 1,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616016',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '14/14',
                allele: 'A',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000616016.5:c.*143G>A',
                existing_variation: 'rs6605067&COSV58988623',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                mane_select: 'NM_001385641.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.59,
                cadd_raw: 0.151347,
                spliceai_pred_dp_ag: -4,
                spliceai_pred_dp_al: 1,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: 0,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000341065',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '12/12',
                allele: 'A',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000341065.8:c.*143G>A',
                existing_variation: 'rs6605067&COSV58988623',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.59,
                cadd_raw: 0.151347,
                spliceai_pred_dp_ag: -4,
                spliceai_pred_dp_al: 1,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: 0,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.7709,
                constraint_oe_lof: 1.0444,
                constraint_oe_lof_lower: 0.745,
                constraint_oe_lof_upper: 1.49,
                constraint_oe_mis: 1.6125,
                constraint_oe_mis_lower: 1.496,
                constraint_oe_mis_upper: 1.738,
                constraint_oe_syn: 1.8448,
                constraint_oe_syn_lower: 1.655,
                constraint_oe_syn_upper: 1.973,
                constraint_pli: 2.1045e-15,
                constraint_syn_z: -7.6532,
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 2.1045e-15,
                loeuf: 1.49,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000342066',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '14/14',
                allele: 'A',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000342066.8:c.*143G>A',
                existing_variation: 'rs6605067&COSV58988623',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                ccds: 'CCDS2.2',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.59,
                cadd_raw: 0.151347,
                spliceai_pred_dp_ag: -4,
                spliceai_pred_dp_al: 1,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: 0,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.4361,
                constraint_oe_lof: 0.89656,
                constraint_oe_lof_lower: 0.64,
                constraint_oe_lof_upper: 1.28,
                constraint_oe_mis: 1.5082,
                constraint_oe_mis_lower: 1.405,
                constraint_oe_mis_upper: 1.619,
                constraint_oe_syn: 1.6983,
                constraint_oe_syn_lower: 1.537,
                constraint_oe_syn_upper: 1.875,
                constraint_pli: 4.7681e-14,
                constraint_syn_z: -6.9212,
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 4.7681e-14,
                loeuf: 1.28,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000455979',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6605067&COSV58988623',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.59,
                cadd_raw: 0.151347,
                spliceai_pred_dp_ag: -4,
                spliceai_pred_dp_al: 1,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: 0,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.7139,
                constraint_oe_lof: 1.0978,
                constraint_oe_lof_lower: 0.733,
                constraint_oe_lof_upper: 1.666,
                constraint_oe_mis: 1.6449,
                constraint_oe_mis_lower: 1.52,
                constraint_oe_mis_upper: 1.781,
                constraint_oe_syn: 1.8162,
                constraint_oe_syn_lower: 1.621,
                constraint_oe_syn_upper: 1.966,
                constraint_pli: 5.471e-11,
                constraint_syn_z: -7.0453,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 5.471e-11,
                loeuf: 1.666,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616125',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6605067&COSV58988623',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.59,
                cadd_raw: 0.151347,
                spliceai_pred_dp_ag: -4,
                spliceai_pred_dp_al: 1,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: 0,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000617307',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6605067&COSV58988623',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.59,
                cadd_raw: 0.151347,
                spliceai_pred_dp_ag: -4,
                spliceai_pred_dp_al: 1,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: 0,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618181',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6605067&COSV58988623',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.59,
                cadd_raw: 0.151347,
                spliceai_pred_dp_ag: -4,
                spliceai_pred_dp_al: 1,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: 0,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618323',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '14/14',
                allele: 'A',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000618323.5:c.*143G>A',
                existing_variation: 'rs6605067&COSV58988623',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.59,
                cadd_raw: 0.151347,
                spliceai_pred_dp_ag: -4,
                spliceai_pred_dp_al: 1,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: 0,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618779',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6605067&COSV58988623',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.59,
                cadd_raw: 0.151347,
                spliceai_pred_dp_ag: -4,
                spliceai_pred_dp_al: 1,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: 0,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000622503',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6605067&COSV58988623',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.59,
                cadd_raw: 0.151347,
                spliceai_pred_dp_ag: -4,
                spliceai_pred_dp_al: 1,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: 0,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: ['ENSG00000188976', 'ENSG00000187634'],
            is_exonic: 1,
            is_coding: 0,
            impact: '3_prime_UTR_variant',
            hgvsc: 'ENST00000327044.7:c.*398C>T',
            existing_variation: 'rs6605067&COSV58988623',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            pheno: '0&1',
            cadd_phred: 2.59,
            cadd_raw: 0.151347,
            spliceai_pred_dp_ag: -13,
            spliceai_pred_dp_al: -46,
            spliceai_pred_dp_dg: 19,
            spliceai_pred_dp_dl: 1,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'NOC2L',
            constraint_mis_z: [-1.901],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147],
            exomiser_ad_exgenespheno: 0.5034,
            exomiser_xr_exgenescombi: 0.00031712,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.8089,
            gnomadv3_af_nfe: 0.9448,
            gnomadv3_af_xy: 0.9052,
            exomiser_xd_exgenescombi: 0.00031712,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9272,
            gnomadv3_af: 0.904,
            exomiser_mt_exgenespheno: 0.5034,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00031712,
            gnomadv3_af_eas: 0.9196,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9271,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00031712,
            exomiser_ad_exgenescombi: 0.00031712,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9638,
            gnomadv3_af_fin: 0.9666,
            exomiser_ar_exgenespheno: 0.5034,
            gnomadv3_af_sas: 0.9215,
            gnomadv3_af_amr: 0.9161,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.9029,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.9077,
            exomiser_xr_exgenespheno: 0.5034,
            exomiser_xd_exgenespheno: 0.5034,
            impact_severity: 'LOW',
            quality: 1840.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [25, 23],
              [19, 18],
              [24, 29],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L', 'SAMD11'],
            ensembl_transcript_id: [
              'ENST00000327044',
              'ENST00000616016',
              'ENST00000341065',
              'ENST00000342066',
              'ENST00000455979',
              'ENST00000616125',
              'ENST00000617307',
              'ENST00000618181',
              'ENST00000618323',
              'ENST00000618779',
              'ENST00000622503',
            ],
            variant_type: 'small',
            highest_af: 0.9196,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 879676,
            hg19_strand: '+',
            hg19_end: 879676,
            entrez_gene_id: ['26155', '148398'],
          },
          {
            is_read: false,
            _id: '64e59ef69c2aec9c8bd9f992',
            variant_id: '15',
            end: 944307,
            ref: 'T',
            alt: 'C',
            start: 944307,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '19/19',
                allele: 'C',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000327044.7:c.*387A>G',
                existing_variation: 'rs2839&COSV58988633',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.782,
                cadd_raw: 0.169365,
                spliceai_pred_dp_ag: 48,
                spliceai_pred_dp_al: -24,
                spliceai_pred_dp_dg: -22,
                spliceai_pred_dp_dl: 8,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616016',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '14/14',
                allele: 'C',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000616016.5:c.*154T>C',
                existing_variation: 'rs2839&COSV58988633',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                mane_select: 'NM_001385641.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.782,
                cadd_raw: 0.169365,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: 22,
                spliceai_pred_dp_dg: 46,
                spliceai_pred_dp_dl: -42,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000341065',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '12/12',
                allele: 'C',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000341065.8:c.*154T>C',
                existing_variation: 'rs2839&COSV58988633',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.782,
                cadd_raw: 0.169365,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: 22,
                spliceai_pred_dp_dg: 46,
                spliceai_pred_dp_dl: -42,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.7709,
                constraint_oe_lof: 1.0444,
                constraint_oe_lof_lower: 0.745,
                constraint_oe_lof_upper: 1.49,
                constraint_oe_mis: 1.6125,
                constraint_oe_mis_lower: 1.496,
                constraint_oe_mis_upper: 1.738,
                constraint_oe_syn: 1.8448,
                constraint_oe_syn_lower: 1.655,
                constraint_oe_syn_upper: 1.973,
                constraint_pli: 2.1045e-15,
                constraint_syn_z: -7.6532,
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 2.1045e-15,
                loeuf: 1.49,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000342066',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '14/14',
                allele: 'C',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000342066.8:c.*154T>C',
                existing_variation: 'rs2839&COSV58988633',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                ccds: 'CCDS2.2',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.782,
                cadd_raw: 0.169365,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: 22,
                spliceai_pred_dp_dg: 46,
                spliceai_pred_dp_dl: -42,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.4361,
                constraint_oe_lof: 0.89656,
                constraint_oe_lof_lower: 0.64,
                constraint_oe_lof_upper: 1.28,
                constraint_oe_mis: 1.5082,
                constraint_oe_mis_lower: 1.405,
                constraint_oe_mis_upper: 1.619,
                constraint_oe_syn: 1.6983,
                constraint_oe_syn_lower: 1.537,
                constraint_oe_syn_upper: 1.875,
                constraint_pli: 4.7681e-14,
                constraint_syn_z: -6.9212,
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 4.7681e-14,
                loeuf: 1.28,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000455979',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2839&COSV58988633',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.782,
                cadd_raw: 0.169365,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: 22,
                spliceai_pred_dp_dg: 46,
                spliceai_pred_dp_dl: -42,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.7139,
                constraint_oe_lof: 1.0978,
                constraint_oe_lof_lower: 0.733,
                constraint_oe_lof_upper: 1.666,
                constraint_oe_mis: 1.6449,
                constraint_oe_mis_lower: 1.52,
                constraint_oe_mis_upper: 1.781,
                constraint_oe_syn: 1.8162,
                constraint_oe_syn_lower: 1.621,
                constraint_oe_syn_upper: 1.966,
                constraint_pli: 5.471e-11,
                constraint_syn_z: -7.0453,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 5.471e-11,
                loeuf: 1.666,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616125',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2839&COSV58988633',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.782,
                cadd_raw: 0.169365,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: 22,
                spliceai_pred_dp_dg: 46,
                spliceai_pred_dp_dl: -42,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000617307',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2839&COSV58988633',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.782,
                cadd_raw: 0.169365,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: 22,
                spliceai_pred_dp_dg: 46,
                spliceai_pred_dp_dl: -42,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618181',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2839&COSV58988633',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.782,
                cadd_raw: 0.169365,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: 22,
                spliceai_pred_dp_dg: 46,
                spliceai_pred_dp_dl: -42,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618323',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '14/14',
                allele: 'C',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000618323.5:c.*154T>C',
                existing_variation: 'rs2839&COSV58988633',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.782,
                cadd_raw: 0.169365,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: 22,
                spliceai_pred_dp_dg: 46,
                spliceai_pred_dp_dl: -42,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618779',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2839&COSV58988633',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.782,
                cadd_raw: 0.169365,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: 22,
                spliceai_pred_dp_dg: 46,
                spliceai_pred_dp_dl: -42,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000622503',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2839&COSV58988633',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.782,
                cadd_raw: 0.169365,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: 22,
                spliceai_pred_dp_dg: 46,
                spliceai_pred_dp_dl: -42,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: ['ENSG00000188976', 'ENSG00000187634'],
            is_exonic: 1,
            is_coding: 0,
            impact: '3_prime_UTR_variant',
            hgvsc: 'ENST00000327044.7:c.*387A>G',
            existing_variation: 'rs2839&COSV58988633',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            pheno: '0&1',
            cadd_phred: 2.782,
            cadd_raw: 0.169365,
            spliceai_pred_dp_ag: 48,
            spliceai_pred_dp_al: -24,
            spliceai_pred_dp_dg: -22,
            spliceai_pred_dp_dl: 8,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'NOC2L',
            constraint_mis_z: [-1.901],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147],
            exomiser_ad_exgenespheno: 0.5034,
            exomiser_xr_exgenescombi: 0.00031712,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.9091,
            gnomadv3_af_nfe: 0.9465,
            gnomadv3_af_xy: 0.934,
            exomiser_xd_exgenescombi: 0.00031712,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9304,
            gnomadv3_af: 0.9341,
            exomiser_mt_exgenespheno: 0.5034,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00031712,
            gnomadv3_af_eas: 0.9194,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9274,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00031712,
            exomiser_ad_exgenescombi: 0.00031712,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9638,
            gnomadv3_af_fin: 0.9666,
            exomiser_ar_exgenespheno: 0.5034,
            gnomadv3_af_sas: 0.9219,
            gnomadv3_af_amr: 0.9329,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.9343,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.9336,
            exomiser_xr_exgenespheno: 0.5034,
            exomiser_xd_exgenespheno: 0.5034,
            impact_severity: 'LOW',
            quality: 1743.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [22, 22],
              [20, 15],
              [25, 24],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L', 'SAMD11'],
            ensembl_transcript_id: [
              'ENST00000327044',
              'ENST00000616016',
              'ENST00000341065',
              'ENST00000342066',
              'ENST00000455979',
              'ENST00000616125',
              'ENST00000617307',
              'ENST00000618181',
              'ENST00000618323',
              'ENST00000618779',
              'ENST00000622503',
            ],
            variant_type: 'small',
            highest_af: 0.9341,
            highest_af_info: {
              source: 'gnomad v3 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 879687,
            hg19_strand: '+',
            hg19_end: 879687,
            entrez_gene_id: ['26155', '148398'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f993',
            variant_id: '16',
            end: 944543,
            ref: 'C',
            alt: 'T',
            start: 944543,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '19/19',
                allele: 'T',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000327044.7:c.*151G>A',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                cadd_phred: 5.033,
                cadd_raw: 0.364874,
                spliceai_pred_dp_ag: -16,
                spliceai_pred_dp_al: 7,
                spliceai_pred_dp_dg: 7,
                spliceai_pred_dp_dl: -7,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.05,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616016',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '14/14',
                allele: 'T',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000616016.5:c.*390C>T',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                mane_select: 'NM_001385641.1',
                source: 'Ensembl',
                cadd_phred: 5.033,
                cadd_raw: 0.364874,
                spliceai_pred_dp_ag: -22,
                spliceai_pred_dp_al: -6,
                spliceai_pred_dp_dg: -8,
                spliceai_pred_dp_dl: -3,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000341065',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '12/12',
                allele: 'T',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000341065.8:c.*390C>T',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 5.033,
                cadd_raw: 0.364874,
                spliceai_pred_dp_ag: -22,
                spliceai_pred_dp_al: -6,
                spliceai_pred_dp_dg: -8,
                spliceai_pred_dp_dl: -3,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.7709,
                constraint_oe_lof: 1.0444,
                constraint_oe_lof_lower: 0.745,
                constraint_oe_lof_upper: 1.49,
                constraint_oe_mis: 1.6125,
                constraint_oe_mis_lower: 1.496,
                constraint_oe_mis_upper: 1.738,
                constraint_oe_syn: 1.8448,
                constraint_oe_syn_lower: 1.655,
                constraint_oe_syn_upper: 1.973,
                constraint_pli: 2.1045e-15,
                constraint_syn_z: -7.6532,
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 2.1045e-15,
                loeuf: 1.49,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000342066',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '14/14',
                allele: 'T',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000342066.8:c.*390C>T',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                ccds: 'CCDS2.2',
                source: 'Ensembl',
                cadd_phred: 5.033,
                cadd_raw: 0.364874,
                spliceai_pred_dp_ag: -22,
                spliceai_pred_dp_al: -6,
                spliceai_pred_dp_dg: -8,
                spliceai_pred_dp_dl: -3,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.4361,
                constraint_oe_lof: 0.89656,
                constraint_oe_lof_lower: 0.64,
                constraint_oe_lof_upper: 1.28,
                constraint_oe_mis: 1.5082,
                constraint_oe_mis_lower: 1.405,
                constraint_oe_mis_upper: 1.619,
                constraint_oe_syn: 1.6983,
                constraint_oe_syn_lower: 1.537,
                constraint_oe_syn_upper: 1.875,
                constraint_pli: 4.7681e-14,
                constraint_syn_z: -6.9212,
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 4.7681e-14,
                loeuf: 1.28,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000455979',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 5.033,
                cadd_raw: 0.364874,
                spliceai_pred_dp_ag: -22,
                spliceai_pred_dp_al: -6,
                spliceai_pred_dp_dg: -8,
                spliceai_pred_dp_dl: -3,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                constraint_mis_z: -3.7139,
                constraint_oe_lof: 1.0978,
                constraint_oe_lof_lower: 0.733,
                constraint_oe_lof_upper: 1.666,
                constraint_oe_mis: 1.6449,
                constraint_oe_mis_lower: 1.52,
                constraint_oe_mis_upper: 1.781,
                constraint_oe_syn: 1.8162,
                constraint_oe_syn_lower: 1.621,
                constraint_oe_syn_upper: 1.966,
                constraint_pli: 5.471e-11,
                constraint_syn_z: -7.0453,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 5.471e-11,
                loeuf: 1.666,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616125',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 5.033,
                cadd_raw: 0.364874,
                spliceai_pred_dp_ag: -22,
                spliceai_pred_dp_al: -6,
                spliceai_pred_dp_dg: -8,
                spliceai_pred_dp_dl: -3,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000617307',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 5.033,
                cadd_raw: 0.364874,
                spliceai_pred_dp_ag: -22,
                spliceai_pred_dp_al: -6,
                spliceai_pred_dp_dg: -8,
                spliceai_pred_dp_dl: -3,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618181',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 5.033,
                cadd_raw: 0.364874,
                spliceai_pred_dp_ag: -22,
                spliceai_pred_dp_al: -6,
                spliceai_pred_dp_dg: -8,
                spliceai_pred_dp_dl: -3,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618323',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '14/14',
                allele: 'T',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000618323.5:c.*390C>T',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 5.033,
                cadd_raw: 0.364874,
                spliceai_pred_dp_ag: -22,
                spliceai_pred_dp_al: -6,
                spliceai_pred_dp_dg: -8,
                spliceai_pred_dp_dl: -3,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618779',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 5.033,
                cadd_raw: 0.364874,
                spliceai_pred_dp_ag: -22,
                spliceai_pred_dp_al: -6,
                spliceai_pred_dp_dg: -8,
                spliceai_pred_dp_dl: -3,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000622503',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 5.033,
                cadd_raw: 0.364874,
                spliceai_pred_dp_ag: -22,
                spliceai_pred_dp_al: -6,
                spliceai_pred_dp_dg: -8,
                spliceai_pred_dp_dl: -3,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'SAMD11',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: ['ENSG00000188976', 'ENSG00000187634'],
            is_exonic: 1,
            is_coding: 0,
            impact: '3_prime_UTR_variant',
            hgvsc: 'ENST00000327044.7:c.*151G>A',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            cadd_phred: 5.033,
            cadd_raw: 0.364874,
            spliceai_pred_dp_ag: -16,
            spliceai_pred_dp_al: 7,
            spliceai_pred_dp_dg: 7,
            spliceai_pred_dp_dl: -7,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0.05, 0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'NOC2L',
            constraint_mis_z: [-1.901],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147],
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: -1,
            gnomadv3_af_nfe: -1,
            gnomadv3_af_xy: -1,
            clnsigconf: 'None',
            gnomadv3_af_mid: -1,
            gnomadv3_af: -1,
            clndisdb: 'None',
            gnomadv3_af_eas: -1,
            gnomadv3_af_asj: -1,
            clnsigincl: 'None',
            clnvi: 'None',
            clnsig: 'None',
            gnomadv3_af_ami: -1,
            gnomadv3_af_fin: -1,
            gnomadv3_af_sas: -1,
            gnomadv3_af_amr: -1,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            gnomadv3_af_xx: -1,
            gnomadv3_af_oth: -1,
            impact_severity: 'LOW',
            quality: 422.2,
            genotypes_index: [
              [0, 0],
              [0, 1],
              [0, 0],
            ],
            allelic_depths: [
              [40, 0],
              [24, 16],
              [40, 0],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L', 'SAMD11'],
            ensembl_transcript_id: [
              'ENST00000327044',
              'ENST00000616016',
              'ENST00000341065',
              'ENST00000342066',
              'ENST00000455979',
              'ENST00000616125',
              'ENST00000617307',
              'ENST00000618181',
              'ENST00000618323',
              'ENST00000618779',
              'ENST00000622503',
            ],
            variant_type: 'small',
            highest_af: -1,
            highest_af_info: {
              source: 'gnomad V2 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 879923,
            hg19_strand: '+',
            hg19_end: 879923,
            entrez_gene_id: ['26155', '148398'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f994',
            variant_id: '17',
            end: 944858,
            ref: 'A',
            alt: 'G',
            start: 944858,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '18/18',
                hgvsc: 'ENST00000327044.7:c.2144-58T>C',
                existing_variation: 'rs3748592&COSV58986945',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 5.936,
                cadd_raw: 0.446489,
                spliceai_pred_dp_ag: -39,
                spliceai_pred_dp_al: -25,
                spliceai_pred_dp_dg: 13,
                spliceai_pred_dp_dl: 32,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616016',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3748592&COSV58986945',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                mane_select: 'NM_001385641.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 5.936,
                cadd_raw: 0.446489,
                spliceai_pred_dp_ag: -39,
                spliceai_pred_dp_al: -25,
                spliceai_pred_dp_dg: 13,
                spliceai_pred_dp_dl: 32,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000341065',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3748592&COSV58986945',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 5.936,
                cadd_raw: 0.446489,
                spliceai_pred_dp_ag: -39,
                spliceai_pred_dp_al: -25,
                spliceai_pred_dp_dg: 13,
                spliceai_pred_dp_dl: 32,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -3.7709,
                constraint_oe_lof: 1.0444,
                constraint_oe_lof_lower: 0.745,
                constraint_oe_lof_upper: 1.49,
                constraint_oe_mis: 1.6125,
                constraint_oe_mis_lower: 1.496,
                constraint_oe_mis_upper: 1.738,
                constraint_oe_syn: 1.8448,
                constraint_oe_syn_lower: 1.655,
                constraint_oe_syn_upper: 1.973,
                constraint_pli: 2.1045e-15,
                constraint_syn_z: -7.6532,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 2.1045e-15,
                loeuf: 1.49,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000342066',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3748592&COSV58986945',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                ccds: 'CCDS2.2',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 5.936,
                cadd_raw: 0.446489,
                spliceai_pred_dp_ag: -39,
                spliceai_pred_dp_al: -25,
                spliceai_pred_dp_dg: 13,
                spliceai_pred_dp_dl: 32,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -3.4361,
                constraint_oe_lof: 0.89656,
                constraint_oe_lof_lower: 0.64,
                constraint_oe_lof_upper: 1.28,
                constraint_oe_mis: 1.5082,
                constraint_oe_mis_lower: 1.405,
                constraint_oe_mis_upper: 1.619,
                constraint_oe_syn: 1.6983,
                constraint_oe_syn_lower: 1.537,
                constraint_oe_syn_upper: 1.875,
                constraint_pli: 4.7681e-14,
                constraint_syn_z: -6.9212,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 4.7681e-14,
                loeuf: 1.28,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000455979',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3748592&COSV58986945',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 5.936,
                cadd_raw: 0.446489,
                spliceai_pred_dp_ag: -39,
                spliceai_pred_dp_al: -25,
                spliceai_pred_dp_dg: 13,
                spliceai_pred_dp_dl: 32,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -3.7139,
                constraint_oe_lof: 1.0978,
                constraint_oe_lof_lower: 0.733,
                constraint_oe_lof_upper: 1.666,
                constraint_oe_mis: 1.6449,
                constraint_oe_mis_lower: 1.52,
                constraint_oe_mis_upper: 1.781,
                constraint_oe_syn: 1.8162,
                constraint_oe_syn_lower: 1.621,
                constraint_oe_syn_upper: 1.966,
                constraint_pli: 5.471e-11,
                constraint_syn_z: -7.0453,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 5.471e-11,
                loeuf: 1.666,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616125',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3748592&COSV58986945',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 5.936,
                cadd_raw: 0.446489,
                spliceai_pred_dp_ag: -39,
                spliceai_pred_dp_al: -25,
                spliceai_pred_dp_dg: 13,
                spliceai_pred_dp_dl: 32,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000617307',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3748592&COSV58986945',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 5.936,
                cadd_raw: 0.446489,
                spliceai_pred_dp_ag: -39,
                spliceai_pred_dp_al: -25,
                spliceai_pred_dp_dg: 13,
                spliceai_pred_dp_dl: 32,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618181',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3748592&COSV58986945',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 5.936,
                cadd_raw: 0.446489,
                spliceai_pred_dp_ag: -39,
                spliceai_pred_dp_al: -25,
                spliceai_pred_dp_dg: 13,
                spliceai_pred_dp_dl: 32,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618323',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3748592&COSV58986945',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 5.936,
                cadd_raw: 0.446489,
                spliceai_pred_dp_ag: -39,
                spliceai_pred_dp_al: -25,
                spliceai_pred_dp_dg: 13,
                spliceai_pred_dp_dl: 32,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618779',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3748592&COSV58986945',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 5.936,
                cadd_raw: 0.446489,
                spliceai_pred_dp_ag: -39,
                spliceai_pred_dp_al: -25,
                spliceai_pred_dp_dg: 13,
                spliceai_pred_dp_dl: 32,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000622503',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3748592&COSV58986945',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 5.936,
                cadd_raw: 0.446489,
                spliceai_pred_dp_ag: -39,
                spliceai_pred_dp_al: -25,
                spliceai_pred_dp_dg: 13,
                spliceai_pred_dp_dl: 32,
                spliceai_pred_ds_ag: 0.02,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: ['ENSG00000188976', 'ENSG00000187634'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant&downstream_gene_variant',
            hgvsc: 'ENST00000327044.7:c.2144-58T>C',
            existing_variation: 'rs3748592&COSV58986945',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            pheno: '0&1',
            cadd_phred: 5.936,
            cadd_raw: 0.446489,
            spliceai_pred_dp_ag: -39,
            spliceai_pred_dp_al: -25,
            spliceai_pred_dp_dg: 13,
            spliceai_pred_dp_dl: 32,
            spliceai_pred_ds_ag: [0.02],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'NOC2L',
            constraint_mis_z: [-1.901],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147],
            exomiser_ad_exgenespheno: 0.5034,
            exomiser_xr_exgenescombi: 0.00031712,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.909,
            gnomadv3_af_nfe: 0.9465,
            gnomadv3_af_xy: 0.9338,
            exomiser_xd_exgenescombi: 0.00031712,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9304,
            gnomadv3_af: 0.9341,
            exomiser_mt_exgenespheno: 0.5034,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00031712,
            gnomadv3_af_eas: 0.9197,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9274,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00031712,
            exomiser_ad_exgenescombi: 0.00031712,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9638,
            gnomadv3_af_fin: 0.9667,
            exomiser_ar_exgenespheno: 0.5034,
            gnomadv3_af_sas: 0.9216,
            gnomadv3_af_amr: 0.9332,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.9343,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.9336,
            exomiser_xr_exgenespheno: 0.5034,
            exomiser_xd_exgenespheno: 0.5034,
            impact_severity: 'LOW',
            quality: 2019.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [14, 28],
              [22, 22],
              [34, 21],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L', 'SAMD11'],
            ensembl_transcript_id: [
              'ENST00000327044',
              'ENST00000616016',
              'ENST00000341065',
              'ENST00000342066',
              'ENST00000455979',
              'ENST00000616125',
              'ENST00000617307',
              'ENST00000618181',
              'ENST00000618323',
              'ENST00000618779',
              'ENST00000622503',
            ],
            variant_type: 'small',
            highest_af: 0.9341,
            highest_af_info: {
              source: 'gnomad v3 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 880238,
            hg19_strand: '+',
            hg19_end: 880238,
            entrez_gene_id: ['26155', '148398'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f995',
            variant_id: '18',
            end: 946247,
            ref: 'G',
            alt: 'A',
            start: 946247,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '16/19',
                codon_change: 'Ctg/Ttg',
                aa_change: 'L',
                aa_length: 615,
                allele: 'A',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000327044.7:c.1843C>T',
                hgvsp: 'ENSP00000317992.6:p.Leu615%3D',
                cds_position: 1843,
                existing_variation: 'rs2272757&COSV58986455',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                domains:
                  'Pfam:PF03715&PANTHER:PTHR12687&PANTHER:PTHR12687:SF10&Superfamily:SSF48371',
                pheno: '0&1',
                cadd_phred: 5.05,
                cadd_raw: 0.366387,
                spliceai_pred_dp_ag: -17,
                spliceai_pred_dp_al: 35,
                spliceai_pred_dp_dg: -7,
                spliceai_pred_dp_dl: -28,
                spliceai_pred_ds_ag: 0.05,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616016',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2272757&COSV58986455',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                mane_select: 'NM_001385641.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 5.05,
                cadd_raw: 0.366387,
                spliceai_pred_dp_ag: -17,
                spliceai_pred_dp_al: 35,
                spliceai_pred_dp_dg: -7,
                spliceai_pred_dp_dl: -28,
                spliceai_pred_ds_ag: 0.05,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000341065',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2272757&COSV58986455',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 5.05,
                cadd_raw: 0.366387,
                spliceai_pred_dp_ag: -17,
                spliceai_pred_dp_al: 35,
                spliceai_pred_dp_dg: -7,
                spliceai_pred_dp_dl: -28,
                spliceai_pred_ds_ag: 0.05,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -3.7709,
                constraint_oe_lof: 1.0444,
                constraint_oe_lof_lower: 0.745,
                constraint_oe_lof_upper: 1.49,
                constraint_oe_mis: 1.6125,
                constraint_oe_mis_lower: 1.496,
                constraint_oe_mis_upper: 1.738,
                constraint_oe_syn: 1.8448,
                constraint_oe_syn_lower: 1.655,
                constraint_oe_syn_upper: 1.973,
                constraint_pli: 2.1045e-15,
                constraint_syn_z: -7.6532,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 2.1045e-15,
                loeuf: 1.49,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000342066',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2272757&COSV58986455',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                ccds: 'CCDS2.2',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 5.05,
                cadd_raw: 0.366387,
                spliceai_pred_dp_ag: -17,
                spliceai_pred_dp_al: 35,
                spliceai_pred_dp_dg: -7,
                spliceai_pred_dp_dl: -28,
                spliceai_pred_ds_ag: 0.05,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -3.4361,
                constraint_oe_lof: 0.89656,
                constraint_oe_lof_lower: 0.64,
                constraint_oe_lof_upper: 1.28,
                constraint_oe_mis: 1.5082,
                constraint_oe_mis_lower: 1.405,
                constraint_oe_mis_upper: 1.619,
                constraint_oe_syn: 1.6983,
                constraint_oe_syn_lower: 1.537,
                constraint_oe_syn_upper: 1.875,
                constraint_pli: 4.7681e-14,
                constraint_syn_z: -6.9212,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 4.7681e-14,
                loeuf: 1.28,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000455979',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2272757&COSV58986455',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 5.05,
                cadd_raw: 0.366387,
                spliceai_pred_dp_ag: -17,
                spliceai_pred_dp_al: 35,
                spliceai_pred_dp_dg: -7,
                spliceai_pred_dp_dl: -28,
                spliceai_pred_ds_ag: 0.05,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -3.7139,
                constraint_oe_lof: 1.0978,
                constraint_oe_lof_lower: 0.733,
                constraint_oe_lof_upper: 1.666,
                constraint_oe_mis: 1.6449,
                constraint_oe_mis_lower: 1.52,
                constraint_oe_mis_upper: 1.781,
                constraint_oe_syn: 1.8162,
                constraint_oe_syn_lower: 1.621,
                constraint_oe_syn_upper: 1.966,
                constraint_pli: 5.471e-11,
                constraint_syn_z: -7.0453,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 5.471e-11,
                loeuf: 1.666,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616125',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2272757&COSV58986455',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 5.05,
                cadd_raw: 0.366387,
                spliceai_pred_dp_ag: -17,
                spliceai_pred_dp_al: 35,
                spliceai_pred_dp_dg: -7,
                spliceai_pred_dp_dl: -28,
                spliceai_pred_ds_ag: 0.05,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000617307',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2272757&COSV58986455',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 5.05,
                cadd_raw: 0.366387,
                spliceai_pred_dp_ag: -17,
                spliceai_pred_dp_al: 35,
                spliceai_pred_dp_dg: -7,
                spliceai_pred_dp_dl: -28,
                spliceai_pred_ds_ag: 0.05,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618181',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2272757&COSV58986455',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 5.05,
                cadd_raw: 0.366387,
                spliceai_pred_dp_ag: -17,
                spliceai_pred_dp_al: 35,
                spliceai_pred_dp_dg: -7,
                spliceai_pred_dp_dl: -28,
                spliceai_pred_ds_ag: 0.05,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618323',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2272757&COSV58986455',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 5.05,
                cadd_raw: 0.366387,
                spliceai_pred_dp_ag: -17,
                spliceai_pred_dp_al: 35,
                spliceai_pred_dp_dg: -7,
                spliceai_pred_dp_dl: -28,
                spliceai_pred_ds_ag: 0.05,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618779',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2272757&COSV58986455',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 5.05,
                cadd_raw: 0.366387,
                spliceai_pred_dp_ag: -17,
                spliceai_pred_dp_al: 35,
                spliceai_pred_dp_dg: -7,
                spliceai_pred_dp_dl: -28,
                spliceai_pred_ds_ag: 0.05,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000622503',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2272757&COSV58986455',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 5.05,
                cadd_raw: 0.366387,
                spliceai_pred_dp_ag: -17,
                spliceai_pred_dp_al: 35,
                spliceai_pred_dp_dg: -7,
                spliceai_pred_dp_dl: -28,
                spliceai_pred_ds_ag: 0.05,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: ['ENSG00000188976', 'ENSG00000187634'],
            is_exonic: 1,
            is_coding: 1,
            aa_change: 'L',
            impact: 'synonymous_variant&downstream_gene_variant',
            hgvsc: 'ENST00000327044.7:c.1843C>T',
            hgvsp: 'ENSP00000317992.6:p.Leu615%3D',
            existing_variation: 'rs2272757&COSV58986455',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            domains:
              'Pfam:PF03715&PANTHER:PTHR12687&PANTHER:PTHR12687:SF10&Superfamily:SSF48371',
            gnomad_af: 0.5696,
            gnomad_afr_af: 0.1314,
            gnomad_amr_af: 0.4859,
            gnomad_asj_af: 0.5436,
            gnomad_eas_af: 0.6586,
            gnomad_fin_af: 0.6262,
            gnomad_nfe_af: 0.6363,
            gnomad_oth_af: 0.5668,
            gnomad_sas_af: 0.5655,
            pheno: '0&1',
            cadd_phred: 5.05,
            cadd_raw: 0.366387,
            spliceai_pred_dp_ag: -17,
            spliceai_pred_dp_al: 35,
            spliceai_pred_dp_dg: -7,
            spliceai_pred_dp_dl: -28,
            spliceai_pred_ds_ag: [0.05],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'NOC2L',
            constraint_mis_z: [-1.901],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147],
            exomiser_ad_exgenespheno: 0.5034,
            exomiser_xr_exgenescombi: 0.00031712,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.145,
            gnomadv3_af_nfe: 0.638,
            gnomadv3_af_xy: 0.4806,
            exomiser_xd_exgenescombi: 0.00031712,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.5669,
            gnomadv3_af: 0.4817,
            exomiser_mt_exgenespheno: 0.5034,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00031712,
            gnomadv3_af_eas: 0.6582,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.5614,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00031712,
            exomiser_ad_exgenescombi: 0.00031712,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.5143,
            gnomadv3_af_fin: 0.6286,
            exomiser_ar_exgenespheno: 0.5034,
            gnomadv3_af_sas: 0.5769,
            gnomadv3_af_amr: 0.4817,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.4828,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.511,
            exomiser_xr_exgenespheno: 0.5034,
            exomiser_xd_exgenespheno: 0.5034,
            impact_severity: 'LOW',
            quality: 1596.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [25, 24],
              [21, 25],
              [24, 13],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L', 'SAMD11'],
            ensembl_transcript_id: [
              'ENST00000327044',
              'ENST00000616016',
              'ENST00000341065',
              'ENST00000342066',
              'ENST00000455979',
              'ENST00000616125',
              'ENST00000617307',
              'ENST00000618181',
              'ENST00000618323',
              'ENST00000618779',
              'ENST00000622503',
            ],
            variant_type: 'small',
            highest_af: 0.6586,
            highest_af_info: {
              source: 'gnomad V2 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 881627,
            hg19_strand: '+',
            hg19_end: 881627,
            entrez_gene_id: ['26155', '148398'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f996',
            variant_id: '19',
            end: 948245,
            ref: 'A',
            alt: 'G',
            start: 948245,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '13/18',
                hgvsc: 'ENST00000327044.7:c.1558-13T>C',
                existing_variation: 'rs4970378',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                cadd_phred: 1.123,
                cadd_raw: -0.036612,
                spliceai_pred_dp_ag: 27,
                spliceai_pred_dp_al: -5,
                spliceai_pred_dp_dg: -9,
                spliceai_pred_dp_dl: -32,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616016',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970378',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                mane_select: 'NM_001385641.1',
                source: 'Ensembl',
                cadd_phred: 1.123,
                cadd_raw: -0.036612,
                spliceai_pred_dp_ag: 27,
                spliceai_pred_dp_al: -5,
                spliceai_pred_dp_dg: -9,
                spliceai_pred_dp_dl: -32,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000341065',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970378',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 1.123,
                cadd_raw: -0.036612,
                spliceai_pred_dp_ag: 27,
                spliceai_pred_dp_al: -5,
                spliceai_pred_dp_dg: -9,
                spliceai_pred_dp_dl: -32,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -3.7709,
                constraint_oe_lof: 1.0444,
                constraint_oe_lof_lower: 0.745,
                constraint_oe_lof_upper: 1.49,
                constraint_oe_mis: 1.6125,
                constraint_oe_mis_lower: 1.496,
                constraint_oe_mis_upper: 1.738,
                constraint_oe_syn: 1.8448,
                constraint_oe_syn_lower: 1.655,
                constraint_oe_syn_upper: 1.973,
                constraint_pli: 2.1045e-15,
                constraint_syn_z: -7.6532,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 2.1045e-15,
                loeuf: 1.49,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000342066',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970378',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                ccds: 'CCDS2.2',
                source: 'Ensembl',
                cadd_phred: 1.123,
                cadd_raw: -0.036612,
                spliceai_pred_dp_ag: 27,
                spliceai_pred_dp_al: -5,
                spliceai_pred_dp_dg: -9,
                spliceai_pred_dp_dl: -32,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -3.4361,
                constraint_oe_lof: 0.89656,
                constraint_oe_lof_lower: 0.64,
                constraint_oe_lof_upper: 1.28,
                constraint_oe_mis: 1.5082,
                constraint_oe_mis_lower: 1.405,
                constraint_oe_mis_upper: 1.619,
                constraint_oe_syn: 1.6983,
                constraint_oe_syn_lower: 1.537,
                constraint_oe_syn_upper: 1.875,
                constraint_pli: 4.7681e-14,
                constraint_syn_z: -6.9212,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 4.7681e-14,
                loeuf: 1.28,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000455979',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970378',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 1.123,
                cadd_raw: -0.036612,
                spliceai_pred_dp_ag: 27,
                spliceai_pred_dp_al: -5,
                spliceai_pred_dp_dg: -9,
                spliceai_pred_dp_dl: -32,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -3.7139,
                constraint_oe_lof: 1.0978,
                constraint_oe_lof_lower: 0.733,
                constraint_oe_lof_upper: 1.666,
                constraint_oe_mis: 1.6449,
                constraint_oe_mis_lower: 1.52,
                constraint_oe_mis_upper: 1.781,
                constraint_oe_syn: 1.8162,
                constraint_oe_syn_lower: 1.621,
                constraint_oe_syn_upper: 1.966,
                constraint_pli: 5.471e-11,
                constraint_syn_z: -7.0453,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                p_li: 5.471e-11,
                loeuf: 1.666,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000616125',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970378',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 1.123,
                cadd_raw: -0.036612,
                spliceai_pred_dp_ag: 27,
                spliceai_pred_dp_al: -5,
                spliceai_pred_dp_dg: -9,
                spliceai_pred_dp_dl: -32,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000617307',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970378',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 1.123,
                cadd_raw: -0.036612,
                spliceai_pred_dp_ag: 27,
                spliceai_pred_dp_al: -5,
                spliceai_pred_dp_dg: -9,
                spliceai_pred_dp_dl: -32,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618181',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970378',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 1.123,
                cadd_raw: -0.036612,
                spliceai_pred_dp_ag: 27,
                spliceai_pred_dp_al: -5,
                spliceai_pred_dp_dg: -9,
                spliceai_pred_dp_dl: -32,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618323',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970378',
                strand: 1,
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 1.123,
                cadd_raw: -0.036612,
                spliceai_pred_dp_ag: 27,
                spliceai_pred_dp_al: -5,
                spliceai_pred_dp_dg: -9,
                spliceai_pred_dp_dl: -32,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000618779',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970378',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 1.123,
                cadd_raw: -0.036612,
                spliceai_pred_dp_ag: 27,
                spliceai_pred_dp_al: -5,
                spliceai_pred_dp_dg: -9,
                spliceai_pred_dp_dl: -32,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'SAMD11',
                ensembl_gene_id: 'ENSG00000187634',
                transcript: 'ENST00000622503',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970378',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:28706',
                source: 'Ensembl',
                cadd_phred: 1.123,
                cadd_raw: -0.036612,
                spliceai_pred_dp_ag: 27,
                spliceai_pred_dp_al: -5,
                spliceai_pred_dp_dg: -9,
                spliceai_pred_dp_dl: -32,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'SAMD11',
                entrez_gene_id: '148398',
                p_haplo: 0.05961,
                p_triplo: 0.08779,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: ['ENSG00000188976', 'ENSG00000187634'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant&downstream_gene_variant',
            hgvsc: 'ENST00000327044.7:c.1558-13T>C',
            existing_variation: 'rs4970378',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            gnomad_af: 1,
            gnomad_afr_af: 0.9996,
            gnomad_amr_af: 1,
            gnomad_asj_af: 1,
            gnomad_eas_af: 1,
            gnomad_fin_af: 1,
            gnomad_nfe_af: 1,
            gnomad_oth_af: 1,
            gnomad_sas_af: 1,
            cadd_phred: 1.123,
            cadd_raw: -0.036612,
            spliceai_pred_dp_ag: 27,
            spliceai_pred_dp_al: -5,
            spliceai_pred_dp_dg: -9,
            spliceai_pred_dp_dl: -32,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'NOC2L',
            constraint_mis_z: [-1.901],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147],
            exomiser_ad_exgenespheno: 0.5034,
            exomiser_xr_exgenescombi: 0.00031712,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.9997,
            gnomadv3_af_nfe: 1,
            gnomadv3_af_xy: 0.9999,
            exomiser_xd_exgenescombi: 0.00031712,
            clnsigconf: 'None',
            gnomadv3_af_mid: 1,
            gnomadv3_af: 0.9999,
            exomiser_mt_exgenespheno: 0.5034,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00031712,
            gnomadv3_af_eas: 1,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 1,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00031712,
            exomiser_ad_exgenescombi: 0.00031712,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 1,
            gnomadv3_af_fin: 1,
            exomiser_ar_exgenespheno: 0.5034,
            gnomadv3_af_sas: 1,
            gnomadv3_af_amr: 1,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.9999,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 1,
            exomiser_xr_exgenespheno: 0.5034,
            exomiser_xd_exgenespheno: 0.5034,
            impact_severity: 'LOW',
            quality: 5574.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 57],
              [0, 49],
              [0, 55],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L', 'SAMD11'],
            ensembl_transcript_id: [
              'ENST00000327044',
              'ENST00000616016',
              'ENST00000341065',
              'ENST00000342066',
              'ENST00000455979',
              'ENST00000616125',
              'ENST00000617307',
              'ENST00000618181',
              'ENST00000618323',
              'ENST00000618779',
              'ENST00000622503',
            ],
            variant_type: 'small',
            highest_af: 1,
            highest_af_info: {
              source: 'gnomad V2 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 883625,
            hg19_strand: '+',
            hg19_end: 883625,
            entrez_gene_id: ['26155', '148398'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f997',
            variant_id: '20',
            end: 952180,
            ref: 'A',
            alt: 'C',
            start: 952180,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '10/18',
                hgvsc: 'ENST00000327044.7:c.1192-41T>G',
                existing_variation: 'rs3748595&COSV58989178',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 2.435,
                cadd_raw: 0.136248,
                spliceai_pred_dp_ag: 23,
                spliceai_pred_dp_al: -18,
                spliceai_pred_dp_dg: 0,
                spliceai_pred_dp_dl: -45,
                spliceai_pred_ds_ag: 0.03,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: ['ENSG00000188976'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant',
            hgvsc: 'ENST00000327044.7:c.1192-41T>G',
            existing_variation: 'rs3748595&COSV58989178',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            gnomad_af: 0.9277,
            gnomad_afr_af: 0.8243,
            gnomad_amr_af: 0.9486,
            gnomad_asj_af: 0.9255,
            gnomad_eas_af: 0.9108,
            gnomad_fin_af: 0.955,
            gnomad_nfe_af: 0.9363,
            gnomad_oth_af: 0.9285,
            gnomad_sas_af: 0.9187,
            pheno: '0&1',
            cadd_phred: 2.435,
            cadd_raw: 0.136248,
            spliceai_pred_dp_ag: 23,
            spliceai_pred_dp_al: -18,
            spliceai_pred_dp_dg: 0,
            spliceai_pred_dp_dl: -45,
            spliceai_pred_ds_ag: [0.03],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'NOC2L',
            constraint_mis_z: [-1.901],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147],
            exomiser_ad_exgenespheno: 0.5034,
            exomiser_xr_exgenescombi: 0.00031712,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.8283,
            gnomadv3_af_nfe: 0.9411,
            gnomadv3_af_xy: 0.9081,
            exomiser_xd_exgenescombi: 0.00031712,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9209,
            gnomadv3_af: 0.9078,
            exomiser_mt_exgenespheno: 0.5034,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00031712,
            gnomadv3_af_eas: 0.9245,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.921,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00031712,
            exomiser_ad_exgenescombi: 0.00031712,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9638,
            gnomadv3_af_fin: 0.9608,
            exomiser_ar_exgenespheno: 0.5034,
            gnomadv3_af_sas: 0.9202,
            gnomadv3_af_amr: 0.9203,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.9074,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.9167,
            exomiser_xr_exgenespheno: 0.5034,
            exomiser_xd_exgenespheno: 0.5034,
            impact_severity: 'LOW',
            quality: 1730.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [29, 16],
              [18, 22],
              [21, 26],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L'],
            ensembl_transcript_id: ['ENST00000327044'],
            variant_type: 'small',
            highest_af: 0.9277,
            highest_af_info: {
              source: 'gnomad V2 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 887560,
            hg19_strand: '+',
            hg19_end: 887560,
            entrez_gene_id: ['26155'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f998',
            variant_id: '21',
            end: 952421,
            ref: 'A',
            alt: 'G',
            start: 952421,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '10/19',
                codon_change: 'acT/acC',
                aa_change: 'T',
                aa_length: 394,
                allele: 'G',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000327044.7:c.1182T>C',
                hgvsp: 'ENSP00000317992.6:p.Thr394%3D',
                cds_position: 1182,
                existing_variation: 'rs3828047&COSV58989914',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                domains:
                  'Pfam:PF03715&PANTHER:PTHR12687&PANTHER:PTHR12687:SF10&Superfamily:SSF48371',
                pheno: '0&1',
                cadd_phred: 0.176,
                cadd_raw: -0.418421,
                spliceai_pred_dp_ag: 32,
                spliceai_pred_dp_al: 41,
                spliceai_pred_dp_dg: -11,
                spliceai_pred_dp_dl: -5,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: ['ENSG00000188976'],
            is_exonic: 1,
            is_coding: 1,
            aa_change: 'T',
            impact: 'synonymous_variant',
            hgvsc: 'ENST00000327044.7:c.1182T>C',
            hgvsp: 'ENSP00000317992.6:p.Thr394%3D',
            existing_variation: 'rs3828047&COSV58989914',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            domains:
              'Pfam:PF03715&PANTHER:PTHR12687&PANTHER:PTHR12687:SF10&Superfamily:SSF48371',
            gnomad_af: 0.9378,
            gnomad_afr_af: 0.9065,
            gnomad_amr_af: 0.9545,
            gnomad_asj_af: 0.9355,
            gnomad_eas_af: 0.911,
            gnomad_fin_af: 0.9606,
            gnomad_nfe_af: 0.9422,
            gnomad_oth_af: 0.9389,
            gnomad_sas_af: 0.9198,
            pheno: '0&1',
            cadd_phred: 0.176,
            cadd_raw: -0.418421,
            spliceai_pred_dp_ag: 32,
            spliceai_pred_dp_al: 41,
            spliceai_pred_dp_dg: -11,
            spliceai_pred_dp_dl: -5,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'NOC2L',
            constraint_mis_z: [-1.901],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147],
            exomiser_ad_exgenespheno: 0.5034,
            exomiser_xr_exgenescombi: 0.00031712,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.9093,
            gnomadv3_af_nfe: 0.9463,
            gnomadv3_af_xy: 0.9338,
            exomiser_xd_exgenescombi: 0.00031712,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9304,
            gnomadv3_af: 0.9341,
            exomiser_mt_exgenespheno: 0.5034,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00031712,
            gnomadv3_af_eas: 0.9242,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9274,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00031712,
            exomiser_ad_exgenescombi: 0.00031712,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9638,
            gnomadv3_af_fin: 0.9648,
            exomiser_ar_exgenespheno: 0.5034,
            gnomadv3_af_sas: 0.9215,
            gnomadv3_af_amr: 0.933,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.9344,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.9336,
            exomiser_xr_exgenespheno: 0.5034,
            exomiser_xd_exgenespheno: 0.5034,
            impact_severity: 'LOW',
            quality: 2558.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [28, 31],
              [21, 23],
              [27, 35],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L'],
            ensembl_transcript_id: ['ENST00000327044'],
            variant_type: 'small',
            highest_af: 0.9378,
            highest_af_info: {
              source: 'gnomad V2 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 887801,
            hg19_strand: '+',
            hg19_end: 887801,
            entrez_gene_id: ['26155'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f999',
            variant_id: '22',
            end: 953259,
            ref: 'T',
            alt: 'C',
            start: 953259,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '9/19',
                codon_change: 'gaA/gaG',
                aa_change: 'E',
                aa_length: 306,
                allele: 'C',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000327044.7:c.918A>G',
                hgvsp: 'ENSP00000317992.6:p.Glu306%3D',
                cds_position: 918,
                existing_variation: 'rs3748596&COSV58989925',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                domains:
                  'PANTHER:PTHR12687&PANTHER:PTHR12687:SF10&Superfamily:SSF48371',
                pheno: '0&1',
                cadd_phred: 3.936,
                cadd_raw: 0.26978,
                spliceai_pred_dp_ag: -1,
                spliceai_pred_dp_al: 49,
                spliceai_pred_dp_dg: 24,
                spliceai_pred_dp_dl: -2,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: ['ENSG00000188976'],
            is_exonic: 1,
            is_coding: 1,
            aa_change: 'E',
            impact: 'synonymous_variant',
            hgvsc: 'ENST00000327044.7:c.918A>G',
            hgvsp: 'ENSP00000317992.6:p.Glu306%3D',
            existing_variation: 'rs3748596&COSV58989925',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            domains:
              'PANTHER:PTHR12687&PANTHER:PTHR12687:SF10&Superfamily:SSF48371',
            gnomad_af: 0.9378,
            gnomad_afr_af: 0.9066,
            gnomad_amr_af: 0.9546,
            gnomad_asj_af: 0.9354,
            gnomad_eas_af: 0.9109,
            gnomad_fin_af: 0.9606,
            gnomad_nfe_af: 0.9423,
            gnomad_oth_af: 0.9386,
            gnomad_sas_af: 0.9198,
            pheno: '0&1',
            cadd_phred: 3.936,
            cadd_raw: 0.26978,
            spliceai_pred_dp_ag: -1,
            spliceai_pred_dp_al: 49,
            spliceai_pred_dp_dg: 24,
            spliceai_pred_dp_dl: -2,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'NOC2L',
            constraint_mis_z: [-1.901],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147],
            exomiser_ad_exgenespheno: 0.5034,
            exomiser_xr_exgenescombi: 0.00031712,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.9092,
            gnomadv3_af_nfe: 0.9464,
            gnomadv3_af_xy: 0.9339,
            exomiser_xd_exgenescombi: 0.00031712,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9304,
            gnomadv3_af: 0.9342,
            exomiser_mt_exgenespheno: 0.5034,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00031712,
            gnomadv3_af_eas: 0.9246,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9274,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00031712,
            exomiser_ad_exgenescombi: 0.00031712,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9638,
            gnomadv3_af_fin: 0.9647,
            exomiser_ar_exgenespheno: 0.5034,
            gnomadv3_af_sas: 0.9216,
            gnomadv3_af_amr: 0.9331,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.9344,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.934,
            exomiser_xr_exgenespheno: 0.5034,
            exomiser_xd_exgenespheno: 0.5034,
            impact_severity: 'LOW',
            quality: 1791.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [22, 22],
              [21, 21],
              [27, 22],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L'],
            ensembl_transcript_id: ['ENST00000327044'],
            variant_type: 'small',
            highest_af: 0.9378,
            highest_af_info: {
              source: 'gnomad V2 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 888639,
            hg19_strand: '+',
            hg19_end: 888639,
            entrez_gene_id: ['26155'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f99a',
            variant_id: '23',
            end: 953279,
            ref: 'T',
            alt: 'C',
            start: 953279,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '9/19',
                codon_change: 'Atc/Gtc',
                aa_change: 'I/V',
                aa_length: 300,
                polyphen_pred: 'benign',
                polyphen_score: 0,
                sift_pred: 'tolerated',
                sift_score: 1,
                allele: 'C',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000327044.7:c.898A>G',
                hgvsp: 'ENSP00000317992.6:p.Ile300Val',
                cds_position: 898,
                existing_variation: 'rs3748597&COSV58989936',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                domains:
                  'PANTHER:PTHR12687&PANTHER:PTHR12687:SF10&Superfamily:SSF48371',
                pheno: '0&1',
                revel: 0.127,
                cadd_phred: 3.893,
                cadd_raw: 0.266107,
                spliceai_pred_dp_ag: -22,
                spliceai_pred_dp_al: 7,
                spliceai_pred_dp_dg: 1,
                spliceai_pred_dp_dl: 4,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.03,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: ['ENSG00000188976'],
            is_exonic: 1,
            is_coding: 1,
            aa_change: 'I/V',
            polyphen_pred: ['benign'],
            polyphen_score: [0],
            sift_pred: ['tolerated'],
            sift_score: [1],
            impact: 'missense_variant',
            hgvsc: 'ENST00000327044.7:c.898A>G',
            hgvsp: 'ENSP00000317992.6:p.Ile300Val',
            existing_variation: 'rs3748597&COSV58989936',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            domains:
              'PANTHER:PTHR12687&PANTHER:PTHR12687:SF10&Superfamily:SSF48371',
            gnomad_af: 0.9378,
            gnomad_afr_af: 0.9066,
            gnomad_amr_af: 0.9546,
            gnomad_asj_af: 0.9355,
            gnomad_eas_af: 0.9109,
            gnomad_fin_af: 0.9606,
            gnomad_nfe_af: 0.9422,
            gnomad_oth_af: 0.9388,
            gnomad_sas_af: 0.9197,
            pheno: '0&1',
            revel: [0.127],
            cadd_phred: 3.893,
            cadd_raw: 0.266107,
            spliceai_pred_dp_ag: -22,
            spliceai_pred_dp_al: 7,
            spliceai_pred_dp_dg: 1,
            spliceai_pred_dp_dl: 4,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0.03],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'NOC2L',
            constraint_mis_z: [-1.901],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147],
            exomiser_ad_exgenespheno: 0.5034,
            exomiser_xr_exgenescombi: 0.00031712,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.9093,
            gnomadv3_af_nfe: 0.9464,
            gnomadv3_af_xy: 0.9338,
            exomiser_xd_exgenescombi: 0.00031712,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9304,
            gnomadv3_af: 0.9342,
            exomiser_mt_exgenespheno: 0.5034,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00031712,
            gnomadv3_af_eas: 0.9245,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9274,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00031712,
            exomiser_ad_exgenescombi: 0.00031712,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9638,
            gnomadv3_af_fin: 0.9647,
            exomiser_ar_exgenespheno: 0.5034,
            gnomadv3_af_sas: 0.9214,
            gnomadv3_af_amr: 0.9331,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.9345,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.9336,
            exomiser_xr_exgenespheno: 0.5034,
            exomiser_xd_exgenespheno: 0.5034,
            impact_severity: 'MED',
            quality: 1884.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [26, 23],
              [23, 23],
              [27, 22],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L'],
            ensembl_transcript_id: ['ENST00000327044'],
            variant_type: 'small',
            highest_af: 0.9378,
            highest_af_info: {
              source: 'gnomad V2 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 888659,
            hg19_strand: '+',
            hg19_end: 888659,
            entrez_gene_id: ['26155'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f99b',
            variant_id: '24',
            end: 953778,
            ref: 'G',
            alt: 'C',
            start: 953778,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: false,
                is_coding: false,
                is_splicing: true,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['splice_region_variant', 'intron_variant'],
                feature_type: 'Transcript',
                intron: '8/18',
                hgvsc: 'ENST00000327044.7:c.888+4C>G',
                existing_variation: 'rs13303056',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                cadd_phred: 0.003,
                cadd_raw: -1.147017,
                spliceai_pred_dp_ag: 21,
                spliceai_pred_dp_al: 24,
                spliceai_pred_dp_dg: 0,
                spliceai_pred_dp_dl: 4,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: 'splice_region_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: ['ENSG00000188976'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'splice_region_variant&intron_variant',
            hgvsc: 'ENST00000327044.7:c.888+4C>G',
            existing_variation: 'rs13303056',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            gnomad_af: 0.9321,
            gnomad_afr_af: 0.8235,
            gnomad_amr_af: 0.9508,
            gnomad_asj_af: 0.9351,
            gnomad_eas_af: 0.9215,
            gnomad_fin_af: 0.9599,
            gnomad_nfe_af: 0.9413,
            gnomad_oth_af: 0.9357,
            gnomad_sas_af: 0.9192,
            cadd_phred: 0.003,
            cadd_raw: -1.147017,
            spliceai_pred_dp_ag: 21,
            spliceai_pred_dp_al: 24,
            spliceai_pred_dp_dg: 0,
            spliceai_pred_dp_dl: 4,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'NOC2L',
            constraint_mis_z: [-1.901],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147],
            exomiser_ad_exgenespheno: 0.5034,
            exomiser_xr_exgenescombi: 0.00031712,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.8285,
            gnomadv3_af_nfe: 0.9461,
            gnomadv3_af_xy: 0.9119,
            exomiser_xd_exgenescombi: 0.00031712,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9236,
            gnomadv3_af: 0.9115,
            exomiser_mt_exgenespheno: 0.5034,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00031712,
            gnomadv3_af_eas: 0.9338,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9274,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00031712,
            exomiser_ad_exgenescombi: 0.00031712,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9627,
            gnomadv3_af_fin: 0.9647,
            exomiser_ar_exgenespheno: 0.5034,
            gnomadv3_af_sas: 0.9208,
            gnomadv3_af_amr: 0.9259,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.9111,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.9191,
            exomiser_xr_exgenespheno: 0.5034,
            exomiser_xd_exgenespheno: 0.5034,
            impact_severity: 'MED',
            quality: 3289.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [1, 1],
            ],
            allelic_depths: [
              [20, 17],
              [24, 22],
              [0, 40],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L'],
            ensembl_transcript_id: ['ENST00000327044'],
            variant_type: 'small',
            highest_af: 0.9338,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 889158,
            hg19_strand: '+',
            hg19_end: 889158,
            entrez_gene_id: ['26155'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f99c',
            variant_id: '25',
            end: 953779,
            ref: 'A',
            alt: 'C',
            start: 953779,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: false,
                is_coding: false,
                is_splicing: true,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['splice_region_variant', 'intron_variant'],
                feature_type: 'Transcript',
                intron: '8/18',
                hgvsc: 'ENST00000327044.7:c.888+3T>G',
                existing_variation: 'rs13302945',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                cadd_phred: 0.601,
                cadd_raw: -0.167013,
                spliceai_pred_dp_ag: 23,
                spliceai_pred_dp_al: 2,
                spliceai_pred_dp_dg: 3,
                spliceai_pred_dp_dl: 2,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: 'splice_region_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: ['ENSG00000188976'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'splice_region_variant&intron_variant',
            hgvsc: 'ENST00000327044.7:c.888+3T>G',
            existing_variation: 'rs13302945',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            gnomad_af: 0.9321,
            gnomad_afr_af: 0.8236,
            gnomad_amr_af: 0.9507,
            gnomad_asj_af: 0.9351,
            gnomad_eas_af: 0.9216,
            gnomad_fin_af: 0.9599,
            gnomad_nfe_af: 0.9413,
            gnomad_oth_af: 0.9357,
            gnomad_sas_af: 0.9192,
            cadd_phred: 0.601,
            cadd_raw: -0.167013,
            spliceai_pred_dp_ag: 23,
            spliceai_pred_dp_al: 2,
            spliceai_pred_dp_dg: 3,
            spliceai_pred_dp_dl: 2,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'NOC2L',
            constraint_mis_z: [-1.901],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147],
            exomiser_ad_exgenespheno: 0.5034,
            exomiser_xr_exgenescombi: 0.00031712,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.8284,
            gnomadv3_af_nfe: 0.9461,
            gnomadv3_af_xy: 0.9118,
            exomiser_xd_exgenescombi: 0.00031712,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9236,
            gnomadv3_af: 0.9114,
            exomiser_mt_exgenespheno: 0.5034,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00031712,
            gnomadv3_af_eas: 0.9339,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9274,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00031712,
            exomiser_ad_exgenescombi: 0.00031712,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9627,
            gnomadv3_af_fin: 0.9646,
            exomiser_ar_exgenespheno: 0.5034,
            gnomadv3_af_sas: 0.9208,
            gnomadv3_af_amr: 0.9259,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.9111,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.9191,
            exomiser_xr_exgenespheno: 0.5034,
            exomiser_xd_exgenespheno: 0.5034,
            impact_severity: 'MED',
            quality: 3289.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [1, 1],
            ],
            allelic_depths: [
              [20, 17],
              [25, 22],
              [0, 40],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L'],
            ensembl_transcript_id: ['ENST00000327044'],
            variant_type: 'small',
            highest_af: 0.9339,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 889159,
            hg19_strand: '+',
            hg19_end: 889159,
            entrez_gene_id: ['26155'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f99d',
            variant_id: '26',
            end: 953858,
            ref: 'G',
            alt: 'A',
            start: 953858,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '8/19',
                codon_change: 'gCg/gTg',
                aa_change: 'A/V',
                aa_length: 271,
                polyphen_pred: 'possibly_damaging',
                polyphen_score: 0.88,
                sift_pred: 'tolerated',
                sift_score: 0.06,
                allele: 'A',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000327044.7:c.812C>T',
                hgvsp: 'ENSP00000317992.6:p.Ala271Val',
                cds_position: 812,
                existing_variation: 'rs3828049&COSV58986908',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                domains:
                  'PANTHER:PTHR12687&PANTHER:PTHR12687:SF10&Superfamily:SSF48371',
                pheno: '0&1',
                revel: 0.101,
                cadd_phred: 20.7,
                cadd_raw: 2.179513,
                spliceai_pred_dp_ag: 13,
                spliceai_pred_dp_al: -2,
                spliceai_pred_dp_dg: 6,
                spliceai_pred_dp_dl: -28,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: ['ENSG00000188976'],
            is_exonic: 1,
            is_coding: 1,
            aa_change: 'A/V',
            polyphen_pred: ['possibly_damaging'],
            polyphen_score: [0.88],
            sift_pred: ['tolerated'],
            sift_score: [0.06],
            impact: 'missense_variant',
            hgvsc: 'ENST00000327044.7:c.812C>T',
            hgvsp: 'ENSP00000317992.6:p.Ala271Val',
            existing_variation: 'rs3828049&COSV58986908',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            domains:
              'PANTHER:PTHR12687&PANTHER:PTHR12687:SF10&Superfamily:SSF48371',
            gnomad_af: 0.05593,
            gnomad_afr_af: 0.02086,
            gnomad_amr_af: 0.04199,
            gnomad_asj_af: 0.06484,
            gnomad_eas_af: 0.08848,
            gnomad_fin_af: 0.03839,
            gnomad_nfe_af: 0.05699,
            gnomad_oth_af: 0.0559,
            gnomad_sas_af: 0.07619,
            pheno: '0&1',
            revel: [0.101],
            cadd_phred: 20.7,
            cadd_raw: 2.179513,
            spliceai_pred_dp_ag: 13,
            spliceai_pred_dp_al: -2,
            spliceai_pred_dp_dg: 6,
            spliceai_pred_dp_dl: -28,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'NOC2L',
            constraint_mis_z: [-1.901],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147],
            exomiser_ad_exgenespheno: 0.5034,
            exomiser_xr_exgenescombi: 0.00031712,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.0207,
            gnomadv3_af_nfe: 0.0528,
            gnomadv3_af_xy: 0.0461,
            exomiser_xd_exgenescombi: 0.00031712,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.0601,
            gnomadv3_af: 0.0451,
            exomiser_mt_exgenespheno: 0.5034,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00031712,
            gnomadv3_af_eas: 0.0747,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.0726,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00031712,
            exomiser_ad_exgenescombi: 0.00031712,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.0362,
            gnomadv3_af_fin: 0.0342,
            exomiser_ar_exgenespheno: 0.5034,
            gnomadv3_af_sas: 0.0755,
            gnomadv3_af_amr: 0.0589,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.0442,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.0468,
            exomiser_xr_exgenespheno: 0.5034,
            exomiser_xd_exgenespheno: 0.5034,
            impact_severity: 'MED',
            quality: 1973.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [22, 19],
              [16, 24],
              [30, 30],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L'],
            ensembl_transcript_id: ['ENST00000327044'],
            variant_type: 'small',
            highest_af: 0.08848,
            highest_af_info: {
              source: 'gnomad V2 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 889238,
            hg19_strand: '+',
            hg19_end: 889238,
            entrez_gene_id: ['26155'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f99e',
            variant_id: '27',
            end: 957365,
            ref: 'G',
            alt: 'A',
            start: 957365,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '2/18',
                hgvsc: 'ENST00000327044.7:c.180-92C>T',
                existing_variation: 'rs13303227',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                cadd_phred: 1.843,
                cadd_raw: 0.070894,
                spliceai_pred_dp_ag: -11,
                spliceai_pred_dp_al: 7,
                spliceai_pred_dp_dg: 5,
                spliceai_pred_dp_dl: 9,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.02,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000338591',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs13303227',
                strand: 1,
                hgnc_id: 'HGNC:24023',
                mane_select: 'NM_198317.3',
                ccds: 'CCDS30550.1',
                source: 'Ensembl',
                cadd_phred: 1.843,
                cadd_raw: 0.070894,
                spliceai_pred_dp_ag: -11,
                spliceai_pred_dp_al: 7,
                spliceai_pred_dp_dg: 5,
                spliceai_pred_dp_dl: 9,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.02,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -0.57094,
                constraint_oe_lof: 0.90749,
                constraint_oe_lof_lower: 0.664,
                constraint_oe_lof_upper: 1.259,
                constraint_oe_mis: 1.0772,
                constraint_oe_mis_lower: 0.998,
                constraint_oe_mis_upper: 1.163,
                constraint_oe_syn: 1.7903,
                constraint_oe_syn_lower: 1.641,
                constraint_oe_syn_upper: 1.939,
                constraint_pli: 1.3052e-16,
                constraint_syn_z: -8.809,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                p_li: 1.3052e-16,
                loeuf: 1.259,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: ['ENSG00000188976', 'ENSG00000187961'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant&upstream_gene_variant',
            hgvsc: 'ENST00000327044.7:c.180-92C>T',
            existing_variation: 'rs13303227',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            cadd_phred: 1.843,
            cadd_raw: 0.070894,
            spliceai_pred_dp_ag: -11,
            spliceai_pred_dp_al: 7,
            spliceai_pred_dp_dg: 5,
            spliceai_pred_dp_dl: 9,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0.02],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'NOC2L',
            constraint_mis_z: [-1.901, -0.57094],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327, 1.259],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338, 1.163],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147, -8.809],
            exomiser_ad_exgenespheno: 0.5034,
            exomiser_xr_exgenescombi: 0.00031712,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.9051,
            gnomadv3_af_nfe: 0.9464,
            gnomadv3_af_xy: 0.9328,
            exomiser_xd_exgenescombi: 0.00031712,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9304,
            gnomadv3_af: 0.933,
            exomiser_mt_exgenespheno: 0.5034,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00031712,
            gnomadv3_af_eas: 0.9246,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9274,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00031712,
            exomiser_ad_exgenescombi: 0.00031712,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9638,
            gnomadv3_af_fin: 0.9648,
            exomiser_ar_exgenespheno: 0.5034,
            gnomadv3_af_sas: 0.9218,
            gnomadv3_af_amr: 0.9328,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.9332,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.9335,
            exomiser_xr_exgenespheno: 0.5034,
            exomiser_xd_exgenespheno: 0.5034,
            impact_severity: 'LOW',
            quality: 2119.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [18, 29],
              [21, 26],
              [32, 18],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L', 'KLHL17'],
            ensembl_transcript_id: ['ENST00000327044', 'ENST00000338591'],
            variant_type: 'small',
            highest_af: 0.933,
            highest_af_info: {
              source: 'gnomad v3 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 892745,
            hg19_strand: '+',
            hg19_end: 892745,
            entrez_gene_id: ['26155', '339451'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f99f',
            variant_id: '28',
            end: 959193,
            ref: 'G',
            alt: 'A',
            start: 959193,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '1/18',
                hgvsc: 'ENST00000327044.7:c.26+22C>T',
                existing_variation: 'rs13303010',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                pheno: 1,
                cadd_phred: 5.591,
                cadd_raw: 0.414883,
                spliceai_pred_dp_ag: -34,
                spliceai_pred_dp_al: -36,
                spliceai_pred_dp_dg: 22,
                spliceai_pred_dp_dl: -14,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.06,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000338591',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs13303010',
                strand: 1,
                hgnc_id: 'HGNC:24023',
                mane_select: 'NM_198317.3',
                ccds: 'CCDS30550.1',
                source: 'Ensembl',
                pheno: 1,
                cadd_phred: 5.591,
                cadd_raw: 0.414883,
                spliceai_pred_dp_ag: -34,
                spliceai_pred_dp_al: -36,
                spliceai_pred_dp_dg: 22,
                spliceai_pred_dp_dl: -14,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.06,
                spliceai_pred_symbol: 'NOC2L',
                constraint_mis_z: -0.57094,
                constraint_oe_lof: 0.90749,
                constraint_oe_lof_lower: 0.664,
                constraint_oe_lof_upper: 1.259,
                constraint_oe_mis: 1.0772,
                constraint_oe_mis_lower: 0.998,
                constraint_oe_mis_upper: 1.163,
                constraint_oe_syn: 1.7903,
                constraint_oe_syn_lower: 1.641,
                constraint_oe_syn_upper: 1.939,
                constraint_pli: 1.3052e-16,
                constraint_syn_z: -8.809,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                p_li: 1.3052e-16,
                loeuf: 1.259,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000466300',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs13303010',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:24023',
                source: 'Ensembl',
                pheno: 1,
                cadd_phred: 5.591,
                cadd_raw: 0.414883,
                spliceai_pred_dp_ag: -34,
                spliceai_pred_dp_al: -36,
                spliceai_pred_dp_dg: 22,
                spliceai_pred_dp_dl: -14,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.06,
                spliceai_pred_symbol: 'NOC2L',
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: ['ENSG00000188976', 'ENSG00000187961'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant&upstream_gene_variant',
            hgvsc: 'ENST00000327044.7:c.26+22C>T',
            existing_variation: 'rs13303010',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            gnomad_af: 0.8198,
            gnomad_afr_af: 0.2474,
            gnomad_amr_af: 0.7665,
            gnomad_asj_af: 0.8776,
            gnomad_eas_af: 0.7319,
            gnomad_fin_af: 0.9013,
            gnomad_nfe_af: 0.8971,
            gnomad_oth_af: 0.8383,
            gnomad_sas_af: 0.8675,
            pheno: '1',
            cadd_phred: 5.591,
            cadd_raw: 0.414883,
            spliceai_pred_dp_ag: -34,
            spliceai_pred_dp_al: -36,
            spliceai_pred_dp_dg: 22,
            spliceai_pred_dp_dl: -14,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0.06],
            spliceai_pred_symbol: 'NOC2L',
            constraint_mis_z: [-1.901, -0.57094],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327, 1.259],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338, 1.163],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147, -8.809],
            exomiser_ad_exgenespheno: 0.5034,
            exomiser_xr_exgenescombi: 0.00031712,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.2655,
            gnomadv3_af_nfe: 0.9017,
            gnomadv3_af_xy: 0.7023,
            exomiser_xd_exgenescombi: 0.00031712,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.8165,
            gnomadv3_af: 0.7019,
            exomiser_mt_exgenespheno: 0.5034,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00031712,
            gnomadv3_af_eas: 0.7332,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.8755,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00031712,
            exomiser_ad_exgenescombi: 0.00031712,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.859,
            gnomadv3_af_fin: 0.9019,
            exomiser_ar_exgenespheno: 0.5034,
            gnomadv3_af_sas: 0.8654,
            gnomadv3_af_amr: 0.7344,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.7015,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.7541,
            exomiser_xr_exgenespheno: 0.5034,
            exomiser_xd_exgenespheno: 0.5034,
            impact_severity: 'LOW',
            quality: 2162.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [24, 23],
              [18, 26],
              [27, 27],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L', 'KLHL17'],
            ensembl_transcript_id: [
              'ENST00000327044',
              'ENST00000338591',
              'ENST00000466300',
            ],
            variant_type: 'small',
            highest_af: 0.8198,
            highest_af_info: {
              source: 'gnomad V2 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 894573,
            hg19_strand: '+',
            hg19_end: 894573,
            entrez_gene_id: ['26155', '339451'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9a0',
            variant_id: '29',
            end: 960891,
            ref: 'C',
            alt: 'T',
            start: 960891,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs112905931',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                cadd_phred: 8.836,
                cadd_raw: 0.741697,
                spliceai_pred_dp_ag: -12,
                spliceai_pred_dp_al: 18,
                spliceai_pred_dp_dg: 18,
                spliceai_pred_dp_dl: 30,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.04,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000338591',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'T',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '1/11',
                hgvsc: 'ENST00000338591.8:c.107+91C>T',
                existing_variation: 'rs112905931',
                strand: 1,
                hgnc_id: 'HGNC:24023',
                mane_select: 'NM_198317.3',
                ccds: 'CCDS30550.1',
                source: 'Ensembl',
                cadd_phred: 8.836,
                cadd_raw: 0.741697,
                spliceai_pred_dp_ag: -12,
                spliceai_pred_dp_al: 18,
                spliceai_pred_dp_dg: 18,
                spliceai_pred_dp_dl: 30,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.04,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -0.57094,
                constraint_oe_lof: 0.90749,
                constraint_oe_lof_lower: 0.664,
                constraint_oe_lof_upper: 1.259,
                constraint_oe_mis: 1.0772,
                constraint_oe_mis_lower: 0.998,
                constraint_oe_mis_upper: 1.163,
                constraint_oe_syn: 1.7903,
                constraint_oe_syn_lower: 1.641,
                constraint_oe_syn_upper: 1.939,
                constraint_pli: 1.3052e-16,
                constraint_syn_z: -8.809,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                p_li: 1.3052e-16,
                loeuf: 1.259,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000466300',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs112905931',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:24023',
                source: 'Ensembl',
                cadd_phred: 8.836,
                cadd_raw: 0.741697,
                spliceai_pred_dp_ag: -12,
                spliceai_pred_dp_al: 18,
                spliceai_pred_dp_dg: 18,
                spliceai_pred_dp_dl: 30,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.04,
                spliceai_pred_symbol: 'KLHL17',
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: ['ENSG00000188976', 'ENSG00000187961'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'upstream_gene_variant&intron_variant',
            existing_variation: 'rs112905931',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            cadd_phred: 8.836,
            cadd_raw: 0.741697,
            spliceai_pred_dp_ag: -12,
            spliceai_pred_dp_al: 18,
            spliceai_pred_dp_dg: 18,
            spliceai_pred_dp_dl: 30,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0.04],
            spliceai_pred_symbol: 'KLHL17',
            constraint_mis_z: [-1.901, -0.57094],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327, 1.259],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338, 1.163],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147, -8.809],
            exomiser_ad_exgenespheno: 0.5001,
            exomiser_xr_exgenescombi: 0.00030641,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.1194,
            gnomadv3_af_nfe: 0.6233,
            gnomadv3_af_xy: 0.4544,
            exomiser_xd_exgenescombi: 0.00030641,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.5355,
            gnomadv3_af: 0.4582,
            exomiser_mt_exgenespheno: 0.5001,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00030641,
            gnomadv3_af_eas: 0.3829,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.5407,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00030641,
            exomiser_ad_exgenescombi: 0.00030641,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.4867,
            gnomadv3_af_fin: 0.595,
            exomiser_ar_exgenespheno: 0.5001,
            gnomadv3_af_sas: 0.5334,
            gnomadv3_af_amr: 0.5309,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.4618,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.4908,
            exomiser_xr_exgenespheno: 0.5001,
            exomiser_xd_exgenespheno: 0.5001,
            impact_severity: 'LOW',
            quality: 983.1,
            genotypes_index: [
              [0, 1],
              [0, 0],
              [0, 1],
            ],
            allelic_depths: [
              [10, 20],
              [39, 0],
              [17, 19],
            ],
            genotype_qualities: [[99, 81, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L', 'KLHL17'],
            ensembl_transcript_id: [
              'ENST00000327044',
              'ENST00000338591',
              'ENST00000466300',
            ],
            variant_type: 'small',
            highest_af: 0.4582,
            highest_af_info: {
              source: 'gnomad v3 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 896271,
            hg19_strand: '+',
            hg19_end: 896271,
            entrez_gene_id: ['26155', '339451'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9a1',
            variant_id: '30',
            end: 961945,
            ref: 'G',
            alt: 'C',
            start: 961945,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970441&COSV58532012',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 1.293,
                cadd_raw: -0.006862,
                spliceai_pred_dp_ag: 12,
                spliceai_pred_dp_al: 36,
                spliceai_pred_dp_dg: -23,
                spliceai_pred_dp_dl: 35,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.01,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000338591',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '4/12',
                codon_change: 'gcG/gcC',
                aa_change: 'A',
                aa_length: 203,
                allele: 'C',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000338591.8:c.609G>C',
                hgvsp: 'ENSP00000343930.3:p.Ala203%3D',
                cds_position: 609,
                existing_variation: 'rs4970441&COSV58532012',
                strand: 1,
                hgnc_id: 'HGNC:24023',
                mane_select: 'NM_198317.3',
                ccds: 'CCDS30550.1',
                source: 'Ensembl',
                domains:
                  'PIRSF:PIRSF037037&SMART:SM00875&Gene3D:1.25.40.420&Pfam:PF07707&PANTHER:PTHR24412&PANTHER:PTHR24412:SF454&CDD:cd18456',
                pheno: '0&1',
                cadd_phred: 1.293,
                cadd_raw: -0.006862,
                spliceai_pred_dp_ag: 12,
                spliceai_pred_dp_al: 36,
                spliceai_pred_dp_dg: -23,
                spliceai_pred_dp_dl: 35,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.01,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -0.57094,
                constraint_oe_lof: 0.90749,
                constraint_oe_lof_lower: 0.664,
                constraint_oe_lof_upper: 1.259,
                constraint_oe_mis: 1.0772,
                constraint_oe_mis_lower: 0.998,
                constraint_oe_mis_upper: 1.163,
                constraint_oe_syn: 1.7903,
                constraint_oe_syn_lower: 1.641,
                constraint_oe_syn_upper: 1.939,
                constraint_pli: 1.3052e-16,
                constraint_syn_z: -8.809,
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                p_li: 1.3052e-16,
                loeuf: 1.259,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970441&COSV58532012',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 1.293,
                cadd_raw: -0.006862,
                spliceai_pred_dp_ag: 12,
                spliceai_pred_dp_al: 36,
                spliceai_pred_dp_dg: -23,
                spliceai_pred_dp_dl: 35,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.01,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970441&COSV58532012',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 1.293,
                cadd_raw: -0.006862,
                spliceai_pred_dp_ag: 12,
                spliceai_pred_dp_al: 36,
                spliceai_pred_dp_dg: -23,
                spliceai_pred_dp_dl: 35,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.01,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970441&COSV58532012',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 1.293,
                cadd_raw: -0.006862,
                spliceai_pred_dp_ag: 12,
                spliceai_pred_dp_al: 36,
                spliceai_pred_dp_dg: -23,
                spliceai_pred_dp_dl: 35,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.01,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000466300',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970441&COSV58532012',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:24023',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 1.293,
                cadd_raw: -0.006862,
                spliceai_pred_dp_ag: 12,
                spliceai_pred_dp_al: 36,
                spliceai_pred_dp_dg: -23,
                spliceai_pred_dp_dl: 35,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.01,
                spliceai_pred_symbol: 'KLHL17',
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: [
              'ENSG00000188976',
              'ENSG00000187961',
              'ENSG00000187583',
            ],
            is_exonic: 1,
            is_coding: 1,
            impact: 'upstream_gene_variant&synonymous_variant',
            existing_variation: 'rs4970441&COSV58532012',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            gnomad_af: 0.9127,
            gnomad_afr_af: 0.7143,
            gnomad_amr_af: 0.937,
            gnomad_asj_af: 0.9166,
            gnomad_eas_af: 0.9061,
            gnomad_fin_af: 0.9374,
            gnomad_nfe_af: 0.9286,
            gnomad_oth_af: 0.9134,
            gnomad_sas_af: 0.9168,
            pheno: '0&1',
            cadd_phred: 1.293,
            cadd_raw: -0.006862,
            spliceai_pred_dp_ag: 12,
            spliceai_pred_dp_al: 36,
            spliceai_pred_dp_dg: -23,
            spliceai_pred_dp_dl: 35,
            spliceai_pred_ds_ag: [0.01],
            spliceai_pred_ds_al: [0.01],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0.01],
            spliceai_pred_symbol: 'KLHL17',
            constraint_mis_z: [-1.901, -0.57094, -1.7988],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327, 1.259, 1.124],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338, 1.163, 1.358],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147, -8.809, -2.4681],
            exomiser_ad_exgenespheno: 0.5001,
            exomiser_xr_exgenescombi: 0.00030641,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.7168,
            gnomadv3_af_nfe: 0.9335,
            gnomadv3_af_xy: 0.8706,
            exomiser_xd_exgenescombi: 0.00030641,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9082,
            gnomadv3_af: 0.8693,
            exomiser_mt_exgenespheno: 0.5001,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00030641,
            gnomadv3_af_eas: 0.9206,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9122,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00030641,
            exomiser_ad_exgenescombi: 0.00030641,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.8855,
            gnomadv3_af_fin: 0.9404,
            exomiser_ar_exgenespheno: 0.5001,
            gnomadv3_af_sas: 0.92,
            gnomadv3_af_amr: 0.9003,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.8681,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.8865,
            exomiser_xr_exgenespheno: 0.5001,
            exomiser_xd_exgenespheno: 0.5001,
            impact_severity: 'LOW',
            quality: 2063.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [24, 25],
              [15, 28],
              [32, 19],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L', 'KLHL17', 'PLEKHN1'],
            ensembl_transcript_id: [
              'ENST00000327044',
              'ENST00000338591',
              'ENST00000379410',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000466300',
            ],
            variant_type: 'small',
            highest_af: 0.9206,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 897325,
            hg19_strand: '+',
            hg19_end: 897325,
            entrez_gene_id: ['26155', '339451', '84069'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9a2',
            variant_id: '31',
            end: 962358,
            ref: 'C',
            alt: 'T',
            start: 962358,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6696971&COSV58532909',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 8.302,
                cadd_raw: 0.683433,
                spliceai_pred_dp_ag: 17,
                spliceai_pred_dp_al: -44,
                spliceai_pred_dp_dg: -26,
                spliceai_pred_dp_dl: 47,
                spliceai_pred_ds_ag: 0.12,
                spliceai_pred_ds_al: 0.02,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000338591',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '5/12',
                codon_change: 'Ctg/Ttg',
                aa_change: 'L',
                aa_length: 239,
                allele: 'T',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000338591.8:c.715C>T',
                hgvsp: 'ENSP00000343930.3:p.Leu239%3D',
                cds_position: 715,
                existing_variation: 'rs6696971&COSV58532909',
                strand: 1,
                hgnc_id: 'HGNC:24023',
                mane_select: 'NM_198317.3',
                ccds: 'CCDS30550.1',
                source: 'Ensembl',
                domains:
                  'PIRSF:PIRSF037037&SMART:SM00875&Gene3D:1.25.40.420&Pfam:PF07707&PANTHER:PTHR24412&PANTHER:PTHR24412:SF454&CDD:cd18456',
                pheno: '0&1',
                cadd_phred: 8.302,
                cadd_raw: 0.683433,
                spliceai_pred_dp_ag: 17,
                spliceai_pred_dp_al: -44,
                spliceai_pred_dp_dg: -26,
                spliceai_pred_dp_dl: 47,
                spliceai_pred_ds_ag: 0.12,
                spliceai_pred_ds_al: 0.02,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -0.57094,
                constraint_oe_lof: 0.90749,
                constraint_oe_lof_lower: 0.664,
                constraint_oe_lof_upper: 1.259,
                constraint_oe_mis: 1.0772,
                constraint_oe_mis_lower: 0.998,
                constraint_oe_mis_upper: 1.163,
                constraint_oe_syn: 1.7903,
                constraint_oe_syn_lower: 1.641,
                constraint_oe_syn_upper: 1.939,
                constraint_pli: 1.3052e-16,
                constraint_syn_z: -8.809,
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                p_li: 1.3052e-16,
                loeuf: 1.259,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6696971&COSV58532909',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 8.302,
                cadd_raw: 0.683433,
                spliceai_pred_dp_ag: 17,
                spliceai_pred_dp_al: -44,
                spliceai_pred_dp_dg: -26,
                spliceai_pred_dp_dl: 47,
                spliceai_pred_ds_ag: 0.12,
                spliceai_pred_ds_al: 0.02,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6696971&COSV58532909',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 8.302,
                cadd_raw: 0.683433,
                spliceai_pred_dp_ag: 17,
                spliceai_pred_dp_al: -44,
                spliceai_pred_dp_dg: -26,
                spliceai_pred_dp_dl: 47,
                spliceai_pred_ds_ag: 0.12,
                spliceai_pred_ds_al: 0.02,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6696971&COSV58532909',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 8.302,
                cadd_raw: 0.683433,
                spliceai_pred_dp_ag: 17,
                spliceai_pred_dp_al: -44,
                spliceai_pred_dp_dg: -26,
                spliceai_pred_dp_dl: 47,
                spliceai_pred_ds_ag: 0.12,
                spliceai_pred_ds_al: 0.02,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000466300',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6696971&COSV58532909',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:24023',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 8.302,
                cadd_raw: 0.683433,
                spliceai_pred_dp_ag: 17,
                spliceai_pred_dp_al: -44,
                spliceai_pred_dp_dg: -26,
                spliceai_pred_dp_dl: 47,
                spliceai_pred_ds_ag: 0.12,
                spliceai_pred_ds_al: 0.02,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: [
              'ENSG00000188976',
              'ENSG00000187961',
              'ENSG00000187583',
            ],
            is_exonic: 1,
            is_coding: 1,
            impact: 'upstream_gene_variant&synonymous_variant',
            existing_variation: 'rs6696971&COSV58532909',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            gnomad_af: 0.07713,
            gnomad_afr_af: 0.1868,
            gnomad_amr_af: 0.05395,
            gnomad_asj_af: 0.07939,
            gnomad_eas_af: 0.08371,
            gnomad_fin_af: 0.06242,
            gnomad_nfe_af: 0.06885,
            gnomad_oth_af: 0.07661,
            gnomad_sas_af: 0.08169,
            pheno: '0&1',
            cadd_phred: 8.302,
            cadd_raw: 0.683433,
            spliceai_pred_dp_ag: 17,
            spliceai_pred_dp_al: -44,
            spliceai_pred_dp_dg: -26,
            spliceai_pred_dp_dl: 47,
            spliceai_pred_ds_ag: [0.12],
            spliceai_pred_ds_al: [0.02],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'KLHL17',
            constraint_mis_z: [-1.901, -0.57094, -1.7988],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327, 1.259, 1.124],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338, 1.163, 1.358],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147, -8.809, -2.4681],
            exomiser_ad_exgenespheno: 0.5001,
            exomiser_xr_exgenescombi: 0.00030641,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.183,
            gnomadv3_af_nfe: 0.0639,
            gnomadv3_af_xy: 0.0999,
            exomiser_xd_exgenescombi: 0.00030641,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.0886,
            gnomadv3_af: 0.0999,
            exomiser_mt_exgenespheno: 0.5001,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00030641,
            gnomadv3_af_eas: 0.0733,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.0822,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00030641,
            exomiser_ad_exgenescombi: 0.00030641,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.1184,
            gnomadv3_af_fin: 0.0592,
            exomiser_ar_exgenespheno: 0.5001,
            gnomadv3_af_sas: 0.0787,
            gnomadv3_af_amr: 0.084,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.0999,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.0874,
            exomiser_xr_exgenespheno: 0.5001,
            exomiser_xd_exgenespheno: 0.5001,
            impact_severity: 'LOW',
            quality: 1898.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [15, 17],
              [21, 28],
              [14, 21],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L', 'KLHL17', 'PLEKHN1'],
            ensembl_transcript_id: [
              'ENST00000327044',
              'ENST00000338591',
              'ENST00000379410',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000466300',
            ],
            variant_type: 'small',
            highest_af: 0.0999,
            highest_af_info: {
              source: 'gnomad v3 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 897738,
            hg19_strand: '+',
            hg19_end: 897738,
            entrez_gene_id: ['26155', '339451', '84069'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9a3',
            variant_id: '32',
            end: 962943,
            ref: 'T',
            alt: 'C',
            start: 962943,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6605071',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                cadd_phred: 3.195,
                cadd_raw: 0.206239,
                spliceai_pred_dp_ag: -26,
                spliceai_pred_dp_al: 28,
                spliceai_pred_dp_dg: -26,
                spliceai_pred_dp_dl: -30,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.01,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000338591',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '6/11',
                hgvsc: 'ENST00000338591.8:c.1042+26T>C',
                existing_variation: 'rs6605071',
                strand: 1,
                hgnc_id: 'HGNC:24023',
                mane_select: 'NM_198317.3',
                ccds: 'CCDS30550.1',
                source: 'Ensembl',
                cadd_phred: 3.195,
                cadd_raw: 0.206239,
                spliceai_pred_dp_ag: -26,
                spliceai_pred_dp_al: 28,
                spliceai_pred_dp_dg: -26,
                spliceai_pred_dp_dl: -30,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.01,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -0.57094,
                constraint_oe_lof: 0.90749,
                constraint_oe_lof_lower: 0.664,
                constraint_oe_lof_upper: 1.259,
                constraint_oe_mis: 1.0772,
                constraint_oe_mis_lower: 0.998,
                constraint_oe_mis_upper: 1.163,
                constraint_oe_syn: 1.7903,
                constraint_oe_syn_lower: 1.641,
                constraint_oe_syn_upper: 1.939,
                constraint_pli: 1.3052e-16,
                constraint_syn_z: -8.809,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                p_li: 1.3052e-16,
                loeuf: 1.259,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6605071',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                cadd_phred: 3.195,
                cadd_raw: 0.206239,
                spliceai_pred_dp_ag: -26,
                spliceai_pred_dp_al: 28,
                spliceai_pred_dp_dg: -26,
                spliceai_pred_dp_dl: -30,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.01,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6605071',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                cadd_phred: 3.195,
                cadd_raw: 0.206239,
                spliceai_pred_dp_ag: -26,
                spliceai_pred_dp_al: 28,
                spliceai_pred_dp_dg: -26,
                spliceai_pred_dp_dl: -30,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.01,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6605071',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 3.195,
                cadd_raw: 0.206239,
                spliceai_pred_dp_ag: -26,
                spliceai_pred_dp_al: 28,
                spliceai_pred_dp_dg: -26,
                spliceai_pred_dp_dl: -30,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.01,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000466300',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['intron_variant', 'NMD_transcript_variant'],
                feature_type: 'Transcript',
                intron: '1/5',
                hgvsc: 'ENST00000466300.1:c.191+26T>C',
                existing_variation: 'rs6605071',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:24023',
                source: 'Ensembl',
                cadd_phred: 3.195,
                cadd_raw: 0.206239,
                spliceai_pred_dp_ag: -26,
                spliceai_pred_dp_al: 28,
                spliceai_pred_dp_dg: -26,
                spliceai_pred_dp_dl: -30,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.01,
                spliceai_pred_symbol: 'KLHL17',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: [
              'ENSG00000188976',
              'ENSG00000187961',
              'ENSG00000187583',
            ],
            is_exonic: 0,
            is_coding: 0,
            impact: 'upstream_gene_variant&intron_variant',
            existing_variation: 'rs6605071',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            gnomad_af: 0.94,
            gnomad_afr_af: 0.9218,
            gnomad_amr_af: 0.9554,
            gnomad_asj_af: 0.9621,
            gnomad_eas_af: 0.9126,
            gnomad_fin_af: 0.9589,
            gnomad_nfe_af: 0.9418,
            gnomad_oth_af: 0.9393,
            gnomad_sas_af: 0.9298,
            cadd_phred: 3.195,
            cadd_raw: 0.206239,
            spliceai_pred_dp_ag: -26,
            spliceai_pred_dp_al: 28,
            spliceai_pred_dp_dg: -26,
            spliceai_pred_dp_dl: -30,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0.01],
            spliceai_pred_symbol: 'KLHL17',
            constraint_mis_z: [-1.901, -0.57094, -1.7988],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327, 1.259, 1.124],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338, 1.163, 1.358],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147, -8.809, -2.4681],
            exomiser_ad_exgenespheno: 0.5001,
            exomiser_xr_exgenescombi: 0.00030641,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.9219,
            gnomadv3_af_nfe: 0.9437,
            gnomadv3_af_xy: 0.9359,
            exomiser_xd_exgenescombi: 0.00030641,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9494,
            gnomadv3_af: 0.9371,
            exomiser_mt_exgenespheno: 0.5001,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00030641,
            gnomadv3_af_eas: 0.9241,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9608,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00030641,
            exomiser_ad_exgenescombi: 0.00030641,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9638,
            gnomadv3_af_fin: 0.965,
            exomiser_ar_exgenespheno: 0.5001,
            gnomadv3_af_sas: 0.9292,
            gnomadv3_af_amr: 0.9291,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.9383,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.9388,
            exomiser_xr_exgenespheno: 0.5001,
            exomiser_xd_exgenespheno: 0.5001,
            impact_severity: 'LOW',
            quality: 2053.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [29, 22],
              [27, 32],
              [26, 22],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L', 'KLHL17', 'PLEKHN1'],
            ensembl_transcript_id: [
              'ENST00000327044',
              'ENST00000338591',
              'ENST00000379410',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000466300',
            ],
            variant_type: 'small',
            highest_af: 0.94,
            highest_af_info: {
              source: 'gnomad V2 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 898323,
            hg19_strand: '+',
            hg19_end: 898323,
            entrez_gene_id: ['26155', '339451', '84069'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9a4',
            variant_id: '33',
            end: 963087,
            ref: 'C',
            alt: 'T',
            start: 963087,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'NOC2L',
                ensembl_gene_id: 'ENSG00000188976',
                transcript: 'ENST00000327044',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs41285808&COSV58531429',
                strand: -1,
                hgnc_id: 'HGNC:24517',
                mane_select: 'NM_015658.4',
                ccds: 'CCDS3.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.066,
                cadd_raw: -0.62037,
                spliceai_pred_dp_ag: 22,
                spliceai_pred_dp_al: -43,
                spliceai_pred_dp_dg: 32,
                spliceai_pred_dp_dl: -35,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.901,
                constraint_oe_lof: 1.0291,
                constraint_oe_lof_lower: 0.805,
                constraint_oe_lof_upper: 1.327,
                constraint_oe_mis: 1.2489,
                constraint_oe_mis_lower: 1.166,
                constraint_oe_mis_upper: 1.338,
                constraint_oe_syn: 1.6049,
                constraint_oe_syn_lower: 1.464,
                constraint_oe_syn_upper: 1.761,
                constraint_pli: 2.8933e-29,
                constraint_syn_z: -6.7147,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'NOC2L',
                entrez_gene_id: '26155',
                p_haplo: 0.08547,
                p_triplo: 0.21461,
                p_li: 2.8933e-29,
                loeuf: 1.327,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000338591',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'T',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '6/11',
                hgvsc: 'ENST00000338591.8:c.1043-22C>T',
                existing_variation: 'rs41285808&COSV58531429',
                strand: 1,
                hgnc_id: 'HGNC:24023',
                mane_select: 'NM_198317.3',
                ccds: 'CCDS30550.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.066,
                cadd_raw: -0.62037,
                spliceai_pred_dp_ag: 22,
                spliceai_pred_dp_al: -43,
                spliceai_pred_dp_dg: 32,
                spliceai_pred_dp_dl: -35,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -0.57094,
                constraint_oe_lof: 0.90749,
                constraint_oe_lof_lower: 0.664,
                constraint_oe_lof_upper: 1.259,
                constraint_oe_mis: 1.0772,
                constraint_oe_mis_lower: 0.998,
                constraint_oe_mis_upper: 1.163,
                constraint_oe_syn: 1.7903,
                constraint_oe_syn_lower: 1.641,
                constraint_oe_syn_upper: 1.939,
                constraint_pli: 1.3052e-16,
                constraint_syn_z: -8.809,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                p_li: 1.3052e-16,
                loeuf: 1.259,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs41285808&COSV58531429',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.066,
                cadd_raw: -0.62037,
                spliceai_pred_dp_ag: 22,
                spliceai_pred_dp_al: -43,
                spliceai_pred_dp_dg: 32,
                spliceai_pred_dp_dl: -35,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs41285808&COSV58531429',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.066,
                cadd_raw: -0.62037,
                spliceai_pred_dp_ag: 22,
                spliceai_pred_dp_al: -43,
                spliceai_pred_dp_dg: 32,
                spliceai_pred_dp_dl: -35,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs41285808&COSV58531429',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.066,
                cadd_raw: -0.62037,
                spliceai_pred_dp_ag: 22,
                spliceai_pred_dp_al: -43,
                spliceai_pred_dp_dg: 32,
                spliceai_pred_dp_dl: -35,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000466300',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '2/6',
                codon_change: 'ccC/ccT',
                aa_change: 'P',
                aa_length: 83,
                allele: 'T',
                impact: ['synonymous_variant', 'NMD_transcript_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000466300.1:c.249C>T',
                hgvsp: 'ENSP00000463694.1:p.Pro83%3D',
                cds_position: 249,
                existing_variation: 'rs41285808&COSV58531429',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:24023',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.066,
                cadd_raw: -0.62037,
                spliceai_pred_dp_ag: 22,
                spliceai_pred_dp_al: -43,
                spliceai_pred_dp_dg: 32,
                spliceai_pred_dp_dl: -35,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000327044',
            ensembl_gene_id: [
              'ENSG00000188976',
              'ENSG00000187961',
              'ENSG00000187583',
            ],
            is_exonic: 1,
            is_coding: 1,
            impact: 'upstream_gene_variant&intron_variant',
            existing_variation: 'rs41285808&COSV58531429',
            ccds: 'CCDS3.1',
            source: 'Ensembl',
            gnomad_af: 0.0528,
            gnomad_afr_af: 0.008742,
            gnomad_amr_af: 0.0384,
            gnomad_asj_af: 0.06105,
            gnomad_eas_af: 0.08703,
            gnomad_fin_af: 0.03796,
            gnomad_nfe_af: 0.05546,
            gnomad_oth_af: 0.05352,
            gnomad_sas_af: 0.06866,
            pheno: '0&1',
            cadd_phred: 0.066,
            cadd_raw: -0.62037,
            spliceai_pred_dp_ag: 22,
            spliceai_pred_dp_al: -43,
            spliceai_pred_dp_dg: 32,
            spliceai_pred_dp_dl: -35,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'KLHL17',
            constraint_mis_z: [-1.901, -0.57094, -1.7988],
            constraint_oe_lof: 1.0291,
            constraint_oe_lof_lower: 0.805,
            constraint_oe_lof_upper: [1.327, 1.259, 1.124],
            constraint_oe_mis: 1.2489,
            constraint_oe_mis_lower: 1.166,
            constraint_oe_mis_upper: [1.338, 1.163, 1.358],
            constraint_oe_syn: 1.6049,
            constraint_oe_syn_lower: 1.464,
            constraint_oe_syn_upper: 1.761,
            constraint_pli: 2.8933e-29,
            constraint_syn_z: [-6.7147, -8.809, -2.4681],
            exomiser_ad_exgenespheno: 0.5001,
            exomiser_xr_exgenescombi: 0.00030641,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.010397,
            gnomadv3_af_nfe: 0.0524,
            gnomadv3_af_xy: 0.0427,
            exomiser_xd_exgenescombi: 0.00030641,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.0573,
            gnomadv3_af: 0.0416,
            exomiser_mt_exgenespheno: 0.5001,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00030641,
            gnomadv3_af_eas: 0.0751,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.0703,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00030641,
            exomiser_ad_exgenescombi: 0.00030641,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.017582,
            gnomadv3_af_fin: 0.034,
            exomiser_ar_exgenespheno: 0.5001,
            gnomadv3_af_sas: 0.0681,
            gnomadv3_af_amr: 0.058,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.0406,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.0464,
            exomiser_xr_exgenespheno: 0.5001,
            exomiser_xd_exgenespheno: 0.5001,
            impact_severity: 'LOW',
            quality: 2384.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [34, 28],
              [24, 25],
              [27, 33],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['NOC2L', 'KLHL17', 'PLEKHN1'],
            ensembl_transcript_id: [
              'ENST00000327044',
              'ENST00000338591',
              'ENST00000379410',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000466300',
            ],
            variant_type: 'small',
            highest_af: 0.08703,
            highest_af_info: {
              source: 'gnomad V2 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 898467,
            hg19_strand: '+',
            hg19_end: 898467,
            entrez_gene_id: ['26155', '339451', '84069'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9a5',
            variant_id: '34',
            end: 964548,
            ref: 'G',
            alt: 'C',
            start: 964548,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000338591',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '11/11',
                hgvsc: 'ENST00000338591.8:c.1700+18G>C',
                existing_variation: 'rs6677386',
                strand: 1,
                hgnc_id: 'HGNC:24023',
                mane_select: 'NM_198317.3',
                ccds: 'CCDS30550.1',
                source: 'Ensembl',
                cadd_phred: 0.682,
                cadd_raw: -0.140987,
                spliceai_pred_dp_ag: -17,
                spliceai_pred_dp_al: -41,
                spliceai_pred_dp_dg: 15,
                spliceai_pred_dp_dl: -28,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.01,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -0.57094,
                constraint_oe_lof: 0.90749,
                constraint_oe_lof_lower: 0.664,
                constraint_oe_lof_upper: 1.259,
                constraint_oe_mis: 1.0772,
                constraint_oe_mis_lower: 0.998,
                constraint_oe_mis_upper: 1.163,
                constraint_oe_syn: 1.7903,
                constraint_oe_syn_lower: 1.641,
                constraint_oe_syn_upper: 1.939,
                constraint_pli: 1.3052e-16,
                constraint_syn_z: -8.809,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                p_li: 1.3052e-16,
                loeuf: 1.259,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6677386',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                cadd_phred: 0.682,
                cadd_raw: -0.140987,
                spliceai_pred_dp_ag: -17,
                spliceai_pred_dp_al: -41,
                spliceai_pred_dp_dg: 15,
                spliceai_pred_dp_dl: -28,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.01,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6677386',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                cadd_phred: 0.682,
                cadd_raw: -0.140987,
                spliceai_pred_dp_ag: -17,
                spliceai_pred_dp_al: -41,
                spliceai_pred_dp_dg: 15,
                spliceai_pred_dp_dl: -28,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.01,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6677386',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 0.682,
                cadd_raw: -0.140987,
                spliceai_pred_dp_ag: -17,
                spliceai_pred_dp_al: -41,
                spliceai_pred_dp_dg: 15,
                spliceai_pred_dp_dl: -28,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.01,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000466300',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6677386',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:24023',
                source: 'Ensembl',
                cadd_phred: 0.682,
                cadd_raw: -0.140987,
                spliceai_pred_dp_ag: -17,
                spliceai_pred_dp_al: -41,
                spliceai_pred_dp_dg: 15,
                spliceai_pred_dp_dl: -28,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0.01,
                spliceai_pred_symbol: 'KLHL17',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000338591',
            ensembl_gene_id: ['ENSG00000187961', 'ENSG00000187583'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant&upstream_gene_variant',
            hgvsc: 'ENST00000338591.8:c.1700+18G>C',
            existing_variation: 'rs6677386',
            ccds: 'CCDS30550.1',
            source: 'Ensembl',
            gnomad_af: 0.8984,
            gnomad_afr_af: 0.5296,
            gnomad_amr_af: 0.9279,
            gnomad_asj_af: 0.9169,
            gnomad_eas_af: 0.9089,
            gnomad_fin_af: 0.9205,
            gnomad_nfe_af: 0.9272,
            gnomad_oth_af: 0.903,
            gnomad_sas_af: 0.9193,
            cadd_phred: 0.682,
            cadd_raw: -0.140987,
            spliceai_pred_dp_ag: -17,
            spliceai_pred_dp_al: -41,
            spliceai_pred_dp_dg: 15,
            spliceai_pred_dp_dl: -28,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0.01],
            spliceai_pred_symbol: 'KLHL17',
            constraint_mis_z: [-0.57094, -1.7988],
            constraint_oe_lof: 0.90749,
            constraint_oe_lof_lower: 0.664,
            constraint_oe_lof_upper: [1.259, 1.124],
            constraint_oe_mis: 1.0772,
            constraint_oe_mis_lower: 0.998,
            constraint_oe_mis_upper: [1.163, 1.358],
            constraint_oe_syn: 1.7903,
            constraint_oe_syn_lower: 1.641,
            constraint_oe_syn_upper: 1.939,
            constraint_pli: 1.3052e-16,
            constraint_syn_z: [-8.809, -2.4681],
            exomiser_ad_exgenespheno: 0.5001,
            exomiser_xr_exgenescombi: 0.00030641,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.5727,
            gnomadv3_af_nfe: 0.9124,
            gnomadv3_af_xy: 0.8188,
            exomiser_xd_exgenescombi: 0.00030641,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.8941,
            gnomadv3_af: 0.8175,
            exomiser_mt_exgenespheno: 0.5001,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00030641,
            gnomadv3_af_eas: 0.9189,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.8825,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00030641,
            exomiser_ad_exgenescombi: 0.00030641,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.846,
            gnomadv3_af_fin: 0.9226,
            exomiser_ar_exgenespheno: 0.5001,
            gnomadv3_af_sas: 0.9032,
            gnomadv3_af_amr: 0.8769,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.8162,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.8475,
            exomiser_xr_exgenespheno: 0.5001,
            exomiser_xd_exgenespheno: 0.5001,
            impact_severity: 'LOW',
            quality: 1820.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [23, 17],
              [14, 11],
              [13, 20],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['KLHL17', 'PLEKHN1'],
            ensembl_transcript_id: [
              'ENST00000338591',
              'ENST00000379410',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000466300',
            ],
            variant_type: 'small',
            highest_af: 0.9189,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 899928,
            hg19_strand: '+',
            hg19_end: 899928,
            entrez_gene_id: ['339451', '84069'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9a6',
            variant_id: '35',
            end: 964557,
            ref: 'G',
            alt: 'T',
            start: 964557,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000338591',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'T',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '11/11',
                hgvsc: 'ENST00000338591.8:c.1700+27G>T',
                existing_variation: 'rs143296006',
                strand: 1,
                hgnc_id: 'HGNC:24023',
                mane_select: 'NM_198317.3',
                ccds: 'CCDS30550.1',
                source: 'Ensembl',
                cadd_phred: 2.689,
                cadd_raw: 0.160796,
                spliceai_pred_dp_ag: -26,
                spliceai_pred_dp_al: -45,
                spliceai_pred_dp_dg: -2,
                spliceai_pred_dp_dl: -27,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.01,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -0.57094,
                constraint_oe_lof: 0.90749,
                constraint_oe_lof_lower: 0.664,
                constraint_oe_lof_upper: 1.259,
                constraint_oe_mis: 1.0772,
                constraint_oe_mis_lower: 0.998,
                constraint_oe_mis_upper: 1.163,
                constraint_oe_syn: 1.7903,
                constraint_oe_syn_lower: 1.641,
                constraint_oe_syn_upper: 1.939,
                constraint_pli: 1.3052e-16,
                constraint_syn_z: -8.809,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                p_li: 1.3052e-16,
                loeuf: 1.259,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs143296006',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                cadd_phred: 2.689,
                cadd_raw: 0.160796,
                spliceai_pred_dp_ag: -26,
                spliceai_pred_dp_al: -45,
                spliceai_pred_dp_dg: -2,
                spliceai_pred_dp_dl: -27,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.01,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs143296006',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                cadd_phred: 2.689,
                cadd_raw: 0.160796,
                spliceai_pred_dp_ag: -26,
                spliceai_pred_dp_al: -45,
                spliceai_pred_dp_dg: -2,
                spliceai_pred_dp_dl: -27,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.01,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs143296006',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 2.689,
                cadd_raw: 0.160796,
                spliceai_pred_dp_ag: -26,
                spliceai_pred_dp_al: -45,
                spliceai_pred_dp_dg: -2,
                spliceai_pred_dp_dl: -27,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.01,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000466300',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs143296006',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:24023',
                source: 'Ensembl',
                cadd_phred: 2.689,
                cadd_raw: 0.160796,
                spliceai_pred_dp_ag: -26,
                spliceai_pred_dp_al: -45,
                spliceai_pred_dp_dg: -2,
                spliceai_pred_dp_dl: -27,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.01,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000338591',
            ensembl_gene_id: ['ENSG00000187961', 'ENSG00000187583'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant&upstream_gene_variant',
            hgvsc: 'ENST00000338591.8:c.1700+27G>T',
            existing_variation: 'rs143296006',
            ccds: 'CCDS30550.1',
            source: 'Ensembl',
            gnomad_af: 0.8955,
            gnomad_afr_af: 0.5296,
            gnomad_amr_af: 0.9253,
            gnomad_asj_af: 0.9154,
            gnomad_eas_af: 0.9088,
            gnomad_fin_af: 0.9196,
            gnomad_nfe_af: 0.9249,
            gnomad_oth_af: 0.8996,
            gnomad_sas_af: 0.9172,
            cadd_phred: 2.689,
            cadd_raw: 0.160796,
            spliceai_pred_dp_ag: -26,
            spliceai_pred_dp_al: -45,
            spliceai_pred_dp_dg: -2,
            spliceai_pred_dp_dl: -27,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0.01],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'KLHL17',
            constraint_mis_z: [-0.57094, -1.7988],
            constraint_oe_lof: 0.90749,
            constraint_oe_lof_lower: 0.664,
            constraint_oe_lof_upper: [1.259, 1.124],
            constraint_oe_mis: 1.0772,
            constraint_oe_mis_lower: 0.998,
            constraint_oe_mis_upper: [1.163, 1.358],
            constraint_oe_syn: 1.7903,
            constraint_oe_syn_lower: 1.641,
            constraint_oe_syn_upper: 1.939,
            constraint_pli: 1.3052e-16,
            constraint_syn_z: [-8.809, -2.4681],
            exomiser_ad_exgenespheno: 0.5001,
            exomiser_xr_exgenescombi: 0.00030641,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.5827,
            gnomadv3_af_nfe: 0.9127,
            gnomadv3_af_xy: 0.8233,
            exomiser_xd_exgenescombi: 0.00030641,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.8988,
            gnomadv3_af: 0.8227,
            exomiser_mt_exgenespheno: 0.5001,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00030641,
            gnomadv3_af_eas: 0.9187,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.8844,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00030641,
            exomiser_ad_exgenescombi: 0.00030641,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.847,
            gnomadv3_af_fin: 0.924,
            exomiser_ar_exgenespheno: 0.5001,
            gnomadv3_af_sas: 0.9049,
            gnomadv3_af_amr: 0.8799,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.8222,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.8458,
            exomiser_xr_exgenespheno: 0.5001,
            exomiser_xd_exgenespheno: 0.5001,
            impact_severity: 'LOW',
            quality: 1859.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [20, 17],
              [14, 11],
              [13, 19],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['KLHL17', 'PLEKHN1'],
            ensembl_transcript_id: [
              'ENST00000338591',
              'ENST00000379410',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000466300',
            ],
            variant_type: 'small',
            highest_af: 0.9187,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 899937,
            hg19_strand: '+',
            hg19_end: 899937,
            entrez_gene_id: ['339451', '84069'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9a7',
            variant_id: '36',
            end: 964558,
            ref: 'G',
            alt: 'C',
            start: 964558,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000338591',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '11/11',
                hgvsc: 'ENST00000338591.8:c.1700+28G>C',
                existing_variation: 'rs147467971',
                strand: 1,
                hgnc_id: 'HGNC:24023',
                mane_select: 'NM_198317.3',
                ccds: 'CCDS30550.1',
                source: 'Ensembl',
                cadd_phred: 1.869,
                cadd_raw: 0.074133,
                spliceai_pred_dp_ag: -27,
                spliceai_pred_dp_al: -50,
                spliceai_pred_dp_dg: 5,
                spliceai_pred_dp_dl: -38,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -0.57094,
                constraint_oe_lof: 0.90749,
                constraint_oe_lof_lower: 0.664,
                constraint_oe_lof_upper: 1.259,
                constraint_oe_mis: 1.0772,
                constraint_oe_mis_lower: 0.998,
                constraint_oe_mis_upper: 1.163,
                constraint_oe_syn: 1.7903,
                constraint_oe_syn_lower: 1.641,
                constraint_oe_syn_upper: 1.939,
                constraint_pli: 1.3052e-16,
                constraint_syn_z: -8.809,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                p_li: 1.3052e-16,
                loeuf: 1.259,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs147467971',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                cadd_phred: 1.869,
                cadd_raw: 0.074133,
                spliceai_pred_dp_ag: -27,
                spliceai_pred_dp_al: -50,
                spliceai_pred_dp_dg: 5,
                spliceai_pred_dp_dl: -38,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs147467971',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                cadd_phred: 1.869,
                cadd_raw: 0.074133,
                spliceai_pred_dp_ag: -27,
                spliceai_pred_dp_al: -50,
                spliceai_pred_dp_dg: 5,
                spliceai_pred_dp_dl: -38,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs147467971',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 1.869,
                cadd_raw: 0.074133,
                spliceai_pred_dp_ag: -27,
                spliceai_pred_dp_al: -50,
                spliceai_pred_dp_dg: 5,
                spliceai_pred_dp_dl: -38,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000466300',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs147467971',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:24023',
                source: 'Ensembl',
                cadd_phred: 1.869,
                cadd_raw: 0.074133,
                spliceai_pred_dp_ag: -27,
                spliceai_pred_dp_al: -50,
                spliceai_pred_dp_dg: 5,
                spliceai_pred_dp_dl: -38,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000338591',
            ensembl_gene_id: ['ENSG00000187961', 'ENSG00000187583'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant&upstream_gene_variant',
            hgvsc: 'ENST00000338591.8:c.1700+28G>C',
            existing_variation: 'rs147467971',
            ccds: 'CCDS30550.1',
            source: 'Ensembl',
            gnomad_af: 0.8955,
            gnomad_afr_af: 0.5305,
            gnomad_amr_af: 0.9252,
            gnomad_asj_af: 0.9153,
            gnomad_eas_af: 0.9091,
            gnomad_fin_af: 0.9189,
            gnomad_nfe_af: 0.9249,
            gnomad_oth_af: 0.899,
            gnomad_sas_af: 0.9174,
            cadd_phred: 1.869,
            cadd_raw: 0.074133,
            spliceai_pred_dp_ag: -27,
            spliceai_pred_dp_al: -50,
            spliceai_pred_dp_dg: 5,
            spliceai_pred_dp_dl: -38,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'KLHL17',
            constraint_mis_z: [-0.57094, -1.7988],
            constraint_oe_lof: 0.90749,
            constraint_oe_lof_lower: 0.664,
            constraint_oe_lof_upper: [1.259, 1.124],
            constraint_oe_mis: 1.0772,
            constraint_oe_mis_lower: 0.998,
            constraint_oe_mis_upper: [1.163, 1.358],
            constraint_oe_syn: 1.7903,
            constraint_oe_syn_lower: 1.641,
            constraint_oe_syn_upper: 1.939,
            constraint_pli: 1.3052e-16,
            constraint_syn_z: [-8.809, -2.4681],
            exomiser_ad_exgenespheno: 0.5001,
            exomiser_xr_exgenescombi: 0.00030641,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.5889,
            gnomadv3_af_nfe: 0.9122,
            gnomadv3_af_xy: 0.8254,
            exomiser_xd_exgenescombi: 0.00030641,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.8988,
            gnomadv3_af: 0.8248,
            exomiser_mt_exgenespheno: 0.5001,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00030641,
            gnomadv3_af_eas: 0.9187,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.8844,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00030641,
            exomiser_ad_exgenescombi: 0.00030641,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.8605,
            gnomadv3_af_fin: 0.9243,
            exomiser_ar_exgenespheno: 0.5001,
            gnomadv3_af_sas: 0.9043,
            gnomadv3_af_amr: 0.8798,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.8241,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.8461,
            exomiser_xr_exgenespheno: 0.5001,
            exomiser_xd_exgenespheno: 0.5001,
            impact_severity: 'LOW',
            quality: 1862.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [20, 17],
              [14, 11],
              [13, 19],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['KLHL17', 'PLEKHN1'],
            ensembl_transcript_id: [
              'ENST00000338591',
              'ENST00000379410',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000466300',
            ],
            variant_type: 'small',
            highest_af: 0.9187,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 899938,
            hg19_strand: '+',
            hg19_end: 899938,
            entrez_gene_id: ['339451', '84069'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9a8',
            variant_id: '37',
            end: 964562,
            ref: 'G',
            alt: 'A',
            start: 964562,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000338591',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '11/11',
                hgvsc: 'ENST00000338591.8:c.1700+32G>A',
                existing_variation: 'rs71509448',
                strand: 1,
                hgnc_id: 'HGNC:24023',
                mane_select: 'NM_198317.3',
                ccds: 'CCDS30550.1',
                source: 'Ensembl',
                cadd_phred: 3.076,
                cadd_raw: 0.195848,
                spliceai_pred_dp_ag: -31,
                spliceai_pred_dp_al: 42,
                spliceai_pred_dp_dg: 1,
                spliceai_pred_dp_dl: -32,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.06,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -0.57094,
                constraint_oe_lof: 0.90749,
                constraint_oe_lof_lower: 0.664,
                constraint_oe_lof_upper: 1.259,
                constraint_oe_mis: 1.0772,
                constraint_oe_mis_lower: 0.998,
                constraint_oe_mis_upper: 1.163,
                constraint_oe_syn: 1.7903,
                constraint_oe_syn_lower: 1.641,
                constraint_oe_syn_upper: 1.939,
                constraint_pli: 1.3052e-16,
                constraint_syn_z: -8.809,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                p_li: 1.3052e-16,
                loeuf: 1.259,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509448',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                cadd_phred: 3.076,
                cadd_raw: 0.195848,
                spliceai_pred_dp_ag: -31,
                spliceai_pred_dp_al: 42,
                spliceai_pred_dp_dg: 1,
                spliceai_pred_dp_dl: -32,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.06,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509448',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                cadd_phred: 3.076,
                cadd_raw: 0.195848,
                spliceai_pred_dp_ag: -31,
                spliceai_pred_dp_al: 42,
                spliceai_pred_dp_dg: 1,
                spliceai_pred_dp_dl: -32,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.06,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509448',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 3.076,
                cadd_raw: 0.195848,
                spliceai_pred_dp_ag: -31,
                spliceai_pred_dp_al: 42,
                spliceai_pred_dp_dg: 1,
                spliceai_pred_dp_dl: -32,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.06,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000466300',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs71509448',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:24023',
                source: 'Ensembl',
                cadd_phred: 3.076,
                cadd_raw: 0.195848,
                spliceai_pred_dp_ag: -31,
                spliceai_pred_dp_al: 42,
                spliceai_pred_dp_dg: 1,
                spliceai_pred_dp_dl: -32,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.06,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000338591',
            ensembl_gene_id: ['ENSG00000187961', 'ENSG00000187583'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant&upstream_gene_variant',
            hgvsc: 'ENST00000338591.8:c.1700+32G>A',
            existing_variation: 'rs71509448',
            ccds: 'CCDS30550.1',
            source: 'Ensembl',
            gnomad_af: 0.9021,
            gnomad_afr_af: 0.5447,
            gnomad_amr_af: 0.9289,
            gnomad_asj_af: 0.9214,
            gnomad_eas_af: 0.9102,
            gnomad_fin_af: 0.9208,
            gnomad_nfe_af: 0.9294,
            gnomad_oth_af: 0.9067,
            gnomad_sas_af: 0.9212,
            cadd_phred: 3.076,
            cadd_raw: 0.195848,
            spliceai_pred_dp_ag: -31,
            spliceai_pred_dp_al: 42,
            spliceai_pred_dp_dg: 1,
            spliceai_pred_dp_dl: -32,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0.06],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'KLHL17',
            constraint_mis_z: [-0.57094, -1.7988],
            constraint_oe_lof: 0.90749,
            constraint_oe_lof_lower: 0.664,
            constraint_oe_lof_upper: [1.259, 1.124],
            constraint_oe_mis: 1.0772,
            constraint_oe_mis_lower: 0.998,
            constraint_oe_mis_upper: [1.163, 1.358],
            constraint_oe_syn: 1.7903,
            constraint_oe_syn_lower: 1.641,
            constraint_oe_syn_upper: 1.939,
            constraint_pli: 1.3052e-16,
            constraint_syn_z: [-8.809, -2.4681],
            exomiser_ad_exgenespheno: 0.5001,
            exomiser_xr_exgenescombi: 0.00030641,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.6022,
            gnomadv3_af_nfe: 0.914,
            gnomadv3_af_xy: 0.8298,
            exomiser_xd_exgenescombi: 0.00030641,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9,
            gnomadv3_af: 0.8285,
            exomiser_mt_exgenespheno: 0.5001,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00030641,
            gnomadv3_af_eas: 0.9188,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.8868,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00030641,
            exomiser_ad_exgenescombi: 0.00030641,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.875,
            gnomadv3_af_fin: 0.927,
            exomiser_ar_exgenespheno: 0.5001,
            gnomadv3_af_sas: 0.9065,
            gnomadv3_af_amr: 0.8824,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.8272,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.8539,
            exomiser_xr_exgenespheno: 0.5001,
            exomiser_xd_exgenespheno: 0.5001,
            impact_severity: 'LOW',
            quality: 1830.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [19, 17],
              [13, 12],
              [12, 18],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['KLHL17', 'PLEKHN1'],
            ensembl_transcript_id: [
              'ENST00000338591',
              'ENST00000379410',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000466300',
            ],
            variant_type: 'small',
            highest_af: 0.9188,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 899942,
            hg19_strand: '+',
            hg19_end: 899942,
            entrez_gene_id: ['339451', '84069'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9a9',
            variant_id: '38',
            end: 964609,
            ref: 'A',
            alt: 'C',
            start: 964609,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000338591',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '11/11',
                hgvsc: 'ENST00000338591.8:c.1700+79A>C',
                existing_variation: 'rs6669071&COSV58531691',
                strand: 1,
                hgnc_id: 'HGNC:24023',
                mane_select: 'NM_198317.3',
                ccds: 'CCDS30550.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 3.568,
                cadd_raw: 0.238368,
                spliceai_pred_dp_ag: -5,
                spliceai_pred_dp_al: 24,
                spliceai_pred_dp_dg: -22,
                spliceai_pred_dp_dl: 24,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -0.57094,
                constraint_oe_lof: 0.90749,
                constraint_oe_lof_lower: 0.664,
                constraint_oe_lof_upper: 1.259,
                constraint_oe_mis: 1.0772,
                constraint_oe_mis_lower: 0.998,
                constraint_oe_mis_upper: 1.163,
                constraint_oe_syn: 1.7903,
                constraint_oe_syn_lower: 1.641,
                constraint_oe_syn_upper: 1.939,
                constraint_pli: 1.3052e-16,
                constraint_syn_z: -8.809,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                p_li: 1.3052e-16,
                loeuf: 1.259,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6669071&COSV58531691',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 3.568,
                cadd_raw: 0.238368,
                spliceai_pred_dp_ag: -5,
                spliceai_pred_dp_al: 24,
                spliceai_pred_dp_dg: -22,
                spliceai_pred_dp_dl: 24,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6669071&COSV58531691',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 3.568,
                cadd_raw: 0.238368,
                spliceai_pred_dp_ag: -5,
                spliceai_pred_dp_al: 24,
                spliceai_pred_dp_dg: -22,
                spliceai_pred_dp_dl: 24,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6669071&COSV58531691',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 3.568,
                cadd_raw: 0.238368,
                spliceai_pred_dp_ag: -5,
                spliceai_pred_dp_al: 24,
                spliceai_pred_dp_dg: -22,
                spliceai_pred_dp_dl: 24,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000466300',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6669071&COSV58531691',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:24023',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 3.568,
                cadd_raw: 0.238368,
                spliceai_pred_dp_ag: -5,
                spliceai_pred_dp_al: 24,
                spliceai_pred_dp_dg: -22,
                spliceai_pred_dp_dl: 24,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000338591',
            ensembl_gene_id: ['ENSG00000187961', 'ENSG00000187583'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant&upstream_gene_variant',
            hgvsc: 'ENST00000338591.8:c.1700+79A>C',
            existing_variation: 'rs6669071&COSV58531691',
            ccds: 'CCDS30550.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            pheno: '0&1',
            cadd_phred: 3.568,
            cadd_raw: 0.238368,
            spliceai_pred_dp_ag: -5,
            spliceai_pred_dp_al: 24,
            spliceai_pred_dp_dg: -22,
            spliceai_pred_dp_dl: 24,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'KLHL17',
            constraint_mis_z: [-0.57094, -1.7988],
            constraint_oe_lof: 0.90749,
            constraint_oe_lof_lower: 0.664,
            constraint_oe_lof_upper: [1.259, 1.124],
            constraint_oe_mis: 1.0772,
            constraint_oe_mis_lower: 0.998,
            constraint_oe_mis_upper: [1.163, 1.358],
            constraint_oe_syn: 1.7903,
            constraint_oe_syn_lower: 1.641,
            constraint_oe_syn_upper: 1.939,
            constraint_pli: 1.3052e-16,
            constraint_syn_z: [-8.809, -2.4681],
            exomiser_ad_exgenespheno: 0.5001,
            exomiser_xr_exgenescombi: 0.00030641,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.7449,
            gnomadv3_af_nfe: 0.9434,
            gnomadv3_af_xy: 0.9081,
            exomiser_xd_exgenescombi: 0.00030641,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.934,
            gnomadv3_af: 0.9106,
            exomiser_mt_exgenespheno: 0.5001,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00030641,
            gnomadv3_af_eas: 0.9322,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9174,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00030641,
            exomiser_ad_exgenescombi: 0.00030641,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.925,
            gnomadv3_af_fin: 0.9517,
            exomiser_ar_exgenespheno: 0.5001,
            gnomadv3_af_sas: 0.9323,
            gnomadv3_af_amr: 0.917,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.9128,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.9256,
            exomiser_xr_exgenespheno: 0.5001,
            exomiser_xd_exgenespheno: 0.5001,
            impact_severity: 'LOW',
            quality: 1315.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [9, 12],
              [6, 11],
              [5, 8],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['KLHL17', 'PLEKHN1'],
            ensembl_transcript_id: [
              'ENST00000338591',
              'ENST00000379410',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000466300',
            ],
            variant_type: 'small',
            highest_af: 0.9322,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 899989,
            hg19_strand: '+',
            hg19_end: 899989,
            entrez_gene_id: ['339451', '84069'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9aa',
            variant_id: '39',
            end: 964621,
            ref: 'G',
            alt: 'A',
            start: 964621,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000338591',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '11/11',
                hgvsc: 'ENST00000338591.8:c.1700+91G>A',
                existing_variation: 'rs28754834',
                strand: 1,
                hgnc_id: 'HGNC:24023',
                mane_select: 'NM_198317.3',
                ccds: 'CCDS30550.1',
                source: 'Ensembl',
                cadd_phred: 7.032,
                cadd_raw: 0.551349,
                spliceai_pred_dp_ag: 12,
                spliceai_pred_dp_al: 21,
                spliceai_pred_dp_dg: 12,
                spliceai_pred_dp_dl: -34,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -0.57094,
                constraint_oe_lof: 0.90749,
                constraint_oe_lof_lower: 0.664,
                constraint_oe_lof_upper: 1.259,
                constraint_oe_mis: 1.0772,
                constraint_oe_mis_lower: 0.998,
                constraint_oe_mis_upper: 1.163,
                constraint_oe_syn: 1.7903,
                constraint_oe_syn_lower: 1.641,
                constraint_oe_syn_upper: 1.939,
                constraint_pli: 1.3052e-16,
                constraint_syn_z: -8.809,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                p_li: 1.3052e-16,
                loeuf: 1.259,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs28754834',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                cadd_phred: 7.032,
                cadd_raw: 0.551349,
                spliceai_pred_dp_ag: 12,
                spliceai_pred_dp_al: 21,
                spliceai_pred_dp_dg: 12,
                spliceai_pred_dp_dl: -34,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs28754834',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                cadd_phred: 7.032,
                cadd_raw: 0.551349,
                spliceai_pred_dp_ag: 12,
                spliceai_pred_dp_al: 21,
                spliceai_pred_dp_dg: 12,
                spliceai_pred_dp_dl: -34,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs28754834',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 7.032,
                cadd_raw: 0.551349,
                spliceai_pred_dp_ag: 12,
                spliceai_pred_dp_al: 21,
                spliceai_pred_dp_dg: 12,
                spliceai_pred_dp_dl: -34,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000466300',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs28754834',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:24023',
                source: 'Ensembl',
                cadd_phred: 7.032,
                cadd_raw: 0.551349,
                spliceai_pred_dp_ag: 12,
                spliceai_pred_dp_al: 21,
                spliceai_pred_dp_dg: 12,
                spliceai_pred_dp_dl: -34,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000338591',
            ensembl_gene_id: ['ENSG00000187961', 'ENSG00000187583'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant&upstream_gene_variant',
            hgvsc: 'ENST00000338591.8:c.1700+91G>A',
            existing_variation: 'rs28754834',
            ccds: 'CCDS30550.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            cadd_phred: 7.032,
            cadd_raw: 0.551349,
            spliceai_pred_dp_ag: 12,
            spliceai_pred_dp_al: 21,
            spliceai_pred_dp_dg: 12,
            spliceai_pred_dp_dl: -34,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'KLHL17',
            constraint_mis_z: [-0.57094, -1.7988],
            constraint_oe_lof: 0.90749,
            constraint_oe_lof_lower: 0.664,
            constraint_oe_lof_upper: [1.259, 1.124],
            constraint_oe_mis: 1.0772,
            constraint_oe_mis_lower: 0.998,
            constraint_oe_mis_upper: [1.163, 1.358],
            constraint_oe_syn: 1.7903,
            constraint_oe_syn_lower: 1.641,
            constraint_oe_syn_upper: 1.939,
            constraint_pli: 1.3052e-16,
            constraint_syn_z: [-8.809, -2.4681],
            exomiser_ad_exgenespheno: 0.5001,
            exomiser_xr_exgenescombi: 0.00030641,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.7518,
            gnomadv3_af_nfe: 0.9263,
            gnomadv3_af_xy: 0.9012,
            exomiser_xd_exgenescombi: 0.00030641,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.8511,
            gnomadv3_af: 0.9028,
            exomiser_mt_exgenespheno: 0.5001,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00030641,
            gnomadv3_af_eas: 0.9278,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.8861,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00030641,
            exomiser_ad_exgenescombi: 0.00030641,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.91,
            gnomadv3_af_fin: 0.9448,
            exomiser_ar_exgenespheno: 0.5001,
            gnomadv3_af_sas: 0.9076,
            gnomadv3_af_amr: 0.9103,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.9041,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.9,
            exomiser_xr_exgenespheno: 0.5001,
            exomiser_xd_exgenespheno: 0.5001,
            impact_severity: 'LOW',
            quality: 988.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [7, 8],
              [5, 11],
              [4, 5],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['KLHL17', 'PLEKHN1'],
            ensembl_transcript_id: [
              'ENST00000338591',
              'ENST00000379410',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000466300',
            ],
            variant_type: 'small',
            highest_af: 0.9278,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 900001,
            hg19_strand: '+',
            hg19_end: 900001,
            entrez_gene_id: ['339451', '84069'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9ab',
            variant_id: '40',
            end: 964660,
            ref: 'GGGGGCGCGGGTCCGCAGTGGGGATGTGCTGCCGGGA',
            alt: 'G',
            start: 964624,
            chrom: 'chr1',
            type: 'indel',
            gene_objs: [
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000338591',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '11/11',
                hgvsc: 'ENST00000338591.8:c.1700+99_1700+134del',
                existing_variation: 'rs1557635896',
                strand: 1,
                hgnc_id: 'HGNC:24023',
                mane_select: 'NM_198317.3',
                ccds: 'CCDS30550.1',
                source: 'Ensembl',
                constraint_mis_z: -0.57094,
                constraint_oe_lof: 0.90749,
                constraint_oe_lof_lower: 0.664,
                constraint_oe_lof_upper: 1.259,
                constraint_oe_mis: 1.0772,
                constraint_oe_mis_lower: 0.998,
                constraint_oe_mis_upper: 1.163,
                constraint_oe_syn: 1.7903,
                constraint_oe_syn_lower: 1.641,
                constraint_oe_syn_upper: 1.939,
                constraint_pli: 1.3052e-16,
                constraint_syn_z: -8.809,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                p_li: 1.3052e-16,
                loeuf: 1.259,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: '-',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs1557635896',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs1557635896',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs1557635896',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000466300',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs1557635896',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:24023',
                source: 'Ensembl',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000338591',
            ensembl_gene_id: ['ENSG00000187961', 'ENSG00000187583'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant&upstream_gene_variant',
            hgvsc: 'ENST00000338591.8:c.1700+99_1700+134del',
            existing_variation: 'rs1557635896',
            ccds: 'CCDS30550.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            constraint_mis_z: [-0.57094, -1.7988],
            constraint_oe_lof: 0.90749,
            constraint_oe_lof_lower: 0.664,
            constraint_oe_lof_upper: [1.259, 1.124],
            constraint_oe_mis: 1.0772,
            constraint_oe_mis_lower: 0.998,
            constraint_oe_mis_upper: [1.163, 1.358],
            constraint_oe_syn: 1.7903,
            constraint_oe_syn_lower: 1.641,
            constraint_oe_syn_upper: 1.939,
            constraint_pli: 1.3052e-16,
            constraint_syn_z: [-8.809, -2.4681],
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.002664,
            gnomadv3_af_nfe: 0.0323,
            gnomadv3_af_xy: 0.014849,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.0333,
            gnomadv3_af: 0.018283,
            clndisdb: 'None',
            gnomadv3_af_eas: 0.0259,
            gnomadv3_af_asj: 0.012887,
            clnsigincl: 'None',
            clnvi: 'None',
            clnsig: 'None',
            gnomadv3_af_ami: 0.0714,
            gnomadv3_af_fin: 0.0464,
            gnomadv3_af_sas: 0.026,
            gnomadv3_af_amr: 0.0090498,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            gnomadv3_af_xx: 0.0214,
            gnomadv3_af_oth: 0.0326,
            impact_severity: 'LOW',
            quality: 270.3,
            genotypes_index: [
              [0, 0],
              [0, 0],
              [0, 1],
            ],
            allelic_depths: [
              [14, 0],
              [16, 0],
              [4, 6],
            ],
            genotype_qualities: [[33, 45, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['KLHL17', 'PLEKHN1'],
            ensembl_transcript_id: [
              'ENST00000338591',
              'ENST00000379410',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000466300',
            ],
            variant_type: 'small',
            highest_af: 0.0259,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 900004,
            hg19_strand: '+',
            hg19_end: 900040,
            entrez_gene_id: ['339451', '84069'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9ac',
            variant_id: '41',
            end: 964905,
            ref: 'C',
            alt: 'T',
            start: 964905,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000338591',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'T',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '11/11',
                hgvsc: 'ENST00000338591.8:c.1701-58C>T',
                existing_variation: 'rs4970435',
                strand: 1,
                hgnc_id: 'HGNC:24023',
                mane_select: 'NM_198317.3',
                ccds: 'CCDS30550.1',
                source: 'Ensembl',
                cadd_phred: 1.981,
                cadd_raw: 0.087466,
                spliceai_pred_dp_ag: 30,
                spliceai_pred_dp_al: 3,
                spliceai_pred_dp_dg: -28,
                spliceai_pred_dp_dl: -4,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0.06,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -0.57094,
                constraint_oe_lof: 0.90749,
                constraint_oe_lof_lower: 0.664,
                constraint_oe_lof_upper: 1.259,
                constraint_oe_mis: 1.0772,
                constraint_oe_mis_lower: 0.998,
                constraint_oe_mis_upper: 1.163,
                constraint_oe_syn: 1.7903,
                constraint_oe_syn_lower: 1.641,
                constraint_oe_syn_upper: 1.939,
                constraint_pli: 1.3052e-16,
                constraint_syn_z: -8.809,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                p_li: 1.3052e-16,
                loeuf: 1.259,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970435',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                cadd_phred: 1.981,
                cadd_raw: 0.087466,
                spliceai_pred_dp_ag: 30,
                spliceai_pred_dp_al: 3,
                spliceai_pred_dp_dg: -28,
                spliceai_pred_dp_dl: -4,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0.06,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970435',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                cadd_phred: 1.981,
                cadd_raw: 0.087466,
                spliceai_pred_dp_ag: 30,
                spliceai_pred_dp_al: 3,
                spliceai_pred_dp_dg: -28,
                spliceai_pred_dp_dl: -4,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0.06,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970435',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 1.981,
                cadd_raw: 0.087466,
                spliceai_pred_dp_ag: 30,
                spliceai_pred_dp_al: 3,
                spliceai_pred_dp_dg: -28,
                spliceai_pred_dp_dl: -4,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0.06,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000466300',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970435',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:24023',
                source: 'Ensembl',
                cadd_phred: 1.981,
                cadd_raw: 0.087466,
                spliceai_pred_dp_ag: 30,
                spliceai_pred_dp_al: 3,
                spliceai_pred_dp_dg: -28,
                spliceai_pred_dp_dl: -4,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0.06,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000338591',
            ensembl_gene_id: ['ENSG00000187961', 'ENSG00000187583'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant&upstream_gene_variant',
            hgvsc: 'ENST00000338591.8:c.1701-58C>T',
            existing_variation: 'rs4970435',
            ccds: 'CCDS30550.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            cadd_phred: 1.981,
            cadd_raw: 0.087466,
            spliceai_pred_dp_ag: 30,
            spliceai_pred_dp_al: 3,
            spliceai_pred_dp_dg: -28,
            spliceai_pred_dp_dl: -4,
            spliceai_pred_ds_ag: [0.01],
            spliceai_pred_ds_al: [0.06],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'KLHL17',
            constraint_mis_z: [-0.57094, -1.7988],
            constraint_oe_lof: 0.90749,
            constraint_oe_lof_lower: 0.664,
            constraint_oe_lof_upper: [1.259, 1.124],
            constraint_oe_mis: 1.0772,
            constraint_oe_mis_lower: 0.998,
            constraint_oe_mis_upper: [1.163, 1.358],
            constraint_oe_syn: 1.7903,
            constraint_oe_syn_lower: 1.641,
            constraint_oe_syn_upper: 1.939,
            constraint_pli: 1.3052e-16,
            constraint_syn_z: [-8.809, -2.4681],
            exomiser_ad_exgenespheno: 0.5001,
            exomiser_xr_exgenescombi: 0.00030641,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.5846,
            gnomadv3_af_nfe: 0.9492,
            gnomadv3_af_xy: 0.8438,
            exomiser_xd_exgenescombi: 0.00030641,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9296,
            gnomadv3_af: 0.8425,
            exomiser_mt_exgenespheno: 0.5001,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00030641,
            gnomadv3_af_eas: 0.9279,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9354,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00030641,
            exomiser_ad_exgenescombi: 0.00030641,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.931,
            gnomadv3_af_fin: 0.9632,
            exomiser_ar_exgenespheno: 0.5001,
            gnomadv3_af_sas: 0.932,
            gnomadv3_af_amr: 0.9049,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.8412,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.8771,
            exomiser_xr_exgenespheno: 0.5001,
            exomiser_xd_exgenespheno: 0.5001,
            impact_severity: 'LOW',
            quality: 2838.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [19, 25],
              [11, 27],
              [34, 20],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['KLHL17', 'PLEKHN1'],
            ensembl_transcript_id: [
              'ENST00000338591',
              'ENST00000379410',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000466300',
            ],
            variant_type: 'small',
            highest_af: 0.9279,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 900285,
            hg19_strand: '+',
            hg19_end: 900285,
            entrez_gene_id: ['339451', '84069'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9ad',
            variant_id: '42',
            end: 964906,
            ref: 'A',
            alt: 'G',
            start: 964906,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000338591',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '11/11',
                hgvsc: 'ENST00000338591.8:c.1701-57A>G',
                existing_variation: 'rs4970434',
                strand: 1,
                hgnc_id: 'HGNC:24023',
                mane_select: 'NM_198317.3',
                ccds: 'CCDS30550.1',
                source: 'Ensembl',
                cadd_phred: 0.369,
                cadd_raw: -0.265581,
                spliceai_pred_dp_ag: 29,
                spliceai_pred_dp_al: 2,
                spliceai_pred_dp_dg: -5,
                spliceai_pred_dp_dl: -29,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0.19,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -0.57094,
                constraint_oe_lof: 0.90749,
                constraint_oe_lof_lower: 0.664,
                constraint_oe_lof_upper: 1.259,
                constraint_oe_mis: 1.0772,
                constraint_oe_mis_lower: 0.998,
                constraint_oe_mis_upper: 1.163,
                constraint_oe_syn: 1.7903,
                constraint_oe_syn_lower: 1.641,
                constraint_oe_syn_upper: 1.939,
                constraint_pli: 1.3052e-16,
                constraint_syn_z: -8.809,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                p_li: 1.3052e-16,
                loeuf: 1.259,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970434',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                cadd_phred: 0.369,
                cadd_raw: -0.265581,
                spliceai_pred_dp_ag: 29,
                spliceai_pred_dp_al: 2,
                spliceai_pred_dp_dg: -5,
                spliceai_pred_dp_dl: -29,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0.19,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970434',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                cadd_phred: 0.369,
                cadd_raw: -0.265581,
                spliceai_pred_dp_ag: 29,
                spliceai_pred_dp_al: 2,
                spliceai_pred_dp_dg: -5,
                spliceai_pred_dp_dl: -29,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0.19,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970434',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 0.369,
                cadd_raw: -0.265581,
                spliceai_pred_dp_ag: 29,
                spliceai_pred_dp_al: 2,
                spliceai_pred_dp_dg: -5,
                spliceai_pred_dp_dl: -29,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0.19,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000466300',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970434',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:24023',
                source: 'Ensembl',
                cadd_phred: 0.369,
                cadd_raw: -0.265581,
                spliceai_pred_dp_ag: 29,
                spliceai_pred_dp_al: 2,
                spliceai_pred_dp_dg: -5,
                spliceai_pred_dp_dl: -29,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0.19,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000338591',
            ensembl_gene_id: ['ENSG00000187961', 'ENSG00000187583'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant&upstream_gene_variant',
            hgvsc: 'ENST00000338591.8:c.1701-57A>G',
            existing_variation: 'rs4970434',
            ccds: 'CCDS30550.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            cadd_phred: 0.369,
            cadd_raw: -0.265581,
            spliceai_pred_dp_ag: 29,
            spliceai_pred_dp_al: 2,
            spliceai_pred_dp_dg: -5,
            spliceai_pred_dp_dl: -29,
            spliceai_pred_ds_ag: [0.01],
            spliceai_pred_ds_al: [0.19],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'KLHL17',
            constraint_mis_z: [-0.57094, -1.7988],
            constraint_oe_lof: 0.90749,
            constraint_oe_lof_lower: 0.664,
            constraint_oe_lof_upper: [1.259, 1.124],
            constraint_oe_mis: 1.0772,
            constraint_oe_mis_lower: 0.998,
            constraint_oe_mis_upper: [1.163, 1.358],
            constraint_oe_syn: 1.7903,
            constraint_oe_syn_lower: 1.641,
            constraint_oe_syn_upper: 1.939,
            constraint_pli: 1.3052e-16,
            constraint_syn_z: [-8.809, -2.4681],
            exomiser_ad_exgenespheno: 0.5001,
            exomiser_xr_exgenescombi: 0.00030641,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.9317,
            gnomadv3_af_nfe: 0.9562,
            gnomadv3_af_xy: 0.9456,
            exomiser_xd_exgenescombi: 0.00030641,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.942,
            gnomadv3_af: 0.947,
            exomiser_mt_exgenespheno: 0.5001,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00030641,
            gnomadv3_af_eas: 0.9281,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9396,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00030641,
            exomiser_ad_exgenescombi: 0.00030641,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9705,
            gnomadv3_af_fin: 0.9729,
            exomiser_ar_exgenespheno: 0.5001,
            gnomadv3_af_sas: 0.9338,
            gnomadv3_af_amr: 0.9414,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.9483,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.9451,
            exomiser_xr_exgenespheno: 0.5001,
            exomiser_xd_exgenespheno: 0.5001,
            impact_severity: 'LOW',
            quality: 2838.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [19, 25],
              [11, 28],
              [34, 20],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['KLHL17', 'PLEKHN1'],
            ensembl_transcript_id: [
              'ENST00000338591',
              'ENST00000379410',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000466300',
            ],
            variant_type: 'small',
            highest_af: 0.947,
            highest_af_info: {
              source: 'gnomad v3 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 900286,
            hg19_strand: '+',
            hg19_end: 900286,
            entrez_gene_id: ['339451', '84069'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9ae',
            variant_id: '43',
            end: 965341,
            ref: 'CTTAT',
            alt: 'C',
            start: 965337,
            chrom: 'chr1',
            type: 'indel',
            gene_objs: [
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000338591',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '12/12',
                allele: '-',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000338591.8:c.*152_*155del',
                existing_variation: 'rs142545439',
                strand: 1,
                hgnc_id: 'HGNC:24023',
                mane_select: 'NM_198317.3',
                ccds: 'CCDS30550.1',
                source: 'Ensembl',
                spliceai_pred_dp_ag: -25,
                spliceai_pred_dp_al: 12,
                spliceai_pred_dp_dg: -38,
                spliceai_pred_dp_dl: 13,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -0.57094,
                constraint_oe_lof: 0.90749,
                constraint_oe_lof_lower: 0.664,
                constraint_oe_lof_upper: 1.259,
                constraint_oe_mis: 1.0772,
                constraint_oe_mis_lower: 0.998,
                constraint_oe_mis_upper: 1.163,
                constraint_oe_syn: 1.7903,
                constraint_oe_syn_lower: 1.641,
                constraint_oe_syn_upper: 1.939,
                constraint_pli: 1.3052e-16,
                constraint_syn_z: -8.809,
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                p_li: 1.3052e-16,
                loeuf: 1.259,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: '-',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs142545439',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                spliceai_pred_dp_ag: -25,
                spliceai_pred_dp_al: 12,
                spliceai_pred_dp_dg: -38,
                spliceai_pred_dp_dl: 13,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs142545439',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                spliceai_pred_dp_ag: -25,
                spliceai_pred_dp_al: 12,
                spliceai_pred_dp_dg: -38,
                spliceai_pred_dp_dl: 13,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs142545439',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                spliceai_pred_dp_ag: -25,
                spliceai_pred_dp_al: 12,
                spliceai_pred_dp_dg: -38,
                spliceai_pred_dp_dl: 13,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000466300',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs142545439',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:24023',
                source: 'Ensembl',
                spliceai_pred_dp_ag: -25,
                spliceai_pred_dp_al: 12,
                spliceai_pred_dp_dg: -38,
                spliceai_pred_dp_dl: 13,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000338591',
            ensembl_gene_id: ['ENSG00000187961', 'ENSG00000187583'],
            is_exonic: 1,
            is_coding: 0,
            impact: '3_prime_UTR_variant&upstream_gene_variant',
            hgvsc: 'ENST00000338591.8:c.*152_*155del',
            existing_variation: 'rs142545439',
            ccds: 'CCDS30550.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            spliceai_pred_dp_ag: -25,
            spliceai_pred_dp_al: 12,
            spliceai_pred_dp_dg: -38,
            spliceai_pred_dp_dl: 13,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'KLHL17',
            constraint_mis_z: [-0.57094, -1.7988],
            constraint_oe_lof: 0.90749,
            constraint_oe_lof_lower: 0.664,
            constraint_oe_lof_upper: [1.259, 1.124],
            constraint_oe_mis: 1.0772,
            constraint_oe_mis_lower: 0.998,
            constraint_oe_mis_upper: [1.163, 1.358],
            constraint_oe_syn: 1.7903,
            constraint_oe_syn_lower: 1.641,
            constraint_oe_syn_upper: 1.939,
            constraint_pli: 1.3052e-16,
            constraint_syn_z: [-8.809, -2.4681],
            exomiser_ad_exgenespheno: 0.5001,
            exomiser_xr_exgenescombi: 0.00030641,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.1191,
            gnomadv3_af_nfe: 0.6247,
            gnomadv3_af_xy: 0.4564,
            exomiser_xd_exgenescombi: 0.00030641,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.5411,
            gnomadv3_af: 0.4592,
            exomiser_mt_exgenespheno: 0.5001,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00030641,
            gnomadv3_af_eas: 0.3739,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.5488,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00030641,
            exomiser_ad_exgenescombi: 0.00030641,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.4901,
            gnomadv3_af_fin: 0.6104,
            exomiser_ar_exgenespheno: 0.5001,
            gnomadv3_af_sas: 0.5029,
            gnomadv3_af_amr: 0.5285,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.4619,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.4914,
            exomiser_xr_exgenespheno: 0.5001,
            exomiser_xd_exgenespheno: 0.5001,
            impact_severity: 'LOW',
            quality: 1624.1,
            genotypes_index: [
              [0, 1],
              [0, 0],
              [0, 1],
            ],
            allelic_depths: [
              [17, 23],
              [37, 0],
              [26, 19],
            ],
            genotype_qualities: [[99, 90, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['KLHL17', 'PLEKHN1'],
            ensembl_transcript_id: [
              'ENST00000338591',
              'ENST00000379410',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000466300',
            ],
            variant_type: 'small',
            highest_af: 0.4592,
            highest_af_info: {
              source: 'gnomad v3 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 900717,
            hg19_strand: '+',
            hg19_end: 900721,
            entrez_gene_id: ['339451', '84069'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9af',
            variant_id: '44',
            end: 965350,
            ref: 'G',
            alt: 'A',
            start: 965350,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000338591',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '12/12',
                allele: 'A',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000338591.8:c.*159G>A',
                existing_variation: 'rs3935066',
                strand: 1,
                hgnc_id: 'HGNC:24023',
                mane_select: 'NM_198317.3',
                ccds: 'CCDS30550.1',
                source: 'Ensembl',
                cadd_phred: 2.894,
                cadd_raw: 0.179593,
                spliceai_pred_dp_ag: 2,
                spliceai_pred_dp_al: 0,
                spliceai_pred_dp_dg: 1,
                spliceai_pred_dp_dl: -50,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -0.57094,
                constraint_oe_lof: 0.90749,
                constraint_oe_lof_lower: 0.664,
                constraint_oe_lof_upper: 1.259,
                constraint_oe_mis: 1.0772,
                constraint_oe_mis_lower: 0.998,
                constraint_oe_mis_upper: 1.163,
                constraint_oe_syn: 1.7903,
                constraint_oe_syn_lower: 1.641,
                constraint_oe_syn_upper: 1.939,
                constraint_pli: 1.3052e-16,
                constraint_syn_z: -8.809,
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                p_li: 1.3052e-16,
                loeuf: 1.259,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3935066',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                cadd_phred: 2.894,
                cadd_raw: 0.179593,
                spliceai_pred_dp_ag: 2,
                spliceai_pred_dp_al: 0,
                spliceai_pred_dp_dg: 1,
                spliceai_pred_dp_dl: -50,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3935066',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                cadd_phred: 2.894,
                cadd_raw: 0.179593,
                spliceai_pred_dp_ag: 2,
                spliceai_pred_dp_al: 0,
                spliceai_pred_dp_dg: 1,
                spliceai_pred_dp_dl: -50,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3935066',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 2.894,
                cadd_raw: 0.179593,
                spliceai_pred_dp_ag: 2,
                spliceai_pred_dp_al: 0,
                spliceai_pred_dp_dg: 1,
                spliceai_pred_dp_dl: -50,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000466300',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3935066',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:24023',
                source: 'Ensembl',
                cadd_phred: 2.894,
                cadd_raw: 0.179593,
                spliceai_pred_dp_ag: 2,
                spliceai_pred_dp_al: 0,
                spliceai_pred_dp_dg: 1,
                spliceai_pred_dp_dl: -50,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000338591',
            ensembl_gene_id: ['ENSG00000187961', 'ENSG00000187583'],
            is_exonic: 1,
            is_coding: 0,
            impact: '3_prime_UTR_variant&upstream_gene_variant',
            hgvsc: 'ENST00000338591.8:c.*159G>A',
            existing_variation: 'rs3935066',
            ccds: 'CCDS30550.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            cadd_phred: 2.894,
            cadd_raw: 0.179593,
            spliceai_pred_dp_ag: 2,
            spliceai_pred_dp_al: 0,
            spliceai_pred_dp_dg: 1,
            spliceai_pred_dp_dl: -50,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'KLHL17',
            constraint_mis_z: [-0.57094, -1.7988],
            constraint_oe_lof: 0.90749,
            constraint_oe_lof_lower: 0.664,
            constraint_oe_lof_upper: [1.259, 1.124],
            constraint_oe_mis: 1.0772,
            constraint_oe_mis_lower: 0.998,
            constraint_oe_mis_upper: [1.163, 1.358],
            constraint_oe_syn: 1.7903,
            constraint_oe_syn_lower: 1.641,
            constraint_oe_syn_upper: 1.939,
            constraint_pli: 1.3052e-16,
            constraint_syn_z: [-8.809, -2.4681],
            exomiser_ad_exgenespheno: 0.5001,
            exomiser_xr_exgenescombi: 0.00030641,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.2952,
            gnomadv3_af_nfe: 0.9088,
            gnomadv3_af_xy: 0.7157,
            exomiser_xd_exgenescombi: 0.00030641,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.8228,
            gnomadv3_af: 0.7153,
            exomiser_mt_exgenespheno: 0.5001,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00030641,
            gnomadv3_af_eas: 0.7346,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.8821,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00030641,
            exomiser_ad_exgenescombi: 0.00030641,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.8659,
            gnomadv3_af_fin: 0.9169,
            exomiser_ar_exgenespheno: 0.5001,
            gnomadv3_af_sas: 0.8679,
            gnomadv3_af_amr: 0.7412,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.7148,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.7641,
            exomiser_xr_exgenespheno: 0.5001,
            exomiser_xd_exgenespheno: 0.5001,
            impact_severity: 'LOW',
            quality: 2061.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [18, 24],
              [19, 13],
              [26, 20],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['KLHL17', 'PLEKHN1'],
            ensembl_transcript_id: [
              'ENST00000338591',
              'ENST00000379410',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000466300',
            ],
            variant_type: 'small',
            highest_af: 0.7346,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 900730,
            hg19_strand: '+',
            hg19_end: 900730,
            entrez_gene_id: ['339451', '84069'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9b0',
            variant_id: '45',
            end: 965592,
            ref: 'T',
            alt: 'G',
            start: 965592,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000338591',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '12/12',
                allele: 'G',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000338591.8:c.*401T>G',
                existing_variation: 'rs9697711',
                strand: 1,
                hgnc_id: 'HGNC:24023',
                mane_select: 'NM_198317.3',
                ccds: 'CCDS30550.1',
                source: 'Ensembl',
                cadd_phred: 4.675,
                cadd_raw: 0.333438,
                spliceai_pred_dp_ag: 0,
                spliceai_pred_dp_al: -12,
                spliceai_pred_dp_dg: -1,
                spliceai_pred_dp_dl: 49,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -0.57094,
                constraint_oe_lof: 0.90749,
                constraint_oe_lof_lower: 0.664,
                constraint_oe_lof_upper: 1.259,
                constraint_oe_mis: 1.0772,
                constraint_oe_mis_lower: 0.998,
                constraint_oe_mis_upper: 1.163,
                constraint_oe_syn: 1.7903,
                constraint_oe_syn_lower: 1.641,
                constraint_oe_syn_upper: 1.939,
                constraint_pli: 1.3052e-16,
                constraint_syn_z: -8.809,
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                p_li: 1.3052e-16,
                loeuf: 1.259,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs9697711',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                cadd_phred: 4.675,
                cadd_raw: 0.333438,
                spliceai_pred_dp_ag: 0,
                spliceai_pred_dp_al: -12,
                spliceai_pred_dp_dg: -1,
                spliceai_pred_dp_dl: 49,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs9697711',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                cadd_phred: 4.675,
                cadd_raw: 0.333438,
                spliceai_pred_dp_ag: 0,
                spliceai_pred_dp_al: -12,
                spliceai_pred_dp_dg: -1,
                spliceai_pred_dp_dl: 49,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs9697711',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 4.675,
                cadd_raw: 0.333438,
                spliceai_pred_dp_ag: 0,
                spliceai_pred_dp_al: -12,
                spliceai_pred_dp_dg: -1,
                spliceai_pred_dp_dl: 49,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000466300',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs9697711',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:24023',
                source: 'Ensembl',
                cadd_phred: 4.675,
                cadd_raw: 0.333438,
                spliceai_pred_dp_ag: 0,
                spliceai_pred_dp_al: -12,
                spliceai_pred_dp_dg: -1,
                spliceai_pred_dp_dl: 49,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000338591',
            ensembl_gene_id: ['ENSG00000187961', 'ENSG00000187583'],
            is_exonic: 1,
            is_coding: 0,
            impact: '3_prime_UTR_variant&upstream_gene_variant',
            hgvsc: 'ENST00000338591.8:c.*401T>G',
            existing_variation: 'rs9697711',
            ccds: 'CCDS30550.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            cadd_phred: 4.675,
            cadd_raw: 0.333438,
            spliceai_pred_dp_ag: 0,
            spliceai_pred_dp_al: -12,
            spliceai_pred_dp_dg: -1,
            spliceai_pred_dp_dl: 49,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'KLHL17',
            constraint_mis_z: [-0.57094, -1.7988],
            constraint_oe_lof: 0.90749,
            constraint_oe_lof_lower: 0.664,
            constraint_oe_lof_upper: [1.259, 1.124],
            constraint_oe_mis: 1.0772,
            constraint_oe_mis_lower: 0.998,
            constraint_oe_mis_upper: [1.163, 1.358],
            constraint_oe_syn: 1.7903,
            constraint_oe_syn_lower: 1.641,
            constraint_oe_syn_upper: 1.939,
            constraint_pli: 1.3052e-16,
            constraint_syn_z: [-8.809, -2.4681],
            exomiser_ad_exgenespheno: 0.5001,
            exomiser_xr_exgenescombi: 0.00030641,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.9313,
            gnomadv3_af_nfe: 0.947,
            gnomadv3_af_xy: 0.9396,
            exomiser_xd_exgenescombi: 0.00030641,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9335,
            gnomadv3_af: 0.9407,
            exomiser_mt_exgenespheno: 0.5001,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00030641,
            gnomadv3_af_eas: 0.9246,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.928,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00030641,
            exomiser_ad_exgenescombi: 0.00030641,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9638,
            gnomadv3_af_fin: 0.9646,
            exomiser_ar_exgenespheno: 0.5001,
            gnomadv3_af_sas: 0.9214,
            gnomadv3_af_amr: 0.9353,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.9418,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.9394,
            exomiser_xr_exgenespheno: 0.5001,
            exomiser_xd_exgenespheno: 0.5001,
            impact_severity: 'LOW',
            quality: 2116.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [22, 35],
              [20, 15],
              [25, 27],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['KLHL17', 'PLEKHN1'],
            ensembl_transcript_id: [
              'ENST00000338591',
              'ENST00000379410',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000466300',
            ],
            variant_type: 'small',
            highest_af: 0.9407,
            highest_af_info: {
              source: 'gnomad v3 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 900972,
            hg19_strand: '+',
            hg19_end: 900972,
            entrez_gene_id: ['339451', '84069'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9b1',
            variant_id: '46',
            end: 965643,
            ref: 'T',
            alt: 'C',
            start: 965643,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000338591',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '12/12',
                allele: 'C',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000338591.8:c.*452T>C',
                existing_variation: 'rs13303351',
                strand: 1,
                hgnc_id: 'HGNC:24023',
                mane_select: 'NM_198317.3',
                ccds: 'CCDS30550.1',
                source: 'Ensembl',
                cadd_phred: 10.53,
                cadd_raw: 0.909364,
                spliceai_pred_dp_ag: 0,
                spliceai_pred_dp_al: -2,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: -2,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -0.57094,
                constraint_oe_lof: 0.90749,
                constraint_oe_lof_lower: 0.664,
                constraint_oe_lof_upper: 1.259,
                constraint_oe_mis: 1.0772,
                constraint_oe_mis_lower: 0.998,
                constraint_oe_mis_upper: 1.163,
                constraint_oe_syn: 1.7903,
                constraint_oe_syn_lower: 1.641,
                constraint_oe_syn_upper: 1.939,
                constraint_pli: 1.3052e-16,
                constraint_syn_z: -8.809,
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                p_li: 1.3052e-16,
                loeuf: 1.259,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs13303351',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                cadd_phred: 10.53,
                cadd_raw: 0.909364,
                spliceai_pred_dp_ag: 0,
                spliceai_pred_dp_al: -2,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: -2,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs13303351',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                cadd_phred: 10.53,
                cadd_raw: 0.909364,
                spliceai_pred_dp_ag: 0,
                spliceai_pred_dp_al: -2,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: -2,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs13303351',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 10.53,
                cadd_raw: 0.909364,
                spliceai_pred_dp_ag: 0,
                spliceai_pred_dp_al: -2,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: -2,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000466300',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs13303351',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:24023',
                source: 'Ensembl',
                cadd_phred: 10.53,
                cadd_raw: 0.909364,
                spliceai_pred_dp_ag: 0,
                spliceai_pred_dp_al: -2,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: -2,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'KLHL17',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000338591',
            ensembl_gene_id: ['ENSG00000187961', 'ENSG00000187583'],
            is_exonic: 1,
            is_coding: 0,
            impact: '3_prime_UTR_variant&upstream_gene_variant',
            hgvsc: 'ENST00000338591.8:c.*452T>C',
            existing_variation: 'rs13303351',
            ccds: 'CCDS30550.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            cadd_phred: 10.53,
            cadd_raw: 0.909364,
            spliceai_pred_dp_ag: 0,
            spliceai_pred_dp_al: -2,
            spliceai_pred_dp_dg: 2,
            spliceai_pred_dp_dl: -2,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'KLHL17',
            constraint_mis_z: [-0.57094, -1.7988],
            constraint_oe_lof: 0.90749,
            constraint_oe_lof_lower: 0.664,
            constraint_oe_lof_upper: [1.259, 1.124],
            constraint_oe_mis: 1.0772,
            constraint_oe_mis_lower: 0.998,
            constraint_oe_mis_upper: [1.163, 1.358],
            constraint_oe_syn: 1.7903,
            constraint_oe_syn_lower: 1.641,
            constraint_oe_syn_upper: 1.939,
            constraint_pli: 1.3052e-16,
            constraint_syn_z: [-8.809, -2.4681],
            exomiser_ad_exgenespheno: 0.5001,
            exomiser_xr_exgenescombi: 0.00030641,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.7975,
            gnomadv3_af_nfe: 0.9446,
            gnomadv3_af_xy: 0.9012,
            exomiser_xd_exgenescombi: 0.00030641,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9304,
            gnomadv3_af: 0.9004,
            exomiser_mt_exgenespheno: 0.5001,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00030641,
            gnomadv3_af_eas: 0.9222,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9277,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00030641,
            exomiser_ad_exgenescombi: 0.00030641,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9648,
            gnomadv3_af_fin: 0.9648,
            exomiser_ar_exgenespheno: 0.5001,
            gnomadv3_af_sas: 0.9215,
            gnomadv3_af_amr: 0.9118,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.8996,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.9078,
            exomiser_xr_exgenespheno: 0.5001,
            exomiser_xd_exgenespheno: 0.5001,
            impact_severity: 'LOW',
            quality: 1761.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 1],
            ],
            allelic_depths: [
              [22, 28],
              [23, 16],
              [19, 25],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['KLHL17', 'PLEKHN1'],
            ensembl_transcript_id: [
              'ENST00000338591',
              'ENST00000379410',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000466300',
            ],
            variant_type: 'small',
            highest_af: 0.9222,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 901023,
            hg19_strand: '+',
            hg19_end: 901023,
            entrez_gene_id: ['339451', '84069'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9b2',
            variant_id: '47',
            end: 966493,
            ref: 'G',
            alt: 'A',
            start: 966493,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000338591',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs200516430',
                strand: 1,
                hgnc_id: 'HGNC:24023',
                mane_select: 'NM_198317.3',
                ccds: 'CCDS30550.1',
                source: 'Ensembl',
                cadd_phred: 9.649,
                cadd_raw: 0.825478,
                constraint_mis_z: -0.57094,
                constraint_oe_lof: 0.90749,
                constraint_oe_lof_lower: 0.664,
                constraint_oe_lof_upper: 1.259,
                constraint_oe_mis: 1.0772,
                constraint_oe_mis_lower: 0.998,
                constraint_oe_mis_upper: 1.163,
                constraint_oe_syn: 1.7903,
                constraint_oe_syn_lower: 1.641,
                constraint_oe_syn_upper: 1.939,
                constraint_pli: 1.3052e-16,
                constraint_syn_z: -8.809,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                p_li: 1.3052e-16,
                loeuf: 1.259,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '1/16',
                allele: 'A',
                impact: ['5_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379410.8:c.-39G>A',
                existing_variation: 'rs200516430',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                existing_inframe_oorfs: 0,
                existing_outofframe_oorfs: 0,
                existing_uorfs: 0,
                cadd_phred: 9.649,
                cadd_raw: 0.825478,
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: '5_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs200516430',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                cadd_phred: 9.649,
                cadd_raw: 0.825478,
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs200516430',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 9.649,
                cadd_raw: 0.825478,
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'KLHL17',
                ensembl_gene_id: 'ENSG00000187961',
                transcript: 'ENST00000466300',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs200516430',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:24023',
                source: 'Ensembl',
                cadd_phred: 9.649,
                cadd_raw: 0.825478,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'KLHL17',
                entrez_gene_id: '339451',
                p_haplo: 0.07787,
                p_triplo: 0.24313,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000338591',
            ensembl_gene_id: ['ENSG00000187961', 'ENSG00000187583'],
            is_exonic: 1,
            is_coding: 1,
            impact: 'downstream_gene_variant&5_prime_UTR_variant',
            existing_variation: 'rs200516430',
            ccds: 'CCDS30550.1',
            source: 'Ensembl',
            gnomad_af: 0.0002339,
            gnomad_afr_af: 0,
            gnomad_amr_af: 0,
            gnomad_asj_af: 0,
            gnomad_eas_af: 0.002999,
            gnomad_fin_af: 0,
            gnomad_nfe_af: 0,
            gnomad_oth_af: 0.0001879,
            gnomad_sas_af: 0,
            cadd_phred: 9.649,
            cadd_raw: 0.825478,
            constraint_mis_z: [-0.57094, -1.7988],
            constraint_oe_lof: 0.90749,
            constraint_oe_lof_lower: 0.664,
            constraint_oe_lof_upper: [1.259, 1.124],
            constraint_oe_mis: 1.0772,
            constraint_oe_mis_lower: 0.998,
            constraint_oe_mis_upper: [1.163, 1.358],
            constraint_oe_syn: 1.7903,
            constraint_oe_syn_lower: 1.641,
            constraint_oe_syn_upper: 1.939,
            constraint_pli: 1.3052e-16,
            constraint_syn_z: [-8.809, -2.4681],
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0,
            gnomadv3_af_nfe: 0,
            gnomadv3_af_xy: 0.00012106,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0,
            gnomadv3_af: 0.00011171,
            clndisdb: 'None',
            gnomadv3_af_eas: 0.0032831,
            gnomadv3_af_asj: 0,
            clnsigincl: 'None',
            clnvi: 'None',
            clnsig: 'None',
            gnomadv3_af_ami: 0,
            gnomadv3_af_fin: 0,
            gnomadv3_af_sas: 0,
            gnomadv3_af_amr: 0,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            gnomadv3_af_xx: 0.00010277,
            gnomadv3_af_oth: 0,
            impact_severity: 'LOW',
            quality: 779.2,
            genotypes_index: [
              [0, 0],
              [0, 1],
              [0, 0],
            ],
            allelic_depths: [
              [53, 0],
              [26, 27],
              [52, 0],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['KLHL17', 'PLEKHN1'],
            ensembl_transcript_id: [
              'ENST00000338591',
              'ENST00000379410',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000466300',
            ],
            variant_type: 'small',
            highest_af: 0.0032831,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 901873,
            hg19_strand: '+',
            hg19_end: 901873,
            entrez_gene_id: ['339451', '84069'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9b3',
            variant_id: '48',
            end: 970892,
            ref: 'A',
            alt: 'C',
            start: 970892,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '6/16',
                codon_change: 'gcA/gcC',
                aa_change: 'A',
                aa_length: 166,
                allele: 'C',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379410.8:c.498A>C',
                hgvsp: 'ENSP00000368720.3:p.Ala166%3D',
                cds_position: 498,
                existing_variation: 'rs28507236&COSV58022662',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                domains:
                  'Gene3D:2.30.29.30&PANTHER:PTHR46882&SMART:SM00233&Superfamily:SSF50729',
                pheno: '0&1',
                cadd_phred: 6.972,
                cadd_raw: 0.545437,
                spliceai_pred_dp_ag: 24,
                spliceai_pred_dp_al: -13,
                spliceai_pred_dp_dg: -46,
                spliceai_pred_dp_dl: 9,
                spliceai_pred_ds_ag: 0.03,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000433179',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs28507236&COSV58022662',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                ccds: 'CCDS76083.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 6.972,
                cadd_raw: 0.545437,
                spliceai_pred_dp_ag: 24,
                spliceai_pred_dp_al: -13,
                spliceai_pred_dp_dg: -46,
                spliceai_pred_dp_dl: 9,
                spliceai_pred_ds_ag: 0.03,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.55519,
                constraint_oe_lof: 1.1826,
                constraint_oe_lof_lower: 0.837,
                constraint_oe_lof_upper: 1.686,
                constraint_oe_mis: 1.079,
                constraint_oe_mis_lower: 0.996,
                constraint_oe_mis_upper: 1.169,
                constraint_oe_syn: 1.2299,
                constraint_oe_syn_lower: 1.1,
                constraint_oe_syn_upper: 1.377,
                constraint_pli: 1.1667e-15,
                constraint_syn_z: -2.3898,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 1.1667e-15,
                loeuf: 1.686,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000341290',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs28507236&COSV58022662',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 6.972,
                cadd_raw: 0.545437,
                spliceai_pred_dp_ag: 24,
                spliceai_pred_dp_al: -13,
                spliceai_pred_dp_dg: -46,
                spliceai_pred_dp_dl: 9,
                spliceai_pred_ds_ag: 0.03,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.61287,
                constraint_oe_lof: 1.052,
                constraint_oe_lof_lower: 0.719,
                constraint_oe_lof_upper: 1.571,
                constraint_oe_mis: 1.089,
                constraint_oe_mis_lower: 1.004,
                constraint_oe_mis_upper: 1.182,
                constraint_oe_syn: 1.2437,
                constraint_oe_syn_lower: 1.111,
                constraint_oe_syn_upper: 1.394,
                constraint_pli: 4.2343e-12,
                constraint_syn_z: -2.4954,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 4.2343e-12,
                loeuf: 1.571,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '6/15',
                codon_change: 'gcA/gcC',
                aa_change: 'A',
                aa_length: 166,
                allele: 'C',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379407.7:c.498A>C',
                hgvsp: 'ENSP00000368717.2:p.Ala166%3D',
                cds_position: 498,
                existing_variation: 'rs28507236&COSV58022662',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                domains:
                  'PANTHER:PTHR46882&Gene3D:2.30.29.30&SMART:SM00233&Superfamily:SSF50729',
                pheno: '0&1',
                cadd_phred: 6.972,
                cadd_raw: 0.545437,
                spliceai_pred_dp_ag: 24,
                spliceai_pred_dp_al: -13,
                spliceai_pred_dp_dg: -46,
                spliceai_pred_dp_dl: 9,
                spliceai_pred_ds_ag: 0.03,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '5/15',
                codon_change: 'gcA/gcC',
                aa_change: 'A',
                aa_length: 206,
                allele: 'C',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379409.6:c.618A>C',
                hgvsp: 'ENSP00000368719.2:p.Ala206%3D',
                cds_position: 618,
                existing_variation: 'rs28507236&COSV58022662',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                domains:
                  'Low_complexity_(Seg):seg&PANTHER:PTHR46882&SMART:SM00233',
                pheno: '0&1',
                cadd_phred: 6.972,
                cadd_raw: 0.545437,
                spliceai_pred_dp_ag: 24,
                spliceai_pred_dp_al: -13,
                spliceai_pred_dp_dg: -46,
                spliceai_pred_dp_dl: 9,
                spliceai_pred_ds_ag: 0.03,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000491024',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs28507236&COSV58022662',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 6.972,
                cadd_raw: 0.545437,
                spliceai_pred_dp_ag: 24,
                spliceai_pred_dp_al: -13,
                spliceai_pred_dp_dg: -46,
                spliceai_pred_dp_dl: 9,
                spliceai_pred_ds_ag: 0.03,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.82994,
                constraint_oe_lof: 0.67057,
                constraint_oe_lof_lower: 0.349,
                constraint_oe_lof_upper: 1.397,
                constraint_oe_mis: 1.2196,
                constraint_oe_mis_lower: 1.061,
                constraint_oe_mis_upper: 1.405,
                constraint_oe_syn: 1.1576,
                constraint_oe_syn_lower: 0.938,
                constraint_oe_syn_upper: 1.438,
                constraint_pli: 0.0017329,
                constraint_syn_z: -0.88419,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 0.0017329,
                loeuf: 1.397,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000379410',
            ensembl_gene_id: ['ENSG00000187583', 'ENSG00000187642'],
            is_exonic: 1,
            is_coding: 1,
            aa_change: 'A',
            impact: 'synonymous_variant&downstream_gene_variant',
            hgvsc: 'ENST00000379410.8:c.498A>C',
            hgvsp: 'ENSP00000368720.3:p.Ala166%3D',
            existing_variation: 'rs28507236&COSV58022662',
            ccds: 'CCDS4.1',
            source: 'Ensembl',
            domains:
              'Gene3D:2.30.29.30&PANTHER:PTHR46882&SMART:SM00233&Superfamily:SSF50729',
            gnomad_af: 0.1269,
            gnomad_afr_af: 0.1874,
            gnomad_amr_af: 0.1779,
            gnomad_asj_af: 0.06542,
            gnomad_eas_af: 0.308,
            gnomad_fin_af: 0.1208,
            gnomad_nfe_af: 0.08499,
            gnomad_oth_af: 0.1147,
            gnomad_sas_af: 0.106,
            pheno: '0&1',
            cadd_phred: 6.972,
            cadd_raw: 0.545437,
            spliceai_pred_dp_ag: 24,
            spliceai_pred_dp_al: -13,
            spliceai_pred_dp_dg: -46,
            spliceai_pred_dp_dl: 9,
            spliceai_pred_ds_ag: [0.03],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'PLEKHN1',
            constraint_mis_z: [-1.7988, -0.55519],
            constraint_oe_lof: 0.80467,
            constraint_oe_lof_lower: 0.585,
            constraint_oe_lof_upper: [1.124, 1.686],
            constraint_oe_mis: 1.2593,
            constraint_oe_mis_lower: 1.168,
            constraint_oe_mis_upper: [1.358, 1.169],
            constraint_oe_syn: 1.2377,
            constraint_oe_syn_lower: 1.107,
            constraint_oe_syn_upper: 1.385,
            constraint_pli: 1.1549e-14,
            constraint_syn_z: [-2.4681, -2.3898],
            exomiser_ad_exgenespheno: 0,
            exomiser_xr_exgenescombi: 0,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.1885,
            gnomadv3_af_nfe: 0.0822,
            gnomadv3_af_xy: 0.1335,
            exomiser_xd_exgenescombi: 0,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.0728,
            gnomadv3_af: 0.1308,
            exomiser_mt_exgenespheno: 0,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0,
            gnomadv3_af_eas: 0.3127,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.0679,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0,
            exomiser_ad_exgenescombi: 0,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.0802,
            gnomadv3_af_fin: 0.1238,
            exomiser_ar_exgenespheno: 0,
            gnomadv3_af_sas: 0.1082,
            gnomadv3_af_amr: 0.1613,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.1283,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.1258,
            exomiser_xr_exgenespheno: 0,
            exomiser_xd_exgenespheno: 0,
            impact_severity: 'LOW',
            quality: 1103.1,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 0],
            ],
            allelic_depths: [
              [12, 15],
              [14, 22],
              [42, 0],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['PLEKHN1', 'PERM1'],
            ensembl_transcript_id: [
              'ENST00000379410',
              'ENST00000433179',
              'ENST00000341290',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000491024',
            ],
            variant_type: 'small',
            highest_af: 0.3127,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 906272,
            hg19_strand: '+',
            hg19_end: 906272,
            entrez_gene_id: ['84069', '84808'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9b4',
            variant_id: '49',
            end: 972160,
            ref: 'T',
            alt: 'C',
            start: 972160,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '9/15',
                hgvsc: 'ENST00000379410.8:c.865+10T>C',
                existing_variation: 'rs186774509',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                cadd_phred: 0.404,
                cadd_raw: -0.247297,
                spliceai_pred_dp_ag: -31,
                spliceai_pred_dp_al: 30,
                spliceai_pred_dp_dg: -10,
                spliceai_pred_dp_dl: -5,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000433179',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs186774509',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                ccds: 'CCDS76083.1',
                source: 'Ensembl',
                cadd_phred: 0.404,
                cadd_raw: -0.247297,
                spliceai_pred_dp_ag: -31,
                spliceai_pred_dp_al: 30,
                spliceai_pred_dp_dg: -10,
                spliceai_pred_dp_dl: -5,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.55519,
                constraint_oe_lof: 1.1826,
                constraint_oe_lof_lower: 0.837,
                constraint_oe_lof_upper: 1.686,
                constraint_oe_mis: 1.079,
                constraint_oe_mis_lower: 0.996,
                constraint_oe_mis_upper: 1.169,
                constraint_oe_syn: 1.2299,
                constraint_oe_syn_lower: 1.1,
                constraint_oe_syn_upper: 1.377,
                constraint_pli: 1.1667e-15,
                constraint_syn_z: -2.3898,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 1.1667e-15,
                loeuf: 1.686,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000341290',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs186774509',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                source: 'Ensembl',
                cadd_phred: 0.404,
                cadd_raw: -0.247297,
                spliceai_pred_dp_ag: -31,
                spliceai_pred_dp_al: 30,
                spliceai_pred_dp_dg: -10,
                spliceai_pred_dp_dl: -5,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.61287,
                constraint_oe_lof: 1.052,
                constraint_oe_lof_lower: 0.719,
                constraint_oe_lof_upper: 1.571,
                constraint_oe_mis: 1.089,
                constraint_oe_mis_lower: 1.004,
                constraint_oe_mis_upper: 1.182,
                constraint_oe_syn: 1.2437,
                constraint_oe_syn_lower: 1.111,
                constraint_oe_syn_upper: 1.394,
                constraint_pli: 4.2343e-12,
                constraint_syn_z: -2.4954,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 4.2343e-12,
                loeuf: 1.571,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '9/14',
                hgvsc: 'ENST00000379407.7:c.901+10T>C',
                existing_variation: 'rs186774509',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                cadd_phred: 0.404,
                cadd_raw: -0.247297,
                spliceai_pred_dp_ag: -31,
                spliceai_pred_dp_al: 30,
                spliceai_pred_dp_dg: -10,
                spliceai_pred_dp_dl: -5,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '8/14',
                hgvsc: 'ENST00000379409.6:c.1021+10T>C',
                existing_variation: 'rs186774509',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 0.404,
                cadd_raw: -0.247297,
                spliceai_pred_dp_ag: -31,
                spliceai_pred_dp_al: 30,
                spliceai_pred_dp_dg: -10,
                spliceai_pred_dp_dl: -5,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000491024',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs186774509',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 0.404,
                cadd_raw: -0.247297,
                spliceai_pred_dp_ag: -31,
                spliceai_pred_dp_al: 30,
                spliceai_pred_dp_dg: -10,
                spliceai_pred_dp_dl: -5,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.82994,
                constraint_oe_lof: 0.67057,
                constraint_oe_lof_lower: 0.349,
                constraint_oe_lof_upper: 1.397,
                constraint_oe_mis: 1.2196,
                constraint_oe_mis_lower: 1.061,
                constraint_oe_mis_upper: 1.405,
                constraint_oe_syn: 1.1576,
                constraint_oe_syn_lower: 0.938,
                constraint_oe_syn_upper: 1.438,
                constraint_pli: 0.0017329,
                constraint_syn_z: -0.88419,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 0.0017329,
                loeuf: 1.397,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000379410',
            ensembl_gene_id: ['ENSG00000187583', 'ENSG00000187642'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant&downstream_gene_variant',
            hgvsc: 'ENST00000379410.8:c.865+10T>C',
            existing_variation: 'rs186774509',
            ccds: 'CCDS4.1',
            source: 'Ensembl',
            gnomad_af: 0.001994,
            gnomad_afr_af: 0,
            gnomad_amr_af: 0.00002902,
            gnomad_asj_af: 0,
            gnomad_eas_af: 0.02499,
            gnomad_fin_af: 0,
            gnomad_nfe_af: 0.000008914,
            gnomad_oth_af: 0.0006588,
            gnomad_sas_af: 0.001048,
            cadd_phred: 0.404,
            cadd_raw: -0.247297,
            spliceai_pred_dp_ag: -31,
            spliceai_pred_dp_al: 30,
            spliceai_pred_dp_dg: -10,
            spliceai_pred_dp_dl: -5,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'PLEKHN1',
            constraint_mis_z: [-1.7988, -0.55519],
            constraint_oe_lof: 0.80467,
            constraint_oe_lof_lower: 0.585,
            constraint_oe_lof_upper: [1.124, 1.686],
            constraint_oe_mis: 1.2593,
            constraint_oe_mis_lower: 1.168,
            constraint_oe_mis_upper: [1.358, 1.169],
            constraint_oe_syn: 1.2377,
            constraint_oe_syn_lower: 1.107,
            constraint_oe_syn_upper: 1.385,
            constraint_pli: 1.1549e-14,
            constraint_syn_z: [-2.4681, -2.3898],
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0,
            gnomadv3_af_nfe: 0.000014698,
            gnomadv3_af_xy: 0.00096803,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0,
            gnomadv3_af: 0.0008342,
            clndisdb: 'None',
            gnomadv3_af_eas: 0.0231,
            gnomadv3_af_asj: 0,
            clnsigincl: 'None',
            clnvi: 'None',
            clnsig: 'None',
            gnomadv3_af_ami: 0,
            gnomadv3_af_fin: 0,
            gnomadv3_af_sas: 0.0006206,
            gnomadv3_af_amr: 0.000065411,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            gnomadv3_af_xx: 0.00070636,
            gnomadv3_af_oth: 0.00095602,
            impact_severity: 'LOW',
            quality: 814.2,
            genotypes_index: [
              [0, 0],
              [0, 0],
              [0, 1],
            ],
            allelic_depths: [
              [53, 0],
              [41, 0],
              [29, 31],
            ],
            genotype_qualities: [[92, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['PLEKHN1', 'PERM1'],
            ensembl_transcript_id: [
              'ENST00000379410',
              'ENST00000433179',
              'ENST00000341290',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000491024',
            ],
            variant_type: 'small',
            highest_af: 0.02499,
            highest_af_info: {
              source: 'gnomad V2 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 907540,
            hg19_strand: '+',
            hg19_end: 907540,
            entrez_gene_id: ['84069', '84808'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9b5',
            variant_id: '50',
            end: 973010,
            ref: 'G',
            alt: 'A',
            start: 973010,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: false,
                is_coding: false,
                is_splicing: true,
                is_canonical: true,
                is_lof: false,
                exon: '11/16',
                codon_change: 'ccG/ccA',
                aa_change: 'P',
                aa_length: 384,
                allele: 'A',
                impact: ['splice_region_variant', 'synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379410.8:c.1152G>A',
                hgvsp: 'ENSP00000368720.3:p.Pro384%3D',
                cds_position: 1152,
                existing_variation: 'rs148169601',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                domains:
                  'PANTHER:PTHR46882&MobiDB_lite:mobidb-lite&MobiDB_lite:mobidb-lite',
                cadd_phred: 10.84,
                cadd_raw: 0.937122,
                spliceai_pred_dp_ag: -37,
                spliceai_pred_dp_al: 48,
                spliceai_pred_dp_dg: 48,
                spliceai_pred_dp_dl: 0,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.32,
                spliceai_pred_ds_dl: 0.48,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'splice_region_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000433179',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs148169601',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                ccds: 'CCDS76083.1',
                source: 'Ensembl',
                cadd_phred: 10.84,
                cadd_raw: 0.937122,
                spliceai_pred_dp_ag: -37,
                spliceai_pred_dp_al: 48,
                spliceai_pred_dp_dg: 48,
                spliceai_pred_dp_dl: 0,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.32,
                spliceai_pred_ds_dl: 0.48,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.55519,
                constraint_oe_lof: 1.1826,
                constraint_oe_lof_lower: 0.837,
                constraint_oe_lof_upper: 1.686,
                constraint_oe_mis: 1.079,
                constraint_oe_mis_lower: 0.996,
                constraint_oe_mis_upper: 1.169,
                constraint_oe_syn: 1.2299,
                constraint_oe_syn_lower: 1.1,
                constraint_oe_syn_upper: 1.377,
                constraint_pli: 1.1667e-15,
                constraint_syn_z: -2.3898,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 1.1667e-15,
                loeuf: 1.686,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: true,
                is_canonical: false,
                is_lof: false,
                exon: '11/15',
                codon_change: 'ccG/ccA',
                aa_change: 'P',
                aa_length: 396,
                allele: 'A',
                impact: ['splice_region_variant', 'synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379407.7:c.1188G>A',
                hgvsp: 'ENSP00000368717.2:p.Pro396%3D',
                cds_position: 1188,
                existing_variation: 'rs148169601',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                domains:
                  'MobiDB_lite:mobidb-lite&MobiDB_lite:mobidb-lite&PANTHER:PTHR46882',
                cadd_phred: 10.84,
                cadd_raw: 0.937122,
                spliceai_pred_dp_ag: -37,
                spliceai_pred_dp_al: 48,
                spliceai_pred_dp_dg: 48,
                spliceai_pred_dp_dl: 0,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.32,
                spliceai_pred_ds_dl: 0.48,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'splice_region_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: true,
                is_canonical: false,
                is_lof: false,
                exon: '10/15',
                codon_change: 'ccG/ccA',
                aa_change: 'P',
                aa_length: 436,
                allele: 'A',
                impact: ['splice_region_variant', 'synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379409.6:c.1308G>A',
                hgvsp: 'ENSP00000368719.2:p.Pro436%3D',
                cds_position: 1308,
                existing_variation: 'rs148169601',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                domains:
                  'MobiDB_lite:mobidb-lite&MobiDB_lite:mobidb-lite&PANTHER:PTHR46882',
                cadd_phred: 10.84,
                cadd_raw: 0.937122,
                spliceai_pred_dp_ag: -37,
                spliceai_pred_dp_al: 48,
                spliceai_pred_dp_dg: 48,
                spliceai_pred_dp_dl: 0,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.32,
                spliceai_pred_ds_dl: 0.48,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'splice_region_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000341290',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs148169601',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                source: 'Ensembl',
                cadd_phred: 10.84,
                cadd_raw: 0.937122,
                spliceai_pred_dp_ag: -37,
                spliceai_pred_dp_al: 48,
                spliceai_pred_dp_dg: 48,
                spliceai_pred_dp_dl: 0,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.32,
                spliceai_pred_ds_dl: 0.48,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.61287,
                constraint_oe_lof: 1.052,
                constraint_oe_lof_lower: 0.719,
                constraint_oe_lof_upper: 1.571,
                constraint_oe_mis: 1.089,
                constraint_oe_mis_lower: 1.004,
                constraint_oe_mis_upper: 1.182,
                constraint_oe_syn: 1.2437,
                constraint_oe_syn_lower: 1.111,
                constraint_oe_syn_upper: 1.394,
                constraint_pli: 4.2343e-12,
                constraint_syn_z: -2.4954,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 4.2343e-12,
                loeuf: 1.571,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000491024',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs148169601',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 10.84,
                cadd_raw: 0.937122,
                spliceai_pred_dp_ag: -37,
                spliceai_pred_dp_al: 48,
                spliceai_pred_dp_dg: 48,
                spliceai_pred_dp_dl: 0,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.32,
                spliceai_pred_ds_dl: 0.48,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.82994,
                constraint_oe_lof: 0.67057,
                constraint_oe_lof_lower: 0.349,
                constraint_oe_lof_upper: 1.397,
                constraint_oe_mis: 1.2196,
                constraint_oe_mis_lower: 1.061,
                constraint_oe_mis_upper: 1.405,
                constraint_oe_syn: 1.1576,
                constraint_oe_syn_lower: 0.938,
                constraint_oe_syn_upper: 1.438,
                constraint_pli: 0.0017329,
                constraint_syn_z: -0.88419,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 0.0017329,
                loeuf: 1.397,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000379410',
            ensembl_gene_id: ['ENSG00000187583', 'ENSG00000187642'],
            is_exonic: 1,
            is_coding: 1,
            aa_change: 'P',
            impact:
              'splice_region_variant&synonymous_variant&downstream_gene_variant',
            hgvsc: 'ENST00000379410.8:c.1152G>A',
            hgvsp: 'ENSP00000368720.3:p.Pro384%3D',
            existing_variation: 'rs148169601',
            ccds: 'CCDS4.1',
            source: 'Ensembl',
            domains:
              'PANTHER:PTHR46882&MobiDB_lite:mobidb-lite&MobiDB_lite:mobidb-lite',
            gnomad_af: 0.0008457,
            gnomad_afr_af: 0,
            gnomad_amr_af: 0.00003103,
            gnomad_asj_af: 0,
            gnomad_eas_af: 0.009524,
            gnomad_fin_af: 0,
            gnomad_nfe_af: 0.0001103,
            gnomad_oth_af: 0.0003584,
            gnomad_sas_af: 0.0004267,
            cadd_phred: 10.84,
            cadd_raw: 0.937122,
            spliceai_pred_dp_ag: -37,
            spliceai_pred_dp_al: 48,
            spliceai_pred_dp_dg: 48,
            spliceai_pred_dp_dl: 0,
            spliceai_pred_ds_ag: [0.01],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0.32],
            spliceai_pred_ds_dl: [0.48],
            spliceai_pred_symbol: 'PLEKHN1',
            constraint_mis_z: [-1.7988, -0.55519],
            constraint_oe_lof: 0.80467,
            constraint_oe_lof_lower: 0.585,
            constraint_oe_lof_upper: [1.124, 1.686],
            constraint_oe_mis: 1.2593,
            constraint_oe_mis_lower: 1.168,
            constraint_oe_mis_upper: [1.358, 1.169],
            constraint_oe_syn: 1.2377,
            constraint_oe_syn_lower: 1.107,
            constraint_oe_syn_upper: 1.385,
            constraint_pli: 1.1549e-14,
            constraint_syn_z: [-2.4681, -2.3898],
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0,
            gnomadv3_af_nfe: 0.00010293,
            gnomadv3_af_xy: 0.00056499,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0,
            gnomadv3_af: 0.00050606,
            clndisdb: 'None',
            gnomadv3_af_eas: 0.0129,
            gnomadv3_af_asj: 0,
            clnsigincl: 'None',
            clnvi: 'None',
            clnsig: 'None',
            gnomadv3_af_ami: 0,
            gnomadv3_af_fin: 0,
            gnomadv3_af_sas: 0.00041408,
            gnomadv3_af_amr: 0.000065428,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            gnomadv3_af_xx: 0.00044977,
            gnomadv3_af_oth: 0,
            impact_severity: 'MED',
            quality: 430.2,
            genotypes_index: [
              [0, 0],
              [0, 1],
              [0, 0],
            ],
            allelic_depths: [
              [37, 0],
              [30, 20],
              [37, 0],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['PLEKHN1', 'PERM1'],
            ensembl_transcript_id: [
              'ENST00000379410',
              'ENST00000433179',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000341290',
              'ENST00000491024',
            ],
            variant_type: 'small',
            highest_af: 0.0129,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 908390,
            hg19_strand: '+',
            hg19_end: 908390,
            entrez_gene_id: ['84069', '84808'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9b6',
            variant_id: '51',
            end: 973693,
            ref: 'C',
            alt: 'T',
            start: 973693,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'T',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '13/15',
                hgvsc: 'ENST00000379410.8:c.1434+53C>T',
                existing_variation: 'rs3892467&COSV58023494',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.122,
                cadd_raw: -0.494063,
                spliceai_pred_dp_ag: -50,
                spliceai_pred_dp_al: -25,
                spliceai_pred_dp_dg: -2,
                spliceai_pred_dp_dl: 16,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.02,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000433179',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'T',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3892467&COSV58023494',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                ccds: 'CCDS76083.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.122,
                cadd_raw: -0.494063,
                spliceai_pred_dp_ag: -50,
                spliceai_pred_dp_al: -25,
                spliceai_pred_dp_dg: -2,
                spliceai_pred_dp_dl: 16,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.02,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.55519,
                constraint_oe_lof: 1.1826,
                constraint_oe_lof_lower: 0.837,
                constraint_oe_lof_upper: 1.686,
                constraint_oe_mis: 1.079,
                constraint_oe_mis_lower: 0.996,
                constraint_oe_mis_upper: 1.169,
                constraint_oe_syn: 1.2299,
                constraint_oe_syn_lower: 1.1,
                constraint_oe_syn_upper: 1.377,
                constraint_pli: 1.1667e-15,
                constraint_syn_z: -2.3898,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 1.1667e-15,
                loeuf: 1.686,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000341290',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3892467&COSV58023494',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.122,
                cadd_raw: -0.494063,
                spliceai_pred_dp_ag: -50,
                spliceai_pred_dp_al: -25,
                spliceai_pred_dp_dg: -2,
                spliceai_pred_dp_dl: 16,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.02,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.61287,
                constraint_oe_lof: 1.052,
                constraint_oe_lof_lower: 0.719,
                constraint_oe_lof_upper: 1.571,
                constraint_oe_mis: 1.089,
                constraint_oe_mis_lower: 1.004,
                constraint_oe_mis_upper: 1.182,
                constraint_oe_syn: 1.2437,
                constraint_oe_syn_lower: 1.111,
                constraint_oe_syn_upper: 1.394,
                constraint_pli: 4.2343e-12,
                constraint_syn_z: -2.4954,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 4.2343e-12,
                loeuf: 1.571,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '12/14',
                hgvsc: 'ENST00000379407.7:c.1329+53C>T',
                existing_variation: 'rs3892467&COSV58023494',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.122,
                cadd_raw: -0.494063,
                spliceai_pred_dp_ag: -50,
                spliceai_pred_dp_al: -25,
                spliceai_pred_dp_dg: -2,
                spliceai_pred_dp_dl: 16,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.02,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '12/14',
                hgvsc: 'ENST00000379409.6:c.1590+53C>T',
                existing_variation: 'rs3892467&COSV58023494',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.122,
                cadd_raw: -0.494063,
                spliceai_pred_dp_ag: -50,
                spliceai_pred_dp_al: -25,
                spliceai_pred_dp_dg: -2,
                spliceai_pred_dp_dl: 16,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.02,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000491024',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '1/4',
                hgvsc: 'ENST00000491024.1:c.129+53C>T',
                existing_variation: 'rs3892467&COSV58023494',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 0.122,
                cadd_raw: -0.494063,
                spliceai_pred_dp_ag: -50,
                spliceai_pred_dp_al: -25,
                spliceai_pred_dp_dg: -2,
                spliceai_pred_dp_dl: 16,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.02,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.82994,
                constraint_oe_lof: 0.67057,
                constraint_oe_lof_lower: 0.349,
                constraint_oe_lof_upper: 1.397,
                constraint_oe_mis: 1.2196,
                constraint_oe_mis_lower: 1.061,
                constraint_oe_mis_upper: 1.405,
                constraint_oe_syn: 1.1576,
                constraint_oe_syn_lower: 0.938,
                constraint_oe_syn_upper: 1.438,
                constraint_pli: 0.0017329,
                constraint_syn_z: -0.88419,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 0.0017329,
                loeuf: 1.397,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000379410',
            ensembl_gene_id: ['ENSG00000187583', 'ENSG00000187642'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant&downstream_gene_variant',
            hgvsc: 'ENST00000379410.8:c.1434+53C>T',
            existing_variation: 'rs3892467&COSV58023494',
            ccds: 'CCDS4.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            pheno: '0&1',
            cadd_phred: 0.122,
            cadd_raw: -0.494063,
            spliceai_pred_dp_ag: -50,
            spliceai_pred_dp_al: -25,
            spliceai_pred_dp_dg: -2,
            spliceai_pred_dp_dl: 16,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0.02],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'PLEKHN1',
            constraint_mis_z: [-1.7988, -0.55519],
            constraint_oe_lof: 0.80467,
            constraint_oe_lof_lower: 0.585,
            constraint_oe_lof_upper: [1.124, 1.686],
            constraint_oe_mis: 1.2593,
            constraint_oe_mis_lower: 1.168,
            constraint_oe_mis_upper: [1.358, 1.169],
            constraint_oe_syn: 1.2377,
            constraint_oe_syn_lower: 1.107,
            constraint_oe_syn_upper: 1.385,
            constraint_pli: 1.1549e-14,
            constraint_syn_z: [-2.4681, -2.3898],
            exomiser_ad_exgenespheno: 0,
            exomiser_xr_exgenescombi: 0,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.2594,
            gnomadv3_af_nfe: 0.3847,
            gnomadv3_af_xy: 0.3617,
            exomiser_xd_exgenescombi: 0,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.4551,
            gnomadv3_af: 0.3615,
            exomiser_mt_exgenespheno: 0,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0,
            gnomadv3_af_eas: 0.6602,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.5101,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0,
            exomiser_ad_exgenescombi: 0,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.2874,
            gnomadv3_af_fin: 0.285,
            exomiser_ar_exgenespheno: 0,
            gnomadv3_af_sas: 0.4994,
            gnomadv3_af_amr: 0.4078,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.3613,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.3949,
            exomiser_xr_exgenespheno: 0,
            exomiser_xd_exgenespheno: 0,
            impact_severity: 'LOW',
            quality: 3202.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [1, 1],
            ],
            allelic_depths: [
              [24, 25],
              [19, 22],
              [0, 60],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['PLEKHN1', 'PERM1'],
            ensembl_transcript_id: [
              'ENST00000379410',
              'ENST00000433179',
              'ENST00000341290',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000491024',
            ],
            variant_type: 'small',
            highest_af: 0.6602,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 909073,
            hg19_strand: '+',
            hg19_end: 909073,
            entrez_gene_id: ['84069', '84808'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9b7',
            variant_id: '52',
            end: 973858,
            ref: 'G',
            alt: 'C',
            start: 973858,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '14/16',
                codon_change: 'cGt/cCt',
                aa_change: 'R/P',
                aa_length: 487,
                polyphen_pred: 'benign',
                polyphen_score: 0,
                sift_pred: 'tolerated',
                sift_score: 0.31,
                allele: 'C',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379410.8:c.1460G>C',
                hgvsp: 'ENSP00000368720.3:p.Arg487Pro',
                cds_position: 1460,
                existing_variation: 'rs3829740&COSV58021873',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                domains: 'PANTHER:PTHR46882&MobiDB_lite:mobidb-lite',
                pheno: '0&1',
                revel: 0.042,
                cadd_phred: 9.265,
                cadd_raw: 0.787153,
                spliceai_pred_dp_ag: -43,
                spliceai_pred_dp_al: -50,
                spliceai_pred_dp_dg: 39,
                spliceai_pred_dp_dl: -7,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000433179',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3829740&COSV58021873',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                ccds: 'CCDS76083.1',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 9.265,
                cadd_raw: 0.787153,
                spliceai_pred_dp_ag: -43,
                spliceai_pred_dp_al: -50,
                spliceai_pred_dp_dg: 39,
                spliceai_pred_dp_dl: -7,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.55519,
                constraint_oe_lof: 1.1826,
                constraint_oe_lof_lower: 0.837,
                constraint_oe_lof_upper: 1.686,
                constraint_oe_mis: 1.079,
                constraint_oe_mis_lower: 0.996,
                constraint_oe_mis_upper: 1.169,
                constraint_oe_syn: 1.2299,
                constraint_oe_syn_lower: 1.1,
                constraint_oe_syn_upper: 1.377,
                constraint_pli: 1.1667e-15,
                constraint_syn_z: -2.3898,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 1.1667e-15,
                loeuf: 1.686,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '13/15',
                codon_change: 'cGt/cCt',
                aa_change: 'R/P',
                aa_length: 452,
                polyphen_pred: 'benign',
                polyphen_score: 0,
                sift_pred: 'tolerated',
                sift_score: 0.3,
                allele: 'C',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379407.7:c.1355G>C',
                hgvsp: 'ENSP00000368717.2:p.Arg452Pro',
                cds_position: 1355,
                existing_variation: 'rs3829740&COSV58021873',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                domains: 'MobiDB_lite:mobidb-lite&PANTHER:PTHR46882',
                pheno: '0&1',
                revel: 0.042,
                cadd_phred: 9.265,
                cadd_raw: 0.787153,
                spliceai_pred_dp_ag: -43,
                spliceai_pred_dp_al: -50,
                spliceai_pred_dp_dg: 39,
                spliceai_pred_dp_dl: -7,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '13/15',
                codon_change: 'cGt/cCt',
                aa_change: 'R/P',
                aa_length: 539,
                polyphen_pred: 'benign',
                polyphen_score: 0,
                sift_pred: 'tolerated',
                sift_score: 0.3,
                allele: 'C',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379409.6:c.1616G>C',
                hgvsp: 'ENSP00000368719.2:p.Arg539Pro',
                cds_position: 1616,
                existing_variation: 'rs3829740&COSV58021873',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                domains: 'MobiDB_lite:mobidb-lite&PANTHER:PTHR46882',
                pheno: '0&1',
                revel: 0.042,
                cadd_phred: 9.265,
                cadd_raw: 0.787153,
                spliceai_pred_dp_ag: -43,
                spliceai_pred_dp_al: -50,
                spliceai_pred_dp_dg: 39,
                spliceai_pred_dp_dl: -7,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000491024',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '2/5',
                codon_change: 'cGt/cCt',
                aa_change: 'R/P',
                aa_length: 52,
                polyphen_pred: 'benign',
                polyphen_score: 0,
                sift_pred: 'tolerated',
                sift_score: 0.36,
                allele: 'C',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000491024.1:c.155G>C',
                hgvsp: 'ENSP00000462558.1:p.Arg52Pro',
                cds_position: 155,
                existing_variation: 'rs3829740&COSV58021873',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                domains: 'MobiDB_lite:mobidb-lite&PANTHER:PTHR46882',
                pheno: '0&1',
                revel: 0.042,
                cadd_phred: 9.265,
                cadd_raw: 0.787153,
                spliceai_pred_dp_ag: -43,
                spliceai_pred_dp_al: -50,
                spliceai_pred_dp_dg: 39,
                spliceai_pred_dp_dl: -7,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.82994,
                constraint_oe_lof: 0.67057,
                constraint_oe_lof_lower: 0.349,
                constraint_oe_lof_upper: 1.397,
                constraint_oe_mis: 1.2196,
                constraint_oe_mis_lower: 1.061,
                constraint_oe_mis_upper: 1.405,
                constraint_oe_syn: 1.1576,
                constraint_oe_syn_lower: 0.938,
                constraint_oe_syn_upper: 1.438,
                constraint_pli: 0.0017329,
                constraint_syn_z: -0.88419,
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 0.0017329,
                loeuf: 1.397,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000341290',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3829740&COSV58021873',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 9.265,
                cadd_raw: 0.787153,
                spliceai_pred_dp_ag: -43,
                spliceai_pred_dp_al: -50,
                spliceai_pred_dp_dg: 39,
                spliceai_pred_dp_dl: -7,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.61287,
                constraint_oe_lof: 1.052,
                constraint_oe_lof_lower: 0.719,
                constraint_oe_lof_upper: 1.571,
                constraint_oe_mis: 1.089,
                constraint_oe_mis_lower: 1.004,
                constraint_oe_mis_upper: 1.182,
                constraint_oe_syn: 1.2437,
                constraint_oe_syn_lower: 1.111,
                constraint_oe_syn_upper: 1.394,
                constraint_pli: 4.2343e-12,
                constraint_syn_z: -2.4954,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 4.2343e-12,
                loeuf: 1.571,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000379410',
            ensembl_gene_id: ['ENSG00000187583', 'ENSG00000187642'],
            is_exonic: 1,
            is_coding: 1,
            aa_change: 'R/P',
            polyphen_pred: ['benign'],
            polyphen_score: [0],
            sift_pred: ['tolerated'],
            sift_score: [0.31],
            impact: 'missense_variant&downstream_gene_variant',
            hgvsc: 'ENST00000379410.8:c.1460G>C',
            hgvsp: 'ENSP00000368720.3:p.Arg487Pro',
            existing_variation: 'rs3829740&COSV58021873',
            ccds: 'CCDS4.1',
            source: 'Ensembl',
            domains: 'PANTHER:PTHR46882&MobiDB_lite:mobidb-lite',
            gnomad_af: 0.6694,
            gnomad_afr_af: 0.7878,
            gnomad_amr_af: 0.6975,
            gnomad_asj_af: 0.7014,
            gnomad_eas_af: 0.8778,
            gnomad_fin_af: 0.5654,
            gnomad_nfe_af: 0.5857,
            gnomad_oth_af: 0.6657,
            gnomad_sas_af: 0.811,
            pheno: '0&1',
            revel: [0.042],
            cadd_phred: 9.265,
            cadd_raw: 0.787153,
            spliceai_pred_dp_ag: -43,
            spliceai_pred_dp_al: -50,
            spliceai_pred_dp_dg: 39,
            spliceai_pred_dp_dl: -7,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0.01],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'PLEKHN1',
            constraint_mis_z: [-1.7988, -0.55519],
            constraint_oe_lof: 0.80467,
            constraint_oe_lof_lower: 0.585,
            constraint_oe_lof_upper: [1.124, 1.686],
            constraint_oe_mis: 1.2593,
            constraint_oe_mis_lower: 1.168,
            constraint_oe_mis_upper: [1.358, 1.169],
            constraint_oe_syn: 1.2377,
            constraint_oe_syn_lower: 1.107,
            constraint_oe_syn_upper: 1.385,
            constraint_pli: 1.1549e-14,
            constraint_syn_z: [-2.4681, -2.3898],
            exomiser_ad_exgenespheno: 0,
            exomiser_xr_exgenescombi: 0,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.7855,
            gnomadv3_af_nfe: 0.5793,
            gnomadv3_af_xy: 0.6749,
            exomiser_xd_exgenescombi: 0,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.7658,
            gnomadv3_af: 0.6704,
            exomiser_mt_exgenespheno: 0,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0,
            gnomadv3_af_eas: 0.8871,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.6925,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0,
            exomiser_ad_exgenescombi: 0,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.5846,
            gnomadv3_af_fin: 0.5606,
            exomiser_ar_exgenespheno: 0,
            gnomadv3_af_sas: 0.8109,
            gnomadv3_af_amr: 0.7162,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.6661,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.6997,
            exomiser_xr_exgenespheno: 0,
            exomiser_xd_exgenespheno: 0,
            impact_severity: 'MED',
            quality: 2443.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [1, 1],
            ],
            allelic_depths: [
              [19, 23],
              [15, 22],
              [0, 33],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['PLEKHN1', 'PERM1'],
            ensembl_transcript_id: [
              'ENST00000379410',
              'ENST00000433179',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000491024',
              'ENST00000341290',
            ],
            variant_type: 'small',
            highest_af: 0.8871,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 909238,
            hg19_strand: '+',
            hg19_end: 909238,
            entrez_gene_id: ['84069', '84808'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9b8',
            variant_id: '53',
            end: 973929,
            ref: 'T',
            alt: 'C',
            start: 973929,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '14/16',
                codon_change: 'Tcc/Ccc',
                aa_change: 'S/P',
                aa_length: 511,
                polyphen_pred: 'benign',
                polyphen_score: 0,
                sift_pred: 'tolerated',
                sift_score: 0.21,
                allele: 'C',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379410.8:c.1531T>C',
                hgvsp: 'ENSP00000368720.3:p.Ser511Pro',
                cds_position: 1531,
                existing_variation: 'rs3829738&COSV58021392',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                domains: 'PANTHER:PTHR46882&MobiDB_lite:mobidb-lite',
                pheno: '1&1',
                revel: 0.034,
                cadd_phred: 9.573,
                cadd_raw: 0.818047,
                spliceai_pred_dp_ag: 39,
                spliceai_pred_dp_al: 24,
                spliceai_pred_dp_dg: -40,
                spliceai_pred_dp_dl: 39,
                spliceai_pred_ds_ag: 0.07,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000433179',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3829738&COSV58021392',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                ccds: 'CCDS76083.1',
                source: 'Ensembl',
                pheno: '1&1',
                cadd_phred: 9.573,
                cadd_raw: 0.818047,
                spliceai_pred_dp_ag: 39,
                spliceai_pred_dp_al: 24,
                spliceai_pred_dp_dg: -40,
                spliceai_pred_dp_dl: 39,
                spliceai_pred_ds_ag: 0.07,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.55519,
                constraint_oe_lof: 1.1826,
                constraint_oe_lof_lower: 0.837,
                constraint_oe_lof_upper: 1.686,
                constraint_oe_mis: 1.079,
                constraint_oe_mis_lower: 0.996,
                constraint_oe_mis_upper: 1.169,
                constraint_oe_syn: 1.2299,
                constraint_oe_syn_lower: 1.1,
                constraint_oe_syn_upper: 1.377,
                constraint_pli: 1.1667e-15,
                constraint_syn_z: -2.3898,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 1.1667e-15,
                loeuf: 1.686,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '13/15',
                codon_change: 'Tcc/Ccc',
                aa_change: 'S/P',
                aa_length: 476,
                polyphen_pred: 'benign',
                polyphen_score: 0,
                sift_pred: 'tolerated_low_confidence',
                sift_score: 0.2,
                allele: 'C',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379407.7:c.1426T>C',
                hgvsp: 'ENSP00000368717.2:p.Ser476Pro',
                cds_position: 1426,
                existing_variation: 'rs3829738&COSV58021392',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                domains: 'MobiDB_lite:mobidb-lite&PANTHER:PTHR46882',
                pheno: '1&1',
                revel: 0.034,
                cadd_phred: 9.573,
                cadd_raw: 0.818047,
                spliceai_pred_dp_ag: 39,
                spliceai_pred_dp_al: 24,
                spliceai_pred_dp_dg: -40,
                spliceai_pred_dp_dl: 39,
                spliceai_pred_ds_ag: 0.07,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '13/15',
                codon_change: 'Tcc/Ccc',
                aa_change: 'S/P',
                aa_length: 563,
                polyphen_pred: 'benign',
                polyphen_score: 0,
                sift_pred: 'tolerated',
                sift_score: 0.21,
                allele: 'C',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379409.6:c.1687T>C',
                hgvsp: 'ENSP00000368719.2:p.Ser563Pro',
                cds_position: 1687,
                existing_variation: 'rs3829738&COSV58021392',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                domains: 'MobiDB_lite:mobidb-lite&PANTHER:PTHR46882',
                pheno: '1&1',
                revel: 0.034,
                cadd_phred: 9.573,
                cadd_raw: 0.818047,
                spliceai_pred_dp_ag: 39,
                spliceai_pred_dp_al: 24,
                spliceai_pred_dp_dg: -40,
                spliceai_pred_dp_dl: 39,
                spliceai_pred_ds_ag: 0.07,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000491024',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '2/5',
                codon_change: 'Tcc/Ccc',
                aa_change: 'S/P',
                aa_length: 76,
                polyphen_pred: 'benign',
                polyphen_score: 0,
                sift_pred: 'tolerated',
                sift_score: 0.29,
                allele: 'C',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000491024.1:c.226T>C',
                hgvsp: 'ENSP00000462558.1:p.Ser76Pro',
                cds_position: 226,
                existing_variation: 'rs3829738&COSV58021392',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                domains: 'MobiDB_lite:mobidb-lite&PANTHER:PTHR46882',
                pheno: '1&1',
                revel: 0.034,
                cadd_phred: 9.573,
                cadd_raw: 0.818047,
                spliceai_pred_dp_ag: 39,
                spliceai_pred_dp_al: 24,
                spliceai_pred_dp_dg: -40,
                spliceai_pred_dp_dl: 39,
                spliceai_pred_ds_ag: 0.07,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.82994,
                constraint_oe_lof: 0.67057,
                constraint_oe_lof_lower: 0.349,
                constraint_oe_lof_upper: 1.397,
                constraint_oe_mis: 1.2196,
                constraint_oe_mis_lower: 1.061,
                constraint_oe_mis_upper: 1.405,
                constraint_oe_syn: 1.1576,
                constraint_oe_syn_lower: 0.938,
                constraint_oe_syn_upper: 1.438,
                constraint_pli: 0.0017329,
                constraint_syn_z: -0.88419,
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 0.0017329,
                loeuf: 1.397,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000341290',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3829738&COSV58021392',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                source: 'Ensembl',
                pheno: '1&1',
                cadd_phred: 9.573,
                cadd_raw: 0.818047,
                spliceai_pred_dp_ag: 39,
                spliceai_pred_dp_al: 24,
                spliceai_pred_dp_dg: -40,
                spliceai_pred_dp_dl: 39,
                spliceai_pred_ds_ag: 0.07,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.61287,
                constraint_oe_lof: 1.052,
                constraint_oe_lof_lower: 0.719,
                constraint_oe_lof_upper: 1.571,
                constraint_oe_mis: 1.089,
                constraint_oe_mis_lower: 1.004,
                constraint_oe_mis_upper: 1.182,
                constraint_oe_syn: 1.2437,
                constraint_oe_syn_lower: 1.111,
                constraint_oe_syn_upper: 1.394,
                constraint_pli: 4.2343e-12,
                constraint_syn_z: -2.4954,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 4.2343e-12,
                loeuf: 1.571,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000379410',
            ensembl_gene_id: ['ENSG00000187583', 'ENSG00000187642'],
            is_exonic: 1,
            is_coding: 1,
            aa_change: 'S/P',
            polyphen_pred: ['benign'],
            polyphen_score: [0],
            sift_pred: ['tolerated'],
            sift_score: [0.21],
            impact: 'missense_variant&downstream_gene_variant',
            hgvsc: 'ENST00000379410.8:c.1531T>C',
            hgvsp: 'ENSP00000368720.3:p.Ser511Pro',
            existing_variation: 'rs3829738&COSV58021392',
            ccds: 'CCDS4.1',
            source: 'Ensembl',
            domains: 'PANTHER:PTHR46882&MobiDB_lite:mobidb-lite',
            gnomad_af: 0.222,
            gnomad_afr_af: 0.2551,
            gnomad_amr_af: 0.2225,
            gnomad_asj_af: 0.2067,
            gnomad_eas_af: 0.2569,
            gnomad_fin_af: 0.2126,
            gnomad_nfe_af: 0.1829,
            gnomad_oth_af: 0.2387,
            gnomad_sas_af: 0.3318,
            pheno: '1&1',
            revel: [0.034],
            cadd_phred: 9.573,
            cadd_raw: 0.818047,
            spliceai_pred_dp_ag: 39,
            spliceai_pred_dp_al: 24,
            spliceai_pred_dp_dg: -40,
            spliceai_pred_dp_dl: 39,
            spliceai_pred_ds_ag: [0.07],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'PLEKHN1',
            constraint_mis_z: [-1.7988, -0.55519],
            constraint_oe_lof: 0.80467,
            constraint_oe_lof_lower: 0.585,
            constraint_oe_lof_upper: [1.124, 1.686],
            constraint_oe_mis: 1.2593,
            constraint_oe_mis_lower: 1.168,
            constraint_oe_mis_upper: [1.358, 1.169],
            constraint_oe_syn: 1.2377,
            constraint_oe_syn_lower: 1.107,
            constraint_oe_syn_upper: 1.385,
            constraint_pli: 1.1549e-14,
            constraint_syn_z: [-2.4681, -2.3898],
            exomiser_ad_exgenespheno: 0,
            exomiser_xr_exgenescombi: 0,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.2519,
            gnomadv3_af_nfe: 0.1825,
            gnomadv3_af_xy: 0.2171,
            exomiser_xd_exgenescombi: 0,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.2816,
            gnomadv3_af: 0.2151,
            exomiser_mt_exgenespheno: 0,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0,
            gnomadv3_af_eas: 0.2501,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.1954,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0,
            exomiser_ad_exgenescombi: 0,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.2945,
            gnomadv3_af_fin: 0.2119,
            exomiser_ar_exgenespheno: 0,
            gnomadv3_af_sas: 0.3371,
            gnomadv3_af_amr: 0.2079,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.2131,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.2316,
            exomiser_xr_exgenespheno: 0,
            exomiser_xd_exgenespheno: 0,
            impact_severity: 'MED',
            quality: 1080.1,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 0],
            ],
            allelic_depths: [
              [22, 22],
              [19, 16],
              [38, 0],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['PLEKHN1', 'PERM1'],
            ensembl_transcript_id: [
              'ENST00000379410',
              'ENST00000433179',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000491024',
              'ENST00000341290',
            ],
            variant_type: 'small',
            highest_af: 0.2569,
            highest_af_info: {
              source: 'gnomad V2 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 909309,
            hg19_strand: '+',
            hg19_end: 909309,
            entrez_gene_id: ['84069', '84808'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9b9',
            variant_id: '54',
            end: 973983,
            ref: 'A',
            alt: 'C',
            start: 973983,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '14/16',
                codon_change: 'Aga/Cga',
                aa_change: 'R',
                aa_length: 529,
                allele: 'C',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379410.8:c.1585A>C',
                hgvsp: 'ENSP00000368720.3:p.Arg529%3D',
                cds_position: 1585,
                existing_variation: 'rs62639990',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                domains: 'PANTHER:PTHR46882&MobiDB_lite:mobidb-lite',
                cadd_phred: 2.485,
                cadd_raw: 0.141186,
                spliceai_pred_dp_ag: 48,
                spliceai_pred_dp_al: -18,
                spliceai_pred_dp_dg: -18,
                spliceai_pred_dp_dl: -15,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.05,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000433179',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs62639990',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                ccds: 'CCDS76083.1',
                source: 'Ensembl',
                cadd_phred: 2.485,
                cadd_raw: 0.141186,
                spliceai_pred_dp_ag: 48,
                spliceai_pred_dp_al: -18,
                spliceai_pred_dp_dg: -18,
                spliceai_pred_dp_dl: -15,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.05,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.55519,
                constraint_oe_lof: 1.1826,
                constraint_oe_lof_lower: 0.837,
                constraint_oe_lof_upper: 1.686,
                constraint_oe_mis: 1.079,
                constraint_oe_mis_lower: 0.996,
                constraint_oe_mis_upper: 1.169,
                constraint_oe_syn: 1.2299,
                constraint_oe_syn_lower: 1.1,
                constraint_oe_syn_upper: 1.377,
                constraint_pli: 1.1667e-15,
                constraint_syn_z: -2.3898,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 1.1667e-15,
                loeuf: 1.686,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000341290',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs62639990',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                source: 'Ensembl',
                cadd_phred: 2.485,
                cadd_raw: 0.141186,
                spliceai_pred_dp_ag: 48,
                spliceai_pred_dp_al: -18,
                spliceai_pred_dp_dg: -18,
                spliceai_pred_dp_dl: -15,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.05,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.61287,
                constraint_oe_lof: 1.052,
                constraint_oe_lof_lower: 0.719,
                constraint_oe_lof_upper: 1.571,
                constraint_oe_mis: 1.089,
                constraint_oe_mis_lower: 1.004,
                constraint_oe_mis_upper: 1.182,
                constraint_oe_syn: 1.2437,
                constraint_oe_syn_lower: 1.111,
                constraint_oe_syn_upper: 1.394,
                constraint_pli: 4.2343e-12,
                constraint_syn_z: -2.4954,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 4.2343e-12,
                loeuf: 1.571,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '13/15',
                codon_change: 'Aga/Cga',
                aa_change: 'R',
                aa_length: 494,
                allele: 'C',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379407.7:c.1480A>C',
                hgvsp: 'ENSP00000368717.2:p.Arg494%3D',
                cds_position: 1480,
                existing_variation: 'rs62639990',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                domains: 'MobiDB_lite:mobidb-lite&PANTHER:PTHR46882',
                cadd_phred: 2.485,
                cadd_raw: 0.141186,
                spliceai_pred_dp_ag: 48,
                spliceai_pred_dp_al: -18,
                spliceai_pred_dp_dg: -18,
                spliceai_pred_dp_dl: -15,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.05,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '13/15',
                codon_change: 'Aga/Cga',
                aa_change: 'R',
                aa_length: 581,
                allele: 'C',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379409.6:c.1741A>C',
                hgvsp: 'ENSP00000368719.2:p.Arg581%3D',
                cds_position: 1741,
                existing_variation: 'rs62639990',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                domains: 'MobiDB_lite:mobidb-lite&PANTHER:PTHR46882',
                cadd_phred: 2.485,
                cadd_raw: 0.141186,
                spliceai_pred_dp_ag: 48,
                spliceai_pred_dp_al: -18,
                spliceai_pred_dp_dg: -18,
                spliceai_pred_dp_dl: -15,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.05,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000491024',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '2/5',
                codon_change: 'Aga/Cga',
                aa_change: 'R',
                aa_length: 94,
                allele: 'C',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000491024.1:c.280A>C',
                hgvsp: 'ENSP00000462558.1:p.Arg94%3D',
                cds_position: 280,
                existing_variation: 'rs62639990',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                domains: 'MobiDB_lite:mobidb-lite&PANTHER:PTHR46882',
                cadd_phred: 2.485,
                cadd_raw: 0.141186,
                spliceai_pred_dp_ag: 48,
                spliceai_pred_dp_al: -18,
                spliceai_pred_dp_dg: -18,
                spliceai_pred_dp_dl: -15,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.05,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.82994,
                constraint_oe_lof: 0.67057,
                constraint_oe_lof_lower: 0.349,
                constraint_oe_lof_upper: 1.397,
                constraint_oe_mis: 1.2196,
                constraint_oe_mis_lower: 1.061,
                constraint_oe_mis_upper: 1.405,
                constraint_oe_syn: 1.1576,
                constraint_oe_syn_lower: 0.938,
                constraint_oe_syn_upper: 1.438,
                constraint_pli: 0.0017329,
                constraint_syn_z: -0.88419,
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 0.0017329,
                loeuf: 1.397,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000379410',
            ensembl_gene_id: ['ENSG00000187583', 'ENSG00000187642'],
            is_exonic: 1,
            is_coding: 1,
            aa_change: 'R',
            impact: 'synonymous_variant&downstream_gene_variant',
            hgvsc: 'ENST00000379410.8:c.1585A>C',
            hgvsp: 'ENSP00000368720.3:p.Arg529%3D',
            existing_variation: 'rs62639990',
            ccds: 'CCDS4.1',
            source: 'Ensembl',
            domains: 'PANTHER:PTHR46882&MobiDB_lite:mobidb-lite',
            gnomad_af: 0.02859,
            gnomad_afr_af: 0.1143,
            gnomad_amr_af: 0.04562,
            gnomad_asj_af: 0.01336,
            gnomad_eas_af: 0.08809,
            gnomad_fin_af: 0.01821,
            gnomad_nfe_af: 0.008062,
            gnomad_oth_af: 0.0251,
            gnomad_sas_af: 0.01657,
            cadd_phred: 2.485,
            cadd_raw: 0.141186,
            spliceai_pred_dp_ag: 48,
            spliceai_pred_dp_al: -18,
            spliceai_pred_dp_dg: -18,
            spliceai_pred_dp_dl: -15,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0.05],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'PLEKHN1',
            constraint_mis_z: [-1.7988, -0.55519],
            constraint_oe_lof: 0.80467,
            constraint_oe_lof_lower: 0.585,
            constraint_oe_lof_upper: [1.124, 1.686],
            constraint_oe_mis: 1.2593,
            constraint_oe_mis_lower: 1.168,
            constraint_oe_mis_upper: [1.358, 1.169],
            constraint_oe_syn: 1.2377,
            constraint_oe_syn_lower: 1.107,
            constraint_oe_syn_upper: 1.385,
            constraint_pli: 1.1549e-14,
            constraint_syn_z: [-2.4681, -2.3898],
            exomiser_ad_exgenespheno: 0,
            exomiser_xr_exgenescombi: 0,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.1166,
            gnomadv3_af_nfe: 0.006777,
            gnomadv3_af_xy: 0.0451,
            exomiser_xd_exgenescombi: 0,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.0094937,
            gnomadv3_af: 0.0453,
            exomiser_mt_exgenespheno: 0,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0,
            gnomadv3_af_eas: 0.0927,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.012385,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0,
            exomiser_ad_exgenescombi: 0,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0,
            gnomadv3_af_fin: 0.017228,
            exomiser_ar_exgenespheno: 0,
            gnomadv3_af_sas: 0.017798,
            gnomadv3_af_amr: 0.047,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.0455,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.0411,
            exomiser_xr_exgenespheno: 0,
            exomiser_xd_exgenespheno: 0,
            impact_severity: 'LOW',
            quality: 999.1,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 0],
            ],
            allelic_depths: [
              [22, 26],
              [19, 15],
              [44, 0],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['PLEKHN1', 'PERM1'],
            ensembl_transcript_id: [
              'ENST00000379410',
              'ENST00000433179',
              'ENST00000341290',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000491024',
            ],
            variant_type: 'small',
            highest_af: 0.0927,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 909363,
            hg19_strand: '+',
            hg19_end: 909363,
            entrez_gene_id: ['84069', '84808'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9ba',
            variant_id: '55',
            end: 974388,
            ref: 'A',
            alt: 'G',
            start: 974388,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '15/15',
                hgvsc: 'ENST00000379410.8:c.1702+24A>G',
                existing_variation: 'rs2340593',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                cadd_phred: 1.941,
                cadd_raw: 0.082772,
                spliceai_pred_dp_ag: 7,
                spliceai_pred_dp_al: 26,
                spliceai_pred_dp_dg: -24,
                spliceai_pred_dp_dl: 1,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.29,
                spliceai_pred_ds_dg: 0.13,
                spliceai_pred_ds_dl: 0.03,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000433179',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2340593',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                ccds: 'CCDS76083.1',
                source: 'Ensembl',
                cadd_phred: 1.941,
                cadd_raw: 0.082772,
                spliceai_pred_dp_ag: 7,
                spliceai_pred_dp_al: 26,
                spliceai_pred_dp_dg: -24,
                spliceai_pred_dp_dl: 1,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.29,
                spliceai_pred_ds_dg: 0.13,
                spliceai_pred_ds_dl: 0.03,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.55519,
                constraint_oe_lof: 1.1826,
                constraint_oe_lof_lower: 0.837,
                constraint_oe_lof_upper: 1.686,
                constraint_oe_mis: 1.079,
                constraint_oe_mis_lower: 0.996,
                constraint_oe_mis_upper: 1.169,
                constraint_oe_syn: 1.2299,
                constraint_oe_syn_lower: 1.1,
                constraint_oe_syn_upper: 1.377,
                constraint_pli: 1.1667e-15,
                constraint_syn_z: -2.3898,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 1.1667e-15,
                loeuf: 1.686,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000341290',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2340593',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                source: 'Ensembl',
                cadd_phred: 1.941,
                cadd_raw: 0.082772,
                spliceai_pred_dp_ag: 7,
                spliceai_pred_dp_al: 26,
                spliceai_pred_dp_dg: -24,
                spliceai_pred_dp_dl: 1,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.29,
                spliceai_pred_ds_dg: 0.13,
                spliceai_pred_ds_dl: 0.03,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.61287,
                constraint_oe_lof: 1.052,
                constraint_oe_lof_lower: 0.719,
                constraint_oe_lof_upper: 1.571,
                constraint_oe_mis: 1.089,
                constraint_oe_mis_lower: 1.004,
                constraint_oe_mis_upper: 1.182,
                constraint_oe_syn: 1.2437,
                constraint_oe_syn_lower: 1.111,
                constraint_oe_syn_upper: 1.394,
                constraint_pli: 4.2343e-12,
                constraint_syn_z: -2.4954,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 4.2343e-12,
                loeuf: 1.571,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '14/14',
                hgvsc: 'ENST00000379407.7:c.1597+24A>G',
                existing_variation: 'rs2340593',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                cadd_phred: 1.941,
                cadd_raw: 0.082772,
                spliceai_pred_dp_ag: 7,
                spliceai_pred_dp_al: 26,
                spliceai_pred_dp_dg: -24,
                spliceai_pred_dp_dl: 1,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.29,
                spliceai_pred_ds_dg: 0.13,
                spliceai_pred_ds_dl: 0.03,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '14/14',
                hgvsc: 'ENST00000379409.6:c.1858+24A>G',
                existing_variation: 'rs2340593',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 1.941,
                cadd_raw: 0.082772,
                spliceai_pred_dp_ag: 7,
                spliceai_pred_dp_al: 26,
                spliceai_pred_dp_dg: -24,
                spliceai_pred_dp_dl: 1,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.29,
                spliceai_pred_ds_dg: 0.13,
                spliceai_pred_ds_dl: 0.03,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000491024',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '3/4',
                hgvsc: 'ENST00000491024.1:c.397+24A>G',
                existing_variation: 'rs2340593',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 1.941,
                cadd_raw: 0.082772,
                spliceai_pred_dp_ag: 7,
                spliceai_pred_dp_al: 26,
                spliceai_pred_dp_dg: -24,
                spliceai_pred_dp_dl: 1,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.29,
                spliceai_pred_ds_dg: 0.13,
                spliceai_pred_ds_dl: 0.03,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.82994,
                constraint_oe_lof: 0.67057,
                constraint_oe_lof_lower: 0.349,
                constraint_oe_lof_upper: 1.397,
                constraint_oe_mis: 1.2196,
                constraint_oe_mis_lower: 1.061,
                constraint_oe_mis_upper: 1.405,
                constraint_oe_syn: 1.1576,
                constraint_oe_syn_lower: 0.938,
                constraint_oe_syn_upper: 1.438,
                constraint_pli: 0.0017329,
                constraint_syn_z: -0.88419,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 0.0017329,
                loeuf: 1.397,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000379410',
            ensembl_gene_id: ['ENSG00000187583', 'ENSG00000187642'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant&downstream_gene_variant',
            hgvsc: 'ENST00000379410.8:c.1702+24A>G',
            existing_variation: 'rs2340593',
            ccds: 'CCDS4.1',
            source: 'Ensembl',
            gnomad_af: 1,
            gnomad_afr_af: 0.9994,
            gnomad_amr_af: 1,
            gnomad_asj_af: 1,
            gnomad_eas_af: 1,
            gnomad_fin_af: 1,
            gnomad_nfe_af: 1,
            gnomad_oth_af: 1,
            gnomad_sas_af: 1,
            cadd_phred: 1.941,
            cadd_raw: 0.082772,
            spliceai_pred_dp_ag: 7,
            spliceai_pred_dp_al: 26,
            spliceai_pred_dp_dg: -24,
            spliceai_pred_dp_dl: 1,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0.29],
            spliceai_pred_ds_dg: [0.13],
            spliceai_pred_ds_dl: [0.03],
            spliceai_pred_symbol: 'PLEKHN1',
            constraint_mis_z: [-1.7988, -0.55519],
            constraint_oe_lof: 0.80467,
            constraint_oe_lof_lower: 0.585,
            constraint_oe_lof_upper: [1.124, 1.686],
            constraint_oe_mis: 1.2593,
            constraint_oe_mis_lower: 1.168,
            constraint_oe_mis_upper: [1.358, 1.169],
            constraint_oe_syn: 1.2377,
            constraint_oe_syn_lower: 1.107,
            constraint_oe_syn_upper: 1.385,
            constraint_pli: 1.1549e-14,
            constraint_syn_z: [-2.4681, -2.3898],
            exomiser_ad_exgenespheno: 0,
            exomiser_xr_exgenescombi: 0,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.9997,
            gnomadv3_af_nfe: 1,
            gnomadv3_af_xy: 0.9999,
            exomiser_xd_exgenescombi: 0,
            clnsigconf: 'None',
            gnomadv3_af_mid: 1,
            gnomadv3_af: 0.9999,
            exomiser_mt_exgenespheno: 0,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0,
            gnomadv3_af_eas: 1,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 1,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0,
            exomiser_ad_exgenescombi: 0,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 1,
            gnomadv3_af_fin: 1,
            exomiser_ar_exgenespheno: 0,
            gnomadv3_af_sas: 1,
            gnomadv3_af_amr: 1,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.9999,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 1,
            exomiser_xr_exgenespheno: 0,
            exomiser_xd_exgenespheno: 0,
            impact_severity: 'LOW',
            quality: 5223.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 48],
              [0, 45],
              [0, 59],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['PLEKHN1', 'PERM1'],
            ensembl_transcript_id: [
              'ENST00000379410',
              'ENST00000433179',
              'ENST00000341290',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000491024',
            ],
            variant_type: 'small',
            highest_af: 1,
            highest_af_info: {
              source: 'gnomad V2 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 909768,
            hg19_strand: '+',
            hg19_end: 909768,
            entrez_gene_id: ['84069', '84808'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9bb',
            variant_id: '56',
            end: 975014,
            ref: 'C',
            alt: 'T',
            start: 975014,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '16/16',
                allele: 'T',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379410.8:c.*439C>T',
                existing_variation: 'rs28477686',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                cadd_phred: 2.564,
                cadd_raw: 0.148898,
                spliceai_pred_dp_ag: 10,
                spliceai_pred_dp_al: -4,
                spliceai_pred_dp_dg: -34,
                spliceai_pred_dp_dl: 13,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000433179',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'T',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs28477686',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                ccds: 'CCDS76083.1',
                source: 'Ensembl',
                cadd_phred: 2.564,
                cadd_raw: 0.148898,
                spliceai_pred_dp_ag: 10,
                spliceai_pred_dp_al: -4,
                spliceai_pred_dp_dg: -34,
                spliceai_pred_dp_dl: 13,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.55519,
                constraint_oe_lof: 1.1826,
                constraint_oe_lof_lower: 0.837,
                constraint_oe_lof_upper: 1.686,
                constraint_oe_mis: 1.079,
                constraint_oe_mis_lower: 0.996,
                constraint_oe_mis_upper: 1.169,
                constraint_oe_syn: 1.2299,
                constraint_oe_syn_lower: 1.1,
                constraint_oe_syn_upper: 1.377,
                constraint_pli: 1.1667e-15,
                constraint_syn_z: -2.3898,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 1.1667e-15,
                loeuf: 1.686,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000341290',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs28477686',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                source: 'Ensembl',
                cadd_phred: 2.564,
                cadd_raw: 0.148898,
                spliceai_pred_dp_ag: 10,
                spliceai_pred_dp_al: -4,
                spliceai_pred_dp_dg: -34,
                spliceai_pred_dp_dl: 13,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.61287,
                constraint_oe_lof: 1.052,
                constraint_oe_lof_lower: 0.719,
                constraint_oe_lof_upper: 1.571,
                constraint_oe_mis: 1.089,
                constraint_oe_mis_lower: 1.004,
                constraint_oe_mis_upper: 1.182,
                constraint_oe_syn: 1.2437,
                constraint_oe_syn_lower: 1.111,
                constraint_oe_syn_upper: 1.394,
                constraint_pli: 4.2343e-12,
                constraint_syn_z: -2.4954,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 4.2343e-12,
                loeuf: 1.571,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs28477686',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                cadd_phred: 2.564,
                cadd_raw: 0.148898,
                spliceai_pred_dp_ag: 10,
                spliceai_pred_dp_al: -4,
                spliceai_pred_dp_dg: -34,
                spliceai_pred_dp_dl: 13,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs28477686',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 2.564,
                cadd_raw: 0.148898,
                spliceai_pred_dp_ag: 10,
                spliceai_pred_dp_al: -4,
                spliceai_pred_dp_dg: -34,
                spliceai_pred_dp_dl: 13,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000491024',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '4/4',
                hgvsc: 'ENST00000491024.1:c.*13+426C>T',
                existing_variation: 'rs28477686',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 2.564,
                cadd_raw: 0.148898,
                spliceai_pred_dp_ag: 10,
                spliceai_pred_dp_al: -4,
                spliceai_pred_dp_dg: -34,
                spliceai_pred_dp_dl: 13,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.82994,
                constraint_oe_lof: 0.67057,
                constraint_oe_lof_lower: 0.349,
                constraint_oe_lof_upper: 1.397,
                constraint_oe_mis: 1.2196,
                constraint_oe_mis_lower: 1.061,
                constraint_oe_mis_upper: 1.405,
                constraint_oe_syn: 1.1576,
                constraint_oe_syn_lower: 0.938,
                constraint_oe_syn_upper: 1.438,
                constraint_pli: 0.0017329,
                constraint_syn_z: -0.88419,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 0.0017329,
                loeuf: 1.397,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000379410',
            ensembl_gene_id: ['ENSG00000187583', 'ENSG00000187642'],
            is_exonic: 1,
            is_coding: 0,
            impact: '3_prime_UTR_variant&downstream_gene_variant',
            hgvsc: 'ENST00000379410.8:c.*439C>T',
            existing_variation: 'rs28477686',
            ccds: 'CCDS4.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            cadd_phred: 2.564,
            cadd_raw: 0.148898,
            spliceai_pred_dp_ag: 10,
            spliceai_pred_dp_al: -4,
            spliceai_pred_dp_dg: -34,
            spliceai_pred_dp_dl: 13,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'PLEKHN1',
            constraint_mis_z: [-1.7988, -0.55519],
            constraint_oe_lof: 0.80467,
            constraint_oe_lof_lower: 0.585,
            constraint_oe_lof_upper: [1.124, 1.686],
            constraint_oe_mis: 1.2593,
            constraint_oe_mis_lower: 1.168,
            constraint_oe_mis_upper: [1.358, 1.169],
            constraint_oe_syn: 1.2377,
            constraint_oe_syn_lower: 1.107,
            constraint_oe_syn_upper: 1.385,
            constraint_pli: 1.1549e-14,
            constraint_syn_z: [-2.4681, -2.3898],
            exomiser_ad_exgenespheno: 0,
            exomiser_xr_exgenescombi: 0,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.5255,
            gnomadv3_af_nfe: 0.3259,
            gnomadv3_af_xy: 0.4019,
            exomiser_xd_exgenescombi: 0,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.4108,
            gnomadv3_af: 0.3989,
            exomiser_mt_exgenespheno: 0,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0,
            gnomadv3_af_eas: 0.4351,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.2872,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0,
            exomiser_ad_exgenescombi: 0,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.4659,
            gnomadv3_af_fin: 0.4042,
            exomiser_ar_exgenespheno: 0,
            gnomadv3_af_sas: 0.4252,
            gnomadv3_af_amr: 0.3803,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.3961,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.3931,
            exomiser_xr_exgenespheno: 0,
            exomiser_xd_exgenespheno: 0,
            impact_severity: 'LOW',
            quality: 1211.9,
            genotypes_index: [
              [0, 1],
              [1, 1],
              [0, 0],
            ],
            allelic_depths: [
              [15, 15],
              [0, 24],
              [43, 0],
            ],
            genotype_qualities: [[99, 72, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['PLEKHN1', 'PERM1'],
            ensembl_transcript_id: [
              'ENST00000379410',
              'ENST00000433179',
              'ENST00000341290',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000491024',
            ],
            variant_type: 'small',
            highest_af: 0.4351,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 910394,
            hg19_strand: '+',
            hg19_end: 910394,
            entrez_gene_id: ['84069', '84808'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9bc',
            variant_id: '57',
            end: 975058,
            ref: 'A',
            alt: 'G',
            start: 975058,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '16/16',
                allele: 'G',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379410.8:c.*483A>G',
                existing_variation: 'rs6685581',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                cadd_phred: 0.267,
                cadd_raw: -0.331894,
                spliceai_pred_dp_ag: 47,
                spliceai_pred_dp_al: 7,
                spliceai_pred_dp_dg: -48,
                spliceai_pred_dp_dl: -31,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000433179',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6685581',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                ccds: 'CCDS76083.1',
                source: 'Ensembl',
                cadd_phred: 0.267,
                cadd_raw: -0.331894,
                spliceai_pred_dp_ag: 47,
                spliceai_pred_dp_al: 7,
                spliceai_pred_dp_dg: -48,
                spliceai_pred_dp_dl: -31,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.55519,
                constraint_oe_lof: 1.1826,
                constraint_oe_lof_lower: 0.837,
                constraint_oe_lof_upper: 1.686,
                constraint_oe_mis: 1.079,
                constraint_oe_mis_lower: 0.996,
                constraint_oe_mis_upper: 1.169,
                constraint_oe_syn: 1.2299,
                constraint_oe_syn_lower: 1.1,
                constraint_oe_syn_upper: 1.377,
                constraint_pli: 1.1667e-15,
                constraint_syn_z: -2.3898,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 1.1667e-15,
                loeuf: 1.686,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000341290',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6685581',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                source: 'Ensembl',
                cadd_phred: 0.267,
                cadd_raw: -0.331894,
                spliceai_pred_dp_ag: 47,
                spliceai_pred_dp_al: 7,
                spliceai_pred_dp_dg: -48,
                spliceai_pred_dp_dl: -31,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.61287,
                constraint_oe_lof: 1.052,
                constraint_oe_lof_lower: 0.719,
                constraint_oe_lof_upper: 1.571,
                constraint_oe_mis: 1.089,
                constraint_oe_mis_lower: 1.004,
                constraint_oe_mis_upper: 1.182,
                constraint_oe_syn: 1.2437,
                constraint_oe_syn_lower: 1.111,
                constraint_oe_syn_upper: 1.394,
                constraint_pli: 4.2343e-12,
                constraint_syn_z: -2.4954,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 4.2343e-12,
                loeuf: 1.571,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6685581',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                cadd_phred: 0.267,
                cadd_raw: -0.331894,
                spliceai_pred_dp_ag: 47,
                spliceai_pred_dp_al: 7,
                spliceai_pred_dp_dg: -48,
                spliceai_pred_dp_dl: -31,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs6685581',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 0.267,
                cadd_raw: -0.331894,
                spliceai_pred_dp_ag: 47,
                spliceai_pred_dp_al: 7,
                spliceai_pred_dp_dg: -48,
                spliceai_pred_dp_dl: -31,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000491024',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '4/4',
                hgvsc: 'ENST00000491024.1:c.*13+470A>G',
                existing_variation: 'rs6685581',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 0.267,
                cadd_raw: -0.331894,
                spliceai_pred_dp_ag: 47,
                spliceai_pred_dp_al: 7,
                spliceai_pred_dp_dg: -48,
                spliceai_pred_dp_dl: -31,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PLEKHN1',
                constraint_mis_z: -0.82994,
                constraint_oe_lof: 0.67057,
                constraint_oe_lof_lower: 0.349,
                constraint_oe_lof_upper: 1.397,
                constraint_oe_mis: 1.2196,
                constraint_oe_mis_lower: 1.061,
                constraint_oe_mis_upper: 1.405,
                constraint_oe_syn: 1.1576,
                constraint_oe_syn_lower: 0.938,
                constraint_oe_syn_upper: 1.438,
                constraint_pli: 0.0017329,
                constraint_syn_z: -0.88419,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 0.0017329,
                loeuf: 1.397,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000379410',
            ensembl_gene_id: ['ENSG00000187583', 'ENSG00000187642'],
            is_exonic: 1,
            is_coding: 0,
            impact: '3_prime_UTR_variant&downstream_gene_variant',
            hgvsc: 'ENST00000379410.8:c.*483A>G',
            existing_variation: 'rs6685581',
            ccds: 'CCDS4.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            cadd_phred: 0.267,
            cadd_raw: -0.331894,
            spliceai_pred_dp_ag: 47,
            spliceai_pred_dp_al: 7,
            spliceai_pred_dp_dg: -48,
            spliceai_pred_dp_dl: -31,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0.01],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'PLEKHN1',
            constraint_mis_z: [-1.7988, -0.55519],
            constraint_oe_lof: 0.80467,
            constraint_oe_lof_lower: 0.585,
            constraint_oe_lof_upper: [1.124, 1.686],
            constraint_oe_mis: 1.2593,
            constraint_oe_mis_lower: 1.168,
            constraint_oe_mis_upper: [1.358, 1.169],
            constraint_oe_syn: 1.2377,
            constraint_oe_syn_lower: 1.107,
            constraint_oe_syn_upper: 1.385,
            constraint_pli: 1.1549e-14,
            constraint_syn_z: [-2.4681, -2.3898],
            exomiser_ad_exgenespheno: 0,
            exomiser_xr_exgenescombi: 0,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.8161,
            gnomadv3_af_nfe: 0.974,
            gnomadv3_af_xy: 0.9333,
            exomiser_xd_exgenescombi: 0,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9747,
            gnomadv3_af: 0.9318,
            exomiser_mt_exgenespheno: 0,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0,
            gnomadv3_af_eas: 0.9996,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9767,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0,
            exomiser_ad_exgenescombi: 0,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9857,
            gnomadv3_af_fin: 0.9945,
            exomiser_ar_exgenespheno: 0,
            gnomadv3_af_sas: 0.9975,
            gnomadv3_af_amr: 0.9528,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.9304,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.9434,
            exomiser_xr_exgenespheno: 0,
            exomiser_xd_exgenespheno: 0,
            impact_severity: 'LOW',
            quality: 2421.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 19],
              [0, 26],
              [0, 39],
            ],
            genotype_qualities: [[99, 78, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['PLEKHN1', 'PERM1'],
            ensembl_transcript_id: [
              'ENST00000379410',
              'ENST00000433179',
              'ENST00000341290',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000491024',
            ],
            variant_type: 'small',
            highest_af: 0.9996,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 910438,
            hg19_strand: '+',
            hg19_end: 910438,
            entrez_gene_id: ['84069', '84808'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9bd',
            variant_id: '58',
            end: 975132,
            ref: 'G',
            alt: 'C',
            start: 975132,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '16/16',
                allele: 'C',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379410.8:c.*557G>C',
                existing_variation: 'rs74424855',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                cadd_phred: 2.972,
                cadd_raw: 0.18659,
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000433179',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs74424855',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                ccds: 'CCDS76083.1',
                source: 'Ensembl',
                cadd_phred: 2.972,
                cadd_raw: 0.18659,
                constraint_mis_z: -0.55519,
                constraint_oe_lof: 1.1826,
                constraint_oe_lof_lower: 0.837,
                constraint_oe_lof_upper: 1.686,
                constraint_oe_mis: 1.079,
                constraint_oe_mis_lower: 0.996,
                constraint_oe_mis_upper: 1.169,
                constraint_oe_syn: 1.2299,
                constraint_oe_syn_lower: 1.1,
                constraint_oe_syn_upper: 1.377,
                constraint_pli: 1.1667e-15,
                constraint_syn_z: -2.3898,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 1.1667e-15,
                loeuf: 1.686,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000341290',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs74424855',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                source: 'Ensembl',
                cadd_phred: 2.972,
                cadd_raw: 0.18659,
                constraint_mis_z: -0.61287,
                constraint_oe_lof: 1.052,
                constraint_oe_lof_lower: 0.719,
                constraint_oe_lof_upper: 1.571,
                constraint_oe_mis: 1.089,
                constraint_oe_mis_lower: 1.004,
                constraint_oe_mis_upper: 1.182,
                constraint_oe_syn: 1.2437,
                constraint_oe_syn_lower: 1.111,
                constraint_oe_syn_upper: 1.394,
                constraint_pli: 4.2343e-12,
                constraint_syn_z: -2.4954,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 4.2343e-12,
                loeuf: 1.571,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs74424855',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                cadd_phred: 2.972,
                cadd_raw: 0.18659,
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs74424855',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 2.972,
                cadd_raw: 0.18659,
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000491024',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '4/4',
                hgvsc: 'ENST00000491024.1:c.*13+544G>C',
                existing_variation: 'rs74424855',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 2.972,
                cadd_raw: 0.18659,
                constraint_mis_z: -0.82994,
                constraint_oe_lof: 0.67057,
                constraint_oe_lof_lower: 0.349,
                constraint_oe_lof_upper: 1.397,
                constraint_oe_mis: 1.2196,
                constraint_oe_mis_lower: 1.061,
                constraint_oe_mis_upper: 1.405,
                constraint_oe_syn: 1.1576,
                constraint_oe_syn_lower: 0.938,
                constraint_oe_syn_upper: 1.438,
                constraint_pli: 0.0017329,
                constraint_syn_z: -0.88419,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 0.0017329,
                loeuf: 1.397,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000379410',
            ensembl_gene_id: ['ENSG00000187583', 'ENSG00000187642'],
            is_exonic: 1,
            is_coding: 0,
            impact: '3_prime_UTR_variant&downstream_gene_variant',
            hgvsc: 'ENST00000379410.8:c.*557G>C',
            existing_variation: 'rs74424855',
            ccds: 'CCDS4.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            cadd_phred: 2.972,
            cadd_raw: 0.18659,
            constraint_mis_z: [-1.7988, -0.55519],
            constraint_oe_lof: 0.80467,
            constraint_oe_lof_lower: 0.585,
            constraint_oe_lof_upper: [1.124, 1.686],
            constraint_oe_mis: 1.2593,
            constraint_oe_mis_lower: 1.168,
            constraint_oe_mis_upper: [1.358, 1.169],
            constraint_oe_syn: 1.2377,
            constraint_oe_syn_lower: 1.107,
            constraint_oe_syn_upper: 1.385,
            constraint_pli: 1.1549e-14,
            constraint_syn_z: [-2.4681, -2.3898],
            exomiser_ad_exgenespheno: 0,
            exomiser_xr_exgenescombi: 0,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.016917,
            gnomadv3_af_nfe: 0.01848,
            gnomadv3_af_xy: 0.0268,
            exomiser_xd_exgenescombi: 0,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.0316,
            gnomadv3_af: 0.0251,
            exomiser_mt_exgenespheno: 0,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0,
            gnomadv3_af_eas: 0.0966,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.017311,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0,
            exomiser_ad_exgenescombi: 0,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0,
            gnomadv3_af_fin: 0.0565,
            exomiser_ar_exgenespheno: 0,
            gnomadv3_af_sas: 0.018604,
            gnomadv3_af_amr: 0.0366,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.0235,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.0249,
            exomiser_xr_exgenespheno: 0,
            exomiser_xd_exgenespheno: 0,
            impact_severity: 'LOW',
            quality: 865.1,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 0],
            ],
            allelic_depths: [
              [10, 13],
              [21, 19],
              [38, 0],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['PLEKHN1', 'PERM1'],
            ensembl_transcript_id: [
              'ENST00000379410',
              'ENST00000433179',
              'ENST00000341290',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000491024',
            ],
            variant_type: 'small',
            highest_af: 0.0966,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 910512,
            hg19_strand: '+',
            hg19_end: 910512,
            entrez_gene_id: ['84069', '84808'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9be',
            variant_id: '59',
            end: 975152,
            ref: 'GAGAGA',
            alt: 'G',
            start: 975147,
            chrom: 'chr1',
            type: 'indel',
            gene_objs: [
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '16/16',
                allele: '-',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379410.8:c.*585_*589del',
                existing_variation: 'rs141996759',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000433179',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: '-',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs141996759',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                ccds: 'CCDS76083.1',
                source: 'Ensembl',
                constraint_mis_z: -0.55519,
                constraint_oe_lof: 1.1826,
                constraint_oe_lof_lower: 0.837,
                constraint_oe_lof_upper: 1.686,
                constraint_oe_mis: 1.079,
                constraint_oe_mis_lower: 0.996,
                constraint_oe_mis_upper: 1.169,
                constraint_oe_syn: 1.2299,
                constraint_oe_syn_lower: 1.1,
                constraint_oe_syn_upper: 1.377,
                constraint_pli: 1.1667e-15,
                constraint_syn_z: -2.3898,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 1.1667e-15,
                loeuf: 1.686,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000341290',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs141996759',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                source: 'Ensembl',
                constraint_mis_z: -0.61287,
                constraint_oe_lof: 1.052,
                constraint_oe_lof_lower: 0.719,
                constraint_oe_lof_upper: 1.571,
                constraint_oe_mis: 1.089,
                constraint_oe_mis_lower: 1.004,
                constraint_oe_mis_upper: 1.182,
                constraint_oe_syn: 1.2437,
                constraint_oe_syn_lower: 1.111,
                constraint_oe_syn_upper: 1.394,
                constraint_pli: 4.2343e-12,
                constraint_syn_z: -2.4954,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 4.2343e-12,
                loeuf: 1.571,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs141996759',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs141996759',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000491024',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '4/4',
                hgvsc: 'ENST00000491024.1:c.*14-552_*14-548del',
                existing_variation: 'rs141996759',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                constraint_mis_z: -0.82994,
                constraint_oe_lof: 0.67057,
                constraint_oe_lof_lower: 0.349,
                constraint_oe_lof_upper: 1.397,
                constraint_oe_mis: 1.2196,
                constraint_oe_mis_lower: 1.061,
                constraint_oe_mis_upper: 1.405,
                constraint_oe_syn: 1.1576,
                constraint_oe_syn_lower: 0.938,
                constraint_oe_syn_upper: 1.438,
                constraint_pli: 0.0017329,
                constraint_syn_z: -0.88419,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 0.0017329,
                loeuf: 1.397,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000379410',
            ensembl_gene_id: ['ENSG00000187583', 'ENSG00000187642'],
            is_exonic: 1,
            is_coding: 0,
            impact: '3_prime_UTR_variant&downstream_gene_variant',
            hgvsc: 'ENST00000379410.8:c.*585_*589del',
            existing_variation: 'rs141996759',
            ccds: 'CCDS4.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            constraint_mis_z: [-1.7988, -0.55519],
            constraint_oe_lof: 0.80467,
            constraint_oe_lof_lower: 0.585,
            constraint_oe_lof_upper: [1.124, 1.686],
            constraint_oe_mis: 1.2593,
            constraint_oe_mis_lower: 1.168,
            constraint_oe_mis_upper: [1.358, 1.169],
            constraint_oe_syn: 1.2377,
            constraint_oe_syn_lower: 1.107,
            constraint_oe_syn_upper: 1.385,
            constraint_pli: 1.1549e-14,
            constraint_syn_z: [-2.4681, -2.3898],
            exomiser_ad_exgenespheno: 0,
            exomiser_xr_exgenescombi: 0,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.0628,
            gnomadv3_af_nfe: 0.02,
            gnomadv3_af_xy: 0.042,
            exomiser_xd_exgenescombi: 0,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.0316,
            gnomadv3_af: 0.0409,
            exomiser_mt_exgenespheno: 0,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0,
            gnomadv3_af_eas: 0.1192,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.01761,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0,
            exomiser_ad_exgenescombi: 0,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.0765,
            gnomadv3_af_fin: 0.0598,
            exomiser_ar_exgenespheno: 0,
            gnomadv3_af_sas: 0.0207,
            gnomadv3_af_amr: 0.0446,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.0398,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.0423,
            exomiser_xr_exgenespheno: 0,
            exomiser_xd_exgenespheno: 0,
            impact_severity: 'LOW',
            quality: 1255.1,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 0],
            ],
            allelic_depths: [
              [8, 13],
              [18, 19],
              [37, 0],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['PLEKHN1', 'PERM1'],
            ensembl_transcript_id: [
              'ENST00000379410',
              'ENST00000433179',
              'ENST00000341290',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000491024',
            ],
            variant_type: 'small',
            highest_af: 0.1192,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 910527,
            hg19_strand: '+',
            hg19_end: 910532,
            entrez_gene_id: ['84069', '84808'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9bf',
            variant_id: '60',
            end: 975523,
            ref: 'T',
            alt: 'C',
            start: 975523,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379410',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '16/16',
                allele: 'C',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379410.8:c.*948T>C',
                existing_variation: 'rs4970429',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                mane_select: 'NM_032129.3',
                ccds: 'CCDS4.1',
                source: 'Ensembl',
                cadd_phred: 0.182,
                cadd_raw: -0.411871,
                spliceai_pred_dp_ag: 6,
                spliceai_pred_dp_al: -2,
                spliceai_pred_dp_dg: -17,
                spliceai_pred_dp_dl: -11,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PERM1',
                constraint_mis_z: -1.7988,
                constraint_oe_lof: 0.80467,
                constraint_oe_lof_lower: 0.585,
                constraint_oe_lof_upper: 1.124,
                constraint_oe_mis: 1.2593,
                constraint_oe_mis_lower: 1.168,
                constraint_oe_mis_upper: 1.358,
                constraint_oe_syn: 1.2377,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.385,
                constraint_pli: 1.1549e-14,
                constraint_syn_z: -2.4681,
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.1549e-14,
                loeuf: 1.124,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000433179',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '3/3',
                allele: 'C',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000433179.3:c.*649A>G',
                existing_variation: 'rs4970429',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                ccds: 'CCDS76083.1',
                source: 'Ensembl',
                cadd_phred: 0.182,
                cadd_raw: -0.411871,
                spliceai_pred_dp_ag: 6,
                spliceai_pred_dp_al: -2,
                spliceai_pred_dp_dg: -17,
                spliceai_pred_dp_dl: -11,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PERM1',
                constraint_mis_z: -0.55519,
                constraint_oe_lof: 1.1826,
                constraint_oe_lof_lower: 0.837,
                constraint_oe_lof_upper: 1.686,
                constraint_oe_mis: 1.079,
                constraint_oe_mis_lower: 0.996,
                constraint_oe_mis_upper: 1.169,
                constraint_oe_syn: 1.2299,
                constraint_oe_syn_lower: 1.1,
                constraint_oe_syn_upper: 1.377,
                constraint_pli: 1.1667e-15,
                constraint_syn_z: -2.3898,
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 1.1667e-15,
                loeuf: 1.686,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PERM1',
                ensembl_gene_id: 'ENSG00000187642',
                transcript: 'ENST00000341290',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '5/5',
                allele: 'C',
                impact: ['3_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000341290.6:c.*649A>G',
                existing_variation: 'rs4970429',
                strand: -1,
                hgnc_id: 'HGNC:28208',
                source: 'Ensembl',
                cadd_phred: 0.182,
                cadd_raw: -0.411871,
                spliceai_pred_dp_ag: 6,
                spliceai_pred_dp_al: -2,
                spliceai_pred_dp_dg: -17,
                spliceai_pred_dp_dl: -11,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PERM1',
                constraint_mis_z: -0.61287,
                constraint_oe_lof: 1.052,
                constraint_oe_lof_lower: 0.719,
                constraint_oe_lof_upper: 1.571,
                constraint_oe_mis: 1.089,
                constraint_oe_mis_lower: 1.004,
                constraint_oe_mis_upper: 1.182,
                constraint_oe_syn: 1.2437,
                constraint_oe_syn_lower: 1.111,
                constraint_oe_syn_upper: 1.394,
                constraint_pli: 4.2343e-12,
                constraint_syn_z: -2.4954,
                impact_so: '3_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PERM1',
                entrez_gene_id: '84808',
                p_li: 4.2343e-12,
                loeuf: 1.571,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379407',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970429',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                ccds: 'CCDS53256.1',
                source: 'Ensembl',
                cadd_phred: 0.182,
                cadd_raw: -0.411871,
                spliceai_pred_dp_ag: 6,
                spliceai_pred_dp_al: -2,
                spliceai_pred_dp_dg: -17,
                spliceai_pred_dp_dl: -11,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PERM1',
                constraint_mis_z: -1.802,
                constraint_oe_lof: 0.76801,
                constraint_oe_lof_lower: 0.552,
                constraint_oe_lof_upper: 1.088,
                constraint_oe_mis: 1.2673,
                constraint_oe_mis_lower: 1.173,
                constraint_oe_mis_upper: 1.369,
                constraint_oe_syn: 1.2595,
                constraint_oe_syn_lower: 1.125,
                constraint_oe_syn_upper: 1.412,
                constraint_pli: 4.2629e-13,
                constraint_syn_z: -2.6338,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 4.2629e-13,
                loeuf: 1.088,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000379409',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4970429',
                strand: 1,
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 0.182,
                cadd_raw: -0.411871,
                spliceai_pred_dp_ag: 6,
                spliceai_pred_dp_al: -2,
                spliceai_pred_dp_dg: -17,
                spliceai_pred_dp_dl: -11,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PERM1',
                constraint_mis_z: -1.7344,
                constraint_oe_lof: 0.71936,
                constraint_oe_lof_lower: 0.517,
                constraint_oe_lof_upper: 1.019,
                constraint_oe_mis: 1.2405,
                constraint_oe_mis_lower: 1.153,
                constraint_oe_mis_upper: 1.335,
                constraint_oe_syn: 1.2324,
                constraint_oe_syn_lower: 1.107,
                constraint_oe_syn_upper: 1.374,
                constraint_pli: 1.6783e-12,
                constraint_syn_z: -2.5115,
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 1.6783e-12,
                loeuf: 1.019,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'PLEKHN1',
                ensembl_gene_id: 'ENSG00000187583',
                transcript: 'ENST00000491024',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '4/4',
                hgvsc: 'ENST00000491024.1:c.*14-189T>C',
                existing_variation: 'rs4970429',
                strand: 1,
                flags: 'cds_start_NF',
                hgnc_id: 'HGNC:25284',
                source: 'Ensembl',
                cadd_phred: 0.182,
                cadd_raw: -0.411871,
                spliceai_pred_dp_ag: 6,
                spliceai_pred_dp_al: -2,
                spliceai_pred_dp_dg: -17,
                spliceai_pred_dp_dl: -11,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'PERM1',
                constraint_mis_z: -0.82994,
                constraint_oe_lof: 0.67057,
                constraint_oe_lof_lower: 0.349,
                constraint_oe_lof_upper: 1.397,
                constraint_oe_mis: 1.2196,
                constraint_oe_mis_lower: 1.061,
                constraint_oe_mis_upper: 1.405,
                constraint_oe_syn: 1.1576,
                constraint_oe_syn_lower: 0.938,
                constraint_oe_syn_upper: 1.438,
                constraint_pli: 0.0017329,
                constraint_syn_z: -0.88419,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'PLEKHN1',
                entrez_gene_id: '84069',
                p_haplo: 0.07047,
                p_triplo: 0.39811,
                p_li: 0.0017329,
                loeuf: 1.397,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000379410',
            ensembl_gene_id: ['ENSG00000187583', 'ENSG00000187642'],
            is_exonic: 1,
            is_coding: 0,
            impact: '3_prime_UTR_variant',
            hgvsc: 'ENST00000379410.8:c.*948T>C',
            existing_variation: 'rs4970429',
            ccds: 'CCDS4.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            cadd_phred: 0.182,
            cadd_raw: -0.411871,
            spliceai_pred_dp_ag: 6,
            spliceai_pred_dp_al: -2,
            spliceai_pred_dp_dg: -17,
            spliceai_pred_dp_dl: -11,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'PERM1',
            constraint_mis_z: [-1.7988, -0.55519],
            constraint_oe_lof: 0.80467,
            constraint_oe_lof_lower: 0.585,
            constraint_oe_lof_upper: [1.124, 1.686],
            constraint_oe_mis: 1.2593,
            constraint_oe_mis_lower: 1.168,
            constraint_oe_mis_upper: [1.358, 1.169],
            constraint_oe_syn: 1.2377,
            constraint_oe_syn_lower: 1.107,
            constraint_oe_syn_upper: 1.385,
            constraint_pli: 1.1549e-14,
            constraint_syn_z: [-2.4681, -2.3898],
            exomiser_ad_exgenespheno: 0,
            exomiser_xr_exgenescombi: 0,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.7787,
            gnomadv3_af_nfe: 0.9683,
            gnomadv3_af_xy: 0.9176,
            exomiser_xd_exgenescombi: 0,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9335,
            gnomadv3_af: 0.916,
            exomiser_mt_exgenespheno: 0,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0,
            gnomadv3_af_eas: 0.9996,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9237,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0,
            exomiser_ad_exgenescombi: 0,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9759,
            gnomadv3_af_fin: 0.9916,
            exomiser_ar_exgenespheno: 0,
            gnomadv3_af_sas: 0.982,
            gnomadv3_af_amr: 0.9468,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.9144,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.924,
            exomiser_xr_exgenespheno: 0,
            exomiser_xd_exgenespheno: 0,
            impact_severity: 'LOW',
            quality: 4339.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 52],
              [0, 40],
              [0, 53],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['PLEKHN1', 'PERM1'],
            ensembl_transcript_id: [
              'ENST00000379410',
              'ENST00000433179',
              'ENST00000341290',
              'ENST00000379407',
              'ENST00000379409',
              'ENST00000491024',
            ],
            variant_type: 'small',
            highest_af: 0.9996,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 910903,
            hg19_strand: '+',
            hg19_end: 910903,
            entrez_gene_id: ['84069', '84808'],
          },
          {
            allelic_depths: [],
            genotype_qualities: [],
            is_read: false,
            _id: '64e5a4cff99efe31718b84a4',
            chrom: 'chr1',
            start: 996282,
            end: 996348,
            len: 66,
            ref: 'G',
            alt: '<DUP>',
            p_lof: 'False',
            constraint_pli: '',
            constraint_oe_lof_upper: [],
            variant_type: 'structural',
            type: 'DUP',
            impact: '',
            gene_objs: [],
            highest_af: 0.959,
            highest_af_info: {
              source: '1KGP - Survtyper - East Asian',
            },
            gnomadv3_af: -1,
            gnomadv3_af_afr: -1,
            gnomadv3_af_amr: -1,
            gnomadv3_af_eas: -1,
            gnomadv3_af_nfe: -1,
            gnomadv3_af_sas: -1,
            gnomadv3_af_xx: -1,
            gnomadv3_af_xy: -1,
            gnomadv3_af_ami: -1,
            gnomadv3_af_asj: -1,
            gnomadv3_af_fin: -1,
            gnomadv3_af_mid: -1,
            gnomadv3_af_oth: -1,
            gnomad_af: -1,
            gnomad_eas_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_nfe_af: -1,
            gnomad_sas_af: -1,
            gnomad_asj_af: -1,
            gnomad_fin_af: -1,
            gnomad_oth_af: -1,
            is_pathogenic: false,
            variant_id: 'S:survindel:2',
            sv_id: 'S:survindel:2',
            afs: [
              {
                source_start: 996283,
                source_end: 996348,
                source_svType: 'DUP',
                svLength: 66,
                reciOverlap: 1,
                annoOverlap: 1,
                source: '1KGP - Survtyper - East Asian',
                source_filter: 'one_kg_sur_eas',
                AF: 0.959,
                SAMPLE_COUNT: 245,
              },
              {
                source_start: 996283,
                source_end: 996348,
                source_svType: 'DUP',
                svLength: 66,
                reciOverlap: 1,
                annoOverlap: 1,
                source: '1KGP - Survtyper - Global',
                source_filter: 'one_kg_sur',
                AF: 0.885,
                SAMPLE_COUNT: 875,
              },
            ],
            exts: [],
            intpns: [],
            caller: 'survindel',
            genotypes_index: [
              [0, 0],
              [0, 0],
              [1, 1],
            ],
            hg19_chrom: 'chr1',
            hg19_start: 931662,
            hg19_strand: '+',
            hg19_end: 931728,
            gene_symbol: [],
            ensembl_gene_id: [],
            entrez_gene_id: [],
            ensembl_transcript_id: [],
          },
          {
            allelic_depths: [],
            genotype_qualities: [],
            is_read: false,
            _id: '64e5a3c599107f1eb02eac4c',
            chrom: 'chr1',
            start: 998763,
            end: 998764,
            len: 58,
            ref: 'G',
            alt: 'GGGGAGGGCGCGGAGCGGAGGGGAGGGCGCGGAGCGGAGGGGAGGGCGCGGAGCGGAGG',
            p_lof: 'False',
            constraint_pli: '',
            constraint_oe_lof_upper: [],
            variant_type: 'structural',
            type: 'INS',
            impact: '',
            gene_objs: [],
            highest_af: 0.655,
            highest_af_info: {
              source: 'HKBC - Manta',
            },
            gnomadv3_af: -1,
            gnomadv3_af_afr: -1,
            gnomadv3_af_amr: -1,
            gnomadv3_af_eas: -1,
            gnomadv3_af_nfe: -1,
            gnomadv3_af_sas: -1,
            gnomadv3_af_xx: -1,
            gnomadv3_af_xy: -1,
            gnomadv3_af_ami: -1,
            gnomadv3_af_asj: -1,
            gnomadv3_af_fin: -1,
            gnomadv3_af_mid: -1,
            gnomadv3_af_oth: -1,
            gnomad_af: -1,
            gnomad_eas_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_nfe_af: -1,
            gnomad_sas_af: -1,
            gnomad_asj_af: -1,
            gnomad_fin_af: -1,
            gnomad_oth_af: -1,
            is_pathogenic: true,
            variant_id: 'S:manta:2',
            sv_id: 'S:manta:2',
            afs: [
              {
                source_start: 998785,
                source_end: 998814,
                source_svType: 'INS',
                svLength: 58,
                source: '1KGP - East Asian',
                source_filter: 'one_kg_eas',
                AF: 0.477,
                SAMPLE_COUNT: 492,
              },
              {
                source_start: 998785,
                source_end: 998814,
                source_svType: 'INS',
                svLength: 58,
                source: '1KGP - Global',
                source_filter: 'one_kg',
                AF: 0.357,
                SAMPLE_COUNT: 2400,
              },
              {
                source_start: 998743,
                source_end: 998743,
                source_svType: 'INS',
                source: '1KGP - Survtyper - East Asian',
                source_filter: 'one_kg_sur_eas',
                AF: 0,
                SAMPLE_COUNT: 504,
              },
              {
                source_start: 998743,
                source_end: 998743,
                source_svType: 'INS',
                source: '1KGP - Survtyper - Global',
                source_filter: 'one_kg_sur',
                AF: 0.0002,
                SAMPLE_COUNT: 2500,
              },
              {
                source_start: 998763,
                source_end: 998764,
                source_svType: 'INS',
                svLength: 166,
                source: '1KGP - Survtyper - East Asian',
                source_filter: 'one_kg_sur_eas',
                AF: 0.00198,
                SAMPLE_COUNT: 504,
              },
              {
                source_start: 998763,
                source_end: 998764,
                source_svType: 'INS',
                svLength: 166,
                source: '1KGP - Survtyper - Global',
                source_filter: 'one_kg_sur',
                AF: 0.000399,
                SAMPLE_COUNT: 2500,
              },
              {
                source_start: 998764,
                source_end: 998764,
                source_svType: 'INS',
                svLength: 76,
                source: '1KGP - Survtyper - East Asian',
                source_filter: 'one_kg_sur_eas',
                AF: 0.546,
                SAMPLE_COUNT: 504,
              },
              {
                source_start: 998764,
                source_end: 998764,
                source_svType: 'INS',
                svLength: 76,
                source: '1KGP - Survtyper - Global',
                source_filter: 'one_kg_sur',
                AF: 0.31,
                SAMPLE_COUNT: 2500,
              },
              {
                source_start: 899422,
                source_end: 1094520,
                source_svType: 'Loss',
                svLength: 195099,
                source: 'DGV Gold (inner)',
                source_filter: 'dgv_gold_inner',
                AF: 0.00133,
              },
              {
                source_start: 902468,
                source_end: 1543403,
                source_svType: 'Gain',
                svLength: 640936,
                source: 'DGV Gold (inner)',
                source_filter: 'dgv_gold_inner',
                AF: 0.0545,
              },
              {
                source_start: 927074,
                source_end: 1134137,
                source_svType: 'Gain',
                svLength: 207064,
                source: 'DGV Gold (inner)',
                source_filter: 'dgv_gold_inner',
                AF: 0.000162,
              },
              {
                source_start: 877619,
                source_end: 1777210,
                source_svType: 'Gain',
                svLength: 899592,
                source: 'DGV Gold (outer)',
                source_filter: 'dgv_gold_outer',
                AF: 0.0545,
              },
              {
                source_start: 899422,
                source_end: 1094520,
                source_svType: 'Loss',
                svLength: 195099,
                source: 'DGV Gold (outer)',
                source_filter: 'dgv_gold_outer',
                AF: 0.00133,
              },
              {
                source_start: 914638,
                source_end: 1161955,
                source_svType: 'Gain',
                svLength: 247318,
                source: 'DGV Gold (outer)',
                source_filter: 'dgv_gold_outer',
                AF: 0.000162,
              },
              {
                source_start: 998763,
                source_end: 998764,
                source_svType: 'INS',
                AF: 0.655,
                SAMPLE_COUNT: 1680,
                source_filter: 'hkbc',
                display_source: 'Hong Kong Birth Cohort',
                source: 'HKBC - Manta',
                svLength: 58,
              },
              {
                source_start: 998764,
                source_end: 998764,
                source_svType: 'INS',
                AF: 0.559,
                SAMPLE_COUNT: 1680,
                source_filter: 'hkbc',
                display_source: 'Hong Kong Birth Cohort',
                source: 'HKBC - INSurVeyor',
                svLength: 76,
              },
            ],
            exts: [
              {
                source_start: 683107,
                source_end: 1051594,
                source: 'DECIPHER',
                svLength: 368488,
              },
            ],
            intpns: [],
            caller: 'manta',
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            hg19_chrom: 'chr1',
            hg19_start: 934143,
            hg19_strand: '+',
            hg19_end: 934144,
            gene_symbol: [],
            ensembl_gene_id: [],
            entrez_gene_id: [],
            ensembl_transcript_id: [],
          },
          {
            allelic_depths: [],
            genotype_qualities: [],
            is_read: false,
            _id: '64e5a42c7df7bcf1e22b4f75',
            chrom: 'chr1',
            start: 998776,
            end: 998776,
            len: 57,
            ref: 'A',
            alt: 'GCGGAGGGGAGGGCGCGGAGCGGAGGGGAGGGCGCGGAGCGGAGGGGAGGGCGCGGT',
            p_lof: 'False',
            constraint_pli: '',
            constraint_oe_lof_upper: [],
            variant_type: 'structural',
            type: 'INS',
            impact: '',
            gene_objs: [],
            highest_af: 0.655,
            highest_af_info: {
              source: 'HKBC - Manta',
            },
            gnomadv3_af: -1,
            gnomadv3_af_afr: -1,
            gnomadv3_af_amr: -1,
            gnomadv3_af_eas: -1,
            gnomadv3_af_nfe: -1,
            gnomadv3_af_sas: -1,
            gnomadv3_af_xx: -1,
            gnomadv3_af_xy: -1,
            gnomadv3_af_ami: -1,
            gnomadv3_af_asj: -1,
            gnomadv3_af_fin: -1,
            gnomadv3_af_mid: -1,
            gnomadv3_af_oth: -1,
            gnomad_af: -1,
            gnomad_eas_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_nfe_af: -1,
            gnomad_sas_af: -1,
            gnomad_asj_af: -1,
            gnomad_fin_af: -1,
            gnomad_oth_af: -1,
            is_pathogenic: true,
            variant_id: 'S:insurveyor:1',
            sv_id: 'S:insurveyor:1',
            afs: [
              {
                source_start: 998785,
                source_end: 998814,
                source_svType: 'INS',
                svLength: 58,
                source: '1KGP - East Asian',
                source_filter: 'one_kg_eas',
                AF: 0.477,
                SAMPLE_COUNT: 492,
              },
              {
                source_start: 998785,
                source_end: 998814,
                source_svType: 'INS',
                svLength: 58,
                source: '1KGP - Global',
                source_filter: 'one_kg',
                AF: 0.357,
                SAMPLE_COUNT: 2400,
              },
              {
                source_start: 998743,
                source_end: 998743,
                source_svType: 'INS',
                source: '1KGP - Survtyper - East Asian',
                source_filter: 'one_kg_sur_eas',
                AF: 0,
                SAMPLE_COUNT: 504,
              },
              {
                source_start: 998743,
                source_end: 998743,
                source_svType: 'INS',
                source: '1KGP - Survtyper - Global',
                source_filter: 'one_kg_sur',
                AF: 0.0002,
                SAMPLE_COUNT: 2500,
              },
              {
                source_start: 998763,
                source_end: 998764,
                source_svType: 'INS',
                svLength: 166,
                source: '1KGP - Survtyper - East Asian',
                source_filter: 'one_kg_sur_eas',
                AF: 0.00198,
                SAMPLE_COUNT: 504,
              },
              {
                source_start: 998763,
                source_end: 998764,
                source_svType: 'INS',
                svLength: 166,
                source: '1KGP - Survtyper - Global',
                source_filter: 'one_kg_sur',
                AF: 0.000399,
                SAMPLE_COUNT: 2500,
              },
              {
                source_start: 998764,
                source_end: 998764,
                source_svType: 'INS',
                svLength: 76,
                source: '1KGP - Survtyper - East Asian',
                source_filter: 'one_kg_sur_eas',
                AF: 0.546,
                SAMPLE_COUNT: 504,
              },
              {
                source_start: 998764,
                source_end: 998764,
                source_svType: 'INS',
                svLength: 76,
                source: '1KGP - Survtyper - Global',
                source_filter: 'one_kg_sur',
                AF: 0.31,
                SAMPLE_COUNT: 2500,
              },
              {
                source_start: 899422,
                source_end: 1094520,
                source_svType: 'Loss',
                svLength: 195099,
                source: 'DGV Gold (inner)',
                source_filter: 'dgv_gold_inner',
                AF: 0.00133,
              },
              {
                source_start: 902468,
                source_end: 1543403,
                source_svType: 'Gain',
                svLength: 640936,
                source: 'DGV Gold (inner)',
                source_filter: 'dgv_gold_inner',
                AF: 0.0545,
              },
              {
                source_start: 927074,
                source_end: 1134137,
                source_svType: 'Gain',
                svLength: 207064,
                source: 'DGV Gold (inner)',
                source_filter: 'dgv_gold_inner',
                AF: 0.000162,
              },
              {
                source_start: 877619,
                source_end: 1777210,
                source_svType: 'Gain',
                svLength: 899592,
                source: 'DGV Gold (outer)',
                source_filter: 'dgv_gold_outer',
                AF: 0.0545,
              },
              {
                source_start: 899422,
                source_end: 1094520,
                source_svType: 'Loss',
                svLength: 195099,
                source: 'DGV Gold (outer)',
                source_filter: 'dgv_gold_outer',
                AF: 0.00133,
              },
              {
                source_start: 914638,
                source_end: 1161955,
                source_svType: 'Gain',
                svLength: 247318,
                source: 'DGV Gold (outer)',
                source_filter: 'dgv_gold_outer',
                AF: 0.000162,
              },
              {
                source_start: 998763,
                source_end: 998764,
                source_svType: 'INS',
                AF: 0.655,
                SAMPLE_COUNT: 1680,
                source_filter: 'hkbc',
                display_source: 'Hong Kong Birth Cohort',
                source: 'HKBC - Manta',
                svLength: 58,
              },
              {
                source_start: 998764,
                source_end: 998764,
                source_svType: 'INS',
                AF: 0.559,
                SAMPLE_COUNT: 1680,
                source_filter: 'hkbc',
                display_source: 'Hong Kong Birth Cohort',
                source: 'HKBC - INSurVeyor',
                svLength: 76,
              },
            ],
            exts: [
              {
                source_start: 683107,
                source_end: 1051594,
                source: 'DECIPHER',
                svLength: 368488,
              },
            ],
            intpns: [],
            caller: 'insurveyor',
            genotypes_index: [
              [0, 0],
              [0, 1],
              [1, 1],
            ],
            hg19_chrom: 'chr1',
            hg19_start: 934156,
            hg19_strand: '+',
            hg19_end: 934156,
            gene_symbol: [],
            ensembl_gene_id: [],
            entrez_gene_id: [],
            ensembl_transcript_id: [],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9c0',
            variant_id: '61',
            end: 999842,
            ref: 'C',
            alt: 'A',
            start: 999842,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'HES4',
                ensembl_gene_id: 'ENSG00000188290',
                transcript: 'ENST00000304952',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '1/3',
                hgvsc: 'ENST00000304952.11:c.108+24G>T',
                existing_variation: 'rs2298214&COSV59241795',
                strand: -1,
                hgnc_id: 'HGNC:24149',
                mane_select: 'NM_021170.4',
                ccds: 'CCDS5.1',
                source: 'Ensembl',
                pheno: '1&1',
                cadd_phred: 6.212,
                cadd_raw: 0.472275,
                spliceai_pred_dp_ag: 6,
                spliceai_pred_dp_al: -37,
                spliceai_pred_dp_dg: 6,
                spliceai_pred_dp_dl: 24,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.01,
                spliceai_pred_ds_dl: 0.02,
                spliceai_pred_symbol: 'HES4',
                constraint_mis_z: -0.43509,
                constraint_oe_lof: 1.7186,
                constraint_oe_lof_lower: 0.868,
                constraint_oe_lof_upper: 1.949,
                constraint_oe_mis: 1.1419,
                constraint_oe_mis_lower: 0.957,
                constraint_oe_mis_upper: 1.368,
                constraint_oe_syn: 1.3468,
                constraint_oe_syn_lower: 1.057,
                constraint_oe_syn_upper: 1.724,
                constraint_pli: 0.0000021108,
                constraint_syn_z: -1.5581,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'HES4',
                entrez_gene_id: '57801',
                p_haplo: 0.07906,
                p_triplo: 0.06819,
                p_li: 0.0000021108,
                loeuf: 1.949,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'HES4',
                ensembl_gene_id: 'ENSG00000188290',
                transcript: 'ENST00000428771',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '1/3',
                codon_change: 'agG/agT',
                aa_change: 'R/S',
                aa_length: 44,
                polyphen_pred: 'benign',
                polyphen_score: 0,
                sift_pred: 'tolerated_low_confidence',
                sift_score: 0.72,
                allele: 'A',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000428771.6:c.132G>T',
                hgvsp: 'ENSP00000393198.2:p.Arg44Ser',
                cds_position: 132,
                existing_variation: 'rs2298214&COSV59241795',
                strand: -1,
                hgnc_id: 'HGNC:24149',
                ccds: 'CCDS44034.1',
                source: 'Ensembl',
                domains: 'MobiDB_lite:mobidb-lite&Low_complexity_(Seg):seg',
                pheno: '1&1',
                revel: 0.079,
                cadd_phred: 6.212,
                cadd_raw: 0.472275,
                spliceai_pred_dp_ag: 6,
                spliceai_pred_dp_al: -37,
                spliceai_pred_dp_dg: 6,
                spliceai_pred_dp_dl: 24,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.01,
                spliceai_pred_ds_dl: 0.02,
                spliceai_pred_symbol: 'HES4',
                constraint_mis_z: -0.46973,
                constraint_oe_lof: 1.6997,
                constraint_oe_lof_lower: 0.816,
                constraint_oe_lof_upper: 1.947,
                constraint_oe_mis: 1.139,
                constraint_oe_mis_lower: 0.97,
                constraint_oe_mis_upper: 1.342,
                constraint_oe_syn: 1.2689,
                constraint_oe_syn_lower: 1.017,
                constraint_oe_syn_upper: 1.595,
                constraint_pli: 0.000013746,
                constraint_syn_z: -1.3661,
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'HES4',
                entrez_gene_id: '57801',
                p_haplo: 0.07906,
                p_triplo: 0.06819,
                p_li: 0.000013746,
                loeuf: 1.947,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'HES4',
                ensembl_gene_id: 'ENSG00000188290',
                transcript: 'ENST00000484667',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '1/2',
                hgvsc: 'ENST00000484667.2:c.108+24G>T',
                existing_variation: 'rs2298214&COSV59241795',
                strand: -1,
                hgnc_id: 'HGNC:24149',
                source: 'Ensembl',
                pheno: '1&1',
                cadd_phred: 6.212,
                cadd_raw: 0.472275,
                spliceai_pred_dp_ag: 6,
                spliceai_pred_dp_al: -37,
                spliceai_pred_dp_dg: 6,
                spliceai_pred_dp_dl: 24,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.01,
                spliceai_pred_ds_dl: 0.02,
                spliceai_pred_symbol: 'HES4',
                constraint_mis_z: -0.28894,
                constraint_oe_lof: 0.97454,
                constraint_oe_lof_lower: 0.358,
                constraint_oe_lof_upper: 1.872,
                constraint_oe_mis: 1.1097,
                constraint_oe_mis_lower: 0.902,
                constraint_oe_mis_upper: 1.373,
                constraint_oe_syn: 1.1765,
                constraint_oe_syn_lower: 0.875,
                constraint_oe_syn_upper: 1.601,
                constraint_pli: 0.025843,
                constraint_syn_z: -0.68893,
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'HES4',
                entrez_gene_id: '57801',
                p_haplo: 0.07906,
                p_triplo: 0.06819,
                p_li: 0.025843,
                loeuf: 1.872,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000624652',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2298214&COSV59241795',
                strand: 1,
                flags: 'cds_end_NF',
                hgnc_id: 'HGNC:4053',
                source: 'Ensembl',
                gene_pheno: 1,
                pheno: '1&1',
                cadd_phred: 6.212,
                cadd_raw: 0.472275,
                spliceai_pred_dp_ag: 6,
                spliceai_pred_dp_al: -37,
                spliceai_pred_dp_dg: 6,
                spliceai_pred_dp_dl: 24,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.01,
                spliceai_pred_ds_dl: 0.02,
                spliceai_pred_symbol: 'HES4',
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000624697',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2298214&COSV59241795',
                strand: 1,
                hgnc_id: 'HGNC:4053',
                source: 'Ensembl',
                gene_pheno: 1,
                pheno: '1&1',
                cadd_phred: 6.212,
                cadd_raw: 0.472275,
                spliceai_pred_dp_ag: 6,
                spliceai_pred_dp_al: -37,
                spliceai_pred_dp_dg: 6,
                spliceai_pred_dp_dl: 24,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.01,
                spliceai_pred_ds_dl: 0.02,
                spliceai_pred_symbol: 'HES4',
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000304952',
            ensembl_gene_id: ['ENSG00000188290', 'ENSG00000187608'],
            is_exonic: 1,
            is_coding: 1,
            impact: 'intron_variant',
            hgvsc: 'ENST00000304952.11:c.108+24G>T',
            existing_variation: 'rs2298214&COSV59241795',
            ccds: 'CCDS5.1',
            source: 'Ensembl',
            gnomad_af: 0.6136,
            gnomad_afr_af: 0.1208,
            gnomad_amr_af: 0.7139,
            gnomad_asj_af: 0.6344,
            gnomad_eas_af: 0.786,
            gnomad_fin_af: 0.5555,
            gnomad_nfe_af: 0.595,
            gnomad_oth_af: 0.5882,
            gnomad_sas_af: 0.6625,
            pheno: '1&1',
            cadd_phred: 6.212,
            cadd_raw: 0.472275,
            spliceai_pred_dp_ag: 6,
            spliceai_pred_dp_al: -37,
            spliceai_pred_dp_dg: 6,
            spliceai_pred_dp_dl: 24,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0.01],
            spliceai_pred_ds_dl: [0.02],
            spliceai_pred_symbol: 'HES4',
            constraint_mis_z: [-0.43509],
            constraint_oe_lof: 1.7186,
            constraint_oe_lof_lower: 0.868,
            constraint_oe_lof_upper: [1.949],
            constraint_oe_mis: 1.1419,
            constraint_oe_mis_lower: 0.957,
            constraint_oe_mis_upper: [1.368],
            constraint_oe_syn: 1.3468,
            constraint_oe_syn_lower: 1.057,
            constraint_oe_syn_upper: 1.724,
            constraint_pli: 0.0000021108,
            constraint_syn_z: [-1.5581],
            exomiser_ad_exgenespheno: 0.5001,
            exomiser_xr_exgenescombi: 0.00030642,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.1137,
            gnomadv3_af_nfe: 0.5694,
            gnomadv3_af_xy: 0.4637,
            exomiser_xd_exgenescombi: 0.00030642,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.5573,
            gnomadv3_af: 0.4573,
            exomiser_mt_exgenespheno: 0.5001,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00030642,
            gnomadv3_af_eas: 0.767,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.6331,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00030642,
            exomiser_ad_exgenescombi: 0.00030642,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.5154,
            gnomadv3_af_fin: 0.5288,
            exomiser_ar_exgenespheno: 0.5001,
            gnomadv3_af_sas: 0.6384,
            gnomadv3_af_amr: 0.6295,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.4513,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.4981,
            exomiser_xr_exgenespheno: 0.5001,
            exomiser_xd_exgenespheno: 0.5001,
            impact_severity: 'LOW',
            quality: 2571.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [1, 1],
            ],
            allelic_depths: [
              [26, 25],
              [20, 21],
              [1, 46],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['HES4', 'ISG15'],
            ensembl_transcript_id: [
              'ENST00000304952',
              'ENST00000428771',
              'ENST00000484667',
              'ENST00000624652',
              'ENST00000624697',
            ],
            variant_type: 'small',
            highest_af: 0.786,
            highest_af_info: {
              source: 'gnomad V2 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 935222,
            hg19_strand: '+',
            hg19_end: 935222,
            entrez_gene_id: ['57801', '9636'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9c1',
            variant_id: '62',
            end: 1000079,
            ref: 'A',
            alt: 'G',
            start: 1000079,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'HES4',
                ensembl_gene_id: 'ENSG00000188290',
                transcript: 'ENST00000304952',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '1/4',
                allele: 'G',
                impact: ['5_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000304952.11:c.-106T>C',
                existing_variation: 'rs3128113&COSV59242288',
                strand: -1,
                hgnc_id: 'HGNC:24149',
                mane_select: 'NM_021170.4',
                ccds: 'CCDS5.1',
                source: 'Ensembl',
                pheno: '0&1',
                existing_inframe_oorfs: 0,
                existing_outofframe_oorfs: 0,
                existing_uorfs: 0,
                cadd_phred: 7.83,
                cadd_raw: 0.633083,
                spliceai_pred_dp_ag: -48,
                spliceai_pred_dp_al: -10,
                spliceai_pred_dp_dg: 25,
                spliceai_pred_dp_dl: -11,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'HES4',
                constraint_mis_z: -0.43509,
                constraint_oe_lof: 1.7186,
                constraint_oe_lof_lower: 0.868,
                constraint_oe_lof_upper: 1.949,
                constraint_oe_mis: 1.1419,
                constraint_oe_mis_lower: 0.957,
                constraint_oe_mis_upper: 1.368,
                constraint_oe_syn: 1.3468,
                constraint_oe_syn_lower: 1.057,
                constraint_oe_syn_upper: 1.724,
                constraint_pli: 0.0000021108,
                constraint_syn_z: -1.5581,
                impact_so: '5_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'HES4',
                entrez_gene_id: '57801',
                p_haplo: 0.07906,
                p_triplo: 0.06819,
                p_li: 0.0000021108,
                loeuf: 1.949,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'HES4',
                ensembl_gene_id: 'ENSG00000188290',
                transcript: 'ENST00000428771',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '1/3',
                allele: 'G',
                impact: ['5_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000428771.6:c.-106T>C',
                existing_variation: 'rs3128113&COSV59242288',
                strand: -1,
                hgnc_id: 'HGNC:24149',
                ccds: 'CCDS44034.1',
                source: 'Ensembl',
                pheno: '0&1',
                existing_inframe_oorfs: 0,
                existing_outofframe_oorfs: 0,
                existing_uorfs: 1,
                cadd_phred: 7.83,
                cadd_raw: 0.633083,
                spliceai_pred_dp_ag: -48,
                spliceai_pred_dp_al: -10,
                spliceai_pred_dp_dg: 25,
                spliceai_pred_dp_dl: -11,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'HES4',
                constraint_mis_z: -0.46973,
                constraint_oe_lof: 1.6997,
                constraint_oe_lof_lower: 0.816,
                constraint_oe_lof_upper: 1.947,
                constraint_oe_mis: 1.139,
                constraint_oe_mis_lower: 0.97,
                constraint_oe_mis_upper: 1.342,
                constraint_oe_syn: 1.2689,
                constraint_oe_syn_lower: 1.017,
                constraint_oe_syn_upper: 1.595,
                constraint_pli: 0.000013746,
                constraint_syn_z: -1.3661,
                impact_so: '5_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'HES4',
                entrez_gene_id: '57801',
                p_haplo: 0.07906,
                p_triplo: 0.06819,
                p_li: 0.000013746,
                loeuf: 1.947,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'HES4',
                ensembl_gene_id: 'ENSG00000188290',
                transcript: 'ENST00000484667',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3128113&COSV59242288',
                strand: -1,
                hgnc_id: 'HGNC:24149',
                source: 'Ensembl',
                pheno: '0&1',
                cadd_phred: 7.83,
                cadd_raw: 0.633083,
                spliceai_pred_dp_ag: -48,
                spliceai_pred_dp_al: -10,
                spliceai_pred_dp_dg: 25,
                spliceai_pred_dp_dl: -11,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'HES4',
                constraint_mis_z: -0.28894,
                constraint_oe_lof: 0.97454,
                constraint_oe_lof_lower: 0.358,
                constraint_oe_lof_upper: 1.872,
                constraint_oe_mis: 1.1097,
                constraint_oe_mis_lower: 0.902,
                constraint_oe_mis_upper: 1.373,
                constraint_oe_syn: 1.1765,
                constraint_oe_syn_lower: 0.875,
                constraint_oe_syn_upper: 1.601,
                constraint_pli: 0.025843,
                constraint_syn_z: -0.68893,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'HES4',
                entrez_gene_id: '57801',
                p_haplo: 0.07906,
                p_triplo: 0.06819,
                p_li: 0.025843,
                loeuf: 1.872,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000624652',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3128113&COSV59242288',
                strand: 1,
                flags: 'cds_end_NF',
                hgnc_id: 'HGNC:4053',
                source: 'Ensembl',
                gene_pheno: 1,
                pheno: '0&1',
                cadd_phred: 7.83,
                cadd_raw: 0.633083,
                spliceai_pred_dp_ag: -48,
                spliceai_pred_dp_al: -10,
                spliceai_pred_dp_dg: 25,
                spliceai_pred_dp_dl: -11,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'HES4',
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000624697',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3128113&COSV59242288',
                strand: 1,
                hgnc_id: 'HGNC:4053',
                source: 'Ensembl',
                gene_pheno: 1,
                pheno: '0&1',
                cadd_phred: 7.83,
                cadd_raw: 0.633083,
                spliceai_pred_dp_ag: -48,
                spliceai_pred_dp_al: -10,
                spliceai_pred_dp_dg: 25,
                spliceai_pred_dp_dl: -11,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'HES4',
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000304952',
            ensembl_gene_id: ['ENSG00000188290', 'ENSG00000187608'],
            is_exonic: 1,
            is_coding: 0,
            impact: '5_prime_UTR_variant',
            hgvsc: 'ENST00000304952.11:c.-106T>C',
            existing_variation: 'rs3128113&COSV59242288',
            ccds: 'CCDS5.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            pheno: '0&1',
            existing_inframe_oorfs: '0',
            existing_outofframe_oorfs: '0',
            existing_uorfs: '0',
            cadd_phred: 7.83,
            cadd_raw: 0.633083,
            spliceai_pred_dp_ag: -48,
            spliceai_pred_dp_al: -10,
            spliceai_pred_dp_dg: 25,
            spliceai_pred_dp_dl: -11,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'HES4',
            constraint_mis_z: [-0.43509],
            constraint_oe_lof: 1.7186,
            constraint_oe_lof_lower: 0.868,
            constraint_oe_lof_upper: [1.949],
            constraint_oe_mis: 1.1419,
            constraint_oe_mis_lower: 0.957,
            constraint_oe_mis_upper: [1.368],
            constraint_oe_syn: 1.3468,
            constraint_oe_syn_lower: 1.057,
            constraint_oe_syn_upper: 1.724,
            constraint_pli: 0.0000021108,
            constraint_syn_z: [-1.5581],
            exomiser_ad_exgenespheno: 0.5001,
            exomiser_xr_exgenescombi: 0.00030642,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.3764,
            gnomadv3_af_nfe: 0.7973,
            gnomadv3_af_xy: 0.6845,
            exomiser_xd_exgenescombi: 0.00030642,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.7166,
            gnomadv3_af: 0.6804,
            exomiser_mt_exgenespheno: 0.5001,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00030642,
            gnomadv3_af_eas: 0.8667,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.7662,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00030642,
            exomiser_ad_exgenescombi: 0.00030642,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.7682,
            gnomadv3_af_fin: 0.7782,
            exomiser_ar_exgenespheno: 0.5001,
            gnomadv3_af_sas: 0.7899,
            gnomadv3_af_amr: 0.7929,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.6765,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.6949,
            exomiser_xr_exgenespheno: 0.5001,
            exomiser_xd_exgenespheno: 0.5001,
            impact_severity: 'LOW',
            quality: 4822.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 50],
              [0, 41],
              [0, 55],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['HES4', 'ISG15'],
            ensembl_transcript_id: [
              'ENST00000304952',
              'ENST00000428771',
              'ENST00000484667',
              'ENST00000624652',
              'ENST00000624697',
            ],
            variant_type: 'small',
            highest_af: 0.8667,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 935459,
            hg19_strand: '+',
            hg19_end: 935459,
            entrez_gene_id: ['57801', '9636'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9c2',
            variant_id: '63',
            end: 1000112,
            ref: 'G',
            alt: 'T',
            start: 1000112,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'HES4',
                ensembl_gene_id: 'ENSG00000188290',
                transcript: 'ENST00000304952',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3121571',
                strand: -1,
                hgnc_id: 'HGNC:24149',
                mane_select: 'NM_021170.4',
                ccds: 'CCDS5.1',
                source: 'Ensembl',
                cadd_phred: 10.72,
                cadd_raw: 0.926617,
                constraint_mis_z: -0.43509,
                constraint_oe_lof: 1.7186,
                constraint_oe_lof_lower: 0.868,
                constraint_oe_lof_upper: 1.949,
                constraint_oe_mis: 1.1419,
                constraint_oe_mis_lower: 0.957,
                constraint_oe_mis_upper: 1.368,
                constraint_oe_syn: 1.3468,
                constraint_oe_syn_lower: 1.057,
                constraint_oe_syn_upper: 1.724,
                constraint_pli: 0.0000021108,
                constraint_syn_z: -1.5581,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'HES4',
                entrez_gene_id: '57801',
                p_haplo: 0.07906,
                p_triplo: 0.06819,
                p_li: 0.0000021108,
                loeuf: 1.949,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'HES4',
                ensembl_gene_id: 'ENSG00000188290',
                transcript: 'ENST00000428771',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '1/3',
                allele: 'T',
                impact: ['5_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000428771.6:c.-139C>A',
                existing_variation: 'rs3121571',
                strand: -1,
                hgnc_id: 'HGNC:24149',
                ccds: 'CCDS44034.1',
                source: 'Ensembl',
                existing_inframe_oorfs: 0,
                existing_outofframe_oorfs: 0,
                existing_uorfs: 1,
                cadd_phred: 10.72,
                cadd_raw: 0.926617,
                constraint_mis_z: -0.46973,
                constraint_oe_lof: 1.6997,
                constraint_oe_lof_lower: 0.816,
                constraint_oe_lof_upper: 1.947,
                constraint_oe_mis: 1.139,
                constraint_oe_mis_lower: 0.97,
                constraint_oe_mis_upper: 1.342,
                constraint_oe_syn: 1.2689,
                constraint_oe_syn_lower: 1.017,
                constraint_oe_syn_upper: 1.595,
                constraint_pli: 0.000013746,
                constraint_syn_z: -1.3661,
                impact_so: '5_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'HES4',
                entrez_gene_id: '57801',
                p_haplo: 0.07906,
                p_triplo: 0.06819,
                p_li: 0.000013746,
                loeuf: 1.947,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'HES4',
                ensembl_gene_id: 'ENSG00000188290',
                transcript: 'ENST00000484667',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3121571',
                strand: -1,
                hgnc_id: 'HGNC:24149',
                source: 'Ensembl',
                cadd_phred: 10.72,
                cadd_raw: 0.926617,
                constraint_mis_z: -0.28894,
                constraint_oe_lof: 0.97454,
                constraint_oe_lof_lower: 0.358,
                constraint_oe_lof_upper: 1.872,
                constraint_oe_mis: 1.1097,
                constraint_oe_mis_lower: 0.902,
                constraint_oe_mis_upper: 1.373,
                constraint_oe_syn: 1.1765,
                constraint_oe_syn_lower: 0.875,
                constraint_oe_syn_upper: 1.601,
                constraint_pli: 0.025843,
                constraint_syn_z: -0.68893,
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'HES4',
                entrez_gene_id: '57801',
                p_haplo: 0.07906,
                p_triplo: 0.06819,
                p_li: 0.025843,
                loeuf: 1.872,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000624652',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3121571',
                strand: 1,
                flags: 'cds_end_NF',
                hgnc_id: 'HGNC:4053',
                source: 'Ensembl',
                gene_pheno: 1,
                cadd_phred: 10.72,
                cadd_raw: 0.926617,
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000624697',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3121571',
                strand: 1,
                hgnc_id: 'HGNC:4053',
                source: 'Ensembl',
                gene_pheno: 1,
                cadd_phred: 10.72,
                cadd_raw: 0.926617,
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000304952',
            ensembl_gene_id: ['ENSG00000188290', 'ENSG00000187608'],
            is_exonic: 1,
            is_coding: 0,
            impact: 'upstream_gene_variant',
            existing_variation: 'rs3121571',
            ccds: 'CCDS5.1',
            source: 'Ensembl',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            cadd_phred: 10.72,
            cadd_raw: 0.926617,
            constraint_mis_z: [-0.43509],
            constraint_oe_lof: 1.7186,
            constraint_oe_lof_lower: 0.868,
            constraint_oe_lof_upper: [1.949],
            constraint_oe_mis: 1.1419,
            constraint_oe_mis_lower: 0.957,
            constraint_oe_mis_upper: [1.368],
            constraint_oe_syn: 1.3468,
            constraint_oe_syn_lower: 1.057,
            constraint_oe_syn_upper: 1.724,
            constraint_pli: 0.0000021108,
            constraint_syn_z: [-1.5581],
            exomiser_ad_exgenespheno: 0.5001,
            exomiser_xr_exgenescombi: 0.00030642,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.3749,
            gnomadv3_af_nfe: 0.7972,
            gnomadv3_af_xy: 0.6841,
            exomiser_xd_exgenescombi: 0.00030642,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.7166,
            gnomadv3_af: 0.68,
            exomiser_mt_exgenespheno: 0.5001,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00030642,
            gnomadv3_af_eas: 0.8662,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.7665,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00030642,
            exomiser_ad_exgenescombi: 0.00030642,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.7704,
            gnomadv3_af_fin: 0.7787,
            exomiser_ar_exgenespheno: 0.5001,
            gnomadv3_af_sas: 0.79,
            gnomadv3_af_amr: 0.793,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.6761,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.6939,
            exomiser_xr_exgenespheno: 0.5001,
            exomiser_xd_exgenespheno: 0.5001,
            impact_severity: 'LOW',
            quality: 4441.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 45],
              [0, 47],
              [0, 46],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['HES4', 'ISG15'],
            ensembl_transcript_id: [
              'ENST00000304952',
              'ENST00000428771',
              'ENST00000484667',
              'ENST00000624652',
              'ENST00000624697',
            ],
            variant_type: 'small',
            highest_af: 0.8662,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 935492,
            hg19_strand: '+',
            hg19_end: 935492,
            entrez_gene_id: ['57801', '9636'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9c3',
            variant_id: '64',
            end: 1013466,
            ref: 'T',
            alt: 'TA',
            start: 1013466,
            chrom: 'chr1',
            type: 'indel',
            gene_objs: [
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000649529',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'A',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs3841266',
                strand: 1,
                hgnc_id: 'HGNC:4053',
                mane_select: 'NM_005101.4',
                ccds: 'CCDS6.1',
                source: 'Ensembl',
                gene_pheno: 1,
                flaglrg: 'LRG_1231t1',
                spliceai_pred_dp_ag: -39,
                spliceai_pred_dp_al: -4,
                spliceai_pred_dp_dg: -18,
                spliceai_pred_dp_dl: -39,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'ISG15',
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000624652',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '2/2',
                hgvsc: 'ENST00000624652.1:c.-21-517dup',
                existing_variation: 'rs3841266',
                strand: 1,
                flags: 'cds_end_NF',
                hgnc_id: 'HGNC:4053',
                source: 'Ensembl',
                gene_pheno: 1,
                spliceai_pred_dp_ag: -39,
                spliceai_pred_dp_al: -4,
                spliceai_pred_dp_dg: -18,
                spliceai_pred_dp_dl: -39,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'ISG15',
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000624697',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '2/2',
                hgvsc: 'ENST00000624697.4:c.-21-517dup',
                existing_variation: 'rs3841266',
                strand: 1,
                hgnc_id: 'HGNC:4053',
                source: 'Ensembl',
                gene_pheno: 1,
                spliceai_pred_dp_ag: -39,
                spliceai_pred_dp_al: -4,
                spliceai_pred_dp_dg: -18,
                spliceai_pred_dp_dl: -39,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'ISG15',
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000649529',
            ensembl_gene_id: ['ENSG00000187608'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'upstream_gene_variant',
            existing_variation: 'rs3841266',
            ccds: 'CCDS6.1',
            source: 'Ensembl',
            gene_pheno: '1',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            flaglrg: 'LRG_1231t1',
            spliceai_pred_dp_ag: -39,
            spliceai_pred_dp_al: -4,
            spliceai_pred_dp_dg: -18,
            spliceai_pred_dp_dl: -39,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'ISG15',
            cgd_agegroup: 'Pediatric',
            cgd_condition:
              'Immunodeficiency_38&_with_basal_ganglia_calcification',
            cgd_entrezid: '9636',
            cgd_gene: 'ISG15',
            cgd_hgncid: '4053',
            cgd_inheritance: 'AR',
            cgd_interventioncategories: 'Allergy/Immunology/Infectious',
            cgd_interventionrationale:
              'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
            cgd_manifestationcategories:
              'Allergy/Immunology/Infectious&_Neurologic',
            cgd_references: '22859821&_25307056',
            exomiser_ad_exgenespheno: 0.5035,
            exomiser_xr_exgenescombi: 0.00031754,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'criteria_provided,_single_submitter',
            clndn: 'Immunodeficiency_38_with_basal_ganglia_calcification',
            gnomadv3_af_afr: 0.5946,
            gnomadv3_af_nfe: 0.9741,
            gnomadv3_af_xy: 0.8672,
            exomiser_xd_exgenescombi: 0.00031754,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9082,
            clnvcso: 'SO:1000035',
            gnomadv3_af: 0.8639,
            exomiser_mt_exgenespheno: 0.5035,
            clndisdb:
              'MONDO:MONDO:0014502,MedGen:C4015293,OMIM:616126,Orphanet:ORPHA319563',
            exomiser_ar_exgenescombi: 0.00031754,
            gnomadv3_af_eas: 0.9509,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.928,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00031754,
            exomiser_ad_exgenescombi: 0.00031754,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'Benign',
            clnvc: 'Duplication',
            gnomadv3_af_ami: 0.9945,
            gnomadv3_af_fin: 0.9859,
            exomiser_ar_exgenespheno: 0.5035,
            gnomadv3_af_sas: 0.9622,
            gnomadv3_af_amr: 0.9308,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'NC_000001.11:g.1013467dup',
            exomiser_ad_exgenesvar: 0,
            clnid: '1185392',
            gnomadv3_af_xx: 0.8608,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.8819,
            exomiser_xr_exgenespheno: 0.5035,
            exomiser_xd_exgenespheno: 0.5035,
            impact_severity: 'LOW',
            quality: 5870.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 37],
              [0, 38],
              [0, 52],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['ISG15'],
            ensembl_transcript_id: [
              'ENST00000649529',
              'ENST00000624652',
              'ENST00000624697',
            ],
            variant_type: 'small',
            highest_af: 0.9509,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 948846,
            hg19_strand: '+',
            hg19_end: 948846,
            entrez_gene_id: ['9636'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9c4',
            variant_id: '65',
            end: 1013490,
            ref: 'C',
            alt: 'G',
            start: 1013490,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000649529',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'G',
                impact: ['upstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs4615788',
                strand: 1,
                hgnc_id: 'HGNC:4053',
                mane_select: 'NM_005101.4',
                ccds: 'CCDS6.1',
                source: 'Ensembl',
                gene_pheno: 1,
                flaglrg: 'LRG_1231t1',
                cadd_phred: 1.399,
                cadd_raw: 0.009998,
                spliceai_pred_dp_ag: 46,
                spliceai_pred_dp_al: -27,
                spliceai_pred_dp_dg: 39,
                spliceai_pred_dp_dl: -27,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'ISG15',
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'upstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000624652',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '2/2',
                hgvsc: 'ENST00000624652.1:c.-21-494C>G',
                existing_variation: 'rs4615788',
                strand: 1,
                flags: 'cds_end_NF',
                hgnc_id: 'HGNC:4053',
                source: 'Ensembl',
                gene_pheno: 1,
                cadd_phred: 1.399,
                cadd_raw: 0.009998,
                spliceai_pred_dp_ag: 46,
                spliceai_pred_dp_al: -27,
                spliceai_pred_dp_dg: 39,
                spliceai_pred_dp_dl: -27,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'ISG15',
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000624697',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '2/2',
                hgvsc: 'ENST00000624697.4:c.-21-494C>G',
                existing_variation: 'rs4615788',
                strand: 1,
                hgnc_id: 'HGNC:4053',
                source: 'Ensembl',
                gene_pheno: 1,
                cadd_phred: 1.399,
                cadd_raw: 0.009998,
                spliceai_pred_dp_ag: 46,
                spliceai_pred_dp_al: -27,
                spliceai_pred_dp_dg: 39,
                spliceai_pred_dp_dl: -27,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'ISG15',
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000649529',
            ensembl_gene_id: ['ENSG00000187608'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'upstream_gene_variant',
            existing_variation: 'rs4615788',
            ccds: 'CCDS6.1',
            source: 'Ensembl',
            gene_pheno: '1',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            flaglrg: 'LRG_1231t1',
            cadd_phred: 1.399,
            cadd_raw: 0.009998,
            spliceai_pred_dp_ag: 46,
            spliceai_pred_dp_al: -27,
            spliceai_pred_dp_dg: 39,
            spliceai_pred_dp_dl: -27,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'ISG15',
            cgd_agegroup: 'Pediatric',
            cgd_condition:
              'Immunodeficiency_38&_with_basal_ganglia_calcification',
            cgd_entrezid: '9636',
            cgd_gene: 'ISG15',
            cgd_hgncid: '4053',
            cgd_inheritance: 'AR',
            cgd_interventioncategories: 'Allergy/Immunology/Infectious',
            cgd_interventionrationale:
              'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
            cgd_manifestationcategories:
              'Allergy/Immunology/Infectious&_Neurologic',
            cgd_references: '22859821&_25307056',
            exomiser_ad_exgenespheno: 0.5035,
            exomiser_xr_exgenescombi: 0.00031754,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'criteria_provided,_single_submitter',
            clndn: 'Immunodeficiency_38_with_basal_ganglia_calcification',
            gnomadv3_af_afr: 0.7417,
            gnomadv3_af_nfe: 0.9587,
            gnomadv3_af_xy: 0.9007,
            exomiser_xd_exgenescombi: 0.00031754,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.8861,
            clnvcso: 'SO:0001483',
            gnomadv3_af: 0.8985,
            exomiser_mt_exgenespheno: 0.5035,
            clndisdb:
              'MONDO:MONDO:0014502,MedGen:C4015293,OMIM:616126,Orphanet:ORPHA319563',
            exomiser_ar_exgenescombi: 0.00031754,
            gnomadv3_af_eas: 0.9998,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9101,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00031754,
            exomiser_ad_exgenescombi: 0.00031754,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'Benign',
            clnvc: 'single_nucleotide_variant',
            gnomadv3_af_ami: 0.9813,
            gnomadv3_af_fin: 0.9755,
            exomiser_ar_exgenespheno: 0.5035,
            gnomadv3_af_sas: 0.9675,
            gnomadv3_af_amr: 0.9357,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'NC_000001.11:g.1013490C>G',
            exomiser_ad_exgenesvar: 0,
            clnid: '1185393',
            gnomadv3_af_xx: 0.8964,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.9067,
            exomiser_xr_exgenespheno: 0.5035,
            exomiser_xd_exgenespheno: 0.5035,
            impact_severity: 'LOW',
            quality: 5822.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 42],
              [0, 36],
              [0, 57],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['ISG15'],
            ensembl_transcript_id: [
              'ENST00000649529',
              'ENST00000624652',
              'ENST00000624697',
            ],
            variant_type: 'small',
            highest_af: 0.9998,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 948870,
            hg19_strand: '+',
            hg19_end: 948870,
            entrez_gene_id: ['9636'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9c5',
            variant_id: '66',
            end: 1013541,
            ref: 'T',
            alt: 'C',
            start: 1013541,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000649529',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '1/2',
                allele: 'C',
                impact: ['5_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000649529.1:c.-33T>C',
                existing_variation: 'rs15842',
                strand: 1,
                hgnc_id: 'HGNC:4053',
                mane_select: 'NM_005101.4',
                ccds: 'CCDS6.1',
                source: 'Ensembl',
                gene_pheno: 1,
                existing_inframe_oorfs: 0,
                existing_outofframe_oorfs: 0,
                existing_uorfs: 0,
                flaglrg: 'LRG_1231t1',
                cadd_phred: 6.025,
                cadd_raw: 0.454751,
                spliceai_pred_dp_ag: -10,
                spliceai_pred_dp_al: 17,
                spliceai_pred_dp_dg: 35,
                spliceai_pred_dp_dl: -12,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'ISG15',
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: '5_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000624652',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '2/2',
                hgvsc: 'ENST00000624652.1:c.-21-443T>C',
                existing_variation: 'rs15842',
                strand: 1,
                flags: 'cds_end_NF',
                hgnc_id: 'HGNC:4053',
                source: 'Ensembl',
                gene_pheno: 1,
                cadd_phred: 6.025,
                cadd_raw: 0.454751,
                spliceai_pred_dp_ag: -10,
                spliceai_pred_dp_al: 17,
                spliceai_pred_dp_dg: 35,
                spliceai_pred_dp_dl: -12,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'ISG15',
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000624697',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '2/2',
                hgvsc: 'ENST00000624697.4:c.-21-443T>C',
                existing_variation: 'rs15842',
                strand: 1,
                hgnc_id: 'HGNC:4053',
                source: 'Ensembl',
                gene_pheno: 1,
                cadd_phred: 6.025,
                cadd_raw: 0.454751,
                spliceai_pred_dp_ag: -10,
                spliceai_pred_dp_al: 17,
                spliceai_pred_dp_dg: 35,
                spliceai_pred_dp_dl: -12,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'ISG15',
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000649529',
            ensembl_gene_id: ['ENSG00000187608'],
            is_exonic: 1,
            is_coding: 0,
            impact: '5_prime_UTR_variant',
            hgvsc: 'ENST00000649529.1:c.-33T>C',
            existing_variation: 'rs15842',
            ccds: 'CCDS6.1',
            source: 'Ensembl',
            gene_pheno: '1',
            gnomad_af: 0.9457,
            gnomad_afr_af: 0.7382,
            gnomad_amr_af: 0.9581,
            gnomad_asj_af: 0.9043,
            gnomad_eas_af: 0.9998,
            gnomad_fin_af: 0.9713,
            gnomad_nfe_af: 0.9578,
            gnomad_oth_af: 0.9377,
            gnomad_sas_af: 0.9608,
            existing_inframe_oorfs: '0',
            existing_outofframe_oorfs: '0',
            existing_uorfs: '0',
            flaglrg: 'LRG_1231t1',
            cadd_phred: 6.025,
            cadd_raw: 0.454751,
            spliceai_pred_dp_ag: -10,
            spliceai_pred_dp_al: 17,
            spliceai_pred_dp_dg: 35,
            spliceai_pred_dp_dl: -12,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'ISG15',
            cgd_agegroup: 'Pediatric',
            cgd_condition:
              'Immunodeficiency_38&_with_basal_ganglia_calcification',
            cgd_entrezid: '9636',
            cgd_gene: 'ISG15',
            cgd_hgncid: '4053',
            cgd_inheritance: 'AR',
            cgd_interventioncategories: 'Allergy/Immunology/Infectious',
            cgd_interventionrationale:
              'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
            cgd_manifestationcategories:
              'Allergy/Immunology/Infectious&_Neurologic',
            cgd_references: '22859821&_25307056',
            exomiser_ad_exgenespheno: 0.5035,
            exomiser_xr_exgenescombi: 0.00031754,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'criteria_provided,_single_submitter',
            clndn: 'Immunodeficiency_38_with_basal_ganglia_calcification',
            gnomadv3_af_afr: 0.7432,
            gnomadv3_af_nfe: 0.9588,
            gnomadv3_af_xy: 0.9009,
            exomiser_xd_exgenescombi: 0.00031754,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.8861,
            clnvcso: 'SO:0001483',
            gnomadv3_af: 0.8988,
            exomiser_mt_exgenespheno: 0.5035,
            clndisdb:
              'MONDO:MONDO:0014502,MedGen:C4015293,OMIM:616126,Orphanet:ORPHA319563',
            exomiser_ar_exgenescombi: 0.00031754,
            gnomadv3_af_eas: 0.9998,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9101,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00031754,
            exomiser_ad_exgenescombi: 0.00031754,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'Benign',
            clnvc: 'single_nucleotide_variant',
            gnomadv3_af_ami: 0.9802,
            gnomadv3_af_fin: 0.9755,
            exomiser_ar_exgenespheno: 0.5035,
            gnomadv3_af_sas: 0.9679,
            gnomadv3_af_amr: 0.9357,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'NC_000001.11:g.1013541T>C',
            exomiser_ad_exgenesvar: 0,
            clnid: '1185394',
            gnomadv3_af_xx: 0.8968,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.9067,
            exomiser_xr_exgenespheno: 0.5035,
            exomiser_xd_exgenespheno: 0.5035,
            impact_severity: 'LOW',
            quality: 3849.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 43],
              [0, 34],
              [0, 50],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['ISG15'],
            ensembl_transcript_id: [
              'ENST00000649529',
              'ENST00000624652',
              'ENST00000624697',
            ],
            variant_type: 'small',
            highest_af: 0.9998,
            highest_af_info: {
              source: 'gnomad V2 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 948921,
            hg19_strand: '+',
            hg19_end: 948921,
            entrez_gene_id: ['9636'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9c6',
            variant_id: '67',
            end: 1014228,
            ref: 'G',
            alt: 'A',
            start: 1014228,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000649529',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '2/2',
                codon_change: 'aGc/aAc',
                aa_change: 'S/N',
                aa_length: 83,
                polyphen_pred: 'benign',
                polyphen_score: 0.003,
                sift_pred: 'tolerated',
                sift_score: 0.38,
                allele: 'A',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000649529.1:c.248G>A',
                hgvsp: 'ENSP00000496832.1:p.Ser83Asn',
                cds_position: 248,
                existing_variation: 'rs1921&COSV65106391',
                strand: 1,
                hgnc_id: 'HGNC:4053',
                mane_select: 'NM_005101.4',
                ccds: 'CCDS6.1',
                source: 'Ensembl',
                gene_pheno: 1,
                domains:
                  'PDB-ENSP_mappings:1z2m.A&PDB-ENSP_mappings:2hj8.A&PDB-ENSP_mappings:3phx.B&PDB-ENSP_mappings:3pse.B&PDB-ENSP_mappings:3r66.C&PDB-ENSP_mappings:3r66.D&PDB-ENSP_mappings:3rt3.B&PDB-ENSP_mappings:3sdl.C&PDB-ENSP_mappings:3sdl.D&PDB-ENSP_mappings:5tl6.A&PDB-ENSP_mappings:5tl6.C&PDB-ENSP_mappings:5w8t.B&PDB-ENSP_mappings:5w8t.D&PDB-ENSP_mappings:5w8u.B&PDB-ENSP_mappings:5w8u.D&PDB-ENSP_mappings:6bi8.C&PDB-ENSP_mappings:6bi8.D&PDB-ENSP_mappings:6ffa.B&PDB-ENSP_mappings:6xa9.B&PDB-ENSP_mappings:6xa9.D&PDB-ENSP_mappings:6xa9.F&PROSITE_profiles:PS50053&CDD:cd01810&PANTHER:PTHR10666:SF267&PANTHER:PTHR10666&Gene3D:3.10.20.90&SMART:SM00213&Superfamily:SSF54236',
                clin_sig: 'benign',
                pheno: '1&1',
                revel: 0.047,
                flaglrg: 'LRG_1231t1',
                cadd_phred: 1.515,
                cadd_raw: 0.027262,
                spliceai_pred_dp_ag: 13,
                spliceai_pred_dp_al: 1,
                spliceai_pred_dp_dg: 43,
                spliceai_pred_dp_dl: -6,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'ISG15',
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000624652',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '3/3',
                codon_change: 'aGc/aAc',
                aa_change: 'S/N',
                aa_length: 75,
                polyphen_pred: 'benign',
                polyphen_score: 0.003,
                sift_pred: 'tolerated',
                sift_score: 0.36,
                allele: 'A',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000624652.1:c.224G>A',
                hgvsp: 'ENSP00000485313.1:p.Ser75Asn',
                cds_position: 224,
                existing_variation: 'rs1921&COSV65106391',
                strand: 1,
                flags: 'cds_end_NF',
                hgnc_id: 'HGNC:4053',
                source: 'Ensembl',
                gene_pheno: 1,
                domains:
                  'Gene3D:3.10.20.90&PROSITE_profiles:PS50053&PANTHER:PTHR10666&PANTHER:PTHR10666:SF267&SMART:SM00213&Superfamily:SSF54236',
                clin_sig: 'benign',
                pheno: '1&1',
                revel: 0.047,
                cadd_phred: 1.515,
                cadd_raw: 0.027262,
                spliceai_pred_dp_ag: 13,
                spliceai_pred_dp_al: 1,
                spliceai_pred_dp_dg: 43,
                spliceai_pred_dp_dl: -6,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'ISG15',
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000624697',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '3/3',
                codon_change: 'aGc/aAc',
                aa_change: 'S/N',
                aa_length: 75,
                polyphen_pred: 'benign',
                polyphen_score: 0.003,
                sift_pred: 'tolerated',
                sift_score: 0.39,
                allele: 'A',
                impact: ['missense_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000624697.4:c.224G>A',
                hgvsp: 'ENSP00000485643.1:p.Ser75Asn',
                cds_position: 224,
                existing_variation: 'rs1921&COSV65106391',
                strand: 1,
                hgnc_id: 'HGNC:4053',
                source: 'Ensembl',
                gene_pheno: 1,
                domains:
                  'Gene3D:3.10.20.90&PROSITE_profiles:PS50053&PANTHER:PTHR10666&PANTHER:PTHR10666:SF267&SMART:SM00213&Superfamily:SSF54236&CDD:cd01810',
                clin_sig: 'benign',
                pheno: '1&1',
                revel: 0.047,
                cadd_phred: 1.515,
                cadd_raw: 0.027262,
                spliceai_pred_dp_ag: 13,
                spliceai_pred_dp_al: 1,
                spliceai_pred_dp_dg: 43,
                spliceai_pred_dp_dl: -6,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'ISG15',
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'missense_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000649529',
            ensembl_gene_id: ['ENSG00000187608'],
            is_exonic: 1,
            is_coding: 1,
            aa_change: 'S/N',
            polyphen_pred: ['benign'],
            polyphen_score: [0.003],
            sift_pred: ['tolerated'],
            sift_score: [0.38],
            impact: 'missense_variant',
            hgvsc: 'ENST00000649529.1:c.248G>A',
            hgvsp: 'ENSP00000496832.1:p.Ser83Asn',
            existing_variation: 'rs1921&COSV65106391',
            ccds: 'CCDS6.1',
            source: 'Ensembl',
            gene_pheno: '1',
            domains:
              'PDB-ENSP_mappings:1z2m.A&PDB-ENSP_mappings:2hj8.A&PDB-ENSP_mappings:3phx.B&PDB-ENSP_mappings:3pse.B&PDB-ENSP_mappings:3r66.C&PDB-ENSP_mappings:3r66.D&PDB-ENSP_mappings:3rt3.B&PDB-ENSP_mappings:3sdl.C&PDB-ENSP_mappings:3sdl.D&PDB-ENSP_mappings:5tl6.A&PDB-ENSP_mappings:5tl6.C&PDB-ENSP_mappings:5w8t.B&PDB-ENSP_mappings:5w8t.D&PDB-ENSP_mappings:5w8u.B&PDB-ENSP_mappings:5w8u.D&PDB-ENSP_mappings:6bi8.C&PDB-ENSP_mappings:6bi8.D&PDB-ENSP_mappings:6ffa.B&PDB-ENSP_mappings:6xa9.B&PDB-ENSP_mappings:6xa9.D&PDB-ENSP_mappings:6xa9.F&PROSITE_profiles:PS50053&CDD:cd01810&PANTHER:PTHR10666:SF267&PANTHER:PTHR10666&Gene3D:3.10.20.90&SMART:SM00213&Superfamily:SSF54236',
            gnomad_af: 0.3593,
            gnomad_afr_af: 0.4146,
            gnomad_amr_af: 0.2452,
            gnomad_asj_af: 0.3142,
            gnomad_eas_af: 0.1998,
            gnomad_fin_af: 0.4674,
            gnomad_nfe_af: 0.4004,
            gnomad_oth_af: 0.3684,
            gnomad_sas_af: 0.3404,
            pheno: '1&1',
            revel: [0.047],
            flaglrg: 'LRG_1231t1',
            cadd_phred: 1.515,
            cadd_raw: 0.027262,
            spliceai_pred_dp_ag: 13,
            spliceai_pred_dp_al: 1,
            spliceai_pred_dp_dg: 43,
            spliceai_pred_dp_dl: -6,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'ISG15',
            cgd_agegroup: 'Pediatric',
            cgd_condition:
              'Immunodeficiency_38&_with_basal_ganglia_calcification',
            cgd_entrezid: '9636',
            cgd_gene: 'ISG15',
            cgd_hgncid: '4053',
            cgd_inheritance: 'AR',
            cgd_interventioncategories: 'Allergy/Immunology/Infectious',
            cgd_interventionrationale:
              'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
            cgd_manifestationcategories:
              'Allergy/Immunology/Infectious&_Neurologic',
            cgd_references: '22859821&_25307056',
            exomiser_ad_exgenespheno: 0.5035,
            exomiser_xr_exgenescombi: 0.00031754,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'criteria_provided,_multiple_submitters,_no_conflicts',
            clndn:
              'Immunodeficiency_38_with_basal_ganglia_calcification|not_specified',
            gnomadv3_af_afr: 0.4058,
            gnomadv3_af_nfe: 0.3989,
            gnomadv3_af_xy: 0.3835,
            exomiser_xd_exgenescombi: 0.00031754,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.328,
            clnvcso: 'SO:0001483',
            gnomadv3_af: 0.3838,
            exomiser_mt_exgenespheno: 0.5035,
            clndisdb:
              'MONDO:MONDO:0014502,MedGen:C4015293,OMIM:616126,Orphanet:ORPHA319563|MedGen:CN169374',
            exomiser_ar_exgenescombi: 0.00031754,
            gnomadv3_af_eas: 0.2027,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.3095,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00031754,
            exomiser_ad_exgenescombi: 0.00031754,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'Benign',
            clnvc: 'single_nucleotide_variant',
            gnomadv3_af_ami: 0.3893,
            gnomadv3_af_fin: 0.4815,
            exomiser_ar_exgenespheno: 0.5035,
            gnomadv3_af_sas: 0.3489,
            gnomadv3_af_amr: 0.2813,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'NC_000001.11:g.1014228G>A',
            exomiser_ad_exgenesvar: 0,
            clnid: '402986',
            gnomadv3_af_xx: 0.3841,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.3676,
            exomiser_xr_exgenespheno: 0.5035,
            exomiser_xd_exgenespheno: 0.5035,
            impact_severity: 'MED',
            quality: 1653.1,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 0],
            ],
            allelic_depths: [
              [20, 34],
              [27, 24],
              [48, 0],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['ISG15'],
            ensembl_transcript_id: [
              'ENST00000649529',
              'ENST00000624652',
              'ENST00000624697',
            ],
            variant_type: 'small',
            highest_af: 0.3838,
            highest_af_info: {
              source: 'gnomad v3 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 949608,
            hg19_strand: '+',
            hg19_end: 949608,
            entrez_gene_id: ['9636'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9c7',
            variant_id: '68',
            end: 1014274,
            ref: 'A',
            alt: 'G',
            start: 1014274,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000649529',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '2/2',
                codon_change: 'gtA/gtG',
                aa_change: 'V',
                aa_length: 98,
                allele: 'G',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000649529.1:c.294A>G',
                hgvsp: 'ENSP00000496832.1:p.Val98%3D',
                cds_position: 294,
                existing_variation: 'rs8997&COSV65106735',
                strand: 1,
                hgnc_id: 'HGNC:4053',
                mane_select: 'NM_005101.4',
                ccds: 'CCDS6.1',
                source: 'Ensembl',
                gene_pheno: 1,
                domains:
                  'PDB-ENSP_mappings:1z2m.A&PDB-ENSP_mappings:2hj8.A&PDB-ENSP_mappings:3phx.B&PDB-ENSP_mappings:3pse.B&PDB-ENSP_mappings:3r66.C&PDB-ENSP_mappings:3r66.D&PDB-ENSP_mappings:3rt3.B&PDB-ENSP_mappings:3sdl.C&PDB-ENSP_mappings:3sdl.D&PDB-ENSP_mappings:5tl6.A&PDB-ENSP_mappings:5tl6.C&PDB-ENSP_mappings:5w8t.B&PDB-ENSP_mappings:5w8t.D&PDB-ENSP_mappings:5w8u.B&PDB-ENSP_mappings:5w8u.D&PDB-ENSP_mappings:6bi8.C&PDB-ENSP_mappings:6bi8.D&PDB-ENSP_mappings:6ffa.B&PDB-ENSP_mappings:6xa9.B&PDB-ENSP_mappings:6xa9.D&PDB-ENSP_mappings:6xa9.F&PROSITE_profiles:PS50053&CDD:cd01810&PANTHER:PTHR10666:SF267&PANTHER:PTHR10666&Gene3D:3.10.20.90&Pfam:PF00240&SMART:SM00213&Superfamily:SSF54236&Prints:PR00348',
                pheno: '0&1',
                flaglrg: 'LRG_1231t1',
                cadd_phred: 5.555,
                cadd_raw: 0.411615,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: -3,
                spliceai_pred_dp_dg: 36,
                spliceai_pred_dp_dl: -3,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.01,
                spliceai_pred_ds_dl: 0.06,
                spliceai_pred_symbol: 'ISG15',
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000624652',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '3/3',
                codon_change: 'gtA/gtG',
                aa_change: 'V',
                aa_length: 90,
                allele: 'G',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000624652.1:c.270A>G',
                hgvsp: 'ENSP00000485313.1:p.Val90%3D',
                cds_position: 270,
                existing_variation: 'rs8997&COSV65106735',
                strand: 1,
                flags: 'cds_end_NF',
                hgnc_id: 'HGNC:4053',
                source: 'Ensembl',
                gene_pheno: 1,
                domains:
                  'Gene3D:3.10.20.90&Pfam:PF00240&PROSITE_profiles:PS50053&PANTHER:PTHR10666&PANTHER:PTHR10666:SF267&SMART:SM00213&Superfamily:SSF54236',
                pheno: '0&1',
                cadd_phred: 5.555,
                cadd_raw: 0.411615,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: -3,
                spliceai_pred_dp_dg: 36,
                spliceai_pred_dp_dl: -3,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.01,
                spliceai_pred_ds_dl: 0.06,
                spliceai_pred_symbol: 'ISG15',
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000624697',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '3/3',
                codon_change: 'gtA/gtG',
                aa_change: 'V',
                aa_length: 90,
                allele: 'G',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000624697.4:c.270A>G',
                hgvsp: 'ENSP00000485643.1:p.Val90%3D',
                cds_position: 270,
                existing_variation: 'rs8997&COSV65106735',
                strand: 1,
                hgnc_id: 'HGNC:4053',
                source: 'Ensembl',
                gene_pheno: 1,
                domains:
                  'Gene3D:3.10.20.90&Pfam:PF00240&Prints:PR00348&PROSITE_profiles:PS50053&PANTHER:PTHR10666&PANTHER:PTHR10666:SF267&SMART:SM00213&Superfamily:SSF54236&CDD:cd01810',
                pheno: '0&1',
                cadd_phred: 5.555,
                cadd_raw: 0.411615,
                spliceai_pred_dp_ag: -15,
                spliceai_pred_dp_al: -3,
                spliceai_pred_dp_dg: 36,
                spliceai_pred_dp_dl: -3,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.01,
                spliceai_pred_ds_dl: 0.06,
                spliceai_pred_symbol: 'ISG15',
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000649529',
            ensembl_gene_id: ['ENSG00000187608'],
            is_exonic: 1,
            is_coding: 1,
            aa_change: 'V',
            impact: 'synonymous_variant',
            hgvsc: 'ENST00000649529.1:c.294A>G',
            hgvsp: 'ENSP00000496832.1:p.Val98%3D',
            existing_variation: 'rs8997&COSV65106735',
            ccds: 'CCDS6.1',
            source: 'Ensembl',
            gene_pheno: '1',
            domains:
              'PDB-ENSP_mappings:1z2m.A&PDB-ENSP_mappings:2hj8.A&PDB-ENSP_mappings:3phx.B&PDB-ENSP_mappings:3pse.B&PDB-ENSP_mappings:3r66.C&PDB-ENSP_mappings:3r66.D&PDB-ENSP_mappings:3rt3.B&PDB-ENSP_mappings:3sdl.C&PDB-ENSP_mappings:3sdl.D&PDB-ENSP_mappings:5tl6.A&PDB-ENSP_mappings:5tl6.C&PDB-ENSP_mappings:5w8t.B&PDB-ENSP_mappings:5w8t.D&PDB-ENSP_mappings:5w8u.B&PDB-ENSP_mappings:5w8u.D&PDB-ENSP_mappings:6bi8.C&PDB-ENSP_mappings:6bi8.D&PDB-ENSP_mappings:6ffa.B&PDB-ENSP_mappings:6xa9.B&PDB-ENSP_mappings:6xa9.D&PDB-ENSP_mappings:6xa9.F&PROSITE_profiles:PS50053&CDD:cd01810&PANTHER:PTHR10666:SF267&PANTHER:PTHR10666&Gene3D:3.10.20.90&Pfam:PF00240&SMART:SM00213&Superfamily:SSF54236&Prints:PR00348',
            gnomad_af: 0.9188,
            gnomad_afr_af: 0.561,
            gnomad_amr_af: 0.9466,
            gnomad_asj_af: 0.8982,
            gnomad_eas_af: 0.9262,
            gnomad_fin_af: 0.9498,
            gnomad_nfe_af: 0.9478,
            gnomad_oth_af: 0.9208,
            gnomad_sas_af: 0.9495,
            pheno: '0&1',
            flaglrg: 'LRG_1231t1',
            cadd_phred: 5.555,
            cadd_raw: 0.411615,
            spliceai_pred_dp_ag: -15,
            spliceai_pred_dp_al: -3,
            spliceai_pred_dp_dg: 36,
            spliceai_pred_dp_dl: -3,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0.01],
            spliceai_pred_ds_dl: [0.06],
            spliceai_pred_symbol: 'ISG15',
            cgd_agegroup: 'Pediatric',
            cgd_condition:
              'Immunodeficiency_38&_with_basal_ganglia_calcification',
            cgd_entrezid: '9636',
            cgd_gene: 'ISG15',
            cgd_hgncid: '4053',
            cgd_inheritance: 'AR',
            cgd_interventioncategories: 'Allergy/Immunology/Infectious',
            cgd_interventionrationale:
              'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
            cgd_manifestationcategories:
              'Allergy/Immunology/Infectious&_Neurologic',
            cgd_references: '22859821&_25307056',
            exomiser_ad_exgenespheno: 0.5035,
            exomiser_xr_exgenescombi: 0.00031754,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'criteria_provided,_multiple_submitters,_no_conflicts',
            clndn: 'Immunodeficiency_38_with_basal_ganglia_calcification',
            gnomadv3_af_afr: 0.5693,
            gnomadv3_af_nfe: 0.9489,
            gnomadv3_af_xy: 0.8434,
            exomiser_xd_exgenescombi: 0.00031754,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.8703,
            clnvcso: 'SO:0001483',
            gnomadv3_af: 0.8398,
            exomiser_mt_exgenespheno: 0.5035,
            clndisdb:
              'MONDO:MONDO:0014502,MedGen:C4015293,OMIM:616126,Orphanet:ORPHA319563',
            exomiser_ar_exgenescombi: 0.00031754,
            gnomadv3_af_eas: 0.9274,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9055,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00031754,
            exomiser_ad_exgenescombi: 0.00031754,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'Benign',
            clnvc: 'single_nucleotide_variant',
            gnomadv3_af_ami: 0.9737,
            gnomadv3_af_fin: 0.9551,
            exomiser_ar_exgenespheno: 0.5035,
            gnomadv3_af_sas: 0.9518,
            gnomadv3_af_amr: 0.9146,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'NC_000001.11:g.1014274A>G',
            exomiser_ad_exgenesvar: 0,
            clnid: '1166514',
            gnomadv3_af_xx: 0.8364,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.8628,
            exomiser_xr_exgenespheno: 0.5035,
            exomiser_xd_exgenespheno: 0.5035,
            impact_severity: 'LOW',
            quality: 5889.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 55],
              [0, 59],
              [0, 57],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['ISG15'],
            ensembl_transcript_id: [
              'ENST00000649529',
              'ENST00000624652',
              'ENST00000624697',
            ],
            variant_type: 'small',
            highest_af: 0.9274,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 949654,
            hg19_strand: '+',
            hg19_end: 949654,
            entrez_gene_id: ['9636'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9c8',
            variant_id: '69',
            end: 1014545,
            ref: 'C',
            alt: 'T',
            start: 1014545,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000649529',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'T',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2799070',
                strand: 1,
                hgnc_id: 'HGNC:4053',
                mane_select: 'NM_005101.4',
                ccds: 'CCDS6.1',
                source: 'Ensembl',
                gene_pheno: 1,
                flaglrg: 'LRG_1231t1',
                cadd_phred: 4.962,
                cadd_raw: 0.358576,
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000624652',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2799070',
                strand: 1,
                flags: 'cds_end_NF',
                hgnc_id: 'HGNC:4053',
                source: 'Ensembl',
                gene_pheno: 1,
                cadd_phred: 4.962,
                cadd_raw: 0.358576,
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
              {
                gene: 'ISG15',
                ensembl_gene_id: 'ENSG00000187608',
                transcript: 'ENST00000624697',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['downstream_gene_variant'],
                feature_type: 'Transcript',
                existing_variation: 'rs2799070',
                strand: 1,
                hgnc_id: 'HGNC:4053',
                source: 'Ensembl',
                gene_pheno: 1,
                cadd_phred: 4.962,
                cadd_raw: 0.358576,
                cgd_agegroup: 'Pediatric',
                cgd_condition:
                  'Immunodeficiency_38&_with_basal_ganglia_calcification',
                cgd_entrezid: 9636,
                cgd_gene: 'ISG15',
                cgd_hgncid: 4053,
                cgd_inheritance: 'AR',
                cgd_interventioncategories: 'Allergy/Immunology/Infectious',
                cgd_interventionrationale:
                  'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
                cgd_manifestationcategories:
                  'Allergy/Immunology/Infectious&_Neurologic',
                cgd_references: '22859821&_25307056',
                impact_so: 'downstream_gene_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'ISG15',
                entrez_gene_id: '9636',
                p_haplo: 0.08611,
                p_triplo: 0.27092,
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display: null,
                clingen_ts_display: null,
              },
            ],
            mane_select: 'ENST00000649529',
            ensembl_gene_id: ['ENSG00000187608'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'downstream_gene_variant',
            existing_variation: 'rs2799070',
            ccds: 'CCDS6.1',
            source: 'Ensembl',
            gene_pheno: '1',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            flaglrg: 'LRG_1231t1',
            cadd_phred: 4.962,
            cadd_raw: 0.358576,
            cgd_agegroup: 'Pediatric',
            cgd_condition:
              'Immunodeficiency_38&_with_basal_ganglia_calcification',
            cgd_entrezid: '9636',
            cgd_gene: 'ISG15',
            cgd_hgncid: '4053',
            cgd_inheritance: 'AR',
            cgd_interventioncategories: 'Allergy/Immunology/Infectious',
            cgd_interventionrationale:
              'Individuals_may_sufffer_from_severe_infections_(including_Mycobacterial_and_Salmonella)&_and_antiinfectious_prophylaxis_and_early_and_aggressive_treatment_of_infections_may_be_beneficial&_BCG_vaccine_has_been_reported_as_resulting_in_adverse_reactions',
            cgd_manifestationcategories:
              'Allergy/Immunology/Infectious&_Neurologic',
            cgd_references: '22859821&_25307056',
            exomiser_ad_exgenespheno: 0.5615,
            exomiser_xr_exgenescombi: 0.00058018,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.6715,
            gnomadv3_af_nfe: 0.9575,
            gnomadv3_af_xy: 0.8774,
            exomiser_xd_exgenescombi: 0.00058018,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.8703,
            gnomadv3_af: 0.8751,
            exomiser_mt_exgenespheno: 0.5615,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00058018,
            gnomadv3_af_eas: 0.971,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.8998,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00058018,
            exomiser_ad_exgenescombi: 0.00058018,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.9802,
            gnomadv3_af_fin: 0.9565,
            exomiser_ar_exgenespheno: 0.5615,
            gnomadv3_af_sas: 0.9625,
            gnomadv3_af_amr: 0.928,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.8729,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.8929,
            exomiser_xr_exgenespheno: 0.5615,
            exomiser_xd_exgenespheno: 0.5615,
            impact_severity: 'LOW',
            quality: 4849.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 45],
              [0, 41],
              [0, 55],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['ISG15'],
            ensembl_transcript_id: [
              'ENST00000649529',
              'ENST00000624652',
              'ENST00000624697',
            ],
            variant_type: 'small',
            highest_af: 0.971,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 949925,
            hg19_strand: '+',
            hg19_end: 949925,
            entrez_gene_id: ['9636'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9c9',
            variant_id: '70',
            end: 1020217,
            ref: 'G',
            alt: 'T',
            start: 1020217,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000379370',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '1/36',
                codon_change: 'ccG/ccT',
                aa_change: 'P',
                aa_length: 15,
                allele: 'T',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379370.7:c.45G>T',
                hgvsp: 'ENSP00000368678.2:p.Pro15%3D',
                cds_position: 45,
                existing_variation: 'rs115173026',
                strand: 1,
                hgnc_id: 'HGNC:329',
                mane_select: 'NM_198576.4',
                ccds: 'CCDS30551.1',
                source: 'Ensembl',
                gene_pheno: 1,
                domains:
                  'PROSITE_profiles:PS51257&Cleavage_site_(Signalp):SignalP-noTM&Low_complexity_(Seg):seg',
                clin_sig: 'benign',
                pheno: 1,
                flaglrg: 'LRG_198t1',
                cadd_phred: 11.22,
                cadd_raw: 0.967963,
                spliceai_pred_dp_ag: 35,
                spliceai_pred_dp_al: 1,
                spliceai_pred_dp_dg: 9,
                spliceai_pred_dp_dl: 17,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                constraint_mis_z: 0.22552,
                constraint_oe_lof: 0.31914,
                constraint_oe_lof_lower: 0.237,
                constraint_oe_lof_upper: 0.435,
                constraint_oe_mis: 0.98257,
                constraint_oe_mis_lower: 0.938,
                constraint_oe_mis_upper: 1.029,
                constraint_oe_syn: 1.2049,
                constraint_oe_syn_lower: 1.133,
                constraint_oe_syn_upper: 1.281,
                constraint_pli: 5.4727e-7,
                constraint_syn_z: -3.9572,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                p_li: 5.4727e-7,
                loeuf: 0.435,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000620552',
                is_exonic: true,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '1/39',
                allele: 'T',
                impact: ['5_prime_UTR_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000620552.4:c.-370G>T',
                existing_variation: 'rs115173026',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                existing_inframe_oorfs: 2,
                existing_outofframe_oorfs: 0,
                existing_uorfs: 1,
                cadd_phred: 11.22,
                cadd_raw: 0.967963,
                spliceai_pred_dp_ag: 35,
                spliceai_pred_dp_al: 1,
                spliceai_pred_dp_dg: 9,
                spliceai_pred_dp_dl: 17,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: '5_prime_UTR_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
            ],
            mane_select: 'ENST00000379370',
            ensembl_gene_id: ['ENSG00000188157'],
            is_exonic: 1,
            is_coding: 1,
            aa_change: 'P',
            impact: 'synonymous_variant',
            hgvsc: 'ENST00000379370.7:c.45G>T',
            hgvsp: 'ENSP00000368678.2:p.Pro15%3D',
            existing_variation: 'rs115173026',
            ccds: 'CCDS30551.1',
            source: 'Ensembl',
            gene_pheno: '1',
            domains:
              'PROSITE_profiles:PS51257&Cleavage_site_(Signalp):SignalP-noTM&Low_complexity_(Seg):seg',
            gnomad_af: 0.3428,
            gnomad_afr_af: 0.3172,
            gnomad_amr_af: 0.2569,
            gnomad_asj_af: 0.3322,
            gnomad_eas_af: 0.2351,
            gnomad_fin_af: 0.4483,
            gnomad_nfe_af: 0.3665,
            gnomad_oth_af: 0.3535,
            gnomad_sas_af: 0.3367,
            pheno: '1',
            flaglrg: 'LRG_198t1',
            cadd_phred: 11.22,
            cadd_raw: 0.967963,
            spliceai_pred_dp_ag: 35,
            spliceai_pred_dp_al: 1,
            spliceai_pred_dp_dg: 9,
            spliceai_pred_dp_dl: 17,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'AGRN',
            constraint_mis_z: [0.22552],
            constraint_oe_lof: 0.31914,
            constraint_oe_lof_lower: 0.237,
            constraint_oe_lof_upper: [0.435],
            constraint_oe_mis: 0.98257,
            constraint_oe_mis_lower: 0.938,
            constraint_oe_mis_upper: [1.029],
            constraint_oe_syn: 1.2049,
            constraint_oe_syn_lower: 1.133,
            constraint_oe_syn_upper: 1.281,
            constraint_pli: 5.4727e-7,
            constraint_syn_z: [-3.9572],
            cgd_agegroup: 'Pediatric',
            cgd_condition: 'Myasthenic_syndrome&_congenital_8',
            cgd_entrezid: '375790',
            cgd_gene: 'AGRN',
            cgd_hgncid: '329',
            cgd_inheritance: 'AR',
            cgd_interventioncategories:
              'Musculoskeletal&_Neurologic&_Pharmacogenomic',
            cgd_interventionrationale:
              'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
            cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
            cgd_references: '19631309&_20301347&_22205389&_22678886',
            exomiser_ad_exgenespheno: 0.5615,
            exomiser_xr_exgenescombi: 0.00058018,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'criteria_provided,_multiple_submitters,_no_conflicts',
            clndn: 'Myasthenic_syndrome,_congenital,_8|not_specified',
            gnomadv3_af_afr: 0.2628,
            gnomadv3_af_nfe: 0.3409,
            gnomadv3_af_xy: 0.3099,
            exomiser_xd_exgenescombi: 0.00058018,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.2962,
            clnvcso: 'SO:0001483',
            gnomadv3_af: 0.3072,
            exomiser_mt_exgenespheno: 0.5615,
            clndisdb:
              'MONDO:MONDO:0014052,MedGen:C3808739,OMIM:615120|MedGen:CN169374',
            exomiser_ar_exgenescombi: 0.00058018,
            gnomadv3_af_eas: 0.1771,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.282,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00058018,
            exomiser_ad_exgenescombi: 0.00058018,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'Benign',
            clnvc: 'single_nucleotide_variant',
            gnomadv3_af_ami: 0.3918,
            gnomadv3_af_fin: 0.4346,
            exomiser_ar_exgenespheno: 0.5615,
            gnomadv3_af_sas: 0.3192,
            gnomadv3_af_amr: 0.2337,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'NC_000001.11:g.1020217G>T',
            exomiser_ad_exgenesvar: 0,
            clnid: '128310',
            gnomadv3_af_xx: 0.3047,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.2964,
            exomiser_xr_exgenespheno: 0.5615,
            exomiser_xd_exgenespheno: 0.5615,
            impact_severity: 'LOW',
            quality: 721.1,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 0],
            ],
            allelic_depths: [
              [19, 14],
              [20, 12],
              [37, 0],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['AGRN'],
            ensembl_transcript_id: ['ENST00000379370', 'ENST00000620552'],
            variant_type: 'small',
            highest_af: 0.3428,
            highest_af_info: {
              source: 'gnomad V2 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 955597,
            hg19_strand: '+',
            hg19_end: 955597,
            entrez_gene_id: ['375790'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9ca',
            variant_id: '71',
            end: 1022518,
            ref: 'G',
            alt: 'T',
            start: 1022518,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000379370',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'T',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '2/35',
                hgvsc: 'ENST00000379370.7:c.463+56G>T',
                existing_variation: 'rs2799064&COSV65071508',
                strand: 1,
                hgnc_id: 'HGNC:329',
                mane_select: 'NM_198576.4',
                ccds: 'CCDS30551.1',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: '1&1',
                flaglrg: 'LRG_198t1',
                cadd_phred: 2.941,
                cadd_raw: 0.183806,
                spliceai_pred_dp_ag: 14,
                spliceai_pred_dp_al: -1,
                spliceai_pred_dp_dg: -2,
                spliceai_pred_dp_dl: -20,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                constraint_mis_z: 0.22552,
                constraint_oe_lof: 0.31914,
                constraint_oe_lof_lower: 0.237,
                constraint_oe_lof_upper: 0.435,
                constraint_oe_mis: 0.98257,
                constraint_oe_mis_lower: 0.938,
                constraint_oe_mis_upper: 1.029,
                constraint_oe_syn: 1.2049,
                constraint_oe_syn_lower: 1.133,
                constraint_oe_syn_upper: 1.281,
                constraint_pli: 5.4727e-7,
                constraint_syn_z: -3.9572,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                p_li: 5.4727e-7,
                loeuf: 0.435,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000620552',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'T',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '2/38',
                hgvsc: 'ENST00000620552.4:c.49+56G>T',
                existing_variation: 'rs2799064&COSV65071508',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: '1&1',
                cadd_phred: 2.941,
                cadd_raw: 0.183806,
                spliceai_pred_dp_ag: 14,
                spliceai_pred_dp_al: -1,
                spliceai_pred_dp_dg: -2,
                spliceai_pred_dp_dl: -20,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
            ],
            mane_select: 'ENST00000379370',
            ensembl_gene_id: ['ENSG00000188157'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant',
            hgvsc: 'ENST00000379370.7:c.463+56G>T',
            existing_variation: 'rs2799064&COSV65071508',
            ccds: 'CCDS30551.1',
            source: 'Ensembl',
            gene_pheno: '1',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            pheno: '1&1',
            flaglrg: 'LRG_198t1',
            cadd_phred: 2.941,
            cadd_raw: 0.183806,
            spliceai_pred_dp_ag: 14,
            spliceai_pred_dp_al: -1,
            spliceai_pred_dp_dg: -2,
            spliceai_pred_dp_dl: -20,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'AGRN',
            constraint_mis_z: [0.22552],
            constraint_oe_lof: 0.31914,
            constraint_oe_lof_lower: 0.237,
            constraint_oe_lof_upper: [0.435],
            constraint_oe_mis: 0.98257,
            constraint_oe_mis_lower: 0.938,
            constraint_oe_mis_upper: [1.029],
            constraint_oe_syn: 1.2049,
            constraint_oe_syn_lower: 1.133,
            constraint_oe_syn_upper: 1.281,
            constraint_pli: 5.4727e-7,
            constraint_syn_z: [-3.9572],
            cgd_agegroup: 'Pediatric',
            cgd_condition: 'Myasthenic_syndrome&_congenital_8',
            cgd_entrezid: '375790',
            cgd_gene: 'AGRN',
            cgd_hgncid: '329',
            cgd_inheritance: 'AR',
            cgd_interventioncategories:
              'Musculoskeletal&_Neurologic&_Pharmacogenomic',
            cgd_interventionrationale:
              'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
            cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
            cgd_references: '19631309&_20301347&_22205389&_22678886',
            exomiser_ad_exgenespheno: 0.5615,
            exomiser_xr_exgenescombi: 0.00058018,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'criteria_provided,_single_submitter',
            clndn: 'not_provided',
            gnomadv3_af_afr: 0.3594,
            gnomadv3_af_nfe: 0.3545,
            gnomadv3_af_xy: 0.3471,
            exomiser_xd_exgenescombi: 0.00058018,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.3006,
            clnvcso: 'SO:0001483',
            gnomadv3_af: 0.3451,
            exomiser_mt_exgenespheno: 0.5615,
            clndisdb: 'MedGen:CN517202',
            exomiser_ar_exgenescombi: 0.00058018,
            gnomadv3_af_eas: 0.1891,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.31,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00058018,
            exomiser_ad_exgenescombi: 0.00058018,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'Benign',
            clnvc: 'single_nucleotide_variant',
            gnomadv3_af_ami: 0.389,
            gnomadv3_af_fin: 0.4412,
            exomiser_ar_exgenespheno: 0.5615,
            gnomadv3_af_sas: 0.3717,
            gnomadv3_af_amr: 0.2505,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'NC_000001.11:g.1022518G>T',
            exomiser_ad_exgenesvar: 0,
            clnid: '677944',
            gnomadv3_af_xx: 0.3432,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.3285,
            exomiser_xr_exgenespheno: 0.5615,
            exomiser_xd_exgenespheno: 0.5615,
            impact_severity: 'LOW',
            quality: 891.1,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [0, 0],
            ],
            allelic_depths: [
              [23, 16],
              [14, 17],
              [52, 0],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['AGRN'],
            ensembl_transcript_id: ['ENST00000379370', 'ENST00000620552'],
            variant_type: 'small',
            highest_af: 0.3451,
            highest_af_info: {
              source: 'gnomad v3 AF',
            },
            hg19_chrom: 'chr1',
            hg19_start: 957898,
            hg19_strand: '+',
            hg19_end: 957898,
            entrez_gene_id: ['375790'],
          },
          {
            allelic_depths: [],
            genotype_qualities: [],
            is_read: false,
            _id: '64e5a4cff99efe31718b84a5',
            chrom: 'chr1',
            start: 1028671,
            end: 1029008,
            len: 337,
            ref: 'A',
            alt: '<DEL>',
            p_lof: 'False',
            constraint_pli: '5.4727e-7',
            constraint_oe_lof_upper: [0.435],
            mane_select: 'ENST00000379370',
            variant_type: 'structural',
            type: 'DEL',
            impact: '',
            gene_objs: [
              {
                is_related: true,
                transcript: 'ENST00000379370',
                gene: 'AGRN',
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                ensembl_gene_id: 'ENSG00000188157',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_hi_order: 2,
                clingen_ts: 'no',
                p_li: 5.4727e-7,
                loeuf: 0.435,
                is_canonical: true,
                is_mane_select: true,
                annotation_overlap: 0.00936,
                reciprocal_overlap: 0.00936,
                bkpt_start: 1020120,
                bkpt_end: 1056116,
                default_show: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                is_related: true,
                transcript: 'ENST00000620552',
                gene: 'AGRN',
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                ensembl_gene_id: 'ENSG00000188157',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_hi_order: 2,
                clingen_ts: 'no',
                annotation_overlap: 0.00936,
                reciprocal_overlap: 0.00936,
                bkpt_start: 1020123,
                bkpt_end: 1056118,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
            ],
            highest_af: -1,
            gnomadv3_af: -1,
            gnomadv3_af_afr: -1,
            gnomadv3_af_amr: -1,
            gnomadv3_af_eas: -1,
            gnomadv3_af_nfe: -1,
            gnomadv3_af_sas: -1,
            gnomadv3_af_xx: -1,
            gnomadv3_af_xy: -1,
            gnomadv3_af_ami: -1,
            gnomadv3_af_asj: -1,
            gnomadv3_af_fin: -1,
            gnomadv3_af_mid: -1,
            gnomadv3_af_oth: -1,
            gnomad_af: -1,
            gnomad_eas_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_nfe_af: -1,
            gnomad_sas_af: -1,
            gnomad_asj_af: -1,
            gnomad_fin_af: -1,
            gnomad_oth_af: -1,
            is_pathogenic: false,
            variant_id: 'S:survindel:3',
            sv_id: 'S:survindel:3',
            afs: [],
            exts: [],
            intpns: [],
            caller: 'survindel',
            genotypes_index: [
              [1, 1],
              [0, 1],
              [0, 0],
            ],
            hg19_chrom: 'chr1',
            hg19_start: 964051,
            hg19_strand: '+',
            hg19_end: 964388,
            gene_symbol: ['AGRN'],
            ensembl_gene_id: ['ENSG00000188157'],
            entrez_gene_id: ['375790'],
            ensembl_transcript_id: ['ENST00000379370', 'ENST00000620552'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9cb',
            variant_id: '72',
            end: 1041122,
            ref: 'GGGC',
            alt: 'G',
            start: 1041119,
            chrom: 'chr1',
            type: 'indel',
            gene_objs: [
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000379370',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '4/35',
                hgvsc: 'ENST00000379370.7:c.728-51_728-49del',
                existing_variation: 'rs1266764583',
                strand: 1,
                hgnc_id: 'HGNC:329',
                mane_select: 'NM_198576.4',
                ccds: 'CCDS30551.1',
                source: 'Ensembl',
                gene_pheno: 1,
                flaglrg: 'LRG_198t1',
                spliceai_pred_dp_ag: 41,
                spliceai_pred_dp_al: 8,
                spliceai_pred_dp_dg: -39,
                spliceai_pred_dp_dl: -30,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                constraint_mis_z: 0.22552,
                constraint_oe_lof: 0.31914,
                constraint_oe_lof_lower: 0.237,
                constraint_oe_lof_upper: 0.435,
                constraint_oe_mis: 0.98257,
                constraint_oe_mis_lower: 0.938,
                constraint_oe_mis_upper: 1.029,
                constraint_oe_syn: 1.2049,
                constraint_oe_syn_lower: 1.133,
                constraint_oe_syn_upper: 1.281,
                constraint_pli: 5.4727e-7,
                constraint_syn_z: -3.9572,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                p_li: 5.4727e-7,
                loeuf: 0.435,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000620552',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '4/38',
                hgvsc: 'ENST00000620552.4:c.314-51_314-49del',
                existing_variation: 'rs1266764583',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                spliceai_pred_dp_ag: 41,
                spliceai_pred_dp_al: 8,
                spliceai_pred_dp_dg: -39,
                spliceai_pred_dp_dl: -30,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000651234',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '3/37',
                hgvsc: 'ENST00000651234.1:c.413-51_413-49del',
                existing_variation: 'rs1266764583',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                spliceai_pred_dp_ag: 41,
                spliceai_pred_dp_al: 8,
                spliceai_pred_dp_dg: -39,
                spliceai_pred_dp_dl: -30,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000652369',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '3/34',
                hgvsc: 'ENST00000652369.1:c.413-51_413-49del',
                existing_variation: 'rs1266764583',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                spliceai_pred_dp_ag: 41,
                spliceai_pred_dp_al: 8,
                spliceai_pred_dp_dg: -39,
                spliceai_pred_dp_dl: -30,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
            ],
            mane_select: 'ENST00000379370',
            ensembl_gene_id: ['ENSG00000188157'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant',
            hgvsc: 'ENST00000379370.7:c.728-51_728-49del',
            existing_variation: 'rs1266764583',
            ccds: 'CCDS30551.1',
            source: 'Ensembl',
            gene_pheno: '1',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            flaglrg: 'LRG_198t1',
            spliceai_pred_dp_ag: 41,
            spliceai_pred_dp_al: 8,
            spliceai_pred_dp_dg: -39,
            spliceai_pred_dp_dl: -30,
            spliceai_pred_ds_ag: [0.01],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'AGRN',
            constraint_mis_z: [0.22552],
            constraint_oe_lof: 0.31914,
            constraint_oe_lof_lower: 0.237,
            constraint_oe_lof_upper: [0.435],
            constraint_oe_mis: 0.98257,
            constraint_oe_mis_lower: 0.938,
            constraint_oe_mis_upper: [1.029],
            constraint_oe_syn: 1.2049,
            constraint_oe_syn_lower: 1.133,
            constraint_oe_syn_upper: 1.281,
            constraint_pli: 5.4727e-7,
            constraint_syn_z: [-3.9572],
            cgd_agegroup: 'Pediatric',
            cgd_condition: 'Myasthenic_syndrome&_congenital_8',
            cgd_entrezid: '375790',
            cgd_gene: 'AGRN',
            cgd_hgncid: '329',
            cgd_inheritance: 'AR',
            cgd_interventioncategories:
              'Musculoskeletal&_Neurologic&_Pharmacogenomic',
            cgd_interventionrationale:
              'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
            cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
            cgd_references: '19631309&_20301347&_22205389&_22678886',
            exomiser_ad_exgenespheno: 0.5615,
            exomiser_xr_exgenescombi: 0.00058018,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.3525,
            gnomadv3_af_nfe: 0.5836,
            gnomadv3_af_xy: 0.5241,
            exomiser_xd_exgenescombi: 0.00058018,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.5324,
            gnomadv3_af: 0.524,
            exomiser_mt_exgenespheno: 0.5615,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00058018,
            gnomadv3_af_eas: 0.6205,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.6382,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00058018,
            exomiser_ad_exgenescombi: 0.00058018,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.407,
            gnomadv3_af_fin: 0.5035,
            exomiser_ar_exgenespheno: 0.5615,
            gnomadv3_af_sas: 0.5919,
            gnomadv3_af_amr: 0.6531,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.5238,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.5329,
            exomiser_xr_exgenespheno: 0.5615,
            exomiser_xd_exgenespheno: 0.5615,
            impact_severity: 'LOW',
            quality: 2939.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 23],
              [0, 22],
              [0, 22],
            ],
            genotype_qualities: [[99, 66, 66]],
            pass_filter: 'PASS',
            gene_symbol: ['AGRN'],
            ensembl_transcript_id: [
              'ENST00000379370',
              'ENST00000620552',
              'ENST00000651234',
              'ENST00000652369',
            ],
            variant_type: 'small',
            highest_af: 0.6205,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 976499,
            hg19_strand: '+',
            hg19_end: 976502,
            entrez_gene_id: ['375790'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9cc',
            variant_id: '73',
            end: 1041134,
            ref: 'AGCGGGGGC',
            alt: 'A',
            start: 1041126,
            chrom: 'chr1',
            type: 'indel',
            gene_objs: [
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000379370',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '4/35',
                hgvsc: 'ENST00000379370.7:c.728-45_728-38del',
                existing_variation: 'rs768504677',
                strand: 1,
                hgnc_id: 'HGNC:329',
                mane_select: 'NM_198576.4',
                ccds: 'CCDS30551.1',
                source: 'Ensembl',
                gene_pheno: 1,
                flaglrg: 'LRG_198t1',
                constraint_mis_z: 0.22552,
                constraint_oe_lof: 0.31914,
                constraint_oe_lof_lower: 0.237,
                constraint_oe_lof_upper: 0.435,
                constraint_oe_mis: 0.98257,
                constraint_oe_mis_lower: 0.938,
                constraint_oe_mis_upper: 1.029,
                constraint_oe_syn: 1.2049,
                constraint_oe_syn_lower: 1.133,
                constraint_oe_syn_upper: 1.281,
                constraint_pli: 5.4727e-7,
                constraint_syn_z: -3.9572,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                p_li: 5.4727e-7,
                loeuf: 0.435,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000620552',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '4/38',
                hgvsc: 'ENST00000620552.4:c.314-45_314-38del',
                existing_variation: 'rs768504677',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000651234',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '3/37',
                hgvsc: 'ENST00000651234.1:c.413-45_413-38del',
                existing_variation: 'rs768504677',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000652369',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '3/34',
                hgvsc: 'ENST00000652369.1:c.413-45_413-38del',
                existing_variation: 'rs768504677',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
            ],
            mane_select: 'ENST00000379370',
            ensembl_gene_id: ['ENSG00000188157'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant',
            hgvsc: 'ENST00000379370.7:c.728-45_728-38del',
            existing_variation: 'rs768504677',
            ccds: 'CCDS30551.1',
            source: 'Ensembl',
            gene_pheno: '1',
            gnomad_af: 0.3124,
            gnomad_afr_af: 0.4318,
            gnomad_amr_af: 0.4173,
            gnomad_asj_af: 0.4583,
            gnomad_eas_af: 0.3684,
            gnomad_fin_af: 0.4722,
            gnomad_nfe_af: 0.2111,
            gnomad_oth_af: 0.3065,
            gnomad_sas_af: 0.2639,
            flaglrg: 'LRG_198t1',
            constraint_mis_z: [0.22552],
            constraint_oe_lof: 0.31914,
            constraint_oe_lof_lower: 0.237,
            constraint_oe_lof_upper: [0.435],
            constraint_oe_mis: 0.98257,
            constraint_oe_mis_lower: 0.938,
            constraint_oe_mis_upper: [1.029],
            constraint_oe_syn: 1.2049,
            constraint_oe_syn_lower: 1.133,
            constraint_oe_syn_upper: 1.281,
            constraint_pli: 5.4727e-7,
            constraint_syn_z: [-3.9572],
            cgd_agegroup: 'Pediatric',
            cgd_condition: 'Myasthenic_syndrome&_congenital_8',
            cgd_entrezid: '375790',
            cgd_gene: 'AGRN',
            cgd_hgncid: '329',
            cgd_inheritance: 'AR',
            cgd_interventioncategories:
              'Musculoskeletal&_Neurologic&_Pharmacogenomic',
            cgd_interventionrationale:
              'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
            cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
            cgd_references: '19631309&_20301347&_22205389&_22678886',
            exomiser_ad_exgenespheno: 0.5615,
            exomiser_xr_exgenescombi: 0.00058018,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.7932,
            gnomadv3_af_nfe: 0.9095,
            gnomadv3_af_xy: 0.8936,
            exomiser_xd_exgenescombi: 0.00058018,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9085,
            gnomadv3_af: 0.8922,
            exomiser_mt_exgenespheno: 0.5615,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00058018,
            gnomadv3_af_eas: 0.9375,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9263,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00058018,
            exomiser_ad_exgenescombi: 0.00058018,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.8665,
            gnomadv3_af_fin: 0.9463,
            exomiser_ar_exgenespheno: 0.5615,
            gnomadv3_af_sas: 0.939,
            gnomadv3_af_amr: 0.9297,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.8908,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.9002,
            exomiser_xr_exgenespheno: 0.5615,
            exomiser_xd_exgenespheno: 0.5615,
            impact_severity: 'LOW',
            quality: 2898.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [1, 22],
              [0, 21],
              [0, 22],
            ],
            genotype_qualities: [[99, 66, 66]],
            pass_filter: 'PASS',
            gene_symbol: ['AGRN'],
            ensembl_transcript_id: [
              'ENST00000379370',
              'ENST00000620552',
              'ENST00000651234',
              'ENST00000652369',
            ],
            variant_type: 'small',
            highest_af: 0.9375,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 976506,
            hg19_strand: '+',
            hg19_end: 976514,
            entrez_gene_id: ['375790'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9cd',
            variant_id: '74',
            end: 1041823,
            ref: 'CCGGCCAGTGCCAGGGTCGAGGTGGGCGGCTCCCCCGGGGGAGGGCTG',
            alt: 'C',
            start: 1041776,
            chrom: 'chr1',
            type: 'indel',
            gene_objs: [
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000379370',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '6/35',
                hgvsc: 'ENST00000379370.7:c.1177+109_1178-99del',
                existing_variation: 'rs70949546',
                strand: 1,
                hgnc_id: 'HGNC:329',
                mane_select: 'NM_198576.4',
                ccds: 'CCDS30551.1',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                flaglrg: 'LRG_198t1',
                constraint_mis_z: 0.22552,
                constraint_oe_lof: 0.31914,
                constraint_oe_lof_lower: 0.237,
                constraint_oe_lof_upper: 0.435,
                constraint_oe_mis: 0.98257,
                constraint_oe_mis_lower: 0.938,
                constraint_oe_mis_upper: 1.029,
                constraint_oe_syn: 1.2049,
                constraint_oe_syn_lower: 1.133,
                constraint_oe_syn_upper: 1.281,
                constraint_pli: 5.4727e-7,
                constraint_syn_z: -3.9572,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                p_li: 5.4727e-7,
                loeuf: 0.435,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000620552',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '6/38',
                hgvsc: 'ENST00000620552.4:c.763+109_764-99del',
                existing_variation: 'rs70949546',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000651234',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '5/37',
                hgvsc: 'ENST00000651234.1:c.862+109_863-99del',
                existing_variation: 'rs70949546',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000652369',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '5/34',
                hgvsc: 'ENST00000652369.1:c.862+109_863-99del',
                existing_variation: 'rs70949546',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
            ],
            mane_select: 'ENST00000379370',
            ensembl_gene_id: ['ENSG00000188157'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant',
            hgvsc: 'ENST00000379370.7:c.1177+109_1178-99del',
            existing_variation: 'rs70949546',
            ccds: 'CCDS30551.1',
            source: 'Ensembl',
            gene_pheno: '1',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            pheno: '1',
            flaglrg: 'LRG_198t1',
            constraint_mis_z: [0.22552],
            constraint_oe_lof: 0.31914,
            constraint_oe_lof_lower: 0.237,
            constraint_oe_lof_upper: [0.435],
            constraint_oe_mis: 0.98257,
            constraint_oe_mis_lower: 0.938,
            constraint_oe_mis_upper: [1.029],
            constraint_oe_syn: 1.2049,
            constraint_oe_syn_lower: 1.133,
            constraint_oe_syn_upper: 1.281,
            constraint_pli: 5.4727e-7,
            constraint_syn_z: [-3.9572],
            cgd_agegroup: 'Pediatric',
            cgd_condition: 'Myasthenic_syndrome&_congenital_8',
            cgd_entrezid: '375790',
            cgd_gene: 'AGRN',
            cgd_hgncid: '329',
            cgd_inheritance: 'AR',
            cgd_interventioncategories:
              'Musculoskeletal&_Neurologic&_Pharmacogenomic',
            cgd_interventionrationale:
              'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
            cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
            cgd_references: '19631309&_20301347&_22205389&_22678886',
            exomiser_ad_exgenespheno: 0.5615,
            exomiser_xr_exgenescombi: 0.00058018,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'criteria_provided,_single_submitter',
            clndn: 'not_provided',
            gnomadv3_af_afr: 0.2483,
            gnomadv3_af_nfe: 0.6895,
            gnomadv3_af_xy: 0.5695,
            exomiser_xd_exgenescombi: 0.00058018,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.5385,
            clnvcso: 'SO:0000159',
            gnomadv3_af: 0.5706,
            exomiser_mt_exgenespheno: 0.5615,
            clndisdb: 'MedGen:CN517202',
            exomiser_ar_exgenescombi: 0.00058018,
            gnomadv3_af_eas: 0.7442,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.701,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00058018,
            exomiser_ad_exgenescombi: 0.00058018,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'Benign',
            clnvc: 'Deletion',
            gnomadv3_af_ami: 0.7102,
            gnomadv3_af_fin: 0.7113,
            exomiser_ar_exgenespheno: 0.5615,
            gnomadv3_af_sas: 0.68,
            gnomadv3_af_amr: 0.6996,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'NC_000001.11:g.1041811_1041857del',
            exomiser_ad_exgenesvar: 0,
            clnid: '679581',
            gnomadv3_af_xx: 0.5717,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.5802,
            exomiser_xr_exgenespheno: 0.5615,
            exomiser_xd_exgenespheno: 0.5615,
            impact_severity: 'LOW',
            quality: 2295.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [1, 1],
            ],
            allelic_depths: [
              [3, 22],
              [0, 17],
              [0, 28],
            ],
            genotype_qualities: [[99, 7, 35]],
            pass_filter: 'PASS',
            gene_symbol: ['AGRN'],
            ensembl_transcript_id: [
              'ENST00000379370',
              'ENST00000620552',
              'ENST00000651234',
              'ENST00000652369',
            ],
            variant_type: 'small',
            highest_af: 0.7442,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 977156,
            hg19_strand: '+',
            hg19_end: 977203,
            entrez_gene_id: ['375790'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9ce',
            variant_id: '75',
            end: 1041950,
            ref: 'T',
            alt: 'C',
            start: 1041950,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000379370',
                is_exonic: false,
                is_coding: false,
                is_splicing: true,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['splice_region_variant', 'intron_variant'],
                feature_type: 'Transcript',
                intron: '6/35',
                hgvsc: 'ENST00000379370.7:c.1178-6T>C',
                existing_variation: 'rs2799066',
                strand: 1,
                hgnc_id: 'HGNC:329',
                mane_select: 'NM_198576.4',
                ccds: 'CCDS30551.1',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                flaglrg: 'LRG_198t1',
                cadd_phred: 0.67,
                cadd_raw: -0.144509,
                spliceai_pred_dp_ag: -7,
                spliceai_pred_dp_al: 11,
                spliceai_pred_dp_dg: 11,
                spliceai_pred_dp_dl: 39,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.12,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                constraint_mis_z: 0.22552,
                constraint_oe_lof: 0.31914,
                constraint_oe_lof_lower: 0.237,
                constraint_oe_lof_upper: 0.435,
                constraint_oe_mis: 0.98257,
                constraint_oe_mis_lower: 0.938,
                constraint_oe_mis_upper: 1.029,
                constraint_oe_syn: 1.2049,
                constraint_oe_syn_lower: 1.133,
                constraint_oe_syn_upper: 1.281,
                constraint_pli: 5.4727e-7,
                constraint_syn_z: -3.9572,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'splice_region_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                p_li: 5.4727e-7,
                loeuf: 0.435,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000620552',
                is_exonic: false,
                is_coding: false,
                is_splicing: true,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['splice_region_variant', 'intron_variant'],
                feature_type: 'Transcript',
                intron: '6/38',
                hgvsc: 'ENST00000620552.4:c.764-6T>C',
                existing_variation: 'rs2799066',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                cadd_phred: 0.67,
                cadd_raw: -0.144509,
                spliceai_pred_dp_ag: -7,
                spliceai_pred_dp_al: 11,
                spliceai_pred_dp_dg: 11,
                spliceai_pred_dp_dl: 39,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.12,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'splice_region_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000651234',
                is_exonic: false,
                is_coding: false,
                is_splicing: true,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['splice_region_variant', 'intron_variant'],
                feature_type: 'Transcript',
                intron: '5/37',
                hgvsc: 'ENST00000651234.1:c.863-6T>C',
                existing_variation: 'rs2799066',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                cadd_phred: 0.67,
                cadd_raw: -0.144509,
                spliceai_pred_dp_ag: -7,
                spliceai_pred_dp_al: 11,
                spliceai_pred_dp_dg: 11,
                spliceai_pred_dp_dl: 39,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.12,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'splice_region_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000652369',
                is_exonic: false,
                is_coding: false,
                is_splicing: true,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['splice_region_variant', 'intron_variant'],
                feature_type: 'Transcript',
                intron: '5/34',
                hgvsc: 'ENST00000652369.1:c.863-6T>C',
                existing_variation: 'rs2799066',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                cadd_phred: 0.67,
                cadd_raw: -0.144509,
                spliceai_pred_dp_ag: -7,
                spliceai_pred_dp_al: 11,
                spliceai_pred_dp_dg: 11,
                spliceai_pred_dp_dl: 39,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.12,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'splice_region_variant',
                impact_severity: 'MED',
                impact_severity_score: 1,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
            ],
            mane_select: 'ENST00000379370',
            ensembl_gene_id: ['ENSG00000188157'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'splice_region_variant&intron_variant',
            hgvsc: 'ENST00000379370.7:c.1178-6T>C',
            existing_variation: 'rs2799066',
            ccds: 'CCDS30551.1',
            source: 'Ensembl',
            gene_pheno: '1',
            gnomad_af: 0.9233,
            gnomad_afr_af: 0.7112,
            gnomad_amr_af: 0.9515,
            gnomad_asj_af: 0.9248,
            gnomad_eas_af: 0.9998,
            gnomad_fin_af: 0.9543,
            gnomad_nfe_af: 0.918,
            gnomad_oth_af: 0.919,
            gnomad_sas_af: 0.953,
            pheno: '1',
            flaglrg: 'LRG_198t1',
            cadd_phred: 0.67,
            cadd_raw: -0.144509,
            spliceai_pred_dp_ag: -7,
            spliceai_pred_dp_al: 11,
            spliceai_pred_dp_dg: 11,
            spliceai_pred_dp_dl: 39,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0.12],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'AGRN',
            constraint_mis_z: [0.22552],
            constraint_oe_lof: 0.31914,
            constraint_oe_lof_lower: 0.237,
            constraint_oe_lof_upper: [0.435],
            constraint_oe_mis: 0.98257,
            constraint_oe_mis_lower: 0.938,
            constraint_oe_mis_upper: [1.029],
            constraint_oe_syn: 1.2049,
            constraint_oe_syn_lower: 1.133,
            constraint_oe_syn_upper: 1.281,
            constraint_pli: 5.4727e-7,
            constraint_syn_z: [-3.9572],
            cgd_agegroup: 'Pediatric',
            cgd_condition: 'Myasthenic_syndrome&_congenital_8',
            cgd_entrezid: '375790',
            cgd_gene: 'AGRN',
            cgd_hgncid: '329',
            cgd_inheritance: 'AR',
            cgd_interventioncategories:
              'Musculoskeletal&_Neurologic&_Pharmacogenomic',
            cgd_interventionrationale:
              'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
            cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
            cgd_references: '19631309&_20301347&_22205389&_22678886',
            exomiser_ad_exgenespheno: 0.5615,
            exomiser_xr_exgenescombi: 0.00058018,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'criteria_provided,_multiple_submitters,_no_conflicts',
            clndn: 'Myasthenic_syndrome,_congenital,_8|not_specified',
            gnomadv3_af_afr: 0.7171,
            gnomadv3_af_nfe: 0.9218,
            gnomadv3_af_xy: 0.8771,
            exomiser_xd_exgenescombi: 0.00058018,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.8924,
            clnvcso: 'SO:0001483',
            gnomadv3_af: 0.8731,
            exomiser_mt_exgenespheno: 0.5615,
            clndisdb:
              'MONDO:MONDO:0014052,MedGen:C3808739,OMIM:615120|MedGen:CN169374',
            exomiser_ar_exgenescombi: 0.00058018,
            gnomadv3_af_eas: 0.9998,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9266,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00058018,
            exomiser_ad_exgenescombi: 0.00058018,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'Benign',
            clnvc: 'single_nucleotide_variant',
            gnomadv3_af_ami: 0.9857,
            gnomadv3_af_fin: 0.9589,
            exomiser_ar_exgenespheno: 0.5615,
            gnomadv3_af_sas: 0.9573,
            gnomadv3_af_amr: 0.9217,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'NC_000001.11:g.1041950T>C',
            exomiser_ad_exgenesvar: 0,
            clnid: '128292',
            gnomadv3_af_xx: 0.8694,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.886,
            exomiser_xr_exgenespheno: 0.5615,
            exomiser_xd_exgenespheno: 0.5615,
            impact_severity: 'MED',
            quality: 4241.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 34],
              [0, 32],
              [0, 56],
            ],
            genotype_qualities: [[99, 96, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['AGRN'],
            ensembl_transcript_id: [
              'ENST00000379370',
              'ENST00000620552',
              'ENST00000651234',
              'ENST00000652369',
            ],
            variant_type: 'small',
            highest_af: 0.9998,
            highest_af_info: {
              source: 'gnomad V2 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 977330,
            hg19_strand: '+',
            hg19_end: 977330,
            entrez_gene_id: ['375790'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9cf',
            variant_id: '76',
            end: 1042190,
            ref: 'G',
            alt: 'A',
            start: 1042190,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000379370',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '7/35',
                hgvsc: 'ENST00000379370.7:c.1384+28G>A',
                existing_variation: 'rs2710876',
                strand: 1,
                hgnc_id: 'HGNC:329',
                mane_select: 'NM_198576.4',
                ccds: 'CCDS30551.1',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                flaglrg: 'LRG_198t1',
                cadd_phred: 0.178,
                cadd_raw: -0.415592,
                spliceai_pred_dp_ag: 2,
                spliceai_pred_dp_al: -2,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: -28,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.02,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                constraint_mis_z: 0.22552,
                constraint_oe_lof: 0.31914,
                constraint_oe_lof_lower: 0.237,
                constraint_oe_lof_upper: 0.435,
                constraint_oe_mis: 0.98257,
                constraint_oe_mis_lower: 0.938,
                constraint_oe_mis_upper: 1.029,
                constraint_oe_syn: 1.2049,
                constraint_oe_syn_lower: 1.133,
                constraint_oe_syn_upper: 1.281,
                constraint_pli: 5.4727e-7,
                constraint_syn_z: -3.9572,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                p_li: 5.4727e-7,
                loeuf: 0.435,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000620552',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '7/38',
                hgvsc: 'ENST00000620552.4:c.970+28G>A',
                existing_variation: 'rs2710876',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                cadd_phred: 0.178,
                cadd_raw: -0.415592,
                spliceai_pred_dp_ag: 2,
                spliceai_pred_dp_al: -2,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: -28,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.02,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000651234',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '6/37',
                hgvsc: 'ENST00000651234.1:c.1069+28G>A',
                existing_variation: 'rs2710876',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                cadd_phred: 0.178,
                cadd_raw: -0.415592,
                spliceai_pred_dp_ag: 2,
                spliceai_pred_dp_al: -2,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: -28,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.02,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000652369',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '6/34',
                hgvsc: 'ENST00000652369.1:c.1069+28G>A',
                existing_variation: 'rs2710876',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                cadd_phred: 0.178,
                cadd_raw: -0.415592,
                spliceai_pred_dp_ag: 2,
                spliceai_pred_dp_al: -2,
                spliceai_pred_dp_dg: 2,
                spliceai_pred_dp_dl: -28,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.02,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
            ],
            mane_select: 'ENST00000379370',
            ensembl_gene_id: ['ENSG00000188157'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant',
            hgvsc: 'ENST00000379370.7:c.1384+28G>A',
            existing_variation: 'rs2710876',
            ccds: 'CCDS30551.1',
            source: 'Ensembl',
            gene_pheno: '1',
            gnomad_af: 0.8973,
            gnomad_afr_af: 0.4355,
            gnomad_amr_af: 0.9316,
            gnomad_asj_af: 0.9182,
            gnomad_eas_af: 0.9726,
            gnomad_fin_af: 0.9373,
            gnomad_nfe_af: 0.9149,
            gnomad_oth_af: 0.8986,
            gnomad_sas_af: 0.9451,
            pheno: '1',
            flaglrg: 'LRG_198t1',
            cadd_phred: 0.178,
            cadd_raw: -0.415592,
            spliceai_pred_dp_ag: 2,
            spliceai_pred_dp_al: -2,
            spliceai_pred_dp_dg: 2,
            spliceai_pred_dp_dl: -28,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0.02],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'AGRN',
            constraint_mis_z: [0.22552],
            constraint_oe_lof: 0.31914,
            constraint_oe_lof_lower: 0.237,
            constraint_oe_lof_upper: [0.435],
            constraint_oe_mis: 0.98257,
            constraint_oe_mis_lower: 0.938,
            constraint_oe_mis_upper: [1.029],
            constraint_oe_syn: 1.2049,
            constraint_oe_syn_lower: 1.133,
            constraint_oe_syn_upper: 1.281,
            constraint_pli: 5.4727e-7,
            constraint_syn_z: [-3.9572],
            cgd_agegroup: 'Pediatric',
            cgd_condition: 'Myasthenic_syndrome&_congenital_8',
            cgd_entrezid: '375790',
            cgd_gene: 'AGRN',
            cgd_hgncid: '329',
            cgd_inheritance: 'AR',
            cgd_interventioncategories:
              'Musculoskeletal&_Neurologic&_Pharmacogenomic',
            cgd_interventionrationale:
              'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
            cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
            cgd_references: '19631309&_20301347&_22205389&_22678886',
            exomiser_ad_exgenespheno: 0.5615,
            exomiser_xr_exgenescombi: 0.00058018,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'criteria_provided,_multiple_submitters,_no_conflicts',
            clndn:
              'Myasthenic_syndrome,_congenital,_8|not_specified|not_provided',
            gnomadv3_af_afr: 0.4415,
            gnomadv3_af_nfe: 0.9175,
            gnomadv3_af_xy: 0.7937,
            exomiser_xd_exgenescombi: 0.00058018,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.879,
            clnvcso: 'SO:0001483',
            gnomadv3_af: 0.788,
            exomiser_mt_exgenespheno: 0.5615,
            clndisdb:
              'MONDO:MONDO:0014052,MedGen:C3808739,OMIM:615120|MedGen:CN169374|MedGen:CN517202',
            exomiser_ar_exgenescombi: 0.00058018,
            gnomadv3_af_eas: 0.968,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9199,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00058018,
            exomiser_ad_exgenescombi: 0.00058018,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'Benign',
            clnvc: 'single_nucleotide_variant',
            gnomadv3_af_ami: 0.9857,
            gnomadv3_af_fin: 0.9398,
            exomiser_ar_exgenespheno: 0.5615,
            gnomadv3_af_sas: 0.9466,
            gnomadv3_af_amr: 0.8833,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'NC_000001.11:g.1042190G>A',
            exomiser_ad_exgenesvar: 0,
            clnid: '263159',
            gnomadv3_af_xx: 0.7825,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.8246,
            exomiser_xr_exgenespheno: 0.5615,
            exomiser_xd_exgenespheno: 0.5615,
            impact_severity: 'LOW',
            quality: 3833.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 43],
              [0, 29],
              [0, 47],
            ],
            genotype_qualities: [[99, 85, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['AGRN'],
            ensembl_transcript_id: [
              'ENST00000379370',
              'ENST00000620552',
              'ENST00000651234',
              'ENST00000652369',
            ],
            variant_type: 'small',
            highest_af: 0.9726,
            highest_af_info: {
              source: 'gnomad V2 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 977570,
            hg19_strand: '+',
            hg19_end: 977570,
            entrez_gene_id: ['375790'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9d0',
            variant_id: '77',
            end: 1043225,
            ref: 'CCT',
            alt: 'C',
            start: 1043223,
            chrom: 'chr1',
            type: 'indel',
            gene_objs: [
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000379370',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '7/35',
                hgvsc: 'ENST00000379370.7:c.1385-15_1385-14del',
                existing_variation: 'rs35881187',
                strand: 1,
                hgnc_id: 'HGNC:329',
                mane_select: 'NM_198576.4',
                ccds: 'CCDS30551.1',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                flaglrg: 'LRG_198t1',
                spliceai_pred_dp_ag: 48,
                spliceai_pred_dp_al: 16,
                spliceai_pred_dp_dg: 21,
                spliceai_pred_dp_dl: 2,
                spliceai_pred_ds_ag: 0.11,
                spliceai_pred_ds_al: 0.04,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                constraint_mis_z: 0.22552,
                constraint_oe_lof: 0.31914,
                constraint_oe_lof_lower: 0.237,
                constraint_oe_lof_upper: 0.435,
                constraint_oe_mis: 0.98257,
                constraint_oe_mis_lower: 0.938,
                constraint_oe_mis_upper: 1.029,
                constraint_oe_syn: 1.2049,
                constraint_oe_syn_lower: 1.133,
                constraint_oe_syn_upper: 1.281,
                constraint_pli: 5.4727e-7,
                constraint_syn_z: -3.9572,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                p_li: 5.4727e-7,
                loeuf: 0.435,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000620552',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '7/38',
                hgvsc: 'ENST00000620552.4:c.971-15_971-14del',
                existing_variation: 'rs35881187',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                spliceai_pred_dp_ag: 48,
                spliceai_pred_dp_al: 16,
                spliceai_pred_dp_dg: 21,
                spliceai_pred_dp_dl: 2,
                spliceai_pred_ds_ag: 0.11,
                spliceai_pred_ds_al: 0.04,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000651234',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '6/37',
                hgvsc: 'ENST00000651234.1:c.1070-15_1070-14del',
                existing_variation: 'rs35881187',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                spliceai_pred_dp_ag: 48,
                spliceai_pred_dp_al: 16,
                spliceai_pred_dp_dg: 21,
                spliceai_pred_dp_dl: 2,
                spliceai_pred_ds_ag: 0.11,
                spliceai_pred_ds_al: 0.04,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000652369',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '6/34',
                hgvsc: 'ENST00000652369.1:c.1070-15_1070-14del',
                existing_variation: 'rs35881187',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                spliceai_pred_dp_ag: 48,
                spliceai_pred_dp_al: 16,
                spliceai_pred_dp_dg: 21,
                spliceai_pred_dp_dl: 2,
                spliceai_pred_ds_ag: 0.11,
                spliceai_pred_ds_al: 0.04,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
            ],
            mane_select: 'ENST00000379370',
            ensembl_gene_id: ['ENSG00000188157'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant',
            hgvsc: 'ENST00000379370.7:c.1385-15_1385-14del',
            existing_variation: 'rs35881187',
            ccds: 'CCDS30551.1',
            source: 'Ensembl',
            gene_pheno: '1',
            gnomad_af: 0.5514,
            gnomad_afr_af: 0.09425,
            gnomad_amr_af: 0.6972,
            gnomad_asj_af: 0.5461,
            gnomad_eas_af: 0.7858,
            gnomad_fin_af: 0.5349,
            gnomad_nfe_af: 0.5385,
            gnomad_oth_af: 0.5392,
            gnomad_sas_af: 0.5463,
            pheno: '1',
            flaglrg: 'LRG_198t1',
            spliceai_pred_dp_ag: 48,
            spliceai_pred_dp_al: 16,
            spliceai_pred_dp_dg: 21,
            spliceai_pred_dp_dl: 2,
            spliceai_pred_ds_ag: [0.11],
            spliceai_pred_ds_al: [0.04],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'AGRN',
            constraint_mis_z: [0.22552],
            constraint_oe_lof: 0.31914,
            constraint_oe_lof_lower: 0.237,
            constraint_oe_lof_upper: [0.435],
            constraint_oe_mis: 0.98257,
            constraint_oe_mis_lower: 0.938,
            constraint_oe_mis_upper: [1.029],
            constraint_oe_syn: 1.2049,
            constraint_oe_syn_lower: 1.133,
            constraint_oe_syn_upper: 1.281,
            constraint_pli: 5.4727e-7,
            constraint_syn_z: [-3.9572],
            cgd_agegroup: 'Pediatric',
            cgd_condition: 'Myasthenic_syndrome&_congenital_8',
            cgd_entrezid: '375790',
            cgd_gene: 'AGRN',
            cgd_hgncid: '329',
            cgd_inheritance: 'AR',
            cgd_interventioncategories:
              'Musculoskeletal&_Neurologic&_Pharmacogenomic',
            cgd_interventionrationale:
              'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
            cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
            cgd_references: '19631309&_20301347&_22205389&_22678886',
            exomiser_ad_exgenespheno: 0.5615,
            exomiser_xr_exgenescombi: 0.00058018,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'criteria_provided,_multiple_submitters,_no_conflicts',
            clndn: 'Myasthenic_syndrome,_congenital,_8|not_specified',
            gnomadv3_af_afr: 0.1099,
            gnomadv3_af_nfe: 0.5485,
            gnomadv3_af_xy: 0.4499,
            exomiser_xd_exgenescombi: 0.00058018,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.5,
            clnvcso: 'SO:0000159',
            gnomadv3_af: 0.4432,
            exomiser_mt_exgenespheno: 0.5615,
            clndisdb:
              'MONDO:MONDO:0014052,MedGen:C3808739,OMIM:615120|MedGen:CN169374',
            exomiser_ar_exgenescombi: 0.00058018,
            gnomadv3_af_eas: 0.7857,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.5574,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00058018,
            exomiser_ad_exgenescombi: 0.00058018,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'Benign',
            clnvc: 'Deletion',
            gnomadv3_af_ami: 0.5954,
            gnomadv3_af_fin: 0.5365,
            exomiser_ar_exgenespheno: 0.5615,
            gnomadv3_af_sas: 0.5552,
            gnomadv3_af_amr: 0.6221,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'NC_000001.11:g.1043224_1043225del',
            exomiser_ad_exgenesvar: 0,
            clnid: '263160',
            gnomadv3_af_xx: 0.4367,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.4774,
            exomiser_xr_exgenespheno: 0.5615,
            exomiser_xd_exgenespheno: 0.5615,
            impact_severity: 'LOW',
            quality: 4459.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [1, 1],
            ],
            allelic_depths: [
              [23, 9],
              [23, 26],
              [0, 72],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['AGRN'],
            ensembl_transcript_id: [
              'ENST00000379370',
              'ENST00000620552',
              'ENST00000651234',
              'ENST00000652369',
            ],
            variant_type: 'small',
            highest_af: 0.7858,
            highest_af_info: {
              source: 'gnomad V2 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 978603,
            hg19_strand: '+',
            hg19_end: 978605,
            entrez_gene_id: ['375790'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9d1',
            variant_id: '78',
            end: 1045080,
            ref: 'G',
            alt: 'A',
            start: 1045080,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000379370',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '12/35',
                hgvsc: 'ENST00000379370.7:c.2255-81G>A',
                existing_variation: 'rs3128097',
                strand: 1,
                hgnc_id: 'HGNC:329',
                mane_select: 'NM_198576.4',
                ccds: 'CCDS30551.1',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                flaglrg: 'LRG_198t1',
                cadd_phred: 8.34,
                cadd_raw: 0.687543,
                spliceai_pred_dp_ag: 30,
                spliceai_pred_dp_al: -33,
                spliceai_pred_dp_dg: 30,
                spliceai_pred_dp_dl: 0,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.03,
                spliceai_pred_ds_dl: 0.43,
                spliceai_pred_symbol: 'AGRN',
                constraint_mis_z: 0.22552,
                constraint_oe_lof: 0.31914,
                constraint_oe_lof_lower: 0.237,
                constraint_oe_lof_upper: 0.435,
                constraint_oe_mis: 0.98257,
                constraint_oe_mis_lower: 0.938,
                constraint_oe_mis_upper: 1.029,
                constraint_oe_syn: 1.2049,
                constraint_oe_syn_lower: 1.133,
                constraint_oe_syn_upper: 1.281,
                constraint_pli: 5.4727e-7,
                constraint_syn_z: -3.9572,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                p_li: 5.4727e-7,
                loeuf: 0.435,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000620552',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '12/38',
                hgvsc: 'ENST00000620552.4:c.1841-81G>A',
                existing_variation: 'rs3128097',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                cadd_phred: 8.34,
                cadd_raw: 0.687543,
                spliceai_pred_dp_ag: 30,
                spliceai_pred_dp_al: -33,
                spliceai_pred_dp_dg: 30,
                spliceai_pred_dp_dl: 0,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.03,
                spliceai_pred_ds_dl: 0.43,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000651234',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '11/37',
                hgvsc: 'ENST00000651234.1:c.1940-81G>A',
                existing_variation: 'rs3128097',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                cadd_phred: 8.34,
                cadd_raw: 0.687543,
                spliceai_pred_dp_ag: 30,
                spliceai_pred_dp_al: -33,
                spliceai_pred_dp_dg: 30,
                spliceai_pred_dp_dl: 0,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.03,
                spliceai_pred_ds_dl: 0.43,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000652369',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'A',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '11/34',
                hgvsc: 'ENST00000652369.1:c.1940-81G>A',
                existing_variation: 'rs3128097',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                cadd_phred: 8.34,
                cadd_raw: 0.687543,
                spliceai_pred_dp_ag: 30,
                spliceai_pred_dp_al: -33,
                spliceai_pred_dp_dg: 30,
                spliceai_pred_dp_dl: 0,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0.03,
                spliceai_pred_ds_dl: 0.43,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
            ],
            mane_select: 'ENST00000379370',
            ensembl_gene_id: ['ENSG00000188157'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant',
            hgvsc: 'ENST00000379370.7:c.2255-81G>A',
            existing_variation: 'rs3128097',
            ccds: 'CCDS30551.1',
            source: 'Ensembl',
            gene_pheno: '1',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            pheno: '1',
            flaglrg: 'LRG_198t1',
            cadd_phred: 8.34,
            cadd_raw: 0.687543,
            spliceai_pred_dp_ag: 30,
            spliceai_pred_dp_al: -33,
            spliceai_pred_dp_dg: 30,
            spliceai_pred_dp_dl: 0,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0.03],
            spliceai_pred_ds_dl: [0.43],
            spliceai_pred_symbol: 'AGRN',
            constraint_mis_z: [0.22552],
            constraint_oe_lof: 0.31914,
            constraint_oe_lof_lower: 0.237,
            constraint_oe_lof_upper: [0.435],
            constraint_oe_mis: 0.98257,
            constraint_oe_mis_lower: 0.938,
            constraint_oe_mis_upper: [1.029],
            constraint_oe_syn: 1.2049,
            constraint_oe_syn_lower: 1.133,
            constraint_oe_syn_upper: 1.281,
            constraint_pli: 5.4727e-7,
            constraint_syn_z: [-3.9572],
            cgd_agegroup: 'Pediatric',
            cgd_condition: 'Myasthenic_syndrome&_congenital_8',
            cgd_entrezid: '375790',
            cgd_gene: 'AGRN',
            cgd_hgncid: '329',
            cgd_inheritance: 'AR',
            cgd_interventioncategories:
              'Musculoskeletal&_Neurologic&_Pharmacogenomic',
            cgd_interventionrationale:
              'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
            cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
            cgd_references: '19631309&_20301347&_22205389&_22678886',
            exomiser_ad_exgenespheno: 0.5615,
            exomiser_xr_exgenescombi: 0.00058018,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'criteria_provided,_multiple_submitters,_no_conflicts',
            clndn: 'Myasthenic_syndrome,_congenital,_8|not_provided',
            gnomadv3_af_afr: 0.7947,
            gnomadv3_af_nfe: 0.9293,
            gnomadv3_af_xy: 0.9014,
            exomiser_xd_exgenescombi: 0.00058018,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.9051,
            clnvcso: 'SO:0001483',
            gnomadv3_af: 0.8986,
            exomiser_mt_exgenespheno: 0.5615,
            clndisdb:
              'MONDO:MONDO:0014052,MedGen:C3808739,OMIM:615120|MedGen:CN517202',
            exomiser_ar_exgenescombi: 0.00058018,
            gnomadv3_af_eas: 0.9998,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9243,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00058018,
            exomiser_ad_exgenescombi: 0.00058018,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'Benign',
            clnvc: 'single_nucleotide_variant',
            gnomadv3_af_ami: 0.9923,
            gnomadv3_af_fin: 0.9598,
            exomiser_ar_exgenespheno: 0.5615,
            gnomadv3_af_sas: 0.9625,
            gnomadv3_af_amr: 0.933,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'NC_000001.11:g.1045080G>A',
            exomiser_ad_exgenesvar: 0,
            clnid: '677948',
            gnomadv3_af_xx: 0.8959,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.9094,
            exomiser_xr_exgenespheno: 0.5615,
            exomiser_xd_exgenespheno: 0.5615,
            impact_severity: 'LOW',
            quality: 5084.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 53],
              [0, 44],
              [0, 55],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['AGRN'],
            ensembl_transcript_id: [
              'ENST00000379370',
              'ENST00000620552',
              'ENST00000651234',
              'ENST00000652369',
            ],
            variant_type: 'small',
            highest_af: 0.9998,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 980460,
            hg19_strand: '+',
            hg19_end: 980460,
            entrez_gene_id: ['375790'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9d2',
            variant_id: '79',
            end: 1045707,
            ref: 'A',
            alt: 'G',
            start: 1045707,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000379370',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '14/35',
                hgvsc: 'ENST00000379370.7:c.2537-26A>G',
                existing_variation: 'rs3128098',
                strand: 1,
                hgnc_id: 'HGNC:329',
                mane_select: 'NM_198576.4',
                ccds: 'CCDS30551.1',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                flaglrg: 'LRG_198t1',
                cadd_phred: 1.442,
                cadd_raw: 0.016525,
                spliceai_pred_dp_ag: 33,
                spliceai_pred_dp_al: 26,
                spliceai_pred_dp_dg: -12,
                spliceai_pred_dp_dl: 33,
                spliceai_pred_ds_ag: 0.11,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                constraint_mis_z: 0.22552,
                constraint_oe_lof: 0.31914,
                constraint_oe_lof_lower: 0.237,
                constraint_oe_lof_upper: 0.435,
                constraint_oe_mis: 0.98257,
                constraint_oe_mis_lower: 0.938,
                constraint_oe_mis_upper: 1.029,
                constraint_oe_syn: 1.2049,
                constraint_oe_syn_lower: 1.133,
                constraint_oe_syn_upper: 1.281,
                constraint_pli: 5.4727e-7,
                constraint_syn_z: -3.9572,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                p_li: 5.4727e-7,
                loeuf: 0.435,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000620552',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '14/38',
                hgvsc: 'ENST00000620552.4:c.2123-26A>G',
                existing_variation: 'rs3128098',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                cadd_phred: 1.442,
                cadd_raw: 0.016525,
                spliceai_pred_dp_ag: 33,
                spliceai_pred_dp_al: 26,
                spliceai_pred_dp_dg: -12,
                spliceai_pred_dp_dl: 33,
                spliceai_pred_ds_ag: 0.11,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000651234',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '13/37',
                hgvsc: 'ENST00000651234.1:c.2222-26A>G',
                existing_variation: 'rs3128098',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                cadd_phred: 1.442,
                cadd_raw: 0.016525,
                spliceai_pred_dp_ag: 33,
                spliceai_pred_dp_al: 26,
                spliceai_pred_dp_dg: -12,
                spliceai_pred_dp_dl: 33,
                spliceai_pred_ds_ag: 0.11,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000652369',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'G',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '13/34',
                hgvsc: 'ENST00000652369.1:c.2222-26A>G',
                existing_variation: 'rs3128098',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                cadd_phred: 1.442,
                cadd_raw: 0.016525,
                spliceai_pred_dp_ag: 33,
                spliceai_pred_dp_al: 26,
                spliceai_pred_dp_dg: -12,
                spliceai_pred_dp_dl: 33,
                spliceai_pred_ds_ag: 0.11,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
            ],
            mane_select: 'ENST00000379370',
            ensembl_gene_id: ['ENSG00000188157'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant',
            hgvsc: 'ENST00000379370.7:c.2537-26A>G',
            existing_variation: 'rs3128098',
            ccds: 'CCDS30551.1',
            source: 'Ensembl',
            gene_pheno: '1',
            gnomad_af: 0.9206,
            gnomad_afr_af: 0.6365,
            gnomad_amr_af: 0.9462,
            gnomad_asj_af: 0.9185,
            gnomad_eas_af: 0.9999,
            gnomad_fin_af: 0.9559,
            gnomad_nfe_af: 0.9247,
            gnomad_oth_af: 0.9161,
            gnomad_sas_af: 0.9552,
            pheno: '1',
            flaglrg: 'LRG_198t1',
            cadd_phred: 1.442,
            cadd_raw: 0.016525,
            spliceai_pred_dp_ag: 33,
            spliceai_pred_dp_al: 26,
            spliceai_pred_dp_dg: -12,
            spliceai_pred_dp_dl: 33,
            spliceai_pred_ds_ag: [0.11],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'AGRN',
            constraint_mis_z: [0.22552],
            constraint_oe_lof: 0.31914,
            constraint_oe_lof_lower: 0.237,
            constraint_oe_lof_upper: [0.435],
            constraint_oe_mis: 0.98257,
            constraint_oe_mis_lower: 0.938,
            constraint_oe_mis_upper: [1.029],
            constraint_oe_syn: 1.2049,
            constraint_oe_syn_lower: 1.133,
            constraint_oe_syn_upper: 1.281,
            constraint_pli: 5.4727e-7,
            constraint_syn_z: [-3.9572],
            cgd_agegroup: 'Pediatric',
            cgd_condition: 'Myasthenic_syndrome&_congenital_8',
            cgd_entrezid: '375790',
            cgd_gene: 'AGRN',
            cgd_hgncid: '329',
            cgd_inheritance: 'AR',
            cgd_interventioncategories:
              'Musculoskeletal&_Neurologic&_Pharmacogenomic',
            cgd_interventionrationale:
              'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
            cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
            cgd_references: '19631309&_20301347&_22205389&_22678886',
            exomiser_ad_exgenespheno: 0.5615,
            exomiser_xr_exgenescombi: 0.00058018,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'criteria_provided,_multiple_submitters,_no_conflicts',
            clndn:
              'Myasthenic_syndrome,_congenital,_8|not_specified|not_provided',
            gnomadv3_af_afr: 0.644,
            gnomadv3_af_nfe: 0.9278,
            gnomadv3_af_xy: 0.8583,
            exomiser_xd_exgenescombi: 0.00058018,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.8956,
            clnvcso: 'SO:0001483',
            gnomadv3_af: 0.8538,
            exomiser_mt_exgenespheno: 0.5615,
            clndisdb:
              'MONDO:MONDO:0014052,MedGen:C3808739,OMIM:615120|MedGen:CN169374|MedGen:CN517202',
            exomiser_ar_exgenescombi: 0.00058018,
            gnomadv3_af_eas: 0.9998,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9165,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00058018,
            exomiser_ad_exgenescombi: 0.00058018,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'Benign',
            clnvc: 'single_nucleotide_variant',
            gnomadv3_af_ami: 0.9923,
            gnomadv3_af_fin: 0.9596,
            exomiser_ar_exgenespheno: 0.5615,
            gnomadv3_af_sas: 0.9617,
            gnomadv3_af_amr: 0.9103,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'NC_000001.11:g.1045707A>G',
            exomiser_ad_exgenesvar: 0,
            clnid: '263172',
            gnomadv3_af_xx: 0.8495,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.8684,
            exomiser_xr_exgenespheno: 0.5615,
            exomiser_xd_exgenespheno: 0.5615,
            impact_severity: 'LOW',
            quality: 4182.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 58],
              [0, 39],
              [1, 40],
            ],
            genotype_qualities: [[99, 99, 87]],
            pass_filter: 'PASS',
            gene_symbol: ['AGRN'],
            ensembl_transcript_id: [
              'ENST00000379370',
              'ENST00000620552',
              'ENST00000651234',
              'ENST00000652369',
            ],
            variant_type: 'small',
            highest_af: 0.9999,
            highest_af_info: {
              source: 'gnomad V2 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 981087,
            hg19_strand: '+',
            hg19_end: 981087,
            entrez_gene_id: ['375790'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9d3',
            variant_id: '80',
            end: 1046551,
            ref: 'A',
            alt: 'G',
            start: 1046551,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000379370',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '18/36',
                codon_change: 'tcA/tcG',
                aa_change: 'S',
                aa_length: 1022,
                allele: 'G',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379370.7:c.3066A>G',
                hgvsp: 'ENSP00000368678.2:p.Ser1022%3D',
                cds_position: 3066,
                existing_variation: 'rs2465128&COSV65068047',
                strand: 1,
                hgnc_id: 'HGNC:329',
                mane_select: 'NM_198576.4',
                ccds: 'CCDS30551.1',
                source: 'Ensembl',
                gene_pheno: 1,
                domains:
                  'PANTHER:PTHR10913&PANTHER:PTHR10913:SF74&MobiDB_lite:mobidb-lite&MobiDB_lite:mobidb-lite&Low_complexity_(Seg):seg',
                clin_sig: 'benign',
                pheno: '1&1',
                flaglrg: 'LRG_198t1',
                cadd_phred: 0.114,
                cadd_raw: -0.508771,
                spliceai_pred_dp_ag: -36,
                spliceai_pred_dp_al: 21,
                spliceai_pred_dp_dg: 38,
                spliceai_pred_dp_dl: -38,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                constraint_mis_z: 0.22552,
                constraint_oe_lof: 0.31914,
                constraint_oe_lof_lower: 0.237,
                constraint_oe_lof_upper: 0.435,
                constraint_oe_mis: 0.98257,
                constraint_oe_mis_lower: 0.938,
                constraint_oe_mis_upper: 1.029,
                constraint_oe_syn: 1.2049,
                constraint_oe_syn_lower: 1.133,
                constraint_oe_syn_upper: 1.281,
                constraint_pli: 5.4727e-7,
                constraint_syn_z: -3.9572,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                p_li: 5.4727e-7,
                loeuf: 0.435,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000620552',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '18/39',
                codon_change: 'tcA/tcG',
                aa_change: 'S',
                aa_length: 884,
                allele: 'G',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000620552.4:c.2652A>G',
                hgvsp: 'ENSP00000484607.1:p.Ser884%3D',
                cds_position: 2652,
                existing_variation: 'rs2465128&COSV65068047',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                domains:
                  'PANTHER:PTHR10913&PANTHER:PTHR10913:SF74&MobiDB_lite:mobidb-lite&MobiDB_lite:mobidb-lite&Low_complexity_(Seg):seg',
                clin_sig: 'benign',
                pheno: '1&1',
                cadd_phred: 0.114,
                cadd_raw: -0.508771,
                spliceai_pred_dp_ag: -36,
                spliceai_pred_dp_al: 21,
                spliceai_pred_dp_dg: 38,
                spliceai_pred_dp_dl: -38,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000651234',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '17/38',
                codon_change: 'tcA/tcG',
                aa_change: 'S',
                aa_length: 917,
                allele: 'G',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000651234.1:c.2751A>G',
                hgvsp: 'ENSP00000499046.1:p.Ser917%3D',
                cds_position: 2751,
                existing_variation: 'rs2465128&COSV65068047',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                domains:
                  'PANTHER:PTHR10913&PANTHER:PTHR10913:SF74&MobiDB_lite:mobidb-lite&MobiDB_lite:mobidb-lite&Low_complexity_(Seg):seg',
                clin_sig: 'benign',
                pheno: '1&1',
                cadd_phred: 0.114,
                cadd_raw: -0.508771,
                spliceai_pred_dp_ag: -36,
                spliceai_pred_dp_al: 21,
                spliceai_pred_dp_dg: 38,
                spliceai_pred_dp_dl: -38,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000652369',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '17/35',
                codon_change: 'tcA/tcG',
                aa_change: 'S',
                aa_length: 917,
                allele: 'G',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000652369.1:c.2751A>G',
                hgvsp: 'ENSP00000498543.1:p.Ser917%3D',
                cds_position: 2751,
                existing_variation: 'rs2465128&COSV65068047',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                domains:
                  'Low_complexity_(Seg):seg&MobiDB_lite:mobidb-lite&MobiDB_lite:mobidb-lite&PANTHER:PTHR10913&PANTHER:PTHR10913:SF74',
                clin_sig: 'benign',
                pheno: '1&1',
                cadd_phred: 0.114,
                cadd_raw: -0.508771,
                spliceai_pred_dp_ag: -36,
                spliceai_pred_dp_al: 21,
                spliceai_pred_dp_dg: 38,
                spliceai_pred_dp_dl: -38,
                spliceai_pred_ds_ag: 0,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
            ],
            mane_select: 'ENST00000379370',
            ensembl_gene_id: ['ENSG00000188157'],
            is_exonic: 1,
            is_coding: 1,
            aa_change: 'S',
            impact: 'synonymous_variant',
            hgvsc: 'ENST00000379370.7:c.3066A>G',
            hgvsp: 'ENSP00000368678.2:p.Ser1022%3D',
            existing_variation: 'rs2465128&COSV65068047',
            ccds: 'CCDS30551.1',
            source: 'Ensembl',
            gene_pheno: '1',
            domains:
              'PANTHER:PTHR10913&PANTHER:PTHR10913:SF74&MobiDB_lite:mobidb-lite&MobiDB_lite:mobidb-lite&Low_complexity_(Seg):seg',
            gnomad_af: 0.8865,
            gnomad_afr_af: 0.4974,
            gnomad_amr_af: 0.9296,
            gnomad_asj_af: 0.8885,
            gnomad_eas_af: 0.9677,
            gnomad_fin_af: 0.9316,
            gnomad_nfe_af: 0.9059,
            gnomad_oth_af: 0.8841,
            gnomad_sas_af: 0.8885,
            pheno: '1&1',
            flaglrg: 'LRG_198t1',
            cadd_phred: 0.114,
            cadd_raw: -0.508771,
            spliceai_pred_dp_ag: -36,
            spliceai_pred_dp_al: 21,
            spliceai_pred_dp_dg: 38,
            spliceai_pred_dp_dl: -38,
            spliceai_pred_ds_ag: [0],
            spliceai_pred_ds_al: [0.01],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'AGRN',
            constraint_mis_z: [0.22552],
            constraint_oe_lof: 0.31914,
            constraint_oe_lof_lower: 0.237,
            constraint_oe_lof_upper: [0.435],
            constraint_oe_mis: 0.98257,
            constraint_oe_mis_lower: 0.938,
            constraint_oe_mis_upper: [1.029],
            constraint_oe_syn: 1.2049,
            constraint_oe_syn_lower: 1.133,
            constraint_oe_syn_upper: 1.281,
            constraint_pli: 5.4727e-7,
            constraint_syn_z: [-3.9572],
            cgd_agegroup: 'Pediatric',
            cgd_condition: 'Myasthenic_syndrome&_congenital_8',
            cgd_entrezid: '375790',
            cgd_gene: 'AGRN',
            cgd_hgncid: '329',
            cgd_inheritance: 'AR',
            cgd_interventioncategories:
              'Musculoskeletal&_Neurologic&_Pharmacogenomic',
            cgd_interventionrationale:
              'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
            cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
            cgd_references: '19631309&_20301347&_22205389&_22678886',
            exomiser_ad_exgenespheno: 0.5615,
            exomiser_xr_exgenescombi: 0.00058018,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'criteria_provided,_multiple_submitters,_no_conflicts',
            clndn: 'Myasthenic_syndrome,_congenital,_8|not_specified',
            gnomadv3_af_afr: 0.5141,
            gnomadv3_af_nfe: 0.9099,
            gnomadv3_af_xy: 0.8056,
            exomiser_xd_exgenescombi: 0.00058018,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.8513,
            clnvcso: 'SO:0001483',
            gnomadv3_af: 0.8016,
            exomiser_mt_exgenespheno: 0.5615,
            clndisdb:
              'MONDO:MONDO:0014052,MedGen:C3808739,OMIM:615120|MedGen:CN169374',
            exomiser_ar_exgenescombi: 0.00058018,
            gnomadv3_af_eas: 0.9618,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.8928,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00058018,
            exomiser_ad_exgenescombi: 0.00058018,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'Benign',
            clnvc: 'single_nucleotide_variant',
            gnomadv3_af_ami: 0.9857,
            gnomadv3_af_fin: 0.9361,
            exomiser_ar_exgenespheno: 0.5615,
            gnomadv3_af_sas: 0.8918,
            gnomadv3_af_amr: 0.8811,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'NC_000001.11:g.1046551A>G',
            exomiser_ad_exgenesvar: 0,
            clnid: '128297',
            gnomadv3_af_xx: 0.7977,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.8241,
            exomiser_xr_exgenespheno: 0.5615,
            exomiser_xd_exgenespheno: 0.5615,
            impact_severity: 'LOW',
            quality: 3189.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 38],
              [0, 28],
              [0, 37],
            ],
            genotype_qualities: [[99, 84, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['AGRN'],
            ensembl_transcript_id: [
              'ENST00000379370',
              'ENST00000620552',
              'ENST00000651234',
              'ENST00000652369',
            ],
            variant_type: 'small',
            highest_af: 0.9677,
            highest_af_info: {
              source: 'gnomad V2 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 981931,
            hg19_strand: '+',
            hg19_end: 981931,
            entrez_gene_id: ['375790'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9d4',
            variant_id: '81',
            end: 1047561,
            ref: 'T',
            alt: 'C',
            start: 1047561,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000379370',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: 'C',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '20/35',
                hgvsc: 'ENST00000379370.7:c.3517-12T>C',
                existing_variation: 'rs3128102',
                strand: 1,
                hgnc_id: 'HGNC:329',
                mane_select: 'NM_198576.4',
                ccds: 'CCDS30551.1',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                flaglrg: 'LRG_198t1',
                cadd_phred: 2.444,
                cadd_raw: 0.137094,
                spliceai_pred_dp_ag: 37,
                spliceai_pred_dp_al: 12,
                spliceai_pred_dp_dg: 37,
                spliceai_pred_dp_dl: 13,
                spliceai_pred_ds_ag: 0.04,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                constraint_mis_z: 0.22552,
                constraint_oe_lof: 0.31914,
                constraint_oe_lof_lower: 0.237,
                constraint_oe_lof_upper: 0.435,
                constraint_oe_mis: 0.98257,
                constraint_oe_mis_lower: 0.938,
                constraint_oe_mis_upper: 1.029,
                constraint_oe_syn: 1.2049,
                constraint_oe_syn_lower: 1.133,
                constraint_oe_syn_upper: 1.281,
                constraint_pli: 5.4727e-7,
                constraint_syn_z: -3.9572,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                p_li: 5.4727e-7,
                loeuf: 0.435,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000620552',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '20/38',
                hgvsc: 'ENST00000620552.4:c.3103-12T>C',
                existing_variation: 'rs3128102',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                cadd_phred: 2.444,
                cadd_raw: 0.137094,
                spliceai_pred_dp_ag: 37,
                spliceai_pred_dp_al: 12,
                spliceai_pred_dp_dg: 37,
                spliceai_pred_dp_dl: 13,
                spliceai_pred_ds_ag: 0.04,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000651234',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '19/37',
                hgvsc: 'ENST00000651234.1:c.3202-12T>C',
                existing_variation: 'rs3128102',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                cadd_phred: 2.444,
                cadd_raw: 0.137094,
                spliceai_pred_dp_ag: 37,
                spliceai_pred_dp_al: 12,
                spliceai_pred_dp_dg: 37,
                spliceai_pred_dp_dl: 13,
                spliceai_pred_ds_ag: 0.04,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000652369',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: 'C',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '19/34',
                hgvsc: 'ENST00000652369.1:c.3202-12T>C',
                existing_variation: 'rs3128102',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                cadd_phred: 2.444,
                cadd_raw: 0.137094,
                spliceai_pred_dp_ag: 37,
                spliceai_pred_dp_al: 12,
                spliceai_pred_dp_dg: 37,
                spliceai_pred_dp_dl: 13,
                spliceai_pred_ds_ag: 0.04,
                spliceai_pred_ds_al: 0.01,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
            ],
            mane_select: 'ENST00000379370',
            ensembl_gene_id: ['ENSG00000188157'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant',
            hgvsc: 'ENST00000379370.7:c.3517-12T>C',
            existing_variation: 'rs3128102',
            ccds: 'CCDS30551.1',
            source: 'Ensembl',
            gene_pheno: '1',
            gnomad_af: 0.8963,
            gnomad_afr_af: 0.4469,
            gnomad_amr_af: 0.9319,
            gnomad_asj_af: 0.9166,
            gnomad_eas_af: 0.9726,
            gnomad_fin_af: 0.9332,
            gnomad_nfe_af: 0.9133,
            gnomad_oth_af: 0.8989,
            gnomad_sas_af: 0.9443,
            pheno: '1',
            flaglrg: 'LRG_198t1',
            cadd_phred: 2.444,
            cadd_raw: 0.137094,
            spliceai_pred_dp_ag: 37,
            spliceai_pred_dp_al: 12,
            spliceai_pred_dp_dg: 37,
            spliceai_pred_dp_dl: 13,
            spliceai_pred_ds_ag: [0.04],
            spliceai_pred_ds_al: [0.01],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'AGRN',
            constraint_mis_z: [0.22552],
            constraint_oe_lof: 0.31914,
            constraint_oe_lof_lower: 0.237,
            constraint_oe_lof_upper: [0.435],
            constraint_oe_mis: 0.98257,
            constraint_oe_mis_lower: 0.938,
            constraint_oe_mis_upper: [1.029],
            constraint_oe_syn: 1.2049,
            constraint_oe_syn_lower: 1.133,
            constraint_oe_syn_upper: 1.281,
            constraint_pli: 5.4727e-7,
            constraint_syn_z: [-3.9572],
            cgd_agegroup: 'Pediatric',
            cgd_condition: 'Myasthenic_syndrome&_congenital_8',
            cgd_entrezid: '375790',
            cgd_gene: 'AGRN',
            cgd_hgncid: '329',
            cgd_inheritance: 'AR',
            cgd_interventioncategories:
              'Musculoskeletal&_Neurologic&_Pharmacogenomic',
            cgd_interventionrationale:
              'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
            cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
            cgd_references: '19631309&_20301347&_22205389&_22678886',
            exomiser_ad_exgenespheno: 0.5615,
            exomiser_xr_exgenescombi: 0.00058018,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'criteria_provided,_multiple_submitters,_no_conflicts',
            clndn: 'Myasthenic_syndrome,_congenital,_8|not_specified',
            gnomadv3_af_afr: 0.464,
            gnomadv3_af_nfe: 0.9171,
            gnomadv3_af_xy: 0.7993,
            exomiser_xd_exgenescombi: 0.00058018,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.8797,
            clnvcso: 'SO:0001483',
            gnomadv3_af: 0.7939,
            exomiser_mt_exgenespheno: 0.5615,
            clndisdb:
              'MONDO:MONDO:0014052,MedGen:C3808739,OMIM:615120|MedGen:CN169374',
            exomiser_ar_exgenescombi: 0.00058018,
            gnomadv3_af_eas: 0.9678,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9199,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00058018,
            exomiser_ad_exgenescombi: 0.00058018,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'Benign',
            clnvc: 'single_nucleotide_variant',
            gnomadv3_af_ami: 0.9857,
            gnomadv3_af_fin: 0.9387,
            exomiser_ar_exgenespheno: 0.5615,
            gnomadv3_af_sas: 0.947,
            gnomadv3_af_amr: 0.8833,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'NC_000001.11:g.1047561T>C',
            exomiser_ad_exgenesvar: 0,
            clnid: '263181',
            gnomadv3_af_xx: 0.7887,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.8297,
            exomiser_xr_exgenespheno: 0.5615,
            exomiser_xd_exgenespheno: 0.5615,
            impact_severity: 'LOW',
            quality: 4803.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 48],
              [0, 41],
              [0, 49],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['AGRN'],
            ensembl_transcript_id: [
              'ENST00000379370',
              'ENST00000620552',
              'ENST00000651234',
              'ENST00000652369',
            ],
            variant_type: 'small',
            highest_af: 0.9726,
            highest_af_info: {
              source: 'gnomad V2 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 982941,
            hg19_strand: '+',
            hg19_end: 982941,
            entrez_gene_id: ['375790'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9d5',
            variant_id: '82',
            end: 1047614,
            ref: 'T',
            alt: 'C',
            start: 1047614,
            chrom: 'chr1',
            type: 'snp',
            gene_objs: [
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000379370',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                exon: '21/36',
                codon_change: 'ttT/ttC',
                aa_change: 'F',
                aa_length: 1186,
                allele: 'C',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000379370.7:c.3558T>C',
                hgvsp: 'ENSP00000368678.2:p.Phe1186%3D',
                cds_position: 3558,
                existing_variation: 'rs10267&COSV65071416',
                strand: 1,
                hgnc_id: 'HGNC:329',
                mane_select: 'NM_198576.4',
                ccds: 'CCDS30551.1',
                source: 'Ensembl',
                gene_pheno: 1,
                domains:
                  'Gene3D:3.30.70.960&Pfam:PF01390&PROSITE_profiles:PS50024&PANTHER:PTHR10913&PANTHER:PTHR10913:SF74&SMART:SM00200&Superfamily:SSF82671',
                clin_sig: 'benign',
                pheno: '1&1',
                flaglrg: 'LRG_198t1',
                cadd_phred: 7.235,
                cadd_raw: 0.571657,
                spliceai_pred_dp_ag: -16,
                spliceai_pred_dp_al: -8,
                spliceai_pred_dp_dg: 39,
                spliceai_pred_dp_dl: 6,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                constraint_mis_z: 0.22552,
                constraint_oe_lof: 0.31914,
                constraint_oe_lof_lower: 0.237,
                constraint_oe_lof_upper: 0.435,
                constraint_oe_mis: 0.98257,
                constraint_oe_mis_lower: 0.938,
                constraint_oe_mis_upper: 1.029,
                constraint_oe_syn: 1.2049,
                constraint_oe_syn_lower: 1.133,
                constraint_oe_syn_upper: 1.281,
                constraint_pli: 5.4727e-7,
                constraint_syn_z: -3.9572,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                p_li: 5.4727e-7,
                loeuf: 0.435,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000620552',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '21/39',
                codon_change: 'ttT/ttC',
                aa_change: 'F',
                aa_length: 1048,
                allele: 'C',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000620552.4:c.3144T>C',
                hgvsp: 'ENSP00000484607.1:p.Phe1048%3D',
                cds_position: 3144,
                existing_variation: 'rs10267&COSV65071416',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                domains:
                  'Gene3D:3.30.70.960&Pfam:PF01390&PROSITE_profiles:PS50024&PANTHER:PTHR10913&PANTHER:PTHR10913:SF74&SMART:SM00200&Superfamily:SSF82671',
                clin_sig: 'benign',
                pheno: '1&1',
                cadd_phred: 7.235,
                cadd_raw: 0.571657,
                spliceai_pred_dp_ag: -16,
                spliceai_pred_dp_al: -8,
                spliceai_pred_dp_dg: 39,
                spliceai_pred_dp_dl: 6,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000651234',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '20/38',
                codon_change: 'ttT/ttC',
                aa_change: 'F',
                aa_length: 1081,
                allele: 'C',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000651234.1:c.3243T>C',
                hgvsp: 'ENSP00000499046.1:p.Phe1081%3D',
                cds_position: 3243,
                existing_variation: 'rs10267&COSV65071416',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                domains:
                  'Gene3D:3.30.70.960&Pfam:PF01390&PROSITE_profiles:PS50024&PANTHER:PTHR10913&PANTHER:PTHR10913:SF74&SMART:SM00200&Superfamily:SSF82671',
                clin_sig: 'benign',
                pheno: '1&1',
                cadd_phred: 7.235,
                cadd_raw: 0.571657,
                spliceai_pred_dp_ag: -16,
                spliceai_pred_dp_al: -8,
                spliceai_pred_dp_dg: 39,
                spliceai_pred_dp_dl: 6,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000652369',
                is_exonic: true,
                is_coding: true,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                exon: '20/35',
                codon_change: 'ttT/ttC',
                aa_change: 'F',
                aa_length: 1081,
                allele: 'C',
                impact: ['synonymous_variant'],
                feature_type: 'Transcript',
                hgvsc: 'ENST00000652369.1:c.3243T>C',
                hgvsp: 'ENSP00000498543.1:p.Phe1081%3D',
                cds_position: 3243,
                existing_variation: 'rs10267&COSV65071416',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                domains:
                  'Pfam:PF01390&Gene3D:3.30.70.960&SMART:SM00200&Superfamily:SSF82671&PROSITE_profiles:PS50024&PANTHER:PTHR10913&PANTHER:PTHR10913:SF74',
                clin_sig: 'benign',
                pheno: '1&1',
                cadd_phred: 7.235,
                cadd_raw: 0.571657,
                spliceai_pred_dp_ag: -16,
                spliceai_pred_dp_al: -8,
                spliceai_pred_dp_dg: 39,
                spliceai_pred_dp_dl: 6,
                spliceai_pred_ds_ag: 0.01,
                spliceai_pred_ds_al: 0,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'synonymous_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
            ],
            mane_select: 'ENST00000379370',
            ensembl_gene_id: ['ENSG00000188157'],
            is_exonic: 1,
            is_coding: 1,
            aa_change: 'F',
            impact: 'synonymous_variant',
            hgvsc: 'ENST00000379370.7:c.3558T>C',
            hgvsp: 'ENSP00000368678.2:p.Phe1186%3D',
            existing_variation: 'rs10267&COSV65071416',
            ccds: 'CCDS30551.1',
            source: 'Ensembl',
            gene_pheno: '1',
            domains:
              'Gene3D:3.30.70.960&Pfam:PF01390&PROSITE_profiles:PS50024&PANTHER:PTHR10913&PANTHER:PTHR10913:SF74&SMART:SM00200&Superfamily:SSF82671',
            gnomad_af: 0.9083,
            gnomad_afr_af: 0.5251,
            gnomad_amr_af: 0.9388,
            gnomad_asj_af: 0.9227,
            gnomad_eas_af: 0.9995,
            gnomad_fin_af: 0.9532,
            gnomad_nfe_af: 0.9167,
            gnomad_oth_af: 0.9079,
            gnomad_sas_af: 0.9527,
            pheno: '1&1',
            flaglrg: 'LRG_198t1',
            cadd_phred: 7.235,
            cadd_raw: 0.571657,
            spliceai_pred_dp_ag: -16,
            spliceai_pred_dp_al: -8,
            spliceai_pred_dp_dg: 39,
            spliceai_pred_dp_dl: 6,
            spliceai_pred_ds_ag: [0.01],
            spliceai_pred_ds_al: [0],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'AGRN',
            constraint_mis_z: [0.22552],
            constraint_oe_lof: 0.31914,
            constraint_oe_lof_lower: 0.237,
            constraint_oe_lof_upper: [0.435],
            constraint_oe_mis: 0.98257,
            constraint_oe_mis_lower: 0.938,
            constraint_oe_mis_upper: [1.029],
            constraint_oe_syn: 1.2049,
            constraint_oe_syn_lower: 1.133,
            constraint_oe_syn_upper: 1.281,
            constraint_pli: 5.4727e-7,
            constraint_syn_z: [-3.9572],
            cgd_agegroup: 'Pediatric',
            cgd_condition: 'Myasthenic_syndrome&_congenital_8',
            cgd_entrezid: '375790',
            cgd_gene: 'AGRN',
            cgd_hgncid: '329',
            cgd_inheritance: 'AR',
            cgd_interventioncategories:
              'Musculoskeletal&_Neurologic&_Pharmacogenomic',
            cgd_interventionrationale:
              'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
            cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
            cgd_references: '19631309&_20301347&_22205389&_22678886',
            exomiser_ad_exgenespheno: 0.5615,
            exomiser_xr_exgenescombi: 0.00058018,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'criteria_provided,_multiple_submitters,_no_conflicts',
            clndn: 'Myasthenic_syndrome,_congenital,_8|not_specified',
            gnomadv3_af_afr: 0.5382,
            gnomadv3_af_nfe: 0.92,
            gnomadv3_af_xy: 0.8255,
            exomiser_xd_exgenescombi: 0.00058018,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.879,
            clnvcso: 'SO:0001483',
            gnomadv3_af: 0.8196,
            exomiser_mt_exgenespheno: 0.5615,
            clndisdb:
              'MONDO:MONDO:0014052,MedGen:C3808739,OMIM:615120|MedGen:CN169374',
            exomiser_ar_exgenescombi: 0.00058018,
            gnomadv3_af_eas: 0.9992,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.9237,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00058018,
            exomiser_ad_exgenescombi: 0.00058018,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'Benign',
            clnvc: 'single_nucleotide_variant',
            gnomadv3_af_ami: 0.9857,
            gnomadv3_af_fin: 0.958,
            exomiser_ar_exgenespheno: 0.5615,
            gnomadv3_af_sas: 0.9566,
            gnomadv3_af_amr: 0.8959,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'NC_000001.11:g.1047614T>C',
            exomiser_ad_exgenesvar: 0,
            clnid: '128302',
            gnomadv3_af_xx: 0.814,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.8461,
            exomiser_xr_exgenespheno: 0.5615,
            exomiser_xd_exgenespheno: 0.5615,
            impact_severity: 'LOW',
            quality: 4608.7,
            genotypes_index: [
              [1, 1],
              [1, 1],
              [1, 1],
            ],
            allelic_depths: [
              [0, 42],
              [0, 46],
              [0, 47],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['AGRN'],
            ensembl_transcript_id: [
              'ENST00000379370',
              'ENST00000620552',
              'ENST00000651234',
              'ENST00000652369',
            ],
            variant_type: 'small',
            highest_af: 0.9995,
            highest_af_info: {
              source: 'gnomad V2 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 982994,
            hg19_strand: '+',
            hg19_end: 982994,
            entrez_gene_id: ['375790'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9d6',
            variant_id: '83',
            end: 1048778,
            ref: 'CAAAAAAAAAAA',
            alt: 'C',
            start: 1048767,
            chrom: 'chr1',
            type: 'indel',
            gene_objs: [
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000379370',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '23/35',
                hgvsc: 'ENST00000379370.7:c.4106-88_4106-78del',
                existing_variation: 'rs57668569',
                strand: 1,
                hgnc_id: 'HGNC:329',
                mane_select: 'NM_198576.4',
                ccds: 'CCDS30551.1',
                source: 'Ensembl',
                gene_pheno: 1,
                flaglrg: 'LRG_198t1',
                constraint_mis_z: 0.22552,
                constraint_oe_lof: 0.31914,
                constraint_oe_lof_lower: 0.237,
                constraint_oe_lof_upper: 0.435,
                constraint_oe_mis: 0.98257,
                constraint_oe_mis_lower: 0.938,
                constraint_oe_mis_upper: 1.029,
                constraint_oe_syn: 1.2049,
                constraint_oe_syn_lower: 1.133,
                constraint_oe_syn_upper: 1.281,
                constraint_pli: 5.4727e-7,
                constraint_syn_z: -3.9572,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                p_li: 5.4727e-7,
                loeuf: 0.435,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000620552',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '23/38',
                hgvsc: 'ENST00000620552.4:c.3692-88_3692-78del',
                existing_variation: 'rs57668569',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000651234',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '22/37',
                hgvsc: 'ENST00000651234.1:c.3791-88_3791-78del',
                existing_variation: 'rs57668569',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000652369',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '22/34',
                hgvsc: 'ENST00000652369.1:c.3791-88_3791-78del',
                existing_variation: 'rs57668569',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
            ],
            mane_select: 'ENST00000379370',
            ensembl_gene_id: ['ENSG00000188157'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant',
            hgvsc: 'ENST00000379370.7:c.4106-88_4106-78del',
            existing_variation: 'rs57668569',
            ccds: 'CCDS30551.1',
            source: 'Ensembl',
            gene_pheno: '1',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            flaglrg: 'LRG_198t1',
            constraint_mis_z: [0.22552],
            constraint_oe_lof: 0.31914,
            constraint_oe_lof_lower: 0.237,
            constraint_oe_lof_upper: [0.435],
            constraint_oe_mis: 0.98257,
            constraint_oe_mis_lower: 0.938,
            constraint_oe_mis_upper: [1.029],
            constraint_oe_syn: 1.2049,
            constraint_oe_syn_lower: 1.133,
            constraint_oe_syn_upper: 1.281,
            constraint_pli: 5.4727e-7,
            constraint_syn_z: [-3.9572],
            cgd_agegroup: 'Pediatric',
            cgd_condition: 'Myasthenic_syndrome&_congenital_8',
            cgd_entrezid: '375790',
            cgd_gene: 'AGRN',
            cgd_hgncid: '329',
            cgd_inheritance: 'AR',
            cgd_interventioncategories:
              'Musculoskeletal&_Neurologic&_Pharmacogenomic',
            cgd_interventionrationale:
              'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
            cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
            cgd_references: '19631309&_20301347&_22205389&_22678886',
            exomiser_ad_exgenespheno: 0.5615,
            exomiser_xr_exgenescombi: 0.00058018,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'None',
            clndn: 'None',
            gnomadv3_af_afr: 0.3591,
            gnomadv3_af_nfe: 0.623,
            gnomadv3_af_xy: 0.5914,
            exomiser_xd_exgenescombi: 0.00058018,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.6452,
            gnomadv3_af: 0.5838,
            exomiser_mt_exgenespheno: 0.5615,
            clndisdb: 'None',
            exomiser_ar_exgenescombi: 0.00058018,
            gnomadv3_af_eas: 0.8222,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.6241,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00058018,
            exomiser_ad_exgenescombi: 0.00058018,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'None',
            gnomadv3_af_ami: 0.6587,
            gnomadv3_af_fin: 0.6576,
            exomiser_ar_exgenespheno: 0.5615,
            gnomadv3_af_sas: 0.6433,
            gnomadv3_af_amr: 0.7074,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'None',
            exomiser_ad_exgenesvar: 0,
            gnomadv3_af_xx: 0.5765,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.5901,
            exomiser_xr_exgenespheno: 0.5615,
            exomiser_xd_exgenespheno: 0.5615,
            impact_severity: 'LOW',
            quality: 2280.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [1, 1],
            ],
            allelic_depths: [
              [3, 15],
              [3, 8],
              [0, 29],
            ],
            genotype_qualities: [[99, 99, 91]],
            pass_filter: 'PASS',
            gene_symbol: ['AGRN'],
            ensembl_transcript_id: [
              'ENST00000379370',
              'ENST00000620552',
              'ENST00000651234',
              'ENST00000652369',
            ],
            variant_type: 'small',
            highest_af: 0.8222,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 984147,
            hg19_strand: '+',
            hg19_end: 984158,
            entrez_gene_id: ['375790'],
          },
          {
            is_read: false,
            _id: '64e59ef79c2aec9c8bd9f9d7',
            variant_id: '84',
            end: 1048793,
            ref: 'CAG',
            alt: 'C',
            start: 1048791,
            chrom: 'chr1',
            type: 'indel',
            gene_objs: [
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000379370',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: true,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '23/35',
                hgvsc: 'ENST00000379370.7:c.4106-75_4106-74del',
                existing_variation: 'rs140904842',
                strand: 1,
                hgnc_id: 'HGNC:329',
                mane_select: 'NM_198576.4',
                ccds: 'CCDS30551.1',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                flaglrg: 'LRG_198t1',
                spliceai_pred_dp_ag: 50,
                spliceai_pred_dp_al: 3,
                spliceai_pred_dp_dg: -24,
                spliceai_pred_dp_dl: -29,
                spliceai_pred_ds_ag: 0.05,
                spliceai_pred_ds_al: 0.03,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                constraint_mis_z: 0.22552,
                constraint_oe_lof: 0.31914,
                constraint_oe_lof_lower: 0.237,
                constraint_oe_lof_upper: 0.435,
                constraint_oe_mis: 0.98257,
                constraint_oe_mis_lower: 0.938,
                constraint_oe_mis_upper: 1.029,
                constraint_oe_syn: 1.2049,
                constraint_oe_syn_lower: 1.133,
                constraint_oe_syn_upper: 1.281,
                constraint_pli: 5.4727e-7,
                constraint_syn_z: -3.9572,
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                p_li: 5.4727e-7,
                loeuf: 0.435,
                is_mane_select: true,
                is_mane_plus_clinical: false,
                default_show: true,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000620552',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '23/38',
                hgvsc: 'ENST00000620552.4:c.3692-75_3692-74del',
                existing_variation: 'rs140904842',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                spliceai_pred_dp_ag: 50,
                spliceai_pred_dp_al: 3,
                spliceai_pred_dp_dg: -24,
                spliceai_pred_dp_dl: -29,
                spliceai_pred_ds_ag: 0.05,
                spliceai_pred_ds_al: 0.03,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000651234',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '22/37',
                hgvsc: 'ENST00000651234.1:c.3791-75_3791-74del',
                existing_variation: 'rs140904842',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                spliceai_pred_dp_ag: 50,
                spliceai_pred_dp_al: 3,
                spliceai_pred_dp_dg: -24,
                spliceai_pred_dp_dl: -29,
                spliceai_pred_ds_ag: 0.05,
                spliceai_pred_ds_al: 0.03,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
              {
                gene: 'AGRN',
                ensembl_gene_id: 'ENSG00000188157',
                transcript: 'ENST00000652369',
                is_exonic: false,
                is_coding: false,
                is_splicing: false,
                is_canonical: false,
                is_lof: false,
                allele: '-',
                impact: ['intron_variant'],
                feature_type: 'Transcript',
                intron: '22/34',
                hgvsc: 'ENST00000652369.1:c.3791-75_3791-74del',
                existing_variation: 'rs140904842',
                strand: 1,
                hgnc_id: 'HGNC:329',
                source: 'Ensembl',
                gene_pheno: 1,
                clin_sig: 'benign',
                pheno: 1,
                spliceai_pred_dp_ag: 50,
                spliceai_pred_dp_al: 3,
                spliceai_pred_dp_dg: -24,
                spliceai_pred_dp_dl: -29,
                spliceai_pred_ds_ag: 0.05,
                spliceai_pred_ds_al: 0.03,
                spliceai_pred_ds_dg: 0,
                spliceai_pred_ds_dl: 0,
                spliceai_pred_symbol: 'AGRN',
                cgd_agegroup: 'Pediatric',
                cgd_condition: 'Myasthenic_syndrome&_congenital_8',
                cgd_entrezid: 375790,
                cgd_gene: 'AGRN',
                cgd_hgncid: 329,
                cgd_inheritance: 'AR',
                cgd_interventioncategories:
                  'Musculoskeletal&_Neurologic&_Pharmacogenomic',
                cgd_interventionrationale:
                  'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
                cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
                cgd_references: '19631309&_20301347&_22205389&_22678886',
                impact_so: 'intron_variant',
                impact_severity: 'LOW',
                impact_severity_score: 0,
                gene_filter: 'AGRN',
                entrez_gene_id: '375790',
                p_haplo: 0.50164,
                p_triplo: 0.86738,
                clingen_hi: 'autosomal',
                clingen_ts: 'no',
                is_mane_select: false,
                is_mane_plus_clinical: false,
                default_show: false,
                is_related: true,
                clingen_hi_display:
                  '🟣 Gene Associated with Autosomal Recessive Phenotype',
                clingen_ts_display: 'No Evidence for Triplosensitivity',
              },
            ],
            mane_select: 'ENST00000379370',
            ensembl_gene_id: ['ENSG00000188157'],
            is_exonic: 0,
            is_coding: 0,
            impact: 'intron_variant',
            hgvsc: 'ENST00000379370.7:c.4106-75_4106-74del',
            existing_variation: 'rs140904842',
            ccds: 'CCDS30551.1',
            source: 'Ensembl',
            gene_pheno: '1',
            gnomad_af: -1,
            gnomad_afr_af: -1,
            gnomad_amr_af: -1,
            gnomad_asj_af: -1,
            gnomad_eas_af: -1,
            gnomad_fin_af: -1,
            gnomad_nfe_af: -1,
            gnomad_oth_af: -1,
            gnomad_sas_af: -1,
            pheno: '1',
            flaglrg: 'LRG_198t1',
            spliceai_pred_dp_ag: 50,
            spliceai_pred_dp_al: 3,
            spliceai_pred_dp_dg: -24,
            spliceai_pred_dp_dl: -29,
            spliceai_pred_ds_ag: [0.05],
            spliceai_pred_ds_al: [0.03],
            spliceai_pred_ds_dg: [0],
            spliceai_pred_ds_dl: [0],
            spliceai_pred_symbol: 'AGRN',
            constraint_mis_z: [0.22552],
            constraint_oe_lof: 0.31914,
            constraint_oe_lof_lower: 0.237,
            constraint_oe_lof_upper: [0.435],
            constraint_oe_mis: 0.98257,
            constraint_oe_mis_lower: 0.938,
            constraint_oe_mis_upper: [1.029],
            constraint_oe_syn: 1.2049,
            constraint_oe_syn_lower: 1.133,
            constraint_oe_syn_upper: 1.281,
            constraint_pli: 5.4727e-7,
            constraint_syn_z: [-3.9572],
            cgd_agegroup: 'Pediatric',
            cgd_condition: 'Myasthenic_syndrome&_congenital_8',
            cgd_entrezid: '375790',
            cgd_gene: 'AGRN',
            cgd_hgncid: '329',
            cgd_inheritance: 'AR',
            cgd_interventioncategories:
              'Musculoskeletal&_Neurologic&_Pharmacogenomic',
            cgd_interventionrationale:
              'Ephedrine_treatment_has_been_reported_as_effective_in_affected_inviduals&_Medical_treatment_with_acetylcholine_esterase_inhibitors_and/or_potassium_channel_blocker_3&4-diaminopyridine_may_be_beneficial_in_many_individuals&_In_infancy&_the_use_of_apnea_monitors_may_be_indicated&_Agents_that_affect_neuromuscular_transmission_and_exacerbate_myasthenic_manifestations_should_be_avoided_',
            cgd_manifestationcategories: 'Musculoskeletal&_Neurologic',
            cgd_references: '19631309&_20301347&_22205389&_22678886',
            exomiser_ad_exgenespheno: 0.5615,
            exomiser_xr_exgenescombi: 0.00058018,
            exomiser_ar_exgenesvar: 0,
            clnrevstat: 'criteria_provided,_multiple_submitters,_no_conflicts',
            clndn: 'Myasthenic_syndrome,_congenital,_8|not_provided',
            gnomadv3_af_afr: 0.3066,
            gnomadv3_af_nfe: 0.5482,
            gnomadv3_af_xy: 0.5087,
            exomiser_xd_exgenescombi: 0.00058018,
            clnsigconf: 'None',
            gnomadv3_af_mid: 0.5467,
            clnvcso: 'SO:0000159',
            gnomadv3_af: 0.504,
            exomiser_mt_exgenespheno: 0.5615,
            clndisdb:
              'MONDO:MONDO:0014052,MedGen:C3808739,OMIM:615120|MedGen:CN517202',
            exomiser_ar_exgenescombi: 0.00058018,
            gnomadv3_af_eas: 0.78,
            exomiser_mt_exgenesvar: 0,
            gnomadv3_af_asj: 0.5508,
            clnsigincl: 'None',
            exomiser_mt_exgenescombi: 0.00058018,
            exomiser_ad_exgenescombi: 0.00058018,
            clnvi: 'None',
            exomiser_xd_exgenesvar: 0,
            clnsig: 'Benign',
            clnvc: 'Deletion',
            gnomadv3_af_ami: 0.5839,
            gnomadv3_af_fin: 0.5334,
            exomiser_ar_exgenespheno: 0.5615,
            gnomadv3_af_sas: 0.5609,
            gnomadv3_af_amr: 0.6583,
            clndnincl: 'None',
            clndisdbincl: 'None',
            clnhgvs: 'NC_000001.11:g.1048792_1048793del',
            exomiser_ad_exgenesvar: 0,
            clnid: '679293',
            gnomadv3_af_xx: 0.4995,
            exomiser_xr_exgenesvar: 0,
            gnomadv3_af_oth: 0.5195,
            exomiser_xr_exgenespheno: 0.5615,
            exomiser_xd_exgenespheno: 0.5615,
            impact_severity: 'LOW',
            quality: 2446.9,
            genotypes_index: [
              [0, 1],
              [0, 1],
              [1, 1],
            ],
            allelic_depths: [
              [5, 15],
              [5, 9],
              [0, 33],
            ],
            genotype_qualities: [[99, 99, 99]],
            pass_filter: 'PASS',
            gene_symbol: ['AGRN'],
            ensembl_transcript_id: [
              'ENST00000379370',
              'ENST00000620552',
              'ENST00000651234',
              'ENST00000652369',
            ],
            variant_type: 'small',
            highest_af: 0.78,
            highest_af_info: {
              source: 'gnomad v3 AF EAS',
            },
            hg19_chrom: 'chr1',
            hg19_start: 984171,
            hg19_strand: '+',
            hg19_end: 984173,
            entrez_gene_id: ['375790'],
          },
        ],
        page_info: {
          current_page: 1,
          per_page: 300,
          total_count: 181551,
          filtered_total_count: 181551,
          total_page: 606,
        },
      };

      mockFindVariants.mockResolvedValue(expectedVariants);

      // Act
      const result = await controller.findVariants(pagingRequest, userInfo);

      // Assert
      expect(result).toBeInstanceOf(BaseResponse);
      expect(result.data).toBe(expectedVariants);
      expect(result.track_number).toBe(expectedTrackNumber);
      expect(variantService.findVariants).toHaveBeenCalledWith(
        pagingRequest,
        userInfo,
      );
    });
  });

  describe('findVariantExons', () => {
    it('should return a BaseResponse with the correct exons and track number', async () => {
      const expectedTrackNumber = 'bab-test01:1667527361234';
      const findVariantExonsRequest: FindVariantExonsRequest = {
        track_number: expectedTrackNumber,
        selected_database: 'family_trio_20230105-110345',
        variant_id: 'S:manta:12075',
      } as FindVariantExonsRequest;
      const userInfo = { username: 'user' };

      const expectedExons: any = {
        _id: '64e5a3d199107f1eb02f158e',
        exons: [
          {
            start: 100218617,
            end: 100220028,
            ENST_ID: 'ENST00000451963',
            EXON_ID: 6,
            IS_CODING: 1,
            reciprocalOverlap: 0.01053,
            annotationOverlap: 0.94972,
            GENE_SYMBOL: 'STAG3',
            gene_stable_id: 'ENSG00000066923',
          },
          {
            start: 100219249,
            end: 100219400,
            ENST_ID: 'ENST00000317271',
            EXON_ID: 2,
            IS_CODING: 0,
            reciprocalOverlap: 0.00119,
            annotationOverlap: 1,
            GENE_SYMBOL: 'PVRIG',
            gene_stable_id: 'ENSG00000213413',
          },
          {
            start: 100219713,
            end: 100220028,
            ENST_ID: 'ENST00000317271',
            EXON_ID: 3,
            IS_CODING: 1,
            reciprocalOverlap: 0.00248,
            annotationOverlap: 1,
            GENE_SYMBOL: 'PVRIG',
            gene_stable_id: 'ENSG00000213413',
          },
          {
            start: 100219949,
            end: 100220028,
            ENST_ID: 'ENST00000699088',
            EXON_ID: 1,
            IS_CODING: 1,
            reciprocalOverlap: 0.00063,
            annotationOverlap: 1,
            GENE_SYMBOL: 'PVRIG',
            gene_stable_id: 'ENSG00000213413',
          },
          {
            start: 100220114,
            end: 100220464,
            ENST_ID: 'ENST00000317271',
            EXON_ID: 4,
            IS_CODING: 1,
            reciprocalOverlap: 0.00276,
            annotationOverlap: 1,
            GENE_SYMBOL: 'PVRIG',
            gene_stable_id: 'ENSG00000213413',
          },
          {
            start: 100220114,
            end: 100220464,
            ENST_ID: 'ENST00000699088',
            EXON_ID: 2,
            IS_CODING: 1,
            reciprocalOverlap: 0.00276,
            annotationOverlap: 1,
            GENE_SYMBOL: 'PVRIG',
            gene_stable_id: 'ENSG00000213413',
          },
          {
            start: 100220114,
            end: 100220673,
            ENST_ID: 'ENST00000451963',
            EXON_ID: 7,
            IS_CODING: 0,
            reciprocalOverlap: 0.0044,
            annotationOverlap: 1,
            GENE_SYMBOL: 'STAG3',
            gene_stable_id: 'ENSG00000066923',
          },
          {
            start: 100220547,
            end: 100220673,
            ENST_ID: 'ENST00000317271',
            EXON_ID: 5,
            IS_CODING: 1,
            reciprocalOverlap: 0.001,
            annotationOverlap: 1,
            GENE_SYMBOL: 'PVRIG',
            gene_stable_id: 'ENSG00000213413',
          },
          {
            start: 100220547,
            end: 100220673,
            ENST_ID: 'ENST00000699088',
            EXON_ID: 3,
            IS_CODING: 1,
            reciprocalOverlap: 0.001,
            annotationOverlap: 1,
            GENE_SYMBOL: 'PVRIG',
            gene_stable_id: 'ENSG00000213413',
          },
          {
            start: 100220760,
            end: 100220820,
            ENST_ID: 'ENST00000317271',
            EXON_ID: 6,
            IS_CODING: 1,
            reciprocalOverlap: 0.00048,
            annotationOverlap: 1,
            GENE_SYMBOL: 'PVRIG',
            gene_stable_id: 'ENSG00000213413',
          },
          {
            start: 100220760,
            end: 100220820,
            ENST_ID: 'ENST00000451963',
            EXON_ID: 8,
            IS_CODING: 0,
            reciprocalOverlap: 0.00048,
            annotationOverlap: 1,
            GENE_SYMBOL: 'STAG3',
            gene_stable_id: 'ENSG00000066923',
          },
          {
            start: 100220760,
            end: 100220820,
            ENST_ID: 'ENST00000699088',
            EXON_ID: 4,
            IS_CODING: 1,
            reciprocalOverlap: 0.00048,
            annotationOverlap: 1,
            GENE_SYMBOL: 'PVRIG',
            gene_stable_id: 'ENSG00000213413',
          },
          {
            start: 100220928,
            end: 100221488,
            ENST_ID: 'ENST00000451963',
            EXON_ID: 9,
            IS_CODING: 0,
            reciprocalOverlap: 0.0044,
            annotationOverlap: 1,
            GENE_SYMBOL: 'STAG3',
            gene_stable_id: 'ENSG00000066923',
          },
          {
            start: 100220928,
            end: 100221490,
            ENST_ID: 'ENST00000317271',
            EXON_ID: 7,
            IS_CODING: 1,
            reciprocalOverlap: 0.00442,
            annotationOverlap: 1,
            GENE_SYMBOL: 'PVRIG',
            gene_stable_id: 'ENSG00000213413',
          },
          {
            start: 100220928,
            end: 100221490,
            ENST_ID: 'ENST00000699088',
            EXON_ID: 5,
            IS_CODING: 1,
            reciprocalOverlap: 0.00442,
            annotationOverlap: 1,
            GENE_SYMBOL: 'PVRIG',
            gene_stable_id: 'ENSG00000213413',
          },
          {
            start: 100307702,
            end: 100307991,
            ENST_ID: 'ENST00000332397',
            EXON_ID: 1,
            IS_CODING: 1,
            reciprocalOverlap: 0.00228,
            annotationOverlap: 1,
            GENE_SYMBOL: 'SPDYE3',
            gene_stable_id: 'ENSG00000214300',
          },
          {
            start: 100308974,
            end: 100309192,
            ENST_ID: 'ENST00000332397',
            EXON_ID: 2,
            IS_CODING: 1,
            reciprocalOverlap: 0.00172,
            annotationOverlap: 1,
            GENE_SYMBOL: 'SPDYE3',
            gene_stable_id: 'ENSG00000214300',
          },
          {
            start: 100310360,
            end: 100310578,
            ENST_ID: 'ENST00000332397',
            EXON_ID: 3,
            IS_CODING: 1,
            reciprocalOverlap: 0.00172,
            annotationOverlap: 1,
            GENE_SYMBOL: 'SPDYE3',
            gene_stable_id: 'ENSG00000214300',
          },
          {
            start: 100311750,
            end: 100311968,
            ENST_ID: 'ENST00000332397',
            EXON_ID: 4,
            IS_CODING: 1,
            reciprocalOverlap: 0.00172,
            annotationOverlap: 1,
            GENE_SYMBOL: 'SPDYE3',
            gene_stable_id: 'ENSG00000214300',
          },
          {
            start: 100313140,
            end: 100313358,
            ENST_ID: 'ENST00000332397',
            EXON_ID: 5,
            IS_CODING: 1,
            reciprocalOverlap: 0.00172,
            annotationOverlap: 1,
            GENE_SYMBOL: 'SPDYE3',
            gene_stable_id: 'ENSG00000214300',
          },
          {
            start: 100314532,
            end: 100314750,
            ENST_ID: 'ENST00000332397',
            EXON_ID: 6,
            IS_CODING: 1,
            reciprocalOverlap: 0.00172,
            annotationOverlap: 1,
            GENE_SYMBOL: 'SPDYE3',
            gene_stable_id: 'ENSG00000214300',
          },
          {
            start: 100315785,
            end: 100315843,
            ENST_ID: 'ENST00000332397',
            EXON_ID: 7,
            IS_CODING: 1,
            reciprocalOverlap: 0.00046,
            annotationOverlap: 1,
            GENE_SYMBOL: 'SPDYE3',
            gene_stable_id: 'ENSG00000214300',
          },
          {
            start: 100317070,
            end: 100317155,
            ENST_ID: 'ENST00000332397',
            EXON_ID: 8,
            IS_CODING: 1,
            reciprocalOverlap: 0.00068,
            annotationOverlap: 1,
            GENE_SYMBOL: 'SPDYE3',
            gene_stable_id: 'ENSG00000214300',
          },
          {
            start: 100319565,
            end: 100319808,
            ENST_ID: 'ENST00000332397',
            EXON_ID: 9,
            IS_CODING: 1,
            reciprocalOverlap: 0.00192,
            annotationOverlap: 1,
            GENE_SYMBOL: 'SPDYE3',
            gene_stable_id: 'ENSG00000214300',
          },
          {
            start: 100319931,
            end: 100320035,
            ENST_ID: 'ENST00000332397',
            EXON_ID: 10,
            IS_CODING: 1,
            reciprocalOverlap: 0.00082,
            annotationOverlap: 1,
            GENE_SYMBOL: 'SPDYE3',
            gene_stable_id: 'ENSG00000214300',
          },
          {
            start: 100320891,
            end: 100322196,
            ENST_ID: 'ENST00000332397',
            EXON_ID: 11,
            IS_CODING: 0,
            reciprocalOverlap: 0.01025,
            annotationOverlap: 1,
            GENE_SYMBOL: 'SPDYE3',
            gene_stable_id: 'ENSG00000214300',
          },
        ],
        variant_id: 'S:manta:12075',
      };

      mockFindVariantExons.mockResolvedValue(expectedExons);

      // Act
      const result = await controller.findVariantExons(
        findVariantExonsRequest,
        userInfo,
      );

      // Assert
      expect(result).toBeInstanceOf(BaseResponse);
      expect(result.data).toBe(expectedExons);
      expect(result.track_number).toBe(findVariantExonsRequest.track_number);
      expect(variantService.findVariantExons).toHaveBeenCalledWith(
        findVariantExonsRequest,
        userInfo,
      );
    });
  });

  describe('markAsRead', () => {
    it('should return a BaseResponse with the correct result and track number', async () => {
      const expectedTrackNumber = 'bab-test01:61686049198';
      const markReadRequest: MarkReadRequest = {
        selected_database: 'family_trio_20230105-110345',
        track_number: expectedTrackNumber,
        is_read: [
          {
            obj_id: '64e5a2209b592486e3891b9e',
            is_share: false,
            is_read: true,
          },
        ],
      } as MarkReadRequest;
      const userInfo = { username: 'user' };

      const expectedResult: MarkReadResponse = {
        success_list: ['64d99c7821727c5dda8274ac'],
      };

      mockSaveRead.mockResolvedValue(expectedResult);

      const result = await controller.markAsRead(markReadRequest, userInfo);

      expect(result).toBeInstanceOf(BaseResponse);
      expect(result.data).toBe(expectedResult);
      expect(result.track_number).toBe(expectedTrackNumber);
      expect(variantService.saveRead).toHaveBeenCalledWith(
        markReadRequest,
        userInfo,
      );
    });
  });

  describe('saveNote', () => {
    it('should return a BaseResponse with the correct result and track number', async () => {
      const expectedTrackNumber = 'trackNumber';
      const updateNoteRequest: UpdateNoteRequest = {
        track_number: expectedTrackNumber,
      } as UpdateNoteRequest;
      const userInfo = { username: 'user' };

      const expectedResult: Variant = {
        allelic_depths: [],
        alt: '<DUP>',
        chrom: 'chr1',
        constraint_oe_lof_upper: [],
        constraint_pli: '',
        end: 65417,
        genotype_qualities: [],
        genotypes_index: [
          [0, 0],
          [0, 1],
          [0, 0],
        ],
        gnomad_af: -1,
        gnomad_afr_af: -1,
        gnomad_amr_af: -1,
        gnomad_asj_af: -1,
        gnomad_eas_af: -1,
        gnomad_fin_af: -1,
        gnomad_nfe_af: -1,
        gnomad_oth_af: -1,
        gnomad_sas_af: -1,
        gnomadv3_af: -1,
        gnomadv3_af_afr: -1,
        gnomadv3_af_ami: -1,
        gnomadv3_af_amr: -1,
        gnomadv3_af_asj: -1,
        gnomadv3_af_eas: -1,
        gnomadv3_af_fin: -1,
        gnomadv3_af_mid: -1,
        gnomadv3_af_nfe: -1,
        gnomadv3_af_oth: -1,
        gnomadv3_af_sas: -1,
        gnomadv3_af_xx: -1,
        gnomadv3_af_xy: -1,
        impact: '',
        ref: 'A',
        start: 51937,
        type: 'DUP',
        variant_id: 'S:cnvkit:1',
        variant_type: 'structural',
        is_read: false,
        highest_af: 0.00149,
        p_lof: 'False',
        is_pathogenic: false,
        gene_objs: [],
        sv_id: 'S:cnvkit:1',
        note: 'testing',
        hg19_chrom: 'chr1',
        hg19_start: 51937,
        hg19_strand: '+',
        hg19_end: 65417,
        caller: 'cnvkit',
        afs: [
          {
            source_start: 51439,
            source_end: 65417,
            source_svType: 'DUP',
            AF: 0.00149,
            SAMPLE_COUNT: 1680,
            reciOverlap: 0.964,
            annoOverlap: 0.964,
            source_filter: 'hkbc',
            display_source: 'Hong Kong Birth Cohort',
            source: 'HKBC - CNVkit',
            svLength: 13979,
          },
        ],
        exts: [],
        intpns: [],
        len: 13480,
        highest_af_info: {
          source: 'HKBC - CNVkit',
        },
        _id: '64e5a2209b592486e3891b9e',
        gene_symbol: [],
        ensembl_gene_id: [],
        entrez_gene_id: [],
        ensembl_transcript_id: [],
      };

      jest.spyOn(variantService, 'saveNote').mockResolvedValue(expectedResult);

      // Act
      const result = await controller.saveNote(updateNoteRequest, userInfo);

      // Assert
      expect(result).toBeInstanceOf(BaseResponse);
      expect(result.data).toBe(expectedResult);
      expect(result.track_number).toBe(updateNoteRequest.track_number);
      expect(variantService.saveNote).toHaveBeenCalledWith(
        updateNoteRequest,
        userInfo,
      );
    });
  });

  describe('exportTsv', () => {
    it('should call variantService.exportTsv with the correct arguments', async () => {
      // Arrange
      const exportRequest: ExportRequest = {} as ExportRequest;
      const userInfo = { username: 'user' };

      jest.spyOn(variantService, 'exportTsv');

      // ActHere's the continuation of the Jest unit tests for the remaining methods in the `VariantController` class:

      await controller.exportTsv(exportRequest, userInfo);

      // Assert
      expect(variantService.exportTsv).toHaveBeenCalledWith(
        exportRequest,
        userInfo,
      );
    });
  });

  describe('getExportTsvResult', () => {
    it('should set the content headers and return the export result', async () => {
      // Arrange
      const exportResultRequest: ExportResultRequest =
        {} as ExportResultRequest;
      const userInfo = { username: 'user' };
      const res: ExpressResponse = {
        set: jest.fn(),
        send: jest.fn(),
        status: jest.fn(),
      } as any;
      const expectedExportResult = 'export result';

      jest
        .spyOn(variantService, 'getExportResult')
        .mockResolvedValue(expectedExportResult);

      // Act
      await controller.getExportTsvResult(exportResultRequest, userInfo, res);

      // Assert
      expect(variantService.getExportResult).toHaveBeenCalledWith(
        exportResultRequest,
        userInfo,
        ExportType.tsv,
      );
      expect(res.set).toHaveBeenCalledWith(
        'Content-Disposition',
        'attachment; filename="variants.tsv"',
      );
      expect(res.set).toHaveBeenCalledWith('Content-Type', 'text/plain');
      expect(res.send).toHaveBeenCalledWith(expectedExportResult);
      expect(res.status).not.toHaveBeenCalled();
    });

    it('should set status 204 and return null if export result is falsy', async () => {
      // Arrange
      const exportResultRequest: ExportResultRequest =
        {} as ExportResultRequest;
      const userInfo = { username: 'user' };
      const res: ExpressResponse = {
        set: jest.fn(),
        send: jest.fn(),
        status: jest.fn(),
      } as any;
      const expectedExportResult = null;

      jest
        .spyOn(variantService, 'getExportResult')
        .mockResolvedValue(expectedExportResult);

      // Act
      await controller.getExportTsvResult(exportResultRequest, userInfo, res);

      // Assert
      expect(variantService.getExportResult).toHaveBeenCalledWith(
        exportResultRequest,
        userInfo,
        ExportType.tsv,
      );
      expect(res.set).not.toHaveBeenCalled();
      expect(res.send).toHaveBeenCalledWith(null);
      expect(res.status).toHaveBeenCalledWith(204);
    });
  });

  describe('exportVcf', () => {
    it('should call variantService.exportVcf with the correct arguments', async () => {
      // Arrange
      const exportRequest: ExportRequest = {} as ExportRequest;
      const userInfo = { username: 'user' };

      jest.spyOn(variantService, 'exportVcf');

      // Act
      await controller.exportVcf(exportRequest, userInfo);

      // Assert
      expect(variantService.exportVcf).toHaveBeenCalledWith(
        exportRequest,
        userInfo,
      );
    });
  });

  describe('getExportVcfResult', () => {
    it('should set the content headers and return the export result', async () => {
      // Arrange
      const exportResultRequest: ExportResultRequest =
        {} as ExportResultRequest;
      const userInfo = { username: 'user' };
      const res: ExpressResponse = {
        set: jest.fn(),
        send: jest.fn(),
        status: jest.fn(),
      } as any;
      const expectedExportResult = 'export result';

      jest
        .spyOn(variantService, 'getExportResult')
        .mockResolvedValue(expectedExportResult);

      // Act
      await controller.getExportVcfResult(exportResultRequest, userInfo, res);

      // Assert
      expect(variantService.getExportResult).toHaveBeenCalledWith(
        exportResultRequest,
        userInfo,
        ExportType.vcf,
      );
      expect(res.set).toHaveBeenCalledWith(
        'Content-Disposition',
        'attachment; filename="variants.vcf"',
      );
      expect(res.set).toHaveBeenCalledWith('Content-Type', 'text/plain');
      expect(res.send).toHaveBeenCalledWith(expectedExportResult);
      expect(res.status).not.toHaveBeenCalled();
    });

    it('should set status 204 and return null if export result is false', async () => {
      // Arrange
      const exportResultRequest: ExportResultRequest =
        {} as ExportResultRequest;
      const userInfo = { username: 'user' };
      const mockSet = jest.fn();
      const res: ExpressResponse = {
        set: mockSet,
        send: jest.fn(),
        status: jest.fn(),
      } as any;
      const expectedExportResult = null;

      jest
        .spyOn(variantService, 'getExportResult')
        .mockResolvedValue(expectedExportResult);

      // Act
      await controller.getExportVcfResult(exportResultRequest, userInfo, res);

      // Assert
      expect(variantService.getExportResult).toHaveBeenCalledWith(
        exportResultRequest,
        userInfo,
        ExportType.vcf,
      );
      expect(mockSet).not.toHaveBeenCalled();
      expect(res.send).toHaveBeenCalledWith(expectedExportResult);
      expect(res.status).toHaveBeenCalledWith(204);
    });
  });
});
