import { ObjectId } from 'mongodb';
import { Variant } from 'src/common';
import { Variants } from 'src/variantsInfo';

const convertJSONToVariants = (jsonVariants, variants: Variants) => {
  const variantProperties = Object.getOwnPropertyNames(jsonVariants);
  for (const key of variantProperties) {
    if (key == '_id') {
      variants[key] = new ObjectId(jsonVariants[key]['$oid']);
    } else {
      variants[key] = jsonVariants[key];
    }
  }
};
const convertJSONToVariant = (jsonVariant, variant: Variant) => {
  const variantProperties = Object.getOwnPropertyNames(jsonVariant);
  for (const key of variantProperties) {
    variant[key] = jsonVariant[key];
  }
};

const variants1 = {
  _id: {
    $oid: '656914a8754ff9bcbb45f150',
  },
  chrom: 'chr1',
  start: 52032698,
  ref: 'C',
  alt: 'T',
  quality: null,
  pass_filter: null,
  cm: 75.6471,
  genotypes_index: [
    [0, 0],
    [0, 1],
    [0, 0],
  ],
  sample: [1],
  scenario: ['0010'],
  highest_exomiser_scombi: 0,
  highest_exomiser_spheno: 0,
  is_coding: 1,
  is_exonic: 1,
  end: 52032698,
  genotype_qualities: [],
  allelic_depths: [],
  gene_objs: [
    {
      is_exonic: 1,
      allele: 'T',
      impact: ['stop_retained_variant'],
      impact_so: 'stop_retained_variant',
      impact_severity: 'LOW',
      impact_severity_score: 1,
      feature_type: 'Transcript',
      exon: '1/1',
      hgvsc: 'ENST00000371614.2:c.1064G>A',
      hgvsp: 'ENSP00000360676.1:p.Ter355%3D',
      cds_position: 1064,
      aa_length: 355,
      aa_change: '*',
      codon_change: 'tGa/tAa',
      existing_variation: 'rs2809917&COSV65405655',
      strand: -1,
      hgnc_id: 'HGNC:25160',
      is_canonical: true,
      mane_select: 'NM_138417.3',
      ccds: 'CCDS562.1',
      source: 'Ensembl',
      gnomad_af: 0.1447,
      gnomad_afr_af: 0.364,
      gnomad_amr_af: 0.1466,
      gnomad_asj_af: 0.0901,
      gnomad_eas_af: 0.1764,
      gnomad_fin_af: 0.07968,
      gnomad_nfe_af: 0.1122,
      gnomad_oth_af: 0.1212,
      gnomad_sas_af: 0.1902,
      gnomadv3_af: 0.1821,
      gnomadv3_af_afr: 0.3587,
      gnomadv3_af_ami: 0.07895,
      gnomadv3_af_amr: 0.1317,
      gnomadv3_af_asj: 0.09487,
      gnomadv3_af_eas: 0.1646,
      gnomadv3_af_fin: 0.07615,
      gnomadv3_af_mid: 0.1044,
      gnomadv3_af_nfe: 0.1108,
      gnomadv3_af_oth: 0.1571,
      gnomadv3_af_sas: 0.1818,
      pheno: '0&1',
      cadd_phred: 11.05,
      cadd_raw: 0.954327,
      spliceai_pred_dp_ag: -11,
      spliceai_pred_dp_al: -10,
      spliceai_pred_dp_dg: 24,
      spliceai_pred_dp_dl: -10,
      spliceai_pred_ds_ag: 0,
      highest_splice_ai: 0,
      spliceai_pred_ds_al: 0,
      spliceai_pred_ds_dg: 0,
      spliceai_pred_ds_dl: 0,
      spliceai_pred_symbol: 'KTI12',
      constraint_mis_z: -0.49843,
      constraint_oe_lof: 0.72753,
      constraint_oe_lof_lower: 0.397,
      constraint_oe_lof_upper: 1.424,
      constraint_oe_mis: 1.0947,
      constraint_oe_mis_lower: 0.984,
      constraint_oe_mis_upper: 1.218,
      constraint_oe_syn: 1.0718,
      constraint_oe_syn_lower: 0.915,
      constraint_oe_syn_upper: 1.259,
      constraint_pli: 0.00040289,
      constraint_syn_z: -0.56369,
      transcript: 'ENST00000371614',
      is_coding: 1,
      gene: 'KTI12',
      gene_filter: 'KTI12',
      entrez_gene_id: '112970',
      ensembl_gene_id: 'ENSG00000198841',
      p_haplo: 0.0741,
      p_triplo: 0.15418,
      p_li: 0.00040289,
      loeuf: 1.424,
      is_mane_select: true,
      is_mane_plus_clinical: false,
      default_show: true,
      is_related: true,
    },
    {
      is_exonic: 0,
      allele: 'T',
      impact: ['intron_variant'],
      impact_so: 'intron_variant',
      impact_severity: 'MODIFIER',
      impact_severity_score: 0,
      feature_type: 'Transcript',
      intron: '2/6',
      hgvsc: 'ENST00000371626.9:c.159-4068G>A',
      existing_variation: 'rs2809917&COSV65405655',
      strand: -1,
      hgnc_id: 'HGNC:24626',
      is_canonical: true,
      mane_select: 'NM_015913.4',
      ccds: 'CCDS561.1',
      source: 'Ensembl',
      gnomad_af: 0.1447,
      gnomad_afr_af: 0.364,
      gnomad_amr_af: 0.1466,
      gnomad_asj_af: 0.0901,
      gnomad_eas_af: 0.1764,
      gnomad_fin_af: 0.07968,
      gnomad_nfe_af: 0.1122,
      gnomad_oth_af: 0.1212,
      gnomad_sas_af: 0.1902,
      gnomadv3_af: 0.1821,
      gnomadv3_af_afr: 0.3587,
      gnomadv3_af_ami: 0.07895,
      gnomadv3_af_amr: 0.1317,
      gnomadv3_af_asj: 0.09487,
      gnomadv3_af_eas: 0.1646,
      gnomadv3_af_fin: 0.07615,
      gnomadv3_af_mid: 0.1044,
      gnomadv3_af_nfe: 0.1108,
      gnomadv3_af_oth: 0.1571,
      gnomadv3_af_sas: 0.1818,
      pheno: '0&1',
      cadd_phred: 11.05,
      cadd_raw: 0.954327,
      spliceai_pred_dp_ag: -15,
      spliceai_pred_dp_al: -10,
      spliceai_pred_dp_dg: 24,
      spliceai_pred_dp_dl: -10,
      spliceai_pred_ds_ag: 0,
      highest_splice_ai: 0.04,
      spliceai_pred_ds_al: 0,
      spliceai_pred_ds_dg: 0,
      spliceai_pred_ds_dl: 0.04,
      spliceai_pred_symbol: 'TXNDC12',
      constraint_mis_z: 0.51718,
      constraint_oe_lof: 0.73759,
      constraint_oe_lof_lower: 0.432,
      constraint_oe_lof_upper: 1.328,
      constraint_oe_mis: 0.84823,
      constraint_oe_mis_lower: 0.706,
      constraint_oe_mis_upper: 1.024,
      constraint_oe_syn: 0.85717,
      constraint_oe_syn_lower: 0.64,
      constraint_oe_syn_upper: 1.163,
      constraint_pli: 0.000037354,
      constraint_syn_z: 0.66422,
      transcript: 'ENST00000371626',
      is_coding: 0,
      gene: 'TXNDC12',
      gene_filter: 'TXNDC12',
      entrez_gene_id: '51060',
      ensembl_gene_id: 'ENSG00000117862',
      p_haplo: 0.35228,
      p_triplo: 0.38277,
      p_li: 0.000037354,
      loeuf: 1.328,
      is_mane_select: true,
      is_mane_plus_clinical: false,
      default_show: true,
      is_related: true,
    },
    {
      is_exonic: 0,
      allele: 'T',
      impact: ['intron_variant', 'NMD_transcript_variant'],
      impact_so: 'intron_variant',
      impact_severity: 'MODIFIER',
      impact_severity_score: 0,
      feature_type: 'Transcript',
      intron: '2/7',
      hgvsc: 'ENST00000472624.5:c.159-2150G>A',
      existing_variation: 'rs2809917&COSV65405655',
      strand: -1,
      hgnc_id: 'HGNC:24626',
      source: 'Ensembl',
      gnomad_af: 0.1447,
      gnomad_afr_af: 0.364,
      gnomad_amr_af: 0.1466,
      gnomad_asj_af: 0.0901,
      gnomad_eas_af: 0.1764,
      gnomad_fin_af: 0.07968,
      gnomad_nfe_af: 0.1122,
      gnomad_oth_af: 0.1212,
      gnomad_sas_af: 0.1902,
      gnomadv3_af: 0.1821,
      gnomadv3_af_afr: 0.3587,
      gnomadv3_af_ami: 0.07895,
      gnomadv3_af_amr: 0.1317,
      gnomadv3_af_asj: 0.09487,
      gnomadv3_af_eas: 0.1646,
      gnomadv3_af_fin: 0.07615,
      gnomadv3_af_mid: 0.1044,
      gnomadv3_af_nfe: 0.1108,
      gnomadv3_af_oth: 0.1571,
      gnomadv3_af_sas: 0.1818,
      pheno: '0&1',
      cadd_phred: 11.05,
      cadd_raw: 0.954327,
      spliceai_pred_dp_ag: -15,
      spliceai_pred_dp_al: -10,
      spliceai_pred_dp_dg: 24,
      spliceai_pred_dp_dl: -10,
      spliceai_pred_ds_ag: 0,
      highest_splice_ai: 0.04,
      spliceai_pred_ds_al: 0,
      spliceai_pred_ds_dg: 0,
      spliceai_pred_ds_dl: 0.04,
      spliceai_pred_symbol: 'TXNDC12',
      transcript: 'ENST00000472624',
      is_coding: 0,
      gene: 'TXNDC12',
      gene_filter: 'TXNDC12',
      entrez_gene_id: '51060',
      ensembl_gene_id: 'ENSG00000117862',
      p_haplo: 0.35228,
      p_triplo: 0.38277,
      is_canonical: false,
      is_mane_select: false,
      is_mane_plus_clinical: false,
      default_show: false,
      is_related: true,
    },
    {
      is_exonic: 0,
      allele: 'T',
      impact: ['downstream_gene_variant'],
      impact_so: 'downstream_gene_variant',
      impact_severity: 'MODIFIER',
      impact_severity_score: 0,
      feature_type: 'Transcript',
      existing_variation: 'rs2809917&COSV65405655',
      strand: -1,
      hgnc_id: 'HGNC:24626',
      source: 'Ensembl',
      gnomad_af: 0.1447,
      gnomad_afr_af: 0.364,
      gnomad_amr_af: 0.1466,
      gnomad_asj_af: 0.0901,
      gnomad_eas_af: 0.1764,
      gnomad_fin_af: 0.07968,
      gnomad_nfe_af: 0.1122,
      gnomad_oth_af: 0.1212,
      gnomad_sas_af: 0.1902,
      gnomadv3_af: 0.1821,
      gnomadv3_af_afr: 0.3587,
      gnomadv3_af_ami: 0.07895,
      gnomadv3_af_amr: 0.1317,
      gnomadv3_af_asj: 0.09487,
      gnomadv3_af_eas: 0.1646,
      gnomadv3_af_fin: 0.07615,
      gnomadv3_af_mid: 0.1044,
      gnomadv3_af_nfe: 0.1108,
      gnomadv3_af_oth: 0.1571,
      gnomadv3_af_sas: 0.1818,
      pheno: '0&1',
      cadd_phred: 11.05,
      cadd_raw: 0.954327,
      spliceai_pred_dp_ag: -15,
      spliceai_pred_dp_al: -10,
      spliceai_pred_dp_dg: 24,
      spliceai_pred_dp_dl: -10,
      spliceai_pred_ds_ag: 0,
      highest_splice_ai: 0.04,
      spliceai_pred_ds_al: 0,
      spliceai_pred_ds_dg: 0,
      spliceai_pred_ds_dl: 0.04,
      spliceai_pred_symbol: 'TXNDC12',
      constraint_mis_z: 0.1659,
      constraint_oe_lof: 1.0036,
      constraint_oe_lof_lower: 0.366,
      constraint_oe_lof_upper: 1.878,
      constraint_oe_mis: 0.91254,
      constraint_oe_mis_lower: 0.668,
      constraint_oe_mis_upper: 1.266,
      constraint_oe_syn: 1.0515,
      constraint_oe_syn_lower: 0.672,
      constraint_oe_syn_upper: 1.668,
      constraint_pli: 0.024969,
      constraint_syn_z: -0.1367,
      transcript: 'ENST00000610127',
      is_coding: 0,
      gene: 'TXNDC12',
      gene_filter: 'TXNDC12',
      entrez_gene_id: '51060',
      ensembl_gene_id: 'ENSG00000117862',
      p_haplo: 0.35228,
      p_triplo: 0.38277,
      p_li: 0.024969,
      loeuf: 1.878,
      is_canonical: false,
      is_mane_select: false,
      is_mane_plus_clinical: false,
      default_show: false,
      is_related: true,
    },
  ],
  impact: ['stop_retained_variant'],
  impact_severity: 'LOW',
  hgvsc: 'ENST00000371614.2:c.1064G>A',
  hgvsp: 'ENSP00000360676.1:p.Ter355%3D',
  aa_change: '*',
  existing_variation: 'rs2809917&COSV65405655',
  mane_select: 'NM_138417.3',
  ccds: 'CCDS562.1',
  source: 'Ensembl',
  gnomad_af: 0.1447,
  gnomad_afr_af: 0.364,
  gnomad_amr_af: 0.1466,
  gnomad_asj_af: 0.0901,
  gnomad_eas_af: 0.1764,
  gnomad_fin_af: 0.07968,
  gnomad_nfe_af: 0.1122,
  gnomad_oth_af: 0.1212,
  gnomad_sas_af: 0.1902,
  gnomadv3_af: 0.1821,
  gnomadv3_af_afr: 0.3587,
  gnomadv3_af_ami: 0.07895,
  gnomadv3_af_amr: 0.1317,
  gnomadv3_af_asj: 0.09487,
  gnomadv3_af_eas: 0.1646,
  gnomadv3_af_fin: 0.07615,
  gnomadv3_af_mid: 0.1044,
  gnomadv3_af_nfe: 0.1108,
  gnomadv3_af_oth: 0.1571,
  gnomadv3_af_sas: 0.1818,
  pheno: '0&1',
  cadd_phred: 11.05,
  cadd_raw: 0.954327,
  spliceai_pred_dp_ag: -11,
  spliceai_pred_dp_al: -10,
  spliceai_pred_dp_dg: 24,
  spliceai_pred_dp_dl: -10,
  spliceai_pred_symbol: 'KTI12',
  constraint_oe_lof: 0.72753,
  constraint_oe_lof_lower: 0.397,
  constraint_oe_lof_upper: [1.424, 1.328],
  constraint_oe_mis: 1.0947,
  constraint_oe_mis_lower: 0.984,
  constraint_oe_mis_upper: [1.218, 1.024],
  constraint_oe_syn: 1.0718,
  constraint_oe_syn_lower: 0.915,
  constraint_oe_syn_upper: 1.259,
  constraint_pli: 0.00040289,
  constraint_syn_z: [-0.56369, 0.66422],
  ensembl_transcript_id: 'ENST00000371614',
  gene_symbol: 'KTI12',
  ensembl_gene_id: 'ENSG00000198841',
  constraint_mis_z: [-0.49843, 0.51718],
  first_constraint_mis_z: -0.49843,
  first_constraint_syn_z: -0.56369,
  first_constraint_oe_lof_upper: 1.424,
  first_constraint_oe_mis_upper: 1.218,
  highest_splice_ai: [0, 0.04],
  first_highest_splice_ai: 0,
  spliceai_pred_ds_ag: [0],
  first_spliceai_pred_ds_ag: 0,
  spliceai_pred_ds_al: [0],
  first_spliceai_pred_ds_al: 0,
  spliceai_pred_ds_dg: [0],
  first_spliceai_pred_ds_dg: 0,
  spliceai_pred_ds_dl: [0, 0.04],
  first_spliceai_pred_ds_dl: 0,
  first_impact: 'stop_retained_variant',
  gnomadv3_af_xx: -1,
  gnomadv3_af_xy: -1,
  highest_af: 0.1821,
  highest_af_info: {
    source: 'gnomad v3 AF',
  },
  hg19_chrom: 'chr1',
  hg19_start: 52498370,
  hg19_strand: '+',
  hg19_end: 52498370,
  type: 'snp',
  variant_type: 'small',
  variant_id: '70959',
  is_read: ['abc'],
};

const variant1 = {
  is_read: false,
  _id: '656914a8754ff9bcbb45f150',
  chrom: 'chr1',
  start: 52032698,
  ref: 'C',
  alt: 'T',
  quality: null,
  pass_filter: null,
  cm: 75.6471,
  genotypes_index: [
    [0, 0],
    [0, 1],
    [0, 0],
  ],
  highest_exomiser_spheno: 0,
  is_coding: 1,
  is_exonic: 1,
  end: 52032698,
  genotype_qualities: [],
  allelic_depths: [],
  gene_objs: [
    {
      is_exonic: 1,
      allele: 'T',
      impact: ['stop_retained_variant'],
      impact_so: 'stop_retained_variant',
      impact_severity: 'LOW',
      impact_severity_score: 1,
      feature_type: 'Transcript',
      exon: '1/1',
      hgvsc: 'ENST00000371614.2:c.1064G>A',
      hgvsp: 'ENSP00000360676.1:p.Ter355%3D',
      cds_position: 1064,
      aa_length: 355,
      aa_change: '*',
      codon_change: 'tGa/tAa',
      existing_variation: 'rs2809917&COSV65405655',
      strand: -1,
      hgnc_id: 'HGNC:25160',
      is_canonical: true,
      mane_select: 'NM_138417.3',
      ccds: 'CCDS562.1',
      source: 'Ensembl',
      gnomad_af: 0.1447,
      gnomad_afr_af: 0.364,
      gnomad_amr_af: 0.1466,
      gnomad_asj_af: 0.0901,
      gnomad_eas_af: 0.1764,
      gnomad_fin_af: 0.07968,
      gnomad_nfe_af: 0.1122,
      gnomad_oth_af: 0.1212,
      gnomad_sas_af: 0.1902,
      gnomadv3_af: 0.1821,
      gnomadv3_af_afr: 0.3587,
      gnomadv3_af_ami: 0.07895,
      gnomadv3_af_amr: 0.1317,
      gnomadv3_af_asj: 0.09487,
      gnomadv3_af_eas: 0.1646,
      gnomadv3_af_fin: 0.07615,
      gnomadv3_af_mid: 0.1044,
      gnomadv3_af_nfe: 0.1108,
      gnomadv3_af_oth: 0.1571,
      gnomadv3_af_sas: 0.1818,
      pheno: '0&1',
      cadd_phred: 11.05,
      cadd_raw: 0.954327,
      spliceai_pred_dp_ag: -11,
      spliceai_pred_dp_al: -10,
      spliceai_pred_dp_dg: 24,
      spliceai_pred_dp_dl: -10,
      spliceai_pred_ds_ag: 0,
      highest_splice_ai: 0,
      spliceai_pred_ds_al: 0,
      spliceai_pred_ds_dg: 0,
      spliceai_pred_ds_dl: 0,
      spliceai_pred_symbol: 'KTI12',
      constraint_mis_z: -0.49843,
      constraint_oe_lof: 0.72753,
      constraint_oe_lof_lower: 0.397,
      constraint_oe_lof_upper: 1.424,
      constraint_oe_mis: 1.0947,
      constraint_oe_mis_lower: 0.984,
      constraint_oe_mis_upper: 1.218,
      constraint_oe_syn: 1.0718,
      constraint_oe_syn_lower: 0.915,
      constraint_oe_syn_upper: 1.259,
      constraint_pli: 0.00040289,
      constraint_syn_z: -0.56369,
      transcript: 'ENST00000371614',
      is_coding: 1,
      gene: 'KTI12',
      gene_filter: 'KTI12',
      entrez_gene_id: '112970',
      ensembl_gene_id: 'ENSG00000198841',
      p_haplo: 0.0741,
      p_triplo: 0.15418,
      p_li: 0.00040289,
      loeuf: 1.424,
      is_mane_select: true,
      is_mane_plus_clinical: false,
      default_show: true,
      is_related: true,
      clingen_hi_display: null,
      clingen_ts_display: null,
    },
    {
      is_exonic: 0,
      allele: 'T',
      impact: ['intron_variant'],
      impact_so: 'intron_variant',
      impact_severity: 'MODIFIER',
      impact_severity_score: 0,
      feature_type: 'Transcript',
      intron: '2/6',
      hgvsc: 'ENST00000371626.9:c.159-4068G>A',
      existing_variation: 'rs2809917&COSV65405655',
      strand: -1,
      hgnc_id: 'HGNC:24626',
      is_canonical: true,
      mane_select: 'NM_015913.4',
      ccds: 'CCDS561.1',
      source: 'Ensembl',
      gnomad_af: 0.1447,
      gnomad_afr_af: 0.364,
      gnomad_amr_af: 0.1466,
      gnomad_asj_af: 0.0901,
      gnomad_eas_af: 0.1764,
      gnomad_fin_af: 0.07968,
      gnomad_nfe_af: 0.1122,
      gnomad_oth_af: 0.1212,
      gnomad_sas_af: 0.1902,
      gnomadv3_af: 0.1821,
      gnomadv3_af_afr: 0.3587,
      gnomadv3_af_ami: 0.07895,
      gnomadv3_af_amr: 0.1317,
      gnomadv3_af_asj: 0.09487,
      gnomadv3_af_eas: 0.1646,
      gnomadv3_af_fin: 0.07615,
      gnomadv3_af_mid: 0.1044,
      gnomadv3_af_nfe: 0.1108,
      gnomadv3_af_oth: 0.1571,
      gnomadv3_af_sas: 0.1818,
      pheno: '0&1',
      cadd_phred: 11.05,
      cadd_raw: 0.954327,
      spliceai_pred_dp_ag: -15,
      spliceai_pred_dp_al: -10,
      spliceai_pred_dp_dg: 24,
      spliceai_pred_dp_dl: -10,
      spliceai_pred_ds_ag: 0,
      highest_splice_ai: 0.04,
      spliceai_pred_ds_al: 0,
      spliceai_pred_ds_dg: 0,
      spliceai_pred_ds_dl: 0.04,
      spliceai_pred_symbol: 'TXNDC12',
      constraint_mis_z: 0.51718,
      constraint_oe_lof: 0.73759,
      constraint_oe_lof_lower: 0.432,
      constraint_oe_lof_upper: 1.328,
      constraint_oe_mis: 0.84823,
      constraint_oe_mis_lower: 0.706,
      constraint_oe_mis_upper: 1.024,
      constraint_oe_syn: 0.85717,
      constraint_oe_syn_lower: 0.64,
      constraint_oe_syn_upper: 1.163,
      constraint_pli: 0.000037354,
      constraint_syn_z: 0.66422,
      transcript: 'ENST00000371626',
      is_coding: 0,
      gene: 'TXNDC12',
      gene_filter: 'TXNDC12',
      entrez_gene_id: '51060',
      ensembl_gene_id: 'ENSG00000117862',
      p_haplo: 0.35228,
      p_triplo: 0.38277,
      p_li: 0.000037354,
      loeuf: 1.328,
      is_mane_select: true,
      is_mane_plus_clinical: false,
      default_show: true,
      is_related: true,
      clingen_hi_display: null,
      clingen_ts_display: null,
    },
    {
      is_exonic: 0,
      allele: 'T',
      impact: ['intron_variant', 'NMD_transcript_variant'],
      impact_so: 'intron_variant',
      impact_severity: 'MODIFIER',
      impact_severity_score: 0,
      feature_type: 'Transcript',
      intron: '2/7',
      hgvsc: 'ENST00000472624.5:c.159-2150G>A',
      existing_variation: 'rs2809917&COSV65405655',
      strand: -1,
      hgnc_id: 'HGNC:24626',
      source: 'Ensembl',
      gnomad_af: 0.1447,
      gnomad_afr_af: 0.364,
      gnomad_amr_af: 0.1466,
      gnomad_asj_af: 0.0901,
      gnomad_eas_af: 0.1764,
      gnomad_fin_af: 0.07968,
      gnomad_nfe_af: 0.1122,
      gnomad_oth_af: 0.1212,
      gnomad_sas_af: 0.1902,
      gnomadv3_af: 0.1821,
      gnomadv3_af_afr: 0.3587,
      gnomadv3_af_ami: 0.07895,
      gnomadv3_af_amr: 0.1317,
      gnomadv3_af_asj: 0.09487,
      gnomadv3_af_eas: 0.1646,
      gnomadv3_af_fin: 0.07615,
      gnomadv3_af_mid: 0.1044,
      gnomadv3_af_nfe: 0.1108,
      gnomadv3_af_oth: 0.1571,
      gnomadv3_af_sas: 0.1818,
      pheno: '0&1',
      cadd_phred: 11.05,
      cadd_raw: 0.954327,
      spliceai_pred_dp_ag: -15,
      spliceai_pred_dp_al: -10,
      spliceai_pred_dp_dg: 24,
      spliceai_pred_dp_dl: -10,
      spliceai_pred_ds_ag: 0,
      highest_splice_ai: 0.04,
      spliceai_pred_ds_al: 0,
      spliceai_pred_ds_dg: 0,
      spliceai_pred_ds_dl: 0.04,
      spliceai_pred_symbol: 'TXNDC12',
      transcript: 'ENST00000472624',
      is_coding: 0,
      gene: 'TXNDC12',
      gene_filter: 'TXNDC12',
      entrez_gene_id: '51060',
      ensembl_gene_id: 'ENSG00000117862',
      p_haplo: 0.35228,
      p_triplo: 0.38277,
      is_canonical: false,
      is_mane_select: false,
      is_mane_plus_clinical: false,
      default_show: false,
      is_related: true,
      clingen_hi_display: null,
      clingen_ts_display: null,
    },
    {
      is_exonic: 0,
      allele: 'T',
      impact: ['downstream_gene_variant'],
      impact_so: 'downstream_gene_variant',
      impact_severity: 'MODIFIER',
      impact_severity_score: 0,
      feature_type: 'Transcript',
      existing_variation: 'rs2809917&COSV65405655',
      strand: -1,
      hgnc_id: 'HGNC:24626',
      source: 'Ensembl',
      gnomad_af: 0.1447,
      gnomad_afr_af: 0.364,
      gnomad_amr_af: 0.1466,
      gnomad_asj_af: 0.0901,
      gnomad_eas_af: 0.1764,
      gnomad_fin_af: 0.07968,
      gnomad_nfe_af: 0.1122,
      gnomad_oth_af: 0.1212,
      gnomad_sas_af: 0.1902,
      gnomadv3_af: 0.1821,
      gnomadv3_af_afr: 0.3587,
      gnomadv3_af_ami: 0.07895,
      gnomadv3_af_amr: 0.1317,
      gnomadv3_af_asj: 0.09487,
      gnomadv3_af_eas: 0.1646,
      gnomadv3_af_fin: 0.07615,
      gnomadv3_af_mid: 0.1044,
      gnomadv3_af_nfe: 0.1108,
      gnomadv3_af_oth: 0.1571,
      gnomadv3_af_sas: 0.1818,
      pheno: '0&1',
      cadd_phred: 11.05,
      cadd_raw: 0.954327,
      spliceai_pred_dp_ag: -15,
      spliceai_pred_dp_al: -10,
      spliceai_pred_dp_dg: 24,
      spliceai_pred_dp_dl: -10,
      spliceai_pred_ds_ag: 0,
      highest_splice_ai: 0.04,
      spliceai_pred_ds_al: 0,
      spliceai_pred_ds_dg: 0,
      spliceai_pred_ds_dl: 0.04,
      spliceai_pred_symbol: 'TXNDC12',
      constraint_mis_z: 0.1659,
      constraint_oe_lof: 1.0036,
      constraint_oe_lof_lower: 0.366,
      constraint_oe_lof_upper: 1.878,
      constraint_oe_mis: 0.91254,
      constraint_oe_mis_lower: 0.668,
      constraint_oe_mis_upper: 1.266,
      constraint_oe_syn: 1.0515,
      constraint_oe_syn_lower: 0.672,
      constraint_oe_syn_upper: 1.668,
      constraint_pli: 0.024969,
      constraint_syn_z: -0.1367,
      transcript: 'ENST00000610127',
      is_coding: 0,
      gene: 'TXNDC12',
      gene_filter: 'TXNDC12',
      entrez_gene_id: '51060',
      ensembl_gene_id: 'ENSG00000117862',
      p_haplo: 0.35228,
      p_triplo: 0.38277,
      p_li: 0.024969,
      loeuf: 1.878,
      is_canonical: false,
      is_mane_select: false,
      is_mane_plus_clinical: false,
      default_show: false,
      is_related: true,
      clingen_hi_display: null,
      clingen_ts_display: null,
    },
  ],
  impact: 'stop_retained_variant',
  impact_severity: 'LOW',
  hgvsc: 'ENST00000371614.2:c.1064G>A',
  hgvsp: 'ENSP00000360676.1:p.Ter355%3D',
  aa_change: '*',
  existing_variation: 'rs2809917&COSV65405655',
  mane_select: 'NM_138417.3',
  ccds: 'CCDS562.1',
  source: 'Ensembl',
  gnomad_af: 0.1447,
  gnomad_afr_af: 0.364,
  gnomad_amr_af: 0.1466,
  gnomad_asj_af: 0.0901,
  gnomad_eas_af: 0.1764,
  gnomad_fin_af: 0.07968,
  gnomad_nfe_af: 0.1122,
  gnomad_oth_af: 0.1212,
  gnomad_sas_af: 0.1902,
  gnomadv3_af: 0.1821,
  gnomadv3_af_afr: 0.3587,
  gnomadv3_af_ami: 0.07895,
  gnomadv3_af_amr: 0.1317,
  gnomadv3_af_asj: 0.09487,
  gnomadv3_af_eas: 0.1646,
  gnomadv3_af_fin: 0.07615,
  gnomadv3_af_mid: 0.1044,
  gnomadv3_af_nfe: 0.1108,
  gnomadv3_af_oth: 0.1571,
  gnomadv3_af_sas: 0.1818,
  pheno: '0&1',
  cadd_phred: 11.05,
  cadd_raw: 0.954327,
  spliceai_pred_dp_ag: -11,
  spliceai_pred_dp_al: -10,
  spliceai_pred_dp_dg: 24,
  spliceai_pred_dp_dl: -10,
  spliceai_pred_symbol: 'KTI12',
  constraint_oe_lof: 0.72753,
  constraint_oe_lof_lower: 0.397,
  constraint_oe_lof_upper: [1.424, 1.328],
  constraint_oe_mis: 1.0947,
  constraint_oe_mis_lower: 0.984,
  constraint_oe_mis_upper: [1.218, 1.024],
  constraint_oe_syn: 1.0718,
  constraint_oe_syn_lower: 0.915,
  constraint_oe_syn_upper: 1.259,
  constraint_pli: 0.00040289,
  constraint_syn_z: [-0.56369, 0.66422],
  ensembl_transcript_id: [
    'ENST00000371614',
    'ENST00000371626',
    'ENST00000472624',
    'ENST00000610127',
  ],
  gene_symbol: ['KTI12', 'TXNDC12'],
  ensembl_gene_id: ['ENSG00000198841', 'ENSG00000117862'],
  constraint_mis_z: [-0.49843, 0.51718],
  first_constraint_mis_z: -0.49843,
  first_constraint_syn_z: -0.56369,
  first_constraint_oe_lof_upper: 1.424,
  first_constraint_oe_mis_upper: 1.218,
  highest_splice_ai: [0, 0.04],
  first_highest_splice_ai: 0,
  spliceai_pred_ds_ag: [0],
  first_spliceai_pred_ds_ag: 0,
  spliceai_pred_ds_al: [0],
  first_spliceai_pred_ds_al: 0,
  spliceai_pred_ds_dg: [0],
  first_spliceai_pred_ds_dg: 0,
  spliceai_pred_ds_dl: [0, 0.04],
  first_spliceai_pred_ds_dl: 0,
  first_impact: 'stop_retained_variant',
  gnomadv3_af_xx: -1,
  gnomadv3_af_xy: -1,
  highest_af: 0.1821,
  highest_af_info: {
    source: 'gnomad v3 AF',
  },
  hg19_chrom: 'chr1',
  hg19_start: 52498370,
  hg19_strand: '+',
  hg19_end: 52498370,
  type: 'snp',
  variant_type: 'small',
  variant_id: '70959',
  entrez_gene_id: ['112970', '51060'],
};

export const mockVariants1 = new Variants();
export const mockVariant1 = new Variant();
convertJSONToVariants(variants1, mockVariants1);
convertJSONToVariant(variant1, mockVariant1);
(<any>Variants.prototype).toJSON = function () {
  return this;
};
